{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# **DisplaCy visualiser tool for the metadata annotations**"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Imports\n",
    "import json\n",
    "from spacy import displacy\n",
    "import re\n",
    "import os"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Currently the annotations structure is as follows:\n",
    "\n",
    "```python\n",
    "start_index = int # The index of the first character of the entity. \n",
    "end_index = int # The index of the last character of the entity. (not inclusive)\n",
    "label = str # Label attached to the entity. \n",
    "\n",
    "{\n",
    "    \"classes\": [\"TEMP\", \"SOFT\", \"STIME\", \"MOL\", \"FFM\"],\n",
    "    \"annotations\": [\n",
    "        [\n",
    "            \"text_to_fill_in\",\n",
    "            {\n",
    "                \"entities\": [\n",
    "                    [start_index, end_index, label],\n",
    "                ]\n",
    "            }\n",
    "        ]\n",
    "    ]\n",
    "}\n",
    "\n",
    "# Example\n",
    "{\n",
    "    \"classes\": [\"TEMP\", \"SOFT\", \"STIME\", \"MOL\", \"FFM\"],\n",
    "    \"annotations\": [\n",
    "        [\n",
    "            \"POPC Ulmschneider OPLS Verlet Group\",\n",
    "            {\n",
    "                \"entities\": [\n",
    "                    [0, 4, \"MOL\"],\n",
    "                    [18, 22, \"FFM\"]\n",
    "                ]\n",
    "            }\n",
    "        ]\n",
    "    ]\n",
    "}\n",
    "\n",
    "```\n",
    "\n",
    "\n",
    "However, the spaCy  format for `ent` is as follows:\n",
    "\n",
    "```python\n",
    "start_index = int # The index of the first character of the entity. \n",
    "end_index = int # The index of the last character of the entity. (not inclusive)\n",
    "label = str # Label attached to the entity. \n",
    "\n",
    "{\n",
    "  \"text\": \"text_to_fill_in.\",\n",
    "  \"ents\": [{ \"start\": start_index, \"end\": end_index, \"label\": label }]\n",
    "}\n",
    "\n",
    "# Example:\n",
    "{\n",
    "  \"text\": \"POPC molecules.\",\n",
    "  \"ents\": [{ \"start\": 0, \"end\": 4, \"label\": \"MOL\" }]\n",
    "}\n",
    "\n",
    "```\n",
    "This is what it looks like working with the spaCy format:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">\n",
       "<mark class=\"entity\" style=\"background: #a8ff65; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Marie Curie\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">PERS</span>\n",
       "</mark>\n",
       " was born in \n",
       "<mark class=\"entity\" style=\"background: orange; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Warsaw\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">LOC</span>\n",
       "</mark>\n",
       " in \n",
       "<mark class=\"entity\" style=\"background: turquoise; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    1867\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">DATE</span>\n",
       "</mark>\n",
       " and later founded the \n",
       "<mark class=\"entity\" style=\"background: #7361ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Radium Institute\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">ORG</span>\n",
       "</mark>\n",
       " in \n",
       "<mark class=\"entity\" style=\"background: orange; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Paris\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">LOC</span>\n",
       "</mark>\n",
       "</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "text = \"Marie Curie was born in Warsaw in 1867 and later founded the Radium Institute in Paris\"\n",
    "ents = [\n",
    "    {\"start\": 0, \"end\": 11, \"label\": \"PERS\"},\n",
    "    {\"start\": 24, \"end\": 30, \"label\": \"LOC\"},\n",
    "    {\"start\": 34, \"end\": 38, \"label\": \"DATE\"},\n",
    "    {\"start\": 61, \"end\": 77, \"label\": \"ORG\"},\n",
    "    {\"start\": 81, \"end\": 86, \"label\": \"LOC\"},\n",
    "]\n",
    "\n",
    "data = {\"text\": text, \"ents\": ents}\n",
    "\n",
    "colors = {\n",
    "    \"PERS\": \"#a8ff65\",\n",
    "    \"LOC\": \"orange\",\n",
    "    \"DATE\": \"turquoise\",\n",
    "    \"ORG\": \"#7361ff\",\n",
    "}\n",
    "options = {\"colors\": colors}\n",
    "\n",
    "displacy.render(data, style=\"ent\", manual=True, options=options)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "However we don't have this kind of format. \n",
    "We need to therefore convert the ground truth format to something spaCy-compatible."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Let's do this for ground truth texts:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [],
   "source": [
    "def convert_annotations(file_path):\n",
    "    \"\"\"\n",
    "    Convert annotations from the custom format to spaCy's displaCy format.\n",
    "\n",
    "    Parameters:\n",
    "        file_path (str): Path to the JSON file containing \"classes\" and \"annotations\" keys.\n",
    "\n",
    "    Returns:\n",
    "        List[dict]: A list of dictionaries in the spaCy format.\n",
    "    \"\"\"\n",
    "    with open(file_path, \"r\") as file:\n",
    "        data = json.load(file)\n",
    "\n",
    "    converted = []\n",
    "    for item in data[\"annotations\"]:\n",
    "        # Each item is structured as [text, {\"entities\": [[start, end, label], ...]}]\n",
    "        text, annotation_info = item\n",
    "        entities = annotation_info.get(\"entities\", [])\n",
    "        # Convert each entity to the spaCy dictionary format\n",
    "        ents = [\n",
    "            {\"start\": start, \"end\": end, \"label\": label}\n",
    "            for start, end, label in entities\n",
    "        ]\n",
    "        converted.append({\"text\": text, \"ents\": ents})\n",
    "    return converted"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Molecular dynamics simulation data of designed cyclic peptide - ligand 1-3 (receptor-ligand bound)<br>Trajectories of receptor-ligand bound simulation and simulation set-up files of designed cyclic peptide as \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    MDM2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " binders. This dataset contains simulations of ligand 1-3. Ligand 4 data can be found here: https://removed The original paper of these designed cyclic peptide: Danelius, E., Pettersson, M., Bred, M., Min, J., Waddell, M. B., Guy, R. K., et al. (2016). Flexibility is important for inhibition of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    MDM2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "/\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    p53\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " protein protein interaction by cyclic -hairpins. Org. Biomol. Chem., 14 (44), 10386 10393. https://removed</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "# Here we take one example of ground-truth annotation.\n",
    "data = \"../annotations/zenodo_3780463.json\"\n",
    "\n",
    "# Convert the annotations.\n",
    "converted_data = convert_annotations(data)\n",
    "original_text = converted_data[0][\"text\"]\n",
    "\n",
    "# Print the original text, but keep the format with the \\n characters.\n",
    "# print(repr(original_text))\n",
    "\n",
    "colors = {\n",
    "    \"TEMP\": \"#ffb3ba\",\n",
    "    \"SOFT\": \"#ffffba\",\n",
    "    \"STIME\": \"#baffc9\",\n",
    "    \"MOL\": \"#bae1ff\",\n",
    "    \"FFM\": \"#cdb4db\",\n",
    "}\n",
    "options = {\"colors\": colors}\n",
    "\n",
    "# You now have a list of annotation dictionaries. For visualization with displaCy,\n",
    "# you can serve a single annotated text or multiple.\n",
    "displacy.render(converted_data, style=\"ent\", manual=True, options=options)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Let's say we have a ground-truth annotation that we want to use as an example in a prompt.\n",
    "We'll first display the input text so that we can read it properly, then we use spaCy to visualise the ground-truth annotation, and lastly, we modify the annotation format in order to get a text that we can use in prompts.\n",
    "\n",
    "In this following case, we want the ground-truth annotation modified to an annotation using XML-style tags."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [],
   "source": [
    "def modify_annotation(text_path: str):\n",
    "    # Convert the annotations.\n",
    "    converted_data = convert_annotations(data)\n",
    "    original_text = converted_data[0][\"text\"]\n",
    "\n",
    "    # Print the original text, but keep the format with the \\n characters.\n",
    "    print(repr(original_text), \"\\n\")\n",
    "\n",
    "    colors = {\n",
    "        \"TEMP\": \"#ffb3ba\",\n",
    "        \"SOFT\": \"#ffffba\",\n",
    "        \"STIME\": \"#baffc9\",\n",
    "        \"MOL\": \"#bae1ff\",\n",
    "        \"FFM\": \"#cdb4db\",\n",
    "    }\n",
    "    options = {\"colors\": colors}\n",
    "\n",
    "    # You now have a list of annotation dictionaries. For visualization with displaCy,\n",
    "    # you can serve a single annotated text or multiple.\n",
    "    displacy.render(converted_data, style=\"ent\", manual=True, options=options)\n",
    "\n",
    "    # 1. Sort entities by start offset (ascending)\n",
    "    ents = sorted(converted_data[0][\"ents\"], key=lambda e: e[\"start\"])\n",
    "    pieces = []\n",
    "    last_idx = 0\n",
    "\n",
    "    # 2. For each entity, append:\n",
    "    #    – the text from the end of the previous entity (or start of string)\n",
    "    #    – the wrapped entity\n",
    "    for e in ents:\n",
    "        start, end, label = e[\"start\"], e[\"end\"], e[\"label\"]\n",
    "        # plain text before this entity\n",
    "        pieces.append(original_text[last_idx:start])\n",
    "        # the entity wrapped in tags\n",
    "        entity_text = original_text[start:end]\n",
    "        pieces.append(f\"<{label}>{entity_text}</{label}>\")\n",
    "        last_idx = end\n",
    "\n",
    "    # 3. Append any trailing text after the last entity\n",
    "    pieces.append(original_text[last_idx:])\n",
    "    return repr(\"\".join(pieces))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "'256 DPPC Molecules bilayer in pure Water, simulated at 288K (gel) or 358K (fluid)\\nPublication: MLLPA: A Machine Learning-assisted Python module to study phase-specific events in lipid membranes Published on: 08 April 2021 Journal: J Comp Chem, 2021, DOI: 10.1002/jcc.26508 Description: Simulation files used to train our Python module to identify the thermodynamic phase of individual lipid molecules in a bilayer, as well as the simulation files analysed by the machine learning models. More information on the module can be found on its website. The training files are named dppc gel.gro and dppc fluid.gro. They respectively correspond to the final frame of the systems simulated at 288K and 358K. All other files are the files analysed by the module. System composition: DPPC molecules: 256 with 130 atoms each Water molecules: 42,492 with 3 atoms each Simulation box dimensions (approx.): 9 x 9 x 20 nm Simulation details: Software: Gromacs (v. 2020) Forcefield: Charmm36 (v. June 2015) - Water: TIP3P Thermostat: Nose-hoover (0.4ps, 2 groups) Barostat: Parrinello-Rahman semi-isotropic (2.0ps, 1.0 bar on each axis, 4.5e-5 bar-1) Duration: 25 ns (after stabilisation) ' \n",
      "\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">256 \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DPPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Molecules bilayer in pure \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", simulated at \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    288K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       " (gel) or \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    358K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       " (fluid)<br>Publication: MLLPA: A Machine Learning-assisted Python module to study phase-specific events in lipid membranes Published on: 08 April 2021 Journal: J Comp Chem, 2021, DOI: 10.1002/jcc.26508 Description: Simulation files used to train our Python module to identify the thermodynamic phase of individual lipid molecules in a bilayer, as well as the simulation files analysed by the machine learning models. More information on the module can be found on its website. The training files are named \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    dppc\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " gel.gro and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    dppc\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " fluid.gro. They respectively correspond to the final frame of the systems simulated at \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    288K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    358K.\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       " All other files are the files analysed by the module. System composition: \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DPPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " molecules: 256 with 130 atoms each \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " molecules: 42,492 with 3 atoms each Simulation box dimensions (approx.): 9 x 9 x 20 nm Simulation details: Software: \n",
       "<mark class=\"entity\" style=\"background: #ffffba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Gromacs (v. 2020)\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " Forcefield: \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Charmm36 (v. June 2015)\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " - \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ": \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    TIP3P\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " Thermostat: Nose-hoover (0.4ps, 2 groups) Barostat: Parrinello-Rahman semi-isotropic (2.0ps, 1.0 bar on each axis, 4.5e-5 bar-1) Duration: \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    25 ns\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       " (after stabilisation) </div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "'256 <MOL>DPPC</MOL> Molecules bilayer in pure <MOL>Water</MOL>, simulated at <TEMP>288K</TEMP> (gel) or <TEMP>358K</TEMP> (fluid)\\nPublication: MLLPA: A Machine Learning-assisted Python module to study phase-specific events in lipid membranes Published on: 08 April 2021 Journal: J Comp Chem, 2021, DOI: 10.1002/jcc.26508 Description: Simulation files used to train our Python module to identify the thermodynamic phase of individual lipid molecules in a bilayer, as well as the simulation files analysed by the machine learning models. More information on the module can be found on its website. The training files are named <MOL>dppc</MOL> gel.gro and <MOL>dppc</MOL> fluid.gro. They respectively correspond to the final frame of the systems simulated at <TEMP>288K</TEMP> and <TEMP>358K.</TEMP> All other files are the files analysed by the module. System composition: <MOL>DPPC</MOL> molecules: 256 with 130 atoms each <MOL>Water</MOL> molecules: 42,492 with 3 atoms each Simulation box dimensions (approx.): 9 x 9 x 20 nm Simulation details: Software: <SOFT>Gromacs (v. 2020)</SOFT> Forcefield: <FFM>Charmm36 (v. June 2015)</FFM> - <MOL>Water</MOL>: <FFM>TIP3P</FFM> Thermostat: Nose-hoover (0.4ps, 2 groups) Barostat: Parrinello-Rahman semi-isotropic (2.0ps, 1.0 bar on each axis, 4.5e-5 bar-1) Duration: <STIME>25 ns</STIME> (after stabilisation) '\n"
     ]
    }
   ],
   "source": [
    "data = \"../annotations/zenodo_4300706.json\"\n",
    "\n",
    "# Top we will have the input text\n",
    "# Middle we will have the spaCy visualisation\n",
    "# Last we will have the modified annotation using XML-style tags\n",
    "print(modify_annotation(data))"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### **LLM annotations compared to GT annotations**\n",
    "---"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Here we give the functions to adapt the following formats into the spaCy format:\n",
    "- XML-style labels\n",
    "- JSON output\n",
    "- Index based"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "metadata": {},
   "outputs": [],
   "source": [
    "def convert_xml_to_spacy_format(annotated_text):\n",
    "    with open(annotated_text, \"r\") as file:\n",
    "        annotated_text = file.read()\n",
    "\n",
    "    pattern = re.compile(r\"<(MOL|STIME|TEMP|SOFTNAME|SOFTVERS|FFM)>(.+?)<\\1>\")\n",
    "    ents = []\n",
    "    clean_text = \"\"\n",
    "    last_idx = 0  # Position in the annotated text\n",
    "\n",
    "    for match in pattern.finditer(annotated_text):\n",
    "        tag = match.group(1)\n",
    "        entity_text = match.group(2)\n",
    "\n",
    "        # Text between last entity and current\n",
    "        pre_entity_text = annotated_text[last_idx : match.start()]\n",
    "        clean_text += pre_entity_text\n",
    "        entity_start = len(clean_text)\n",
    "        clean_text += entity_text\n",
    "        entity_end = len(clean_text)\n",
    "\n",
    "        ents.append({\"start\": entity_start, \"end\": entity_end, \"label\": tag})\n",
    "\n",
    "        last_idx = match.end()\n",
    "\n",
    "    # Add any remaining text after last tag\n",
    "    clean_text += annotated_text[last_idx:]\n",
    "\n",
    "    return [{\"text\": clean_text, \"ents\": ents}]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 26,
   "metadata": {},
   "outputs": [],
   "source": [
    "def visualize_xml_vs_gt(data: list) -> None:\n",
    "    \"\"\"\n",
    "    Visualize the annotated text using displaCy.\n",
    "\n",
    "    Parameters:\n",
    "        data (list): List of paths to the annotated text files.\n",
    "    \"\"\"\n",
    "    # Convert the annotations.\n",
    "    converted_real_data = convert_annotations(data[0])  # Real annotation\n",
    "    converted_ai_data = convert_xml_to_spacy_format(data[1])  # AI annotated\n",
    "\n",
    "    colors = {\n",
    "        \"TEMP\": \"#ffb3ba\",\n",
    "        \"SOFTNAME\": \"#ffffba\",\n",
    "        \"SOFTVERS\": \"#orange\",\n",
    "        \"STIME\": \"#baffc9\",\n",
    "        \"MOL\": \"#bae1ff\",\n",
    "        \"FFM\": \"#cdb4db\",\n",
    "    }\n",
    "    options = {\"colors\": colors}\n",
    "\n",
    "    print(\"\\nREAL ANNOTATION:\")\n",
    "    displacy.render(converted_real_data, style=\"ent\", manual=True, options=options)\n",
    "    print(\"\\AI XML ANNOTATION:\")\n",
    "    displacy.render(converted_ai_data, style=\"ent\", manual=True, options=options)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><!DOCTYPE html>\n",
       "<html lang=\"en\">\n",
       "    <head>\n",
       "        <title>displaCy</title>\n",
       "    </head>\n",
       "\n",
       "    <body style=\"font-size: 16px; font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol'; padding: 4rem 2rem; direction: ltr\">\n",
       "<figure style=\"margin-bottom: 6rem\">\n",
       "<div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Molecular dynamics simulation data of designed cyclic peptide - ligand 1-3 (receptor-ligand bound)<br>Trajectories of receptor-ligand bound simulation and simulation set-up files of designed cyclic peptide as MDM2 binders.</div>\n",
       "</figure>\n",
       "</body>\n",
       "</html></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Using the 'ent' visualizer\n",
      "Serving on http://0.0.0.0:5000 ...\n",
      "\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "127.0.0.1 - - [19/Jun/2025 13:49:17] \"GET / HTTP/1.1\" 200 662\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Shutting down server on port 5000.\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">cyclic peptide - ligand 1-3 (receptor-ligand bound)\\nTrajectories of receptor-ligand bound simulation and simulation set-up files of designed cyclic peptide as MDM2 binders. This dataset contains simulations of ligand 1-3. Ligand 4 data can be found here: https://removed The original paper of these designed cyclic peptide: Danelius, E., Pettersson, M., Bred, M., Min, J., Waddell, M. B., Guy, R. K., et al. (2016). Flexibility is important for inhibition of the MDM2/p53 protein protein interaction by cyclic -hairpins. Org. Biomol. Chem., 14 (44), 10386 10393. https://removed</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "# ai_path = \"../llm_outputs/annotations_2025-04-22_17-31-26/few_shot/gpt-4.1-2025-04-14/zenodo_7007107.json\"\n",
    "# ai_path = \"../llm_outputs/annotations_2025-04-22_17-31-26/few_shot/gpt-4.1-2025-04-14/zenodo_1219494.json\"\n",
    "# ai_path = \"../llm_outputs/annotations_2025-04-22_17-31-26/one_shot/gpt-4.1-2025-04-14/zenodo_7323535.json\"\n",
    "# ai_path = \"../output_llm_annotations/zero_shot_prompt_template/deepseek-r1-distill-llama-70b/figshare_20300547.json\"\n",
    "\n",
    "# zenodo_6478270\n",
    "# zenodo_3778112\n",
    "\n",
    "\n",
    "# o3-2025-04-16\n",
    "# o3-mini-2025-01-31\n",
    "# o4-mini-2025-04-16\n",
    "ai_path = \"../llm_outputs/2025-04-28_17-32-24/annotations/few_shot/2025-05-27_14-50-19/zenodo_3780463.json\"\n",
    "real_path = \"../annotations/zenodo_3780463.json\"\n",
    "\n",
    "# ai_path = \"../testing/ai_text.json\"\n",
    "# real_path = \"../testing/real_text.json\"\n",
    "\n",
    "converted_real_data = convert_annotations(real_path)  # Real annotation\n",
    "\n",
    "\n",
    "def convert_ai_xml(annotated_text):\n",
    "    with open(annotated_text, \"r\") as file:\n",
    "        annotated_text = json.load(file)\n",
    "\n",
    "    annotated_text = annotated_text[\"response\"]\n",
    "\n",
    "    pattern = re.compile(r\"<(MOL|STIME|TEMP|SOFTNAME|SOFTVERS|FFM)>(.+?)</\\1>\")\n",
    "    ents = []\n",
    "    clean_text = \"\"\n",
    "    last_idx = 0  # Position in the annotated text\n",
    "\n",
    "    for match in pattern.finditer(annotated_text):\n",
    "        tag = match.group(1)\n",
    "        entity_text = match.group(2)\n",
    "\n",
    "        # Text between last entity and current\n",
    "        pre_entity_text = annotated_text[last_idx : match.start()]\n",
    "        clean_text += pre_entity_text\n",
    "        entity_start = len(clean_text)\n",
    "        clean_text += entity_text\n",
    "        entity_end = len(clean_text)\n",
    "\n",
    "        ents.append({\"start\": entity_start, \"end\": entity_end, \"label\": tag})\n",
    "\n",
    "        last_idx = match.end()\n",
    "\n",
    "    # Add any remaining text after last tag\n",
    "    clean_text += annotated_text[last_idx:]\n",
    "    ents = []\n",
    "    return [{\"text\": clean_text, \"ents\": ents}]\n",
    "\n",
    "\n",
    "converted_ai_text = convert_ai_xml(ai_path)\n",
    "\n",
    "colors = {\n",
    "    \"TEMP\": \"#ffb3ba\",\n",
    "    \"SOFTNAME\": \"#ffffba\",\n",
    "    \"SOFTVERS\": \"#orange\",\n",
    "    \"STIME\": \"#baffc9\",\n",
    "    \"MOL\": \"#bae1ff\",\n",
    "    \"FFM\": \"#cdb4db\",\n",
    "}\n",
    "options = {\"colors\": colors}\n",
    "\n",
    "displacy.render(converted_real_data, style=\"ent\", manual=True, options=options)\n",
    "\n",
    "displacy.render(converted_ai_text, style=\"ent\", manual=True, options=options)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "REAL ANNOTATION:\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">212 \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DPPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Molecules bilayer in pure \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", simulated at temperatures ranging from \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    288K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       " to \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    358K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       "<br>Publication: A machine learning study of the two states model for lipid bilayer phase transitions Published on: 12 August 2020 Journal: Phys. Chem. Chem. Phys., 2020, DOI: 10.1039/D0CP02058C Description: Simulation files used to train our machine learning algorithm to identify the thermodynamic phase of individual lipid molecules in a bilayer, as well as the simulation files analysed by the machine learning models. Code source for the ML algorithm can be found on Github. The training files are named gel.gro and fluid.gro. They respectively correspond to the final frame of the systems simulated at \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    288K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    358K.\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       " All other files are the files analysed by the machine learning models. System composition: \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DPPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " molecules: 212 with 130 atoms each \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " molecules: 29,826 with 3 atoms each Simulation box dimensions (approx.): 8 x 8 x 20 nm Simulation details: Software: \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Gromacs (v. 2016.4)\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " Forcefield: \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Charmm36 (v. June 2015)\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " - \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ": \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    TIP3P\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " Thermostat: Nose-hoover (0.4ps, 2 groups) Barostat: Parrinello-Rahman semi-isotropic (2.0ps, 1.0 bar on each axis, 4.5e-5 bar-1) Duration: \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    50 ns\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       " </div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "AI ANNOTATION:\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">212 \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DPPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Molecules bilayer in pure \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", simulated at temperatures ranging from \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    288K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       " to \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    358K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       "\\nPublication: A machine learning study of the two states model for lipid bilayer phase transitions Published on: 12 August 2020 Journal: Phys. Chem. Chem. Phys., 2020, DOI: 10.1039/D0CP02058C Description: Simulation files used to train our machine learning algorithm to identify the thermodynamic phase of individual lipid molecules in a bilayer, as well as the simulation files analysed by the machine learning models. Code source for the ML algorithm can be found on Github. The training files are named gel.gro and fluid.gro. They respectively correspond to the final frame of the systems simulated at \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    288K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    358K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       ". All other files are the files analysed by the machine learning models. System composition: \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DPPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " molecules: 212 with 130 atoms each \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " molecules: 29,826 with 3 atoms each Simulation box dimensions (approx.): 8 x 8 x 20 nm Simulation details: Software: \n",
       "<mark class=\"entity\" style=\"background: #ffffba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Gromacs\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFTNAME</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #orange; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    v. 2016.4\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFTVERS</span>\n",
       "</mark>\n",
       ") Forcefield: \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Charmm36\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #orange; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    June 2015\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFTVERS</span>\n",
       "</mark>\n",
       ") - \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ": \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    TIP3P\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Thermostat: Nose-hoover (0.4ps, 2 groups) Barostat: Parrinello-Rahman semi-isotropic (2.0ps, 1.0 bar on each axis, 4.5e-5 bar-1) Duration: \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    50 ns\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       "</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "real_data = \"../annotations/zenodo_3950029.json\"\n",
    "xml_ai_data = \"../ai_annotations/chatgpt_zenodo_3950029.txt\"\n",
    "\n",
    "data = [real_data, xml_ai_data]\n",
    "\n",
    "visualize_xml_vs_gt(data)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### **REAL ANNOTATION**"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Assessment of mutation probabilities of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    KRAS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " G12 missense mutants and their long-time scale dynamics by atomistic molecular simulations and Markov state modeling: Datasets.<br>Datasets related to the publication [1]. Including: \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    KRAS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " G12X mutations derived from COSMIC v.79 [http://cancer.sanger.ac.uk/cosmic/] (KRAS G12X mut COSMICv79..xlsx) RMSFs (300-2000ns) of GDP-systems (300 2000rmsf GDP systems RAW AVG SE.xlsx) RMSFs (300-2000ns) of GTP-systems (300 2000RMSF GTP systems RAW AVG SE.xlsx) \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PyInteraph\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " analysis data for salt-bridges and hydrophobic clusters (.dat files for each system in the PyInteraph data.zip-file) Backbone trajectories for each system (residues 4-164; frames for every 1ns). Last number (e.g. 1) refers to the replica of the simulated system. backbone 4-164.gro/.pdb/.tpr -files (resid 4-164) [1] Pantsar T et al. Assessment of mutation probabilities of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    KRAS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " G12 missense mutants and their long-time scale dynamics by atomistic molecular simulations and Markov state modeling. PLoS Comput Biol Submitted (2018)</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "real_data = \"../annotations/zenodo_1346073.json\"\n",
    "\n",
    "# Convert the annotations.\n",
    "converted_data = convert_annotations(real_data)\n",
    "\n",
    "colors = {\n",
    "    \"TEMP\": \"#ffb3ba\",\n",
    "    \"SOFTNAME\": \"#ffffba\",\n",
    "    \"SOFTVERS\": \"#orange\",\n",
    "    \"STIME\": \"#baffc9\",\n",
    "    \"MOL\": \"#bae1ff\",\n",
    "    \"FFM\": \"#cdb4db\",\n",
    "}\n",
    "options = {\"colors\": colors}\n",
    "\n",
    "# You now have a list of annotation dictionaries. For visualization with displaCy,\n",
    "# you can serve a single annotated text or multiple.\n",
    "displacy.render(converted_data, style=\"ent\", manual=True, options=options)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### **A.I. ANNOTATIONS**"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "AI ANNOTATION:\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Assessment of mutation probabilities of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    KRAS&lt;MOl&gt; G12 missense mutants and their long-time scale dynamics by atomistic molecular simulations and Markov state modeling: Datasets.\\nDatasets related to the publication [1]. Including: \n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "KRAS&lt;MOl&gt; G12X mutations derived from COSMIC v. &lt;SOFTVERS&gt;79 [http://cancer.sanger.ac.uk/cosmic/] (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    KRAS&lt;MOl&gt; G12X mut COSMICv79..xlsx) RMSFs (&lt;SOFTNAME&gt;NAMD&lt;SOFTNAME&gt; for &lt;STIME&gt;300&lt;STIME&gt;-&lt;STIME&gt;2000ns&lt;STIME&gt;) of \n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "GDP system\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    -systems (&lt;SOFTNAME&gt;NAMD&lt;SOFTNAME&gt; for &lt;STIME&gt;300&lt;STIME&gt;-&lt;STIME&gt;2000ns&lt;STIME&gt; rmsf \n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "GDP system&lt;s|MOL&gt; RAW AVG SE.xlsx) RMSFs (\n",
       "<mark class=\"entity\" style=\"background: #ffffba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    NAMD\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFTNAME</span>\n",
       "</mark>\n",
       " for \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    300\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       "-\n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    2000ns\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       ") of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GTP system&lt;s|MOL&gt;-systems (&lt;SOFTNAME&gt;NAMD&lt;SOFTNAME&gt; for &lt;STIME&gt;300&lt;STIME&gt;-&lt;STIME&gt;2000ns&lt;STIME&gt; RMSF \n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "GTP system&lt;s|MOL&gt; RAW AVG SE.xlsx) PyInteraph analysis data for salt-bridges and hydrophobic clusters (.dat files for each system in the PyInteraph data.zip-file) Backbone trajectories for each system (residues 4-164; frames for every \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    1ns\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       "). Instead of using \n",
       "<mark class=\"entity\" style=\"background: #ffffba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GROMACS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFTNAME</span>\n",
       "</mark>\n",
       ", we used \n",
       "<mark class=\"entity\" style=\"background: #ffffba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    NAMD\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFTNAME</span>\n",
       "</mark>\n",
       " also.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "ai_data = \"../ai_annotations/llama_3_1_b_annotation.txt\"\n",
    "\n",
    "converted_ai_data = convert_annotated_text_to_spacy_format(ai_data)\n",
    "\n",
    "colors = {\n",
    "    \"TEMP\": \"#ffb3ba\",\n",
    "    \"SOFTNAME\": \"#ffffba\",\n",
    "    \"SOFTVERS\": \"#orange\",\n",
    "    \"STIME\": \"#baffc9\",\n",
    "    \"MOL\": \"#bae1ff\",\n",
    "    \"FFM\": \"#cdb4db\",\n",
    "}\n",
    "options = {\"colors\": colors}\n",
    "\n",
    "print(\"\\nAI ANNOTATION:\")\n",
    "displacy.render(converted_ai_data, style=\"ent\", manual=True, options=options)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### **All ground-truth annotations visualised using spaCy**\n",
    "---"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Visualizing figshare_22213635.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Theoretical Calculations of the Thermal Rate Coefficients for the Interstellar \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    NH3+\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " + \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    H2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    NH4+\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " + \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    H\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Reaction on a New Machine Learning Potential Energy Surface<br>The \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    NH3+\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " + \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    H2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    NH4+\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " + H ion molecule hydrogen-exchange reaction is important for understanding the mechanisms of ammonia formation in interstellar medium and circumstellar environments. The thermal rate coefficients of this reaction are calculated using the ring polymer molecular dynamics approach, which can account for the nuclear quantum effects, on a newly developed ab initio potential energy surface (PES). The PES was constructed by fitting a large number of ab initio points at the MP2/cc-pVDZ and CCSD (T)/cc-pVTZ levels using a -machine learning approach. The obtained rate coefficients exhibited a minimum at T \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    130 K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       ", and this characteristic temperature-dependent behavior qualitatively agreed with the experimental measurements. This study confirms that quantum tunneling plays a significant role in the reaction dynamics along with the existence of the metastable \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    NH3+\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    H2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " pre-reactive complex in the entrance channel of the PES.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_4757161.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Enzyme\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Inspired Catalysts<br>\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Polystyrene\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " fragments containing catalytic and hydrophobic residues were built and simulated using \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Amber16\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " package.Parameters for \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    styrene\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    STY\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "), catalyst (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CAT\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") and hydrophobic (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    SPA\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") residues were assigned using \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Antechamber\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    General Amber Force Field\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GAFF\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       "), while the charges were derived using two-stage RESP fitting procedure to the electrostatic potential calculated with \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Gaussian09\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " at HF/6-31G (d) level of theory. This level of theory was used to be consistent with \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GAFF\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " parameterization. Geometry of each residue was optimized in the gas phase and polymer backbone atoms were capped with a \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    methyl\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " group. \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Styrene\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " monomer was modelled as \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    2-phenylpropane\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". The additional \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    methyl\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " group and hydrogen atom were removed in the subsequent step to ensure correct connectivity between the monomers, which was defined with prepgen module of \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    AmberTools\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       ". Charges on these capping groups were required to sum to zero and the excess charge was distributed between the remaining atoms. Terminal \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    styrene\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " residues were capped with \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    methyl\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " groups. Identical approach was employed for creating catalyst and hydrophobic residue parameters, except multi-conformer fitting was performed for these molecules due to their conformational flexibility. We used 10 conformers for each molecule in the RESP procedure of partial charge derivation. The uploaded files include prep libraries required for building polystyrene system:* \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    STY\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ": \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    styrene\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " monomer * HST: terminal (head) \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    styrene\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " monomer capped with \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    methyl\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " group* TST: terminal (tail) \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    styrene\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " monomer capped with \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    methyl\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " group* \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CAT\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ": \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    styrene\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " residue with catalytic moiety attached* \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    SPA\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ": \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    styrene\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " residue with C16 \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    alkyl\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " chain attached* SCL: cross-linking residue (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    p-divinylbenzene\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") with 1 additional branching point* SDB: cross-linking residue (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    p-divinylbenzene\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") with 2 additional branching points (double-branch)Six fragments containing catalyst and hydrophobic residue separated by 0 to 5 \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    styrene\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " units (EIC-0 to EIC-5) were built, with three additional \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    styrene\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " units added on each side of catalyst and hydrophobic residue. These fragments were then solvated with \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    TIP3P\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    waters\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " in a truncated octahedron box with 10 A buffer in each direction from the solute. The solvated fragments were relaxed in 4 steps: 1) steepest descent minimization was performed with maximum of 10 000 cycles; 2) slow heating of the system from 0 to 300 K in 100 ps with harmonic position restraints on solute (2.5 kcal/mol A 2) under constant volume (NVT); 3) constant pressure (NPT) dynamics at \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    300 K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       " for 1 ns with harmonic position restraints placed on solute (2.5 kcal/mol A 2); 4) constant pressure (NPT) dynamics at \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    300 K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       " without restraints for another 1 ns. After relaxation (total of 2.1 ns), each system was simulated for \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    100 ns\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       " at \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    300 K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       " under constant volume (NVT). Periodic boundary conditions were used and long range electrostatic interactions were calculated with the Particle-Mesh Ewald (PME) technique with a cutoff of 8.0 A . The temperature in all simulations was set to \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    300 K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       " and controlled by Langevin thermostat combined with random seed generator.[6] The \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    SHAKE\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " algorithm[7] was employed to constrain bonds involving hydrogen atoms during dynamics and an integration time step of 2 fs was used. The simulations were carried out using GPU version of \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PMEMD\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " program implemented in the \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Amber16\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " package, saving snapshots every 10 ps. Analysis was performed using cpptraj module available in \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    AmberTools\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       ". For clustering, all-atom RMS deviations of catalyst and hydrophobic residues (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    styrene\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " residues were not included) was used in conjunction with k-means algorithm with maximum number of clusters set to 20. An example input file to build a fragment with these residues using \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    LEaP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " module of \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Amber\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " package is provided (build example.leap), as well as the input files for running molecular dynamics (min.in, heat.in, npt-restr.in, npt.in, nvt.in). Conformations representing the two most populated clusters for each fragments is shown in figure named top2clusters EIC0-5.png. Distances calculated between \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CAT\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    SPA\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " residues, as well as the distances measured between head and tail atoms for each of these two residues is shown in figure named Distances&amp;initial conformations.png. Finally, the system topologies and resulting trajectories with stripped solvent are provided (eic-N.prmtop and eic-N.nc files). </div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_21263177.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Extending the Stochastic Titration CpHMD to \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CHARMM36m\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       "<br>The impact of pH on proteins is significant but often neglected in molecular dynamics simulations. Constant-pH Molecular Dynamics (CpHMD) is the state-of-the-art methodology to deal with these effects. However, it still lacks widespread adoption by the scientific community. The stochastic titration CpHMD is one of such methods that, until now, only supported the \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GROMOS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " force field family. Here, we extend this method s implementation to include the \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CHARMM36m\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " force field available in the \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GROMACS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " software package. We test this new implementation with a diverse group of proteins, namely, \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    lysozyme\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Staphylococcal nuclease\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    human and E. coli thioredoxins\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". All proteins were conformationally stable in the simulations, even at extreme pH values. The RMSE values (pKa prediction vs experimental) obtained were very encouraging, in particular for \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    lysozyme\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    human thioredoxin\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". We have also identified a few residues that challenged the CpHMD simulations, highlighting scenarios where the method still needs improvement independently of the force field. The \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CHARMM36m\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " all-atom implementation was more computationally efficient when compared with the \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GROMOS 54A7\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       ", taking advantage of a shorter nonbonded interaction cutoff and a less frequent neighboring list update. The new extension will allow the study of pH effects in many systems for which this force field is particularly suited, i.e., proteins, membrane proteins, lipid bilayers, and nucleic acids.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_6582985.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">MD simulation of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " bilayer with \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    OPLS3e\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " force field. Full hydration 44 w/l<br>MD simulation of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    1-palmitoyl-2-oleoyl-phosphatidylcholine\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") bilayer with \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    OPLS3e\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " force field. Full hydration 44 w/l. Dataset contains trajectories ( trj), topologies (-out.cms) and input files (.cfg, .msj, .cms), and other files. For the ease of the upload, trajectory files ( trj) are divided to \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    100ns\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       " pieces and tarred (named popc44 x-xns trj.tar.gz) \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Gromacs\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " converted files are also included: popchydr44 0-500ns.xtc, \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    desmond\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " md popc200lip500ns-in converted.gro \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    desmond\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " md popc200lip500ns-in converted.top System: \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " bilayer in \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Number of lipids: 200 (100/leaflet) Number of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    waters\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ": 8859 Simulation engine: \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Desmond 2019-4\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       "</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_6478270.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">MD simulation of the crystal unit cell of the second PDZ domain of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    LNX2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " protein<br>This molecular dynamics simulation data is provided as part of the manuscript &quot;LAWS: Local Alignment for \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Sites - a method to analyze crystallographic \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " in simulations&quot;. The code for the algorithm is provided: on github The system contains one unit cell of the crystal (PDB ID: \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    5E11\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") with 4 symmetrically related protein chains. The total simulation length is \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    1 microsecond\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       ". Force field + \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " model: \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CHARMM36m\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " + \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CHARMM\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       "-modified \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    TIP3P\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " Number of atoms: 9650 Number of time frames: 100,000 with 10-ps stride The details of the simulations are provided in the manuscript.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_7783568.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Protonation of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Biliverdin IX\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Chromophore in the Red and Far-Red Photoactive States of a Bacteriophytochrome<br>The \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    tetrapyrrole chromophore\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    biliverdin IX\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    BV\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") in the bacteriophytochrome from Deinococcus radiodurans (DrBphP) is usually assumed to be fully protonated, but this assumption has not been systematically validated by experiments or extensive computations. Here, we use force field molecular dynamics simulations and quantum mechanics/molecular mechanics calculations with density functional theory and XMCQDPT2 methods to investigate the effect of the five most probable protonation forms of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    BV\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " on structural stability, binding pocket interactions, and absorption spectra in the two photochromic states of DrBphP. While agreement with X-ray structural data and measured UV/vis spectra suggest that in both states the protonated form of the chromophore dominates, we also find that a minor population with a deprotonated D-ring could contribute to the red-shifted tail in the absorption spectra.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_20300547.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Structure and Property of Alkylated \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Graphene Oxide\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Depending on the Chain Length: Grand Canonical Monte Carlo-Molecular Dynamics Approach<br>Herein, we present all-atomistic molecular dynamics (MD) simulations of alkylated \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    graphene oxides\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    AGOs\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "), which involve the following two key procedures: (1) the structures of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    AGOs\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " were determined using ab initio and grand canonical Monte Carlo simulations and (2) the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    AGO\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " sheets were assembled and simulated in a layered structure using MD simulation and the physical properties were calculated based on the results. To investigate the effects of the chain length of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    alkyl\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " groups on the structure and properties of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    AGO\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-based materials, we performed two-step simulations, involving (1) equilibration of layered \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    AGOs\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and (2) tensile tests. The results showed that longer \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    alkyl\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " chains reduce the interlayer distance and increase the density of the layered structure. Young s moduli of the layered structures were evaluated using the developed method and were found to increase with increasing \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    alkyl\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " chain length.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_4805388.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">MMPB/GBSA Calculations For \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    TCR\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    pHLA\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "<br>Starting structures, input files and some analysis scripts for the MD simulations and subsequent MMPB/GBSA calculations performed on \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    TCR\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    pHLA\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " complexes. 0 Protein Preperation.zip - Starting protein structures for both wild-type proteins and all variants simulated. 1 MD Simulation Input Files.zip - \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Amber\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " MD simulation input files used for both equilibration and production MD with a readme file informing on the order of operations. 2 MMPB GBSA Calculations.zip - Example MMPB/GBSA calculation inputs alongside two helper scripts to prepare for these calculations after having run the MD simulations (one to retain only local waters and another to generate the topology files needed). 3 nmode \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    amber14\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " modified.tar - Modified version of normal mode analysis that enables the use of the ibelly algorithm. 4 Bootstrapping.zip - Example Bootstrapping calculation script for R. Also provided are exemplar .csv files to show how to arrange your data so R can read these in correctly. </div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_51754.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Molecular dynamics simulation trajectory of a \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    cationic lipid\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " bilayer: 75/25 mol% \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DMTAP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "/\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DMPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " in 0.5 M \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    NaCl\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "<br>System: Symmetric bilayer of cationic \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DMTAP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    dimyristoyltrimethylammoniumpropane\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", 75 mol-%) and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    zwitterionic DMPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    dimyristoylphosphatidylcholine\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", 25 mol-%) lipids in 0.5 M \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    NaCl\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " solution. Number of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DMPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ": 32. Number of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DMTAP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ": 96. Number of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Na+\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " ions: 49. Number of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Cl-\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " ions: 145. Number of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    waters\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ": 5398. Lipid model: '\n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Berger\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       "' united-atom [Biophys. J. 72 2002 (1997)] with \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DMTAP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " modification by Gurtovenko et al. [Biophys. J. 86 3461 (2004)]. \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " model: \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    SPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " [In Intermolecular Forces, ed. Pullman. 331 (1981)]. Simulation engine: \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Gromacs 3.x\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " [https://removed Trajectory length: \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    110 ns\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       ". Previously equilibrated for: 11 ns. Sampling rate: every 10 ps. Time integration step: 2 fs. Thermodynamic ensemble: NpT. Temperature coupling: 'Berendsen' [J. Chem. Phys. 81 3684 (1984)] with lipids and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " coupled separately at T \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    323 K.\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       " Pressure coupling: 'Berendsen' [J. Chem. Phys. 81 3684 (1984)] with xy and z coupled separately at p 1.0 bar. Electrostatics: PME [J. Chem. Phys. 98 10089 (1993); J. Chem. Phys. 103 8577 (1995)], real-space cutoff at 1.0 nm. Van der Waals: Truncated at 1.0 nm. Constraints: Covalent bond lengths in lipids using \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    LINCS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " [J. Comput. Chem. 18 1463 (1997)], in \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " using \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    SETTLE\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " [J. Comput. Chem. 13 952 (1992)]. Used in publications: [1] Markus S. Miettinen, Andrey A. Gurtovenko, Ilpo Vattulainen, and Mikko Karttunen: &quot;Ion Dynamics in Cationic Lipid Bilayer Systems in Saline Solutions&quot;. J. Phys. Chem. B 113 9226 (2009). DOI: 10.1021/jp810233q. [2] Markus S. Miettinen: &quot;Computational Modeling of Cationic Lipid Bilayers in Saline Solutions&quot;. PhD Thesis. Aalto University School of Science and Technology, Helsinki, Finland. (2010). ISBN 978-952-60-3194-1.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_20009556.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Absorption of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Phosphonium\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Cations and Dications into a Hydrated \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Phospholipid Bilayer: A Computational Study<br>Molecular dynamics (MD) based on an empirical force field is applied to investigate the effect of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    phosphonium\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " cations ([P6,6,6,6]+) and geminal dications ([DxC10]2+) inserted at T   \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    300 K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       " into the hydration layer separating planar \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " phospholipid bilayers. Up to high concentration, nearly every added cation and dication becomes absorbed into the lipid phase. Absorption takes place during several microseconds and is virtually irreversible. The neutralizing counterions ([\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Cl\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "] , in the present simulation) remain dissolved in \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", giving origin to the charge separation and the strong electrostatic double layer at the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "/lipid interface. Incorporation of cations and dications changes the properties of the lipid bilayer such as diffusion, viscosity, and the electrostatic pattern. At high ionic concentration, the bilayer acquires a long-wavelength standing undulation, corresponding to a change of phase from fluid planar to ripple. All these changes are potentially able to affect processes relevant in the context of cell biology. The major difference between cations and dications concerns the kinetics of absorption, which takes place nearly two times faster in the [P6,6,6,6]+ case, and for [DxC10]2+ dications displays a marked separation into two-stages, corresponding to the easy absorption of the first \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    phosphonium\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " head of the dication and the somewhat more activated absorption of the second \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    phosphonium\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " head of each dication.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_13499346.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Atomistic Perspective on Biomolecular Adsorption on Functionalized \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Carbon Nanomaterials\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " under Ambient Conditions<br>The use of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    carbon-based nanomaterials\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " is tremendously increasing in various areas of technological, bioengineering, and biomedical applications. The functionality of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    carbon-based nanomaterials\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " can be further broadened via chemical functionalization of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    carbon nanomaterial\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " surfaces. On the other hand, concern is rising on possible adverse effects when nanomaterials are taken up by biological organisms. In order to contribute into understanding of interactions of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    carbon-based nanomaterials\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " with biological matter, we have investigated adsorption of small biomolecules on nanomaterials using enhanced sampling molecular dynamics. The biomolecules included amino acid side chain analogues, fragments of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    lipids\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    sugar monomers\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". The adsorption behavior on unstructured amorphous \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    carbon\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    pristine graphene\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and its derivatives (such as few-layer \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    graphene\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    graphene oxide\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    reduced graphene oxide\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") as well as \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    pristine carbon\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " nanotubes, and those functionalized with \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    OH\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " , \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    COOH\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " , \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    COO\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " , \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    NH2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " , and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    NH3+\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " groups was investigated with respect to surface concentration. An adsorption profile, that is, the free energy as a function of distance from the nanomaterial surfaces, was determined for each molecule and surface using the Metadynamics approach. The results were analyzed in terms of chemical specificity, surface charge, and surface concentration. It was shown that although morphology of the nanomaterial has a limited effect on the adsorption properties, functionalization of the surface by various molecular groups can drastically change the adsorption behavior that can be used in the design of nanosurfaces with highly selective adsorption properties and safe for human health and environment.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_2178850.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Two-Level Spatial Modulation of Vibronic Conductance in Conjugated \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Oligophenylenes\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " on \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Boron Nitride\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "<br>Intramolecular current-induced vibronic excitations are reported in highly ordered monolayers of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    quaterphenylene dicarbonitriles\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " at an electronically patterned \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    boron nitride\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " on \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    copper\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " platform (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    BN\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "/\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Cu\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (111)). A first level of spatially modulated conductance at the nanometer-scale is induced by the substrate. Moreover, a second level of conductance variations at the molecular level is found. Low temperature scanning tunneling microscopy studies in conjunction with molecular dynamics calculations reveal collective amplification of the molecule s interphenylene torsion angles in the monolayer. Librational modes influencing these torsion angles are identified as initial excitations during vibronic conductance. Density functional theory is used to map \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    phenylene\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " breathing modes and other vibrational excitations that are suggested to be at the origin of the submolecular features during vibronic conductance.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_44660.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">MD simulation trajectory and related files for \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " bilayer with 30 mol% of protonated \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    pazePC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Berger\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Gromacs 4.5.\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       ") Simulation trajectory and related files for \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " bilayer with 30mol% of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    pazePC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " in deprotonated form used in &quot;\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Acyl\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " chain disorder and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    azelaoyl\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " orientation in lipid membranes containing oxidised lipids&quot; by T. M. Ferreira, et al. https://removed The simulation with deprotonated \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    pazePC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " from the same publication available at https://removed 90 \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", 38 \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    pazePC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", 7250 \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and 38 \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    potassium\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " molecules. Simulation time \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    167ns\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       ". Force field for \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " is based on \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Berger\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " model and force field for \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    pazePC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " is modified from H. Khandelia and O. G. Mouritsen, Biophysical Journal, 2009, 96 (7), 2734 2743, https://removed</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_1219494.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Simulation \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CER160\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " 50 \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " 512 \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    NaCl\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " 150mM \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    310K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       " (rep1)<br>Title publication: A \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Ceramide\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-Regulated Element in the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Late Endosomal Protein\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    LAPTM4B\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Controls Amino Acid Transporter Interaction Year publication: 2018 DOI publication: 10.1021/acscentsci.7b00556 Description: Simulation \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " membrane containing ~8% \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ceramide\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CER160\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") using \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    charmm36\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " at \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    310K.\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       " MD engine: \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Gromacs 2016.2-dev-20170105-4feb0be\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " Force field: \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    charmm36\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " Temperature: \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    310 K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       " Pressure: 1 bar Simulation time: \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    1069 ns\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       " Saving frequency: 100 ps (A version with saving frequency of 10 ps can be found in the following links) https://removed https://removed Molecular content: \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " 512 \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CER160\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " 50 \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    SOL\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " 28100 \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    NA\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " 75 \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CL\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " 75</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_19082291.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Importance of Three-Body Problems and Protein Protein Interactions in \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Proteolysis-Targeting Chimera\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Modeling: Insights from Molecular Dynamics Simulations<br>\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Proteolysis-targeting chimeras\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PROTACs\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") are a class of bifunctional molecules that can induce the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ubiquitin\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " degradation of its target protein by hijacking the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    E3 ligase\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " to form a target protein-\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PROTAC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    E3 ligase\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " ternary complex. Its underlying principle has inspired the development of a wide range of protein degraders that are similar to or beyond \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PROTACs\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " in recent years. The formation of the ternary complexes is the key to the success of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PROTAC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-induced protein degradation. Nevertheless, the lack of effective ternary complex modeling techniques has limited the application of computer-aided drug discovery tools to this emerging and fast developing new land in drug industry. Thus, in this study, we explored the application of the more physically sound molecular dynamics simulation and the molecular mechanics combined with the generalized Born and surface area continuum solvation (MM/GBSA) method to solve the underlying three-body problem in \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PROTAC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " modeling. We first verified the accuracy of our approach using a series of known \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Brd4 BD2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " degraders. The calculated binding energy showed a good correlation with the experimental Kd values. The modeling of a unique property, namely, the value, for \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PROTACs\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " was also first and accurately performed to our best knowledge. The results also demonstrated the importance of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PROTAC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-induced protein protein interactions in its modeling, either qualitatively or quantitatively. Finally, by standing on the success of earlier docking-based approaches, our protocol was also applied as a rescoring function in pose prediction. The results showed a notable improvement in reranking the initial poses generated from a modified \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Rosetta\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " method, which was reportedly one of the best among a handful of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PROTAC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " modeling approaches available in this field. We hope this work could provide a practical protocol and more insights to study the binding and the design of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PROTACs\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and other \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    protein\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " degraders.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_2644158.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">A1810 raw data<br>Molecular dynamics simulation trajectories of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    kinesin\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " motor domain exposed to electric field and the starting structure (system.gro). List of the files system.gro the initial structure of the system Trajectories, \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    kinesin\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " recentered trajectories with no electric field applied NOFIELD-traj 1 cent.xtc NOFIELD-traj 2 cent.xtc NOFIELD-traj 3 cent.xtc trajectories with 100 MV/m electric field in X direction X-traj 1 cent.xtc X-traj 2 cent.xtc X-traj 3 cent.xtc trajectories with 100 MV/m electric field in -X direction Xrev-traj 1 cent.xtc Xrev-traj 2 cent.xtc Xrev-traj 3 cent.xtc trajectories with 100 MV/m electric field in Y direction Y-traj 1 cent.xtc Y-traj 2 cent.xtc Y-traj 3 cent.xtc trajectories with 100 MV/m electric field in -Y direction Yrev-traj 1 cent.xtc Yrev-traj 2 cent.xtc Yrev-traj 3 cent.xtc </div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_2414671.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Molecular Dynamics Simulations of Depth Distribution of Spin-Labeled \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Phospholipids\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " within Lipid Bilayer<br>Spin-labeled lipids are commonly used as fluorescence quenchers in studies of membrane penetration of dye-labeled proteins and peptides using depth-dependent quenching. Accurate calculations of depth of the fluorophore rely on the use of several spin labels placed in the membrane at various positions. The depth of the quenchers (spin probes) has to be determined independently; however, experimental determination of transverse distributions of spin probe depths is difficult. In this Article, we use molecular dynamics (MD) simulations to study the membrane behavior and depth distributions of spin-labeled \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    phospholipids\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " in a \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    1-palmitoyl-2-oleoyl-sn-glycero-3-phosphocholine\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") bilayer. To probe different depths within the bilayer, a series containing five \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Doxyl-labeled lipids\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    n-Doxyl PC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") has been studied, in which a spin moiety was covalently attached to nth carbon atoms (where n   5, 7, 10, 12, and 14) of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    sn-2 stearoyl\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " chain of the host \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    phospholipid\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". Our results demonstrate that the chain-attached spin labels are broadly distributed across the model membrane and their environment is characterized by a high degree of mobility and structural heterogeneity. Despite the high thermal disorder, the depth distributions of the Doxyl labels were found to correlate well with their attachment positions, indicating that the distribution of the spin label within the model membrane is dictated by the depth of the nth lipid carbon atom and not by intrinsic properties of the label. In contrast, a much broader and heterogeneous distribution was observed for a headgroup-attached \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Tempo\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " spin label of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Tempo-PC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " lipids. MD simulations reveal that, due to the hydrophobic nature, a \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Tempo\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " moiety favors partitioning from the headgroup region deeper into the membrane. Depending on the concentration of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Tempo-PC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " lipids, the probable depth of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Tempo\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " moiety could span a range from 14.4 to 18.2 A from the membrane center. Comparison of the MD-estimated immersion depths of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Tempo\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    n-Doxyl\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " labels with their suggested experimental depth positions allows us to review critically the possible sources of error in depth-dependent fluorescence quenching studies.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_14156828.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">An Active Site Inhibitor Induces Conformational Penalties for \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ACE2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Recognition by the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Spike Protein\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    SARS-CoV 2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "<br>The novel \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    RNA\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " virus, \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    severe acute respiratory syndrome coronavirus II\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    SARS-CoV-2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "), is currently the leading cause of mortality in 2020, having led to over 1.6 million deaths and infecting over 75 million people worldwide by December 2020. While vaccination has started and several clinical trials for a number of vaccines are currently underway, there is a pressing need for a cure for those already infected with the virus. Of particular interest in the design of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    anti-SARS-CoV-2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " therapeutics is the human protein \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    angiotensin converting enzyme II\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ACE2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") to which this virus adheres before entry into the host cell. The \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    SARS-CoV-2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " virion binds to cell-surface bound \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ACE2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " via interactions of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    spike protein\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    s-protein\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") on the viral surface with \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ACE2.\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " In this paper, we use all-atom molecular dynamics simulations and binding enthalpy calculations to determine the effect that a bound \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ACE2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " active site inhibitor (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    MLN-4760\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") would have on the binding affinity of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    SARS-CoV-2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    s-protein\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " with \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ACE2.\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Our analysis indicates that the binding enthalpy could be reduced for \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    s-protein\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " adherence to the active site inhibitor-bound \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ACE2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " protein by as much as 1.48-fold as an upper limit. This weakening of binding strength was observed to be due to the destabilization of the interactions between \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ACE2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " residues \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Glu-35\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Glu-37\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Tyr-83\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Lys-353\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Arg-393\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    SARS-CoV-2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    s-protein\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " receptor binding domain (RBD). The conformational changes were shown to lead to weakening of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ACE2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " interactions with \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    SARS-CoV-2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    s-protein\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", therefore reducing s-protein binding strength. Further, we observed increased conformational lability of the N-terminal helix and a conformational shift of a significant portion of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ACE2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " motifs involved in \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    s-protein\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " binding, which may affect the kinetics of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    s-protein\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " binding when the small molecule inhibitor is bound to the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ACE2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " active site. These observations suggest potential new ways for interfering with the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    SARS-CoV-2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " adhesion by modulating \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ACE2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " conformation through distal active site inhibitor binding.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_53887.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Molecular Dynamics Study of the Vibrational Energy Exchange Between \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Nanodiamond\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    N-butylpyridinium Tetrafluoroborate\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Ionic Liquid<br>Nanofluids are composed of certain nanoscale object immersed in the molecular or ionic liquid. Nanofluids generally exhibit higher thermal conductivities, as compared to conventional fluids. Therefore, they are interesting to applications, in which thermal conductivity is a cornerstone. Hereby, molecular dynamics simulations were used to study an energy exchange in one of the possible nanofluids: \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    N-butylpyridinium tetrafluoroborate\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " based solution of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    nanodiamonds\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". The data are given in the form of energy components versus time and atomistic trajectories versus time at a number of temperatures. The simulations were conducted in the constant energy ensemble to avoid efects of temperature coupling and pressure coupling. The dataset possesses both research and educational values.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_7037843.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">mRNA lipid nanoparticle phase transition<br>The repository contains input files and data from the manuscript: Trollmann, Marius F.W. and Bo ckmann, Rainer A. &quot;mRNA lipid nanoparticle phase transition&quot;, Biophysical Journal (2022) https://doi.org/10.1016/j.bpj.2022.08.037 &gt; Periodic membrane patches Equilibrated structures, .mdp and .top files for the simulations of the periodic Comirnaty membrane patches. (\n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    microsecond\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       "   \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    us\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       ") &gt;&gt; periodic patches/low ph/single patch: ~ System A: Three replicas (\n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    4.1 us\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    3.0 us\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    3.0 us\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       ") of the self-assembled Comirnaty lipid mixture with the protonated aminolipid &gt;&gt; periodic patches/low ph/quad patch: System B: A quadruplicated system A patch simulated for \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    1.0 us\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       " &gt;&gt; periodic patches/\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    dspc\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " chol: System C: Binary membrane including \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DSPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and 43mol% \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    cholesterol\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " &gt;&gt; periodic patches/neutral ph: System D: A deprotonated system B patch simulated for \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    0.633 us\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       " &gt;&gt; periodic patches/mrna selfassembly: System E: Structures of the self-assembled Comirnaty lipid mixture with the modified mRNA strand + periodic patches/mrna selfassembly/selfassembly: Structures of the mRNA-lipid mixture after self-assembly with protonated aminolipids + periodic patches/mrna selfassembly/set1: Quadruplicated simulation systems after deprotonation of distant aminolipids (set 1, see paper) + periodic patches/mrna selfassembly/set2: Quadruplicated simulation systems after deprotonation of random aminolipids (set 2, see paper) + periodic patches/mrna selfassembly/set3: Simulation systems after deprotonated of all aminolipids (systems were not quadruplicated) (set 3, see paper) &gt; Lipid nanoparticles Equilibrated structures, .mdp and .top files for the simulations of the lipid nanoparticles. (\n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    microsecond\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       "   \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    us\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       ") &gt;&gt; nanoparticles/lnp nopegs: System F: Structures of the lipid nanoparticles with capped PEGylated lipids &gt;&gt; nanoparticles/lnp pegs: System G: Structure of the lipid nanoparticle with complete PEGylated lipids &gt; Topologies - topology/DSPC.top - Parameters for the standard phospholipid from the \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CHARMM36\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " forcefield - topology/CHOL.top - Parameters for \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    cholesterol\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " from the \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CHARMM36\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " forcefield - topology/alc.itp, topology/alc.prm - Parametrization files of the PEG-ylated lipid ALC-0159 obtained from the \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CGenFF-Webserver\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " - topology/alc neutral.itp - Parameters for the neutral aminolipid ALC-0315 obtained from the \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CGenFF-Webserver\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " - topology/alc protonated.itp - Parameters for the protonated aminolipid ALC-0315 obtained from the \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CGenFF-Webserver\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " - topology/modRNA.top - Parameters for the short modified mRNA strand. \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Uridine\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " was replaced with \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    N1-Methylpseudouridine\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". The parameters were not included in the standard \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CHARMM36 forcefield (version July 2020)\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " and were manually added to the forcefield. - topology/ALC SHORT CORRECT IDX.itp - Parameters for the capped PEG-ylated lipid ALC-0159 -&gt; Parameters were manually adapted to fit the shortened structure. - topology/charmm36-jul2020.ff - \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CHARMM36\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " forcefield parameters (version July 2020) with included parameters for \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    N1-Methylpseudouridine\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". - topology/cgenff output - Output from the \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CGenFF-Webserver\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " to parametrize the aminolipid (ALC-0315) and the PEGylated-lipid (ALC-0159) </div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_5784447.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Fourier projection algorithm for calculating phonon dispersion relations of crystals<br>We propose a Fourier projection algorithm and demonstrate its superiority for calculation of the phonon dispersion relations of 2D and 3D crystals based on the atomic coordinate trajectories from supercell molecular dynamics/first-principle molecular dynamics simulations. The lattice vibration states are described in a six-dimensional phase space composed of lattice and wave vectors. Phonon dispersion information in the first Brillouin zone sampled with a k-mesh grid of N N N is generated by Fourier projection of the lattice vibration frequencies in the supercell by N N N primitive cells, the dispersion relations along high-symmetry paths are retrieved therefrom. Our algorithm is physically intuitive, computationally convenient, and universally applicable. As examples of application, the successful dispersion relation calculations of 2D \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    graphene\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", 3D BCC-\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Fe\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and FCC-\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Cu\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " are presented. This algorithm shows the correspondence between real and reciprocal lattices, that is, the spatial information in the states of the lattice vibration were preserved. The high-symmetry paths of the first Brillouin zone in the reciprocal lattice primitive cells of the hexagonal, BCC, and FCC lattice were also given in detail. In combination with first-principles molecular dynamics, as long as the interatomic force is accurately known, reliable and accurate dispersion relation can be achieved by this Fourier projection algorithm.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_5680225.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Simulated complex structures of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    h-FBP21\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " tandem WW domain with proline-rich ligand extracted from SmB/B' core-splicing protein<br>The tandem WW domain of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    human formin-binding protein 21\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    h-FBP21\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " tWW) consists of two WW domains separated by a flexible linker. It can bind target sequences in two different orientations and the flexibility of the linker additionally allows the two WW domains to adopt various relative orientations to each other. As consequence, the elucidation of possible complex structures for the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    h-FBP21\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " tWW is very challenging. Here, we present two complex structures for the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    h-FBP21\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " tWW and a proline-rich sequence from its natural binding partner, the core-splicing protein SmB/B . Showing parallel ( 6 ) and antiparallel ( 14 ) binding orientation, the two structures also differ in the relative positioning of the WW domains. For further instructions regarding the files, please refer to README .</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_8046437.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Short molecular dynamics of a peptide inside a pure \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DMPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " membrane<br>\n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    1 ns\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       " of molecular dynamics simulation of a 19-residue peptide inside a pure \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DMPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " membrane, performed at the NPT ensemble - 1 atm at \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    310K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       " using Berendsen (semi-isotropic) and V-rescale for pressure and temperature coupling, respectively, with 1.6 ps and 0.1ps as coupling constants. Van der Waals interactions were treated using the Verlet algorithm with a 1 nm cutoff. Electrostatics where treated with particle-mesh Ewald, with a 1 nm cutoff for the real space calculations. Both the peptide and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DMPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " were parameterized using the \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GROMOS 54A7\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       ", while \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    SPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " was used for \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    LINCS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " restraints were used on all the bonds. Peptide sequence is \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    AAAQAAQAQWAQRQATWQA\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". This sequence was not taken from any real world examples, as this is a test system created for MDAnalysis datasets. The peptide was set to have be -helical secondary structure, and was inserted manually in the membrane before minimization and equilibration. More on MDAnalysis: https://www.mdanalysis.org/</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_21528986.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Conserved \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Networks Identification for Drug Design Using Density Clustering Approaches on Positional and Orientational Data<br>This work describes the development and testing of a method for the identification and classification of conserved \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " molecules and their networks from molecular dynamics (MD) simulations. The conserved \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    waters\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " in the active sites of proteins influence protein ligand binding. Recently, several groups have argued that a \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " network formed from conserved \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    waters\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " can be used to interpret the thermodynamic signature of the binding site. We implemented a novel methodology in which we apply the complex approach to categorize \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " molecules extracted from the MD simulation trajectories using clustering approaches. The main advantage of our methodology as compared to current state of the art approaches is the inclusion of the information on the orientation of hydrogen atoms to further inform the clustering algorithm and to classify the conserved \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    waters\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " into different subtypes depending on how strongly certain orientations are preferred. This information is vital for assessing the stability of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " networks. The newly developed approach is described in detail as well as validated against known results from the scientific literature including comparisons with the experimental data on \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    thermolysin\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    thrombin\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", and Haemophilus influenzae virulence protein \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    SiaP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " as well as with the previous computational results on \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    thermolysin\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". We observed excellent agreement with the literature and were also able to provide additional insights into the orientations of the conserved \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " molecules, highlighting the key interactions which stabilize them. The source code of our approach, as well as the utility tools used for visualization, are freely available on GitHub.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_7007107.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Thriving in the heat \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Lysine\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " acetylation stabilizes the quaternary structure of a Mega-Dalton hyperthermoactive \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PEP-synthase\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "<br>Over time structural adaptations enabled proteins and enzymes to have sufficient stability and flexibility to perform the basic functions of life under various environmental conditions. The catalytic cores of key metabolic enzymes of hyperthermophilic archaea work at a temperature range of \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    80-120 C\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       ", similar to the conditions wher the earliest life forms may have thrived. Here we characterize a key enzyme of the central carbon metabolism of Pyrococcus furious, through an integrative approach combining structural mass spectrometry, cryo-electron microscopy, mass photometry and molecular modelling with molecular dynamics simulations. From our investigation, we unveil the structural organization of phosphoenolpyruvate synthase (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PPSA\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "). Its 24-meric assembly - weighing over 2 MDa - harbors flexible distal domains, whose proper functioning and coordination depends on widespread chemical acetylation of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    lysine\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " residues. This non-enzymatic post-translational modification, along with other types of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    lysine\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " modifications, also occurs on most other major protein complexes of P. furiosus. These modifications likely originated in the chemically favorable primordial conditions and gradually became highly specialized and enzyme-driven in more distantly related mesophiles and Eukaryotes. Molecular dynamics simulations and analysis The all-atom structures of the full c1 (X4) 24-mer \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PPSA\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " models, carrying highly acetylated sites at 14 positions (106, 120,185, 187, 427, 466, 492, 496, 557, 574, 641, 726, 737, 805) or unmodified lysines were coarse grained (CG), mapping their atoms to the \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    SIRAH\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " force field (ff), that uses a classical Hamiltonian common to most all-atom potentials to describe particle particle interactions, and recently extended to support the PTMs most commonly found on proteins (Garay et al., 2020). Both starting structures contained a disulfide bond between Cys42-Cys189 as well as phosphorylation of Thr440. All starting structures, simulation boxes and parameters files used for the minimization, equilibration and production are provided (Supplementary Data 5). Simulation was conducted in \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GROMACS 2020.4\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " (Hess et al., 2008), for which code can be found here: https://doi.org/10.5281/zenodo.5636522. The 24-aly system consisted of 94128 CG atoms for the protein representation, solvated with 179271 WT4 CG \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " beads (Garay et al., 2020). A neutral charge was achieved adding 6617 NaW (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Na+\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") and 5633 ClW (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Cl-\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "), corresponding to a concentration of approximately 150 mM \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    NaCl\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " concentration. The 24-lys system was also prepared accordingly and consisted of 93456 CG atoms for the protein, 186690 WT4 CG \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " beads and neutralized with 6503 NaW (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Na+\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") and 5855 ClW (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Cl-\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "). Solvation was done using the default radii of 0.105 nm for atoms not present in the VdW database (vdwradii.dat) and then removing the WT4 molecules within 0.3 nm from the solute. In all cases, eventual clashes were relaxed during the solute-restrained energy minimization. Due to the length of the loops connecting the 3 domains, an alternative configuration of the tetramers where the CD-NBD domains are sitting on top of the neighboring \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PPSA\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " subunit is also possible (see Supplementary Note 1). This was named alternative (a) configuration, as opposed to the original (o), thus producing 4 starting models: a24-aly, o24-aly, a24-lys and o24-lys (Supplementary Data 5). All these configurations were subjected to a first round of production consisting of \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    125 ns\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       " in duplicate to define the stable configuration for further extension of the simulation time to \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    500 ns\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       ". The main stages of the simulation can be summarized as follows:1) solvent and side-chain relaxation by 2 stages of 20 000 steps of energy minimization, imposing positional restraints of 1000 kJ mol-1 nm-2 on the whole protein (stage 1) and only on backbone beads (GN and GO, stage 2); 2) solvent NVT ensemble equilibration with a first stage where the temperature was slowly increased from \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    303K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       " to \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    363K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       " in 7 steps of \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    4 ns\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       " each, and a second stage to equilibrate the protein by gradually releasing the positional restraints from 1000 kJ mol-1 nm-2 on the backbone beads (GN and GO) to 100 kJ mol-1 nm-2 on the C-terminus to compensate for the missing stabilization effect of the Met799-Fe cluster; 3) production simulation of an additional \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    80 ns\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       " in NPT ensemble at \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    363 K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       " and 1 bar imposing positional restraints of 100 kJ mol-1 nm-2 on the C-terminus. Non-bonded interactions were treated with a 1.2 nm cutoff and PME for long-range electrostatics. An integration time-step of 15 fs was used during MD production runs. The system pressure was controlled by the Parrinello-Rahman barostat (Parrinello and Rahman, 1981) with a coupling time of 4 ps. The positional restraints during the production simulation were necessary to maintain the overall system at the high energy of the particles at \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    363 K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    90 C\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       "), the temperature mimicking the near-optimum temperature for \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PPSA\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " catalytic activity. All simulations were run in duplicate. Although the simulations do not accurately reflect the possible dynamics of the protein in native conditions, the computational challenges of simulating &gt;350000 atoms for the protein only drove us to use a CG representation of the system. To analyze the trajectories, every functional module (tetramer) was extracted from the simulation of the full 24-mer for each replicate, resulting in two sets of 12 independent trajectories (6 for each 24-mer) either with and without acetylated \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Lysine\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " residues. To produce morphed movies and analyze secondary structure and interfaces, the CG tetramers trajectories were back mapped to atomistic detail using \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    SIRAH\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " tools via \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    VMD\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       ". A back mapped atomistic model was produced every 35.7 ns, and subjected to 100 cycles of energy minimization in \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    AMBER\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       ", resulting in 14 atomistic models describing each trajectory of a given tetramer over the \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    500 ns\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       " of simulation, further interpolated in \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ChimeraX\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " using the morph command to produce the atomistic representation of the dynamics (Supplementary Data 5). RMSD calculation and trajectory analysis were done with the \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    MDanalysis\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " suite (Michaud-Agrawal et al., 2011). Each probability density in Figure 5B and 5C is calculated over two independent coarse-grained molecular dynamics simulations of the last \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    250 ns\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       " of production of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PPSA\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " 24-mer in the acetylated and non-acetylated form, using a snapshot frequency of 5 ns. In Figure 5B, the RMSD of the C atoms (equivalent to backbone GC beads in CG MD) of the NBD region (residues 1-365) in the tetramer form was calculated with respect to their conformation in the tetramer \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PPSA\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " resting state. The six trajectories of all six tetramers included in the simulated \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PPSA\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " complex, resulting in \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    3 μs\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       " ([250ns*6]*2) of accumulated tetramer simulation, were used in the probability density calculation. In Figure 5C, the RMSD of the C atoms (equivalent to backbone GC beads in CG MD) was calculated over the CD (residues 379-481) and PBD (residues 510-790) regions with respect to the modelled CD-PBD state of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PPSA\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " in the monomer form. All 24 \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PPSA\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " monomers of which the simulated \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PPSA\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " complex was composed were included in the probability density calculation, resulting in \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    12 μs\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       " ([250ns*24]*2) of accumulated \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PPSA\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " monomer simulation. </div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_14511885.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Datasets supporting Haloi et al. Communications Biology<br>This file includes datasets used in Haloi et al. - Communications Biology. Title of the manuscript &quot;Structural Basis of Complex Formation Between Mitochondrial Anion Channel \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    VDAC1\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Hexokinase-II\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " &quot;.Fig2 PanelA HMMM.psf, Fig2 PanelA HMMM frame 0, Fig2 PanelA HMMM frame \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    200ns\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       ": Initial and final, membrane-bound configurations of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    HKII-N\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " in a representative HMMM MD trajectory. These files can be directly open using \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    VMD\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       ". Fig2 PanelA full.psf, Fig2 PanelA full frame 200ns.pdb: Final, membrane-bound configurations of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    HKII-N\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " in a representative full-membrane MD trajectory. These files can be directly opened using \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    VMD\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       ". Fig2 PanelB.xlsx: Insertion depth of H-anchor residue I2 into the membrane, calculated using the HMMM and full-membrane MD simulations. Fig2 PanelC.xlsx: Insertion depth of the H-anchor residues (1-27) into the membrane, calculated using the full-membrane MD simulation.Fig3 PanelA.xlsx: Tilt angle of the H-anchor, calculated using the BD and MD simulation.Fig3 PanelD.xlsx: X-Y coordinates of the center of mass of H-anchor, calculated using the trajectories from all 100 independent BD replicates.Fig3 PanelE HKV1.psf, Fig3 PanelE HKV1.pdb: Final configuration membrane-embedded \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    HKII\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "/\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    VDAC1\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " complex (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    HKV1\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "). These files can be directly opened using \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    VMD\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       ". Fig4 PanelB HKV1.xls: Radius profile of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    VDAC1\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " pore in HKV1.Fig4 PanelB VDAC1.xls: Radius profile of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    VDAC1\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " pore in \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    VDAC1\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " alone.Fig4 PanelD.xlsx: The cumulative net number of channel-crossing events by \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Chloride\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Sodium\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", tracked over the time course of a representative MD simulation for \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    VDAC1\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " or the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    HKV1\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " complex.Fig5 Electrophysilogy.xlsx: Values of our electrophysiology data points. Fig6 PanelA.xlsx: X-Y coordinates of the center of mass (C.O.M) of H-anchor, calculated using the trajectories from all 100 independent BD replicates of WT- and S215phosphorylated-\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    VDAC1\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " system.Fig6 PanelB.xlsx: Z-coordinate (perpendicular to the membrane normal) values of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    VDAC1\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " residues, calculated using the full-membrane \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    VDAC1\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-alone MD simulation.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_4106413.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Electric-Field Control of Neon Uptake and Release to and from \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Clathrate Hydrates\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "<br>The thermodynamic and kinetic properties of neon uptake and release to and from a sII \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    clathrate hydrate\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " were evaluated using molecular dynamics simulation in the absence and presence of an applied static electric field. In the case of neon leakage , neon clathrate in contact with a vacuum was simulated at temperatures ranging from \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    200\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       " to \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    225 K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       " for \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    0.5 μs\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       ", with progressive neon-emptying of cages monitored. Activation energies for release and uptake were 16.4 and 14.9 kJ/mol, respectively consistent with experimentally determined values; for neon release in a 0.7 V/nm electric field, the value declines to 6.5 kJ/mol, suggesting its use as a control agent in facilitating gas release from already-loaded clathrates. For neon uptake into an initially empty hydrate, loading free energies for 512 and 51264 cages, computed via thermodynamic integration, was lower than that previously reported for methane uptake into sI hydrates.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_11475366.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Ensemble-Based Molecular Simulation of Chemical Reactions under Vibrational Nonequilibrium<br>We present an approach to incorporate the effect of vibrational nonequilibrium in molecular dynamics (MD) simulations. A perturbed canonical ensemble, in which selected modes are excited to higher temperature while all others remain equilibrated at low temperature, is simulated by applying a specifically tailored bias potential. Our method can be readily applied to any (classical or quantum mechanical) MD setup at virtually no additional computational cost and allows the study of reactions of vibrationally excited molecules in nonequilibrium environments such as plasmas. In combination with enhanced sampling methods, the vibrational efficacy and mode selectivity of vibrationally stimulated reactions can then be quantified in terms of chemically relevant observables, such as reaction rates and apparent free energy barriers. We first validate our method for the prototypical hydrogen exchange reaction and then show how it can capture the effect of vibrational excitation on a symmetric SN2 reaction and radical addition on \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CO2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ".</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_1293813.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">\n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Slipids\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPE\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Simulations (versions 1 and 2) \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    310 K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       " (NOTE: hexagonal membrane)<br>Two \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Slipids\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPE\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " bilayer simulations performed using \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GROMACS 5.0.6\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " for \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    200 ns\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       " with different starting velocities. Simulations were performed with a 1.0 nm cut-off with PME for the Coulombic and a 1.0 nm cut-off for the van der Waals interactions interactions. These simulations were performed at \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    310 K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       " with a 336 hexagonal lipid bilayer. The full trajectories are provided bar the initial 100 ns. The starting structure of the membrane was made by reverse mapping an equilibrated \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    MARTINI\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPE\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " bilayer (constructed with the INSANE script).</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_4748350.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Extension of the Highly Mobile Membrane Mimetic to Transmembrane Systems through Customized in Silico Solvents<br>The mechanics of the protein lipid interactions of transmembrane proteins are difficult to capture with conventional atomic molecular dynamics, due to the slow lateral diffusion of lipids restricting sampling to states near the initial membrane configuration. The highly mobile membrane mimetic (HMMM) model accelerates lipid dynamics by modeling the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    acyl\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " tails nearest to the membrane center as a fluid organic solvent while maintaining an atomic description of the lipid headgroups and short \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    acyl\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " tails. The HMMM has been applied to many peripheral protein systems; however, the organic solvent used to date caused deformations in transmembrane proteins by intercalating into the protein and disrupting interactions between individual side chains. We ameliorate the effect of the solvent on transmembrane protein structure through the development of two new in silico Lennard-Jones solvents. The parameters for the new solvents were determined through an extensive parameter search in order to match the bulk properties of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    alkanes\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " in a highly simplified model. Using these new solvents, we substantially improve the insertion free energy profiles of 10 protein side chain analogues across the entire bilayer. In addition, we reduce the intercalation of solvent into transmembrane systems, resulting in native-like transmembrane protein structures from five different topological classes within a HMMM bilayer. The parametrization of the solvents, in addition to their computed physical properties, is discussed. By combining high lipid lateral diffusion with intact transmembrane proteins, we foresee the developed solvents being useful to efficiently identify membrane composition inhomogeneities and lipid binding caused by the presence of membrane proteins.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_7323535.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Unbiased simulation of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Alanine Dipeptide\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " in gas phase<br>\n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    87 microsecond\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       " long unbiased Molecular Dynamics (MD) trajectory of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    alanine dipeptide\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " in gas phase (traj comp.xtc). Temperature   \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    300 K.\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       " Force Field: \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    AMBER99SB-ILDN\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       ". There are 30+ back and forth transitions between the C 7eq and the C 7ax state. Simulations were performed using \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GROMACS 2021.4\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       ". The .tpr file is provided for reproduction. Details of the simulation parameters are accessible from the md.log file. The phi and psi torsion angles and the internal energy is printed in the \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    COLVAR\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " file at 2 ps interval. \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Alanine dipeptide\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " is often used as a model system to test new simulation methods. We hope that sharing our long unbiased trajectory will help other research groups to compare the accuracy of the results obtained from new methods. This trajectory was generated as a part of our recent publication below. Please cite the following paper when using this trajectory: 1. Ray, Dhiman, Narjes Ansari, Valerio Rizzi, Michele Invernizzi, and Michele Parrinello. &quot;Rare event kinetics from adaptive bias enhanced sampling.&quot; Journal of Chemical Theory and Computation (2022). https://doi.org/10.1021/acs.jctc.2c00806 </div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_3779223.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Molecular dynamics simulation data of regulatory ACT domain monomer of human \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    phenylalanine hydroxylase\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PAH\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ")<br>Raw data of molecular dynamics simulations of regulatory ACT domain monomer. binding.zip: simulation starting from 21 monomer conformations with 19 Phe ligand bound.zip: simulation starting from monomer with bound Phe ligand monomer only.zip: simulation starting from 21 monomer conformations simulation Simulation setup files are also included in each folder. Adaptive sampling data are also included in monomer and binding simulations. Details can be found in this paper: Yunhui Ge, Elias Borne, Shannon Stewart, Michael R. Hansen, Emilia C. Arturo, Eileen K. Jaffe and Vincent A. Voelz. Simulation of the regulatory ACT domain of human \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PAH\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " unveil the mechanism of phenylalanine binding. J. Biol. Chem., 2018, 293 (51), pp 19532-19543</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_21304804.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Modeling the Effect of Hydrophobicity on the Passive Permeation of Solutes across a Bacterial Model Membrane<br>Passive diffusion across biomembranes is an important mechanism of permeation for multiple drugs, including antibiotics. However, this process is frequently neglected while studying drug uptake and, in our view, warrants further investigation. Here, we apply molecular dynamics simulations to investigate the impact of changes in molecular hydrophobicity on the permeability of a series of inhibitors of the quorum sensing of Pseudomonas aeruginosa, previously discovered by us, across a membrane model. Overall, we show that permeation across this membrane model does not correlate with the molecule s hydrophobicity. We demonstrate that using a simple model for permeation, based on the difference between the maximum and minimum of the free energy profile, outperforms the inhomogeneous solubility-diffusion model, yielding a permeability ranking that better agrees with the experimental results, especially for hydrophobic permeants. The calculated differences in permeability could not explain differences in in bacterio activity. Nevertheless, substantial differences in molecular orientation along the permeation pathway correlate with the in bacterio activity, emphasizing the importance of analyzing, at an atomistic level, the permeation pathway of these solutes.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_14994624.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Atomistic Molecular Dynamics Simulations of Lipids Near \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    TiO2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Nanosurfaces<br>Understanding of interactions between inorganic nanomaterials and biomolecules, and particularly lipid bilayers, is crucial in many biotechnological and biomedical applications, as well as for the evaluation of possible toxic effects caused by nanoparticles. Here, we present a molecular dynamics study of adsorption of two important constituents of the cell membranes, \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    1,2-dimyristoyl-sn-glycero-3-phosphocholine\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DMPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    1-palmitoyl-2-oleoyl-sn-glycero-3-phosphoethanolamine\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPE\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "), lipids to a number of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    titanium dioxide\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " planar surfaces, and a spherical nanoparticle under physiological conditions. By constructing the number density profiles of the lipid headgroup atoms, we have identified several possible binding modes and calculated their relative prevalence in the simulated systems. Our estimates of the adsorption strength, based on the total fraction of adsorbed lipids, show that \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPE\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " binds to the selected \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    titanium dioxide\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " surfaces stronger than \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DMPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", due to the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ethanolamine\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " group forming hydrogen bonds with the surface. Moreover, while \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPE\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " shows a clear preference toward anatase surfaces over rutile, \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DMPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " has a particularly high affinity to rutile (101) and a lower affinity to other surfaces. Finally, we study how lipid concentration, addition of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    cholesterol\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", as well as \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    titanium dioxide\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " surface curvature may affect overall adsorption.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_6755131.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">All-atom molecular dynamics simulations of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    phenylalanine-4-hydroxylase\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PAH\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") tetramer to investigate the impact of two novel heterozygous mutations, p.Y198N and p.Y204F, observed in a classical phenylketonuria patient<br>\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Phenylalanine-4-hydroxylase\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PAH\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") tetramer system (\n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Robetta\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " modelling to complete the structure with template PDB ID: \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    6hyc\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") with parametrised \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    BH4\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " ligand (parameters are available in the dataset) and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Fe\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (II) metal ions in a \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    TIP3P\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " box ionised with 0.15 M \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    KCl\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " were presented as wild-type and carrying two novel mutations as Y198N on dimeric chains A and B, and Y204F on dimeric chains C and D. In addition, E353 and E422 are protonated as predicted by \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PROPKA\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       ". \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    BH4\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " molecule parametrization was performed by using \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GAFF\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Antechamber\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " and amb2chm par.py program of \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Amber2018\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       ". 5,000-step minimization and 1 ns equilibration were performed by fixing the protein to relax the system. Then, another 5,000-step minimization and 1 ns equilibration were performed without any constraints, except the \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    SHAKE\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " algorithm on \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " molecules, to relax the protein and system. The production simulations were performed along \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    100 ns\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       " trajectory at \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    310 K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       " collected under NpT ensemble. All system preparation and simulation details for this dataset is available with the related background, results and conclusions in the following article: Tolga Aslan, Asl Yenenler-Kutlu, Umut Gerlevik, Ays e C ig dem Aktug lu Zeybek, Ertug rul K yk m, Osman Ug ur Sezerman &amp; Necla Birgul Iyison (2021) Identifying and elucidating the roles of Y198N and Y204F mutations in the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PAH\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " enzyme through molecular dynamic simulations, Journal of Biomolecular Structure and Dynamics, DOI: 10.1080/07391102.2021.1921619</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_16712908.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Just Add \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ": Modulating the Structure-Derived Acidity of Catalytic Hexameric Resorcinarene Capsules<br>The hexameric \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    undecyl-resorcin[4] arene\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " capsule (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    C11R6\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") features eight discrete structural \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " molecules located at the vertices of its cubic suprastructure. Combining NMR spectroscopy with classical molecular dynamics (MD), we identified and characterized two distinct species of this capsule, \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    C11R6\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-A and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    C11R6\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-B, respectively featuring 8 and 15 \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " molecules incorporated into their respective hydrogen-bonded networks. Furthermore, we found that the ratio of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    C11R6\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-A and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    C11R6\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-B found in solution can be modulated by controlling the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " content of the sample. The importance of this supramolecular modulation in \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    C11R6\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " capsules is highlighted by its ability to perform acid-catalyzed transformations, which is an emergent property arising from the hydrogen bonding within the suprastructure. We show that the conversion of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    C11R6\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-A to \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    C11R6\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-B enhances the catalytic rate of a model Diels Alder cyclization by 10-fold, demonstrating the cofactor-derived control of a supramolecular catalytic process that emulates natural enzymatic systems.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_21806690.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Investigating the Solvent Effects on Binding Affinity of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PAHs\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ExBox4+\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Complexes: An Alchemical Approach<br>\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Polycyclic aromatic hydrocarbons\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PAHs\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") are polluting agents, produced naturally or artificially, widely dispersed in the environment and potentially carcinogenic and immunotoxic to humans and animals, mainly for marine life. Recently, a tetracationic box-shaped \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    cyclophane\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ExBox4+\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") was synthesized, fully characterized, and revealed to form host guest complexes with \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PAHs\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " in \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    acetonitrile\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", demonstrating the potential ability for it to act as a \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PAHs\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " scavenger. This work investigates, through Molecular Dynamics (MD) simulations, the binding affinity between different \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PAHs\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ExBox4+\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " in different solvents: \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    chloroform\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (nonpolar), \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    acetonitrile\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (polar protic), and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (polar protic). An alchemical method of simultaneous decoupling-recoupling (SDR) was used and implemented in a newly developed Python program called GHOAT, which fully automates the calculation of binding free energies and invokes the \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    AMBER 2020\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " simulation package. The results showed that the affinity between \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ExBox4+\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PAHs\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " in water is much larger than in organic media, with free energies between 5 and 20 kcal/mol, being able to act as a \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PHAs\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " scavenger with great potential for applications in environmental chemistry such as soil washing. The results also reveal a significant correlation with the experimental available G values. The methodology employed presents itself as an important tool for the in silico determination of binding affinities, not only available for charged \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    cyclophanes\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " but also extensible to several other HG supramolecular systems in condensed media, aiding in the rational design of host guest systems in a significant way.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_8009588.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Potential of mean force and umbrella sampling histograms for MA/CActd and CActd/CActd complexes<br>PMF &amp; Histogram data for CActd/CActd and MA/CActd complexes- md pull.mdp \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Gromacs\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       "' molecular dynamics parameters for pulling process of complexes, used for generating starting configurations for the umbrella sampling windows. These windows are used to obtain histograms of COM separations which are used to calculate the potential of mean force for each of the complexes by the weighted histogram method (WHAM).- distPull-MACA.xvg Table of distance (nm) vs time (ps) for the pulling process of the MA/CActd complex- distPull-CACA.xvg Table of distance (nm) vs time (ps) for the pulling process of the CActd/CActd complex- histoMACA.xvg Table of histograms (count) vs COM separation (nm) from umbrella sampling of the MA/CActd complex- histoCACA.xvg Table of histograms (count) vs COM separation (nm) from umbrella sampling of the CActd/CActd complex- PMF MA CActd.txt Table of potential of mean force (kcal/mol) vs COM separation (nm) for the MA/CActd complex.- PMF CActd CActd.txt Table of potential of mean force (kcal/mol) vs COM separation (nm) for the CActd/CActd complex.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_3780463.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Molecular dynamics simulation data of designed cyclic peptide - ligand 1-3 (receptor-ligand bound)<br>Trajectories of receptor-ligand bound simulation and simulation set-up files of designed cyclic peptide as \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    MDM2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " binders. This dataset contains simulations of ligand 1-3. Ligand 4 data can be found here: https://removed The original paper of these designed cyclic peptide: Danelius, E., Pettersson, M., Bred, M., Min, J., Waddell, M. B., Guy, R. K., et al. (2016). Flexibility is important for inhibition of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    MDM2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "/\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    p53\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " protein protein interaction by cyclic -hairpins. Org. Biomol. Chem., 14 (44), 10386 10393. https://removed</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_2527738.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Interaction of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Hematoporphyrin\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " with Lipid Membranes<br>Natural or synthetic \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    porphyrins\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " are being used as photosensitizers in photodiagnosis (PD) and photodynamic therapy (PDT) of malignancies and some other diseases. Understanding the interactions between \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    porphyrins\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and cell membranes is therefore important to rationalize the uptake of photosensitizers and their passive transport through cell membranes. In this study, we consider the properties of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    hematoporphyrin\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Hp\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "), a well-known photosensitizer for PD and PDT, in the presence of a \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    1-palmitoyl-2-oleoyl-sn-glycero-3-phosphocholine\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") bilayer that we use as a model system for protein-free cell membranes. For this purpose, we employed \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    200 ns\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       " atomic-scale molecular dynamics (MD) simulations for five systems containing the neutral (Hp0) or the dianionic form (Hp2 ) of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Hp\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " bilayer. MD simulations allowed one to estimate the position, orientation, and dynamics of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Hp\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " molecules inside the membrane. The dye molecules were found to reside in the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    phospholipid\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " headgroup area close to the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    carbonyl\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " groups of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPC acyl\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " chains. Their orientations were dependent on the protonation state of two propionic groups. Hp2 was found to have a lower affinity to enter the membrane than the neutral form. The \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    dianions\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", being in the aqueous phase, formed stable dimers with a strictly determined geometry. Our results fully supported the experimental data and provide a more detailed molecular-level description of the interactions of photosensitizers with lipid membranes.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_12374771.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Coarse-Grained Molecular Model for the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Glycosylphosphatidylinositol\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Anchor with and without Protein<br>\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Glycosylphosphatidylinositol\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GPI\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") anchors are a unique class of complex glycolipids that anchor a great variety of proteins to the extracellular leaflet of plasma membranes of eukaryotic cells. These anchors can exist either with or without an attached protein called \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GPI\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-anchored protein (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GPI\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-AP) both in vitro and in vivo. Although \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GPIs\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " are known to participate in a broad range of cellular functions, it is to a large extent unknown how these are related to \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GPI\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " structure and composition. Their conformational flexibility and microheterogeneity make it difficult to study them experimentally. Simplified atomistic models are amenable to all-atom computer simulations in small lipid bilayer patches but not suitable for studying their partitioning and trafficking in complex and heterogeneous membranes. Here, we present a coarse-grained model of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GPI\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " anchor constructed with a modified version of the \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    MARTINI\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " force field that is suited for modeling carbohydrates, proteins, and lipids in an aqueous environment using \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    MARTINI\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " s polarizable \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". The nonbonded interactions for sugars were reparametrized by calculating their partitioning free energies between polar and apolar phases. In addition, sugar sugar interactions were optimized by adjusting the second virial coefficients of osmotic pressures for solutions of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    glucose\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    sucrose\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    trehalose\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " to match with experimental data. With respect to the conformational dynamics of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GPI\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-anchored green fluorescent protein, the accessible time scales are now at least an order of magnitude larger than for the all-atom system. This is particularly important for fine-tuning the mutual interactions of lipids, carbohydrates, and amino acids when comparing to experimental results. We discuss the prospective use of the coarse-grained \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GPI\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " model for studying protein-sorting and trafficking in membrane models.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_16715821.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Diffusion Behavior of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Methane\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " in 3D \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Kerogen\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Models<br>As global energy demand increases, natural gas recovery from source rocks is attracting considerable attention since recent development in shale extraction techniques has made the recovery process economically viable. \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Kerogens\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " are thought to play an important role in gas recovery; however, the interactions between trapped shale gas and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    kerogens\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " remain poorly understood due to the complex, heterogeneous microporous structure of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    kerogens\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". This study examines the diffusive behavior of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    methane\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " molecules in \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    kerogen\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " matrices of different types (Type I, II, and II) and maturity levels (A to D for Type II \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    kerogens\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") on a molecular scale. Models of each \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    kerogen\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " type were developed using simulated annealing. We employed grand canonical Monte Carlo simulations to predict the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    methane\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " loadings of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    kerogen\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " models and then used equilibrium molecular dynamics simulations to compute the mean square displacement of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    methane\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " molecules within the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    kerogen\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " matrices under reservoir-relevant conditions, that is, \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    365 K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       " and 275 bar. Our results show that \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    methane\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " self-diffusivity exhibits some degree of anisotropy in all \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    kerogen\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " types examined here except for Type I-A \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    kerogens\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", where diffusion is the fastest and isotropic diffusion is observed. Self-diffusivity appears to correlate positively with pore volume for Type II \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    kerogens\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", where an increase in diffusivity is observed with increasing maturity. Swelling of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    kerogen\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " matrix up to a 3% volume change is also observed upon \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    methane\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " adsorption. The findings contribute to a better understanding of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    hydrocarbon\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " transport mechanisms in shale and may lead to further development of extraction techniques, fracturing fluids, and recovery predictions.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_13234529.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Serine 477 plays a crucial role in the interaction of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    SARS-CoV-2 spike protein\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " with the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    human receptor ACE2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " -Supporting material<br>The dataset belongs to our manuscript &quot;Serine 477 plays a crucial role in the interaction of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    SARS-CoV-2 spike protein\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " with the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    human receptor ACE2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ".&quot; that is currently under peer review and contains the following files and folders:gmx simulations: Folder consisting the trajectories and structure files generated during the MD simulations using \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Gromacs\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " software. Atomic fluctuations RBD fig2a.xvg: The atomic fluctuations data generated from the Normal Mode Analysis of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    COVID2 Spike\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-RBD, the plotted figure is fig2aRBD entropy fig1b.tsv: The residue wise Shannon entropy values for \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    COVID2Spike\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-RBD. The ploted figure is fig1b</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_3457004.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">\n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    General AMBER Force Field\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " Parameters for \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Diphenyl Diselenides\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Diphenyl Ditellurides\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "<br>The \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    General AMBER Force Field\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GAFF\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       ") has been extended to describe a series of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    selenium\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    tellurium diphenyl dichalcogenides\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". These compounds, besides being eco-friendly catalysts for numerous oxidations in organic chemistry, display peroxidase activity, i.e., can reduce hydrogen peroxide and harmful organic hydroperoxides to water/alcohols and as such are very promising antioxidant drugs. The novel \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GAFF\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " parameters are tested in MD simulations in different solvents and the 77Se NMR chemical shift of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    diphenyl diselenide\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " is computed using structures extracted from MD snapshots and found in nice agreement with the measured value in CDCl3. The whole computational protocol is described in detail and integrated with in-house code to allow easy derivation of the force field parameters for analogous compounds as well as for \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Se\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "/\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Te\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " organocompounds in general.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_5106286.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">PrimaDORAC: A Free Web Interface for the Assignment of Partial Charges, Chemical Topology, and Bonded Parameters in Organic or Drug Molecules<br>We present PrimaDORAC, a simple and freely accessible web interface for generating the topology and the parameter files of organic or drug molecules to be used in molecular mechanics or molecular dynamics calculations. The interface relies on our in-house FORTRAN90 parser, working on the recently released \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Generalized Amber Force Field\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " parameter set (\n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GAFF2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       "). AM1/BCC charges are computed using the Public Domain MOPAC7 program and the bond charge corrections (BCC) reported in Jakalian, A.; Jack, D. B.; Bayly, C. I.; J. Comp. Chem., 2002, 23, 1623 1641. The interface has been tested on about 52,000 compounds (identified with a CAS registry number) taken from the National Cancer Institute (NCI) Open Database. PrimaDORAC has been found to be very reliable, producing \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GAFF2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " minimized structures bearing a mean root square displacement of about 0.01 0.02 nm with respect to the original CORINA-generated 3D NCI structures. As a demonstrative example, we release the full topology and parameter files, along with the AM1/BCC-GAFF2 computed in vacuo IR spectrum, for some recently discovered PARP/MCL1 inhibitors. The web interface and parser, including the sources, are part of the ORAC code (Procacci, P.; J. Chem. Inf. Model., 2016, 56, 1117 1121), distributed under the General Public License at www.chim.unifi.it/orac</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_3667662.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Simulation of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ":\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPE\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " 1:1 membrane, model by Maciejewski and Rog<br>A \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ":\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPE\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " 1:1 bilayer with 64 lipids per leaflet is simulated for \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    300 ns\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       " using the model by Maciejewski and Rog [1,2] and \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Gromacs simulation engine version 2019.4\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       ". The trajectory (.xtc) is saved every 100ps. The simulation parameters (.mdp), the energy file (.edr), the continue point to extend the simulation (.cpt), the run input parameters (.tpr), and the final structure (.gro) are also given. The lipid topologies (.itp) are attached, and they are obtained from the authors. Note that unlike in the original paper [2], buffered Verlet lists are used for efficiency. [1] DOI: 10.1016/j.dib.2016.03.067 [2] DOI: 10.1021/jp5016627</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_10269416.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Adding Anisotropy to the Standard Quasi-Harmonic Approximation Still Fails in Several Ways to Capture Organic Crystal Thermodynamics<br>We evaluate the accuracy of varying thermal expansion models for the quasi-harmonic approximation (QHA) relative to molecular dynamics (MD) for 10 sets of enantiotropic organic polymorphs. Relative to experiment we find that MD, using an off-the-shelf point charge potential, gets the sign of the enthalpic contributions correct for 6 of the 10 pairs of polymorphs and the sign of the entropic contributions correct for all pairs. We find that anisotropic QHA provides little improvement to the error in free energy differences from MD relative to isotropic QHA, but does a better job capturing the thermal expansion of the crystals. A form of entropy enthalpy compensation allows the free energy differences of QHA to deviate less than 0.1 kcal/mol from MD for most polymorphic pairs, despite errors up to 0.4 kcal/mol in the entropy and enthalpy. Deviations in the free energy of QHA and MD do not clearly correlate with molecular flexibility, clarifying a previously published finding. Much of the error previously found between QHA and MD for these flexible molecules is reduced when QHA is run from a lattice minimum consistent with the same basin as MD, rather than the energy-minimized experimental crystal structure. Specifically, performing anisotropic QHA on lattice minimum quenched from low-temperature replica exchange simulations reduced the error previously found by 0.2 kcal/mol on average. However, these conformationally flexible molecules can have many low-temperature conformational minima, and the choice of an inconsistent minima causes free energies estimated from QHA to deviate from MD at temperatures as low as 10 K. We also find finite size errors in the polymorph free energy differences using anisotropic QHA, with free energy differences as large as 0.5 kcal/mol between unit and supercells loosely correlated with differences in anisotropic thermal expansion. These larger system sizes are computationally more accessible using our cheaper 1D variant of anisotropic QHA, which gives free energies within 0.02 kcal/mol of the fully anisotropic approach at all temperatures studied. The errors between MD and experiment are 1 2 orders of magnitude larger than those seen between QHA and MD, so the quality of the force field used is still of primary concern, but this study illustrates a number of other important factors that must be considered to obtain quantitative organic crystal thermodynamics.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_9684887.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Atomistic Simulation of Phase Transitions and Charge Mobility for the Organic Semiconductor \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Ph-BTBT-C10\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "<br>We present a computational study of the molecular organization and charge mobility of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Ph-BTBT-C10\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", a high performance organic semiconductor of considerable current interest. We have observed for the first time by atomistic molecular dynamics the formation of an ordered smectic phase on cooling down from the isotropic melt and upon heating the crystal, for this system, in good agreement with experiment. Although we could observe only a smectic A and not a smectic E phase, the temperature variation of the hole mobility estimated from hopping model calculations reproduces the main features of experiments. The crystal phase is characterized by high mobility bilayers defined by the aromatic -conjugated cores, but it is effectively insulating in the orthogonal direction. The smectic phase is characterized by more disordered monolayers, which have a good in-plane mobility and a lower, but still appreciable, degree of charge transport across the layers. This feature may be advantageous for applications requiring materials with evenly balanced, three-dimensional conduction paths.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_3592499.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Simulations of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DLPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DOPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DEPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " membranes<br>Simulations of three different lipid bilayers using the \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CHARMM36\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " lipid model. The bilayers were built using \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CHARMM-GUI\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " and simulated for \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    500 ns\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       " using \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Gromacs 2019\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       ". The systems contain 200 lipids (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DLPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DOPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", or \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DEPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "), 50 lipids per \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", and 130 mM \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    NaCl\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". For each simulation, the run input file (tpr), topology (top), index (ndx), trajectory (xtc), log file (log), energy file (edr), continue point (cpt), and the final structure (gro) are given. The topology refers to the \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Gromacs\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       "-compatible \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CHARMM36\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " force field, available at https://removed ff.shtml#gromacs and to other itp files provided in this upload. The simulation parameters are given in the md.mdp file.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_5508259.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Reduced Point Charge Models of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Proteins\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ": Effect of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Protein\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Interactions in Molecular Dynamics Simulations of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Ubiquitin\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Systems<br>We investigate the influence of various solvent models on the structural stability and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    protein\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " interface of three \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ubiquitin\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " complexes (PDB access codes: \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    1Q0W\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    2MBB\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    2G3Q\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") modeled using the \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Amber99sb\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " force field (FF) and two different point charge distributions. A previously developed reduced point charge model (\n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    RPCM\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       "), wherein each amino acid residue is described by a limited number of point charges, is tested and compared to its all-atom (AA) version. The complexes are solvated in \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    TIP4P-Ew\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " or \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    TIP3P\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " type \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " molecules, involving either the scaling of the Lennard-Jones \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    protein\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Owater interaction parameters, or the coarse-grain (CG) \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    SIRAH\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " description. The best agreements between the \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    RPCM\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " and AA models were obtained for structural, \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    protein\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", and ligand \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ubiquitin\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " properties when using the \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    TIP4P-Ew\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " FF with a scaling factor of 0.7. At the \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    RPCM\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " level, a decrease in , or the inclusion of \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    SIRAH\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " particles, allows weakening of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    protein\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " interactions. It results in a slight collapse of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    protein\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " structure and a less compact hydration shell and, thus, in a decrease in the number of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    protein\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " H-bonds. The dynamics of the surface \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    protein\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " atoms and of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " shell molecules are also slightly refrained, which allow the generation of stable \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    RPCM\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " trajectories.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_21980305.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Plant \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Terpenoid\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Permeability through Biological Membranes Explored via Molecular Simulations<br>Plants synthesize small molecule \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    diterpenes\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " composed of 20 carbons from precursor \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    isopentenyl diphosphate\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    dimethylallyl disphosphate\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", manufacturing diverse compounds used for defense, signaling, and other functions. Industrially, \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    diterpenes\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " are used as natural aromas and flavoring, as pharmaceuticals, and as natural insecticides or repellents. Despite \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    diterpene\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " ubiquity in plant systems, it remains unknown how plants control \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    diterpene\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " localization and transport. For many other small molecules, plant cells maintain transport proteins that control compound compartmentalization. However, for most \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    diterpene\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " compounds, specific transport proteins have not been identified, and so it has been hypothesized that \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    diterpenes\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " may cross biological membranes passively. Through molecular simulation, we study membrane transport for three complex \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    diterpenes\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " from among the many made by members of the Lamiaceae family to determine their permeability coefficient across plasma membrane models. To facilitate accurate simulation, the intermolecular interactions for \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    leubethanol\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    abietic acid\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    sclareol\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " were parametrized through the standard \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CHARMM\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " methodology for incorporation into molecular simulations. To evaluate the effect of membrane composition on permeability, we simulate the three \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    diterpenes\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " in two membrane models derived from sorghum and yeast lipidomics data. We track permeation events within our unbiased simulations, and compare implied permeation coefficients with those calculated from Replica Exchange Umbrella Sampling calculations using the inhomogeneous solubility diffusion model. The \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    diterpenes\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " are observed to permeate freely through these membranes, indicating that a transport protein may not be needed to export these small molecules from plant cells. Moreover, the permeability is observed to be greater for plant-like membrane compositions when compared against animal-like membrane models. Increased permeability for \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    diterpene\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " molecules in plant membranes suggest that plants have tailored their membranes to facilitate low-energy transport processes for signaling molecules.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_5872566.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Fast Dynamic Docking Guided by Adaptive Electrostatic Bias: The MD-Binding Approach<br>Engineering chemical entities to modify how pharmaceutical targets function, as it is done in drug design, requires a good understanding of molecular recognition and binding. In this context, the limitations of statically describing bimolecular recognition, as done in docking/scoring, call for insightful and efficient dynamical investigations. On the experimental side, the characterization of dynamical binding processes is still in its infancy. Thus, computer simulations, particularly molecular dynamics (MD), are compelled to play a prominent role, allowing a deeper comprehension of the binding process and its causes and thus a more informed compound selection, making more significant the computational contribution to drug discovery (Carlson, H. A. Curr. Opin. Chem. Biol. 2002, 6, 447 452). Unfortunately, MD-based approaches cannot yet describe complex events without incurring prohibitive time and computational costs. Here, we present a new method for fully and dynamically simulating drug target-complex formations, tested against a real world and pharmaceutically relevant benchmark set. The method, based on an adaptive, electrostatics-inspired bias, envisions a campaign of trivially parallel short MD simulations and a strategy to identify a near native binding pose from the sampled configurations. At an affordable computational cost, this method provided predictions of good accuracy also when the starting protein conformation was different from that of the crystal complex, a known hurdle for traditional molecular docking (Lexa, K. W.; Carlson, H. A. Q. Rev. Biophys. 2012, 45, 301 343). Moreover, along the observed binding routes, it identified some key features also found by much more computationally expensive plain-MD simulations. Overall, this methodology represents significant progress in the description of binding phenomena.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_7234728.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Deep potential model and datasets used in the publication &quot;Plastic Deformation of Superionic \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Ices&quot;<br>The file training data for deepmd.zip contains the training set and deep potential model for superionic ices used in the publication &quot;Plastic Deformation of Superionic \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Ices&quot;. The deep potential (DP) neural-network model was generated using the \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DeepMD-kit\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " package. The training set was generated using \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    AIMD\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       "/\n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CP2K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " calculations for both undeformed and deformed states of the fcc superionic phase at temperatures of \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    2500\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    3000\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    3500\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    4000 K.\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       " The file \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    cp2k\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " shear.zip contains instructions to modify the \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CP2K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " code to calculate the shear vs strain deformation in continuous mode. The file \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    lammps\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " dislocation.zip contains the files needed to run the dislocation mobility calculation. </div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_2032860.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Improved Coarse-Grained Modeling of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Cholesterol\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-Containing Lipid Bilayers<br>\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Cholesterol\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " trafficking, which is an essential function in mammalian cells, is intimately connected to molecular-scale interactions through \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    cholesterol\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " modulation of membrane structure and dynamics and interaction with membrane receptors. Since these effects of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    cholesterol\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " occur on \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    micro- to millisecond\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       " time scales, it is essential to develop accurate coarse-grained simulation models that can reach these time scales. \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Cholesterol\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " has been shown experimentally to thicken the membrane and increase \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    phospholipid\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " tail order between 0 and 40% \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    cholesterol\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", above which these effects plateau or slightly decrease. Here, we showed that the published \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    MARTINI\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " coarse-grained force-field for \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    phospholipid\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    cholesterol\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " fails to capture these effects. Using reference atomistic simulations, we systematically modified \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    cholesterol\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " bonded parameters in \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    MARTINI\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " to improve its performance. We showed that the corrections to pseudobond angles between \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    glycerol\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and the lipid tails and around the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    oleoyl\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " double bond particle (the angle-corrected model ) slightly improves the agreement of \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    MARTINI\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " with experimentally measured thermal, elastic, and dynamic properties of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " membranes. The angle-corrected model improves prediction of the thickening and ordering effects up to 40% \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    cholesterol\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " but overestimates these effects at higher \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    cholesterol\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " concentration. In accordance with prior work that showed the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    cholesterol\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " rough face \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    methyl\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " groups are important for limiting \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    cholesterol\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " self-association, we revised the coarse-grained representation of these \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    methyl\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " groups to better match \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    cholesterol\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    cholesterol\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " radial distribution functions from atomistic simulations. In addition, by using a finer-grained representation of the branched \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    cholesterol\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " tail than \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    MARTINI\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       ", we improved predictions of lipid tail order and bilayer thickness across a wide range of concentrations. Finally, transferability testing shows that a model incorporating our revised parameters into \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DOPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " outperforms other CG models in a \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DOPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "/\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    cholesterol\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " simulation series, which further argues for its efficacy and generalizability. These results argue for the importance of systematic optimization for coarse-graining biologically important molecules like \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    cholesterol\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " with complicated molecular structure.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_13836577.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Molecular Dynamics Simulation of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    n Octacosane\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Mixture Confined in \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Graphene\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Mesopores: Comparison of Atomistic and Coarse-Grained Calculations and the Effect of Catalyst Nanoparticle<br>Fischer Tropsch synthesis (FTS) is used extensively in the gas-to-liquids (GTL) process to produce clean, high-quality low emission transportation fuels from synthesis gas. In order to gain a better understanding of the phase behavior of the produced wax \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " mixtures at reaction conditions by accounting for confinement effects of either hydrophilic or hydrophobic characteristics and catalyst nanoparticles (NPs), one needs to closely examine the validity of incorporating coarse grained approaches in the context of FTS. The present study focuses on simulating by means of atomistic (AA) and coarse-grained (CG) molecular dynamics (MD; CGMD) simulations the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    n-octacosane\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    n-C28\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " mixture inside \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    graphene\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    G\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    graphene oxide\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GO\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") mesopores under low-temperature FTS conditions (\n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    473.15 K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       ") with the inclusion of a Co NP. \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Graphene\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " derivatives were selected as they are emerging catalyst support materials in the FTS reaction and allow for direct comparisons between the AA and CG methodologies. We also evaluated the presence of long-chain \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    alcohols\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " such as \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    dodecan-1-ol\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " at 7 wt % on the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    n-C28\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    H2O\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " mixture phase behavior. The AA simulations involved the use of the united atom \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    TraPPE\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    TraPPE-UA\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       ") force field for \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    n-C28\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and the alcohols and the \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    TIP4P/2005\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " force field for \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", while the \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    MARTINI\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " force field was used for the CG description. With regard to the mixtures inside the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    G\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " pore, both AA and CG simulation methods capture the phase separation of the mixture, with \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " molecules occupying the center of the pore and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    n-C28\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " forming layers at the pore walls; inside \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GO\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", both methodologies show that \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " separates near the surface and the model wax lies at the pore center. Both AA and CG simulations accurately capture the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    n-C28\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    H2O\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " diffusivity as a function of the distance from the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    G\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GO\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " pore centers, respectively, being lower closer to the pore surface. \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Dodecan-1-ol\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " is mostly located at the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    n-C28\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    H2O\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " interface showing a higher preference toward the wax as a consequence of increased van der Waals intermolecular interactions, slightly reducing the mobility of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". The agreement between AA and CG MD simulations paved the way for novel CGMD simulations with DFT-derived parameters for Co, to study the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    n-C28\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    H2O\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " behavior in its presence. Our results show that the Co NP does not affect phase separation of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    n-C28\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    H2O\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " mixture inside the pores; however, \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " at such high concentrations covers the Co NP surface extensively. Given the experimental difficulties in exploring the relevant mechanisms at this scale, our results showcase that CGMD using the \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    MARTINI\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " force field can be employed to study FTS-related processes at this level and are expected to open new pathways in the investigation of confined mixtures relevant to the FTS reaction in the presence of supported catalyst NPs.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_11356691.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">GroScore: Accurate Scoring of Protein Protein Binding Poses Using Explicit-Solvent Free-Energy Calculations<br>Protein protein docking algorithms promise a potential relief for the mismatch between the number of experimentally determined complex structures and the number of relevant protein interactions in an organism. To distinguish correctly from wrongly generated poses, it is necessary to score complexes according to their structural similarity to the real complex, which is usually done by computing interaction energies of some sort. Here, we explore the potential of free-energy calculations with statistical-mechanical foundation in the context of molecular dynamics (MD) simulations with explicit solvent to score a large number of complex poses. We introduce an adaptive sampling scheme which ensures that most sampling time is spent on the most promising poses. Our approach is illustrated by scoring of all targets in the CAPRI Score set, a scoring benchmark set, and three additional CAPRI targets, together consisting of more than 22 000 poses. Our scoring scheme shows a performance that is competitive with the most successful approaches that were previously reported. All necessary scripts to run the automated scoring pipeline are available in the Supporting Information for this paper.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_16698491.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Experimental and Theoretical Exploration of Volumetric Properties of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Aminobutyric Acid\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    l Valine\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " in the Electrolytic Environment at T   \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    283.15\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       " to \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    318.15 K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       " and Pressure P   0.1 MPa<br>In the present work, density ( ) calculations, molecular dynamics (MD) simulations, and quantum chemical calculations of pseudo-binary mixtures of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    aminobutyric acid\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    l-valine\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " in aqueous solutions of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    sodium thiosulfate\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Na2S2O3\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") at T   \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    283.15\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       " to \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    318.15 K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       " and atmospheric pressure (0.1 MPa) were performed. MD simulations reveal the formation of molecular aggregates induced by the preferential solvation in the mixture, due to the electrostatic interactions at the first layer of solvation between the charged groups of amino acids and the inorganic salt. The formation of these molecular aggregates is promoted by an increase in the salt concentration. Noncovalent interactions within the pseudo-binary mixtures were studied by the noncovalent interaction index (NCI) and IGM analysis, which well supported the experimental findings.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_4730896.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">SIRAH-CoV2 initiative: T307I Mutation in a \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    SARS-CoV-2 Spike\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " from sub-Antarctic Chile<br>This dataset contains the trajectories of \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    10 microseconds\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       "-long coarse-grained molecular dynamics simulations of the soluble part of the wild-type \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    spike protein\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " of SARS-CoV2, the single mutant D614G, and the double mutant T307I/D614G. Simulations are based on the PDB structure \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    6XR8\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". Loops were completed using the \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    SwissProt\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " sever, and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    glycans\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " were not included. Simulations were performed using the \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    SIRAH\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " force field 2.0 running with the \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Gromacs 2018.3\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " package at the Institut Pasteur de Montevideo under the conditions reported in Machado et al. JCTC 2019, adding 150 mM \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    NaCl\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " according to Machado &amp; Pantano JCTC 2020. The following files in gromacs format are provided for each simulation: i) a .psf containing the topological information (connectivity, etc) ii) a big .xtc file containing \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    10 microseconds\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       " of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CG protein\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " recorded every 200 picoseconds (50000 frames) iii) a smaller .xtc file containing the same trajectory but with only one frame per nanosecond (1000 frames) iv) a .gro file with the initial configuration Finally, the file sirah vmdtk.tcl contains a series of macros and scripts that facilitate the visualization and analysis of the CG trajectories. Step-By-Step tutorials for visualizing and analyzing CG trajectories using SirahTools can be found at www.sirahff.com. To take a quick look at a trajectory using \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    VMD 1.9.3\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       ", run the command: &gt; vmd SpikeWT-SIRAH.psf SpikeWT-SIRAH.gro SpikeWT-SIRAH-1kfrm.xtc -e sirah vmdtk.tcl Note that you can use normal \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    VMD\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " drawing methods as vdw, licorice, etc., and coloring by restype, element, name, etc. This dataset is part of the SIRAH-CoV2 initiative for data sharing. If you use this data, please cite Garay PG, et al &quot;The SIRAH-CoV-2 Initiative: A Coarse-Grained Simulations Dataset of the SARS-CoV-2 Proteome&quot;. Front. Med. Technol. 3:644039. doi: 10.3389/fmedt.2021.644039 For further details, please contact Sergio Pantano (spantano@pasteur.edu.uy).</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_13814.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">MD simulation trajectory and related files for \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " bilayer in low hydration (\n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Berger\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " model delivered by Tieleman, \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Gromacs 4.5\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       ")<br>Equilibrated \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " lipid bilayer simulation in low hydration (7 \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " per lipid molecule) ran with \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Gromacs 4.5\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Berger\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " force field delivered by Peter Tieleman (https://removed with fixed double bond dihedrals, \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    60ns\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       ", T \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    298K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       ", 128 \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " molecules, 896 \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " molecules. This data is used in the nmrlipids.blospot.fi project. More details from nmrlipids.blospot.fi and https://removed If data is used, please cite the nmrlipids.blogspot.fi project and the original publications related to the force field.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_2106349.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">\n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ProBiS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       "-\n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CHARMMing\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       ": Web Interface for Prediction and Optimization of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Ligands\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " in \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Protein\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Binding Sites<br>\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Proteins\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " often exist only as apo structures (unligated) in the Protein Data Bank, with their corresponding holo structures (with \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ligands\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") unavailable. However, apoproteins may not represent the amino-acid residue arrangement upon \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ligand\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " binding well, which is especially problematic for molecular docking. We developed the \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ProBiS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       "-\n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CHARMMing\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " web interface by connecting the \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ProBiS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " (http://probis.cmm.ki.si) and \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CHARMMing\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " (http://www.charmming.org) web servers into one functional unit that enables prediction of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    protein\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ligand\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " complexes and allows for their geometry optimization and interaction energy calculation. The \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ProBiS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " web server predicts \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ligands\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (small compounds, \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    proteins\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", nucleic acids, and single-atom \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ligands\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") that may bind to a query \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    protein\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". This is achieved by comparing its surface structure against a nonredundant database of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    protein\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " structures and finding those that have binding sites similar to that of the query \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    protein\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". Existing \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ligands\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " found in the similar binding sites are then transposed to the query according to predictions from \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ProBiS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " . The \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CHARMMing\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " web server enables, among other things, minimization and potential energy calculation for a wide variety of biomolecular systems, and it is used here to optimize the geometry of the predicted \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    protein\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ligand\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " complex structures using the CHARMM force field and to calculate their interaction energies with the corresponding query proteins. We show how \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ProBiS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       "-\n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CHARMMing\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " can be used to predict ligands and their poses for a particular binding site, and minimize the predicted \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    protein\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ligand\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " complexes to obtain representations of holoproteins. The \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ProBiS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       "-\n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CHARMMing\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " web interface is freely available for academic users at http://probis.nih.gov.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_6010416.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Source molecular simulation data for calculating energy and friction profiles and permeability coefficients through model lipid membranes<br>Energy files from \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GROMACS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " molecular dynamics simulations with enhanced free energy sampling contain time-dependent evolution of the free energy profiles and friction profiles (and other energies and simulation properties) that were used for calculating permeability coefficients in the publication https://www.biorxiv.org/content/10.1101/2021.07.16.452599v1 Simulation system contains a lipid \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " or \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DPPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " bilayer with a varying amount of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    cholesterol\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (specified as mol% in the file name). Hydrophobic level of the permeating particle is specified as &quot;level-I&quot;, &quot;level-II&quot; etc. When unspecified in the file name, the particle is hydrophobic level &quot;III&quot;. Lipids \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    D-C14-PC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " denote \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " lipids with both tails monounsaturated of length 14 carbon atoms. \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DOPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " is equivalent to \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    D-C18-PC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". (Detailed description in the publication) Adaptive Weighted Histogram (AWH) method was used to sample the free energy profile of translocating small molecule through the lipid bilayer. \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GROMACS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " tool 'gmx awh' reads the files and provides the described profiles. Files were generated by \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GROMACS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " 'mdrun' simulation engine version 2019.3. Coarse-grained \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    MARTINI 3.0\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " model was used for modeling the biomolecular interactions. Scripts to perform the simulations and the files with initial configurations and simulation settings are stored in a public GitHub repository depozited on Zenodo.org: https://doi.org/10.5281/zenodo.5082249. Abraham, M. J. et al. \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GROMACS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       ": High performance molecular simulations through multi-level parallelism from laptops to supercomputers. SoftwareX 1 2, 19 25 (2015). Lindahl, V., Lidmar, J. &amp; Hess, B. Accelerated weight histogram method for exploring free energy landscapes. J. Chem. Phys. 141, 044110 (2014). Souza, P. C. T. et al. Martini 3: a general purpose force field for coarse-grained molecular dynamics. Nat. Methods 18, 382 388 (2021). Melcr, J. Git repository with analysis scripts for MD simulations of permeability through lipid membranes. (2021) doi:https://doi.org/10.5281/zenodo.5082249.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_2020311.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">An Extension and Further Validation of an All-Atomistic Force Field for Biological Membranes<br>Biological membranes are versatile in composition and host intriguing molecular processes. In order to be able to study these systems, an accurate model Hamiltonian or force field (FF) is a necessity. Here, we report the results of our extension of earlier developed all-atomistic FF parameters for fully saturated \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    phospholipids\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " that complements an earlier parameter set for saturated \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    phosphatidylcholine\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " lipids (J. Phys. Chem. B, 2012, 116, 3164 3179). The FF, coined \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Slipids\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " (Stockholm lipids), now also includes parameters for unsaturated \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    phosphatidylcholine\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    phosphatidylethanolamine\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " lipids, e.g., \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DOPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    SOPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPE\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DOPE\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". As the extended set of parameters is derived with the same philosophy as previously applied, the resulting FF has been developed in a fully consistent manner. The capabilities of \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Slipids\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " are demonstrated by performing long simulations without applying any surface tension and using the correct isothermal isobaric (NPT) ensemble for a range of temperatures and carefully comparing a number of properties with experimental findings. Results show that several structural properties are very well reproduced, such as scattering form factors, NMR order parameters, thicknesses, and area per lipid. Thermal dependencies of different thicknesses and area per lipid are reproduced as well. Lipid diffusion is systematically slightly underestimated, whereas the normalized lipid diffusion follows the experimental trends. This is believed to be due to the lack of collective movement in the relatively small bilayer patches used. Furthermore, the compatibility with amino acid FFs from the \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    AMBER\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " family is tested in explicit transmembrane complexes of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    WALP23\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " peptide with \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DLPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DOPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " bilayers, and this shows that \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Slipids\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " can be used to study more complex and biologically relevant systems.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_838635.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Large \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DPPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " monolayer simulations with \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Charmm36+OPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " @ \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    298 K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       " (Part 1/2)<br>\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DPPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " monolayers simulated at a varying area per lipid in the NVT ensemble. The \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Charmm36\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " lipid model [1] is employed together with the 4-point \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    OPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " model [2]. Each system contains two monolayers with 256 lipids each. These monolayers are separated by a \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " slab, and surrounded by vacuum.  The number in the filename corresponds to the area per lipid in Ångströms. The starting structures were generated by setting up a \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " monolayer system with an area per lipid of 50 Å^2, followed by a simulation during which the monolayer area was expanded using \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PLUMED\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       ". Next, \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " was replaced by \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DPPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", and the systems were energy minimized. Systems with area smaller than 51 Å^2 were generated by compressing the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DPPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " monolayer system with \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PLUMED\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       ". All trajectories, \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    100–200 ns\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       " long (until convergence of surface tension), are simulated with \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Gromacs 5.0.x\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " [3] using the default \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Charmm36\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " monolayer simulations parameters given in the mdp file. Topology (.top) and index files are also common for all simulations. The topologies (.itp) for the lipids can be obtained from \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Charmm-GUI\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " and for the \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    OPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " model from https://bioinformatics.cs.vt.edu/~izadi/ . Part 2 of this upload is available at https://doi.org/10.5281/zenodo.838635 Matti Javanainen, Antti Lamberg, Lukasz Cwiklik, Ilpo Vattulainen, and O. H. Samuli Ollila. Atomistic Model for Nearly Quantitative Simulations of Langmuir Monolayers, Langmuir 2018 34 (7), 2565-2572, DOI: 10.1021/acs.langmuir.7b02855 [1] DOI: 10.1021/jp101759q [2] DOI: 10.1021/jz501780a [3] DOI: 10.1016/j.softx.2015.06.001</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_1346073.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Assessment of mutation probabilities of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    KRAS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " G12 missense mutants and their long-time scale dynamics by atomistic molecular simulations and Markov state modeling: Datasets.<br>Datasets related to the publication [1]. Including: \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    KRAS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " G12X mutations derived from COSMIC v.79 [http://cancer.sanger.ac.uk/cosmic/] (KRAS G12X mut COSMICv79..xlsx) RMSFs (300-2000ns) of GDP-systems (300 2000rmsf GDP systems RAW AVG SE.xlsx) RMSFs (300-2000ns) of GTP-systems (300 2000RMSF GTP systems RAW AVG SE.xlsx) \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PyInteraph\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " analysis data for salt-bridges and hydrophobic clusters (.dat files for each system in the PyInteraph data.zip-file) Backbone trajectories for each system (residues 4-164; frames for every 1ns). Last number (e.g. 1) refers to the replica of the simulated system. backbone 4-164.gro/.pdb/.tpr -files (resid 4-164) [1] Pantsar T et al. Assessment of mutation probabilities of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    KRAS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " G12 missense mutants and their long-time scale dynamics by atomistic molecular simulations and Markov state modeling. PLoS Comput Biol Submitted (2018)</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_3228177.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Simulation \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " 512 \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    NaCl\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " 150mM \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    310K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       " membrane<br>Title publication: A \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Ceramide\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-Regulated Element in the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Late Endosomal Protein\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    LAPTM4B\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Controls Amino Acid Transporter Interaction Year publication: 2018 DOI publication: 10.1021/acscentsci.7b00582 Description: Simulation \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " membrane using \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    charmm36\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " at \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    310K.\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       " MD engine: 2016.1 Force field: \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    charmm36\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " Temperature: \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    310 K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       " Pressure: 1 bar Simulation time: \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    4200 ns\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       " Saving frequency: 100 ps Molecular content: \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " 512 \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " 20480 \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Cl-\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " 55 \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Na+\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " 55 Other information: * Center of mass motion removed independently for bilayer and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " + \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ions\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " * 01-* are the merged files. * 02-* are the original simulation files. So do not concatenate 02 and 01 files.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_13022698.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Structure and Dynamics of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Dioleoyl-Phosphatidylcholine\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Bilayers under the Influence of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Quercetin\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Rutin\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "<br>\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Quercetin\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    rutin\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", two widely studied flavonoids with applications foreseen in the sectors of pharmaceutical and cosmetic industries, have been chosen as model compounds for a detailed structural and dynamical investigation onto their influence on fluid lipid bilayers. Combining global small angle X-ray scattering analysis with molecular dynamics, various changes in the properties of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    dioleoyl-phosphatidylcholine\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DOPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") bilayers have been determined. The solubility of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    quercetin\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " in \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DOPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " membranes is assured up to 12 mol %, whereas \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    rutin\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", with additional \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    glucose\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    rhamnose\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " groups, are fully soluble only up to 6 mol %. Both flavonoids induce an increase in membrane undulations and thin the bilayers slightly (&lt;1 A ) in a concentration dependent manner, wherein \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    quercetin\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " shows a stronger effect. Concomitantly, in the order of 2 4%, the adjacent bilayer distance increases with the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    flavonoid\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " s concentration. Partial molecular areas of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    quercetin\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    rutin\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " are determined to be 26 and 51 A 2, respectively. Simulated averaged areas per molecule confirm these estimates. A 60 tilted orientation of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    quercetin\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " is observed with respect to the bilayer normal, whereas the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    flavonoid\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " moiety of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    rutin\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " is oriented more perpendicular ( -angle 30 ) to the membrane surface. Both \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    flavonoid\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " moieties are located at a depth of 12 and 16 A for \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    quercetin\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    rutin\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", respectively, while their anionic forms display a location closer to the polar interface. Finally, at both simulated concentrations (1.5 and 12 mol %), \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DOPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    rutin\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " systems induce a stronger packing of the pure \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DOPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " lipid bilayer, mainly due to stronger attractive electrostatic interactions in the polar lipid head region.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_17430576.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">MLCV: Bridging Machine-Learning-Based Dimensionality Reduction and Free-Energy Calculation<br>Importance-sampling algorithms leaning on the definition of a model reaction coordinate (RC) are widely employed to probe processes relevant to chemistry and biology alike, spanning time scales not amenable to common, brute-force molecular dynamics (MD) simulations. In practice, the model RC often consists of a handful of collective variables (CVs) chosen on the basis of chemical intuition. However, constructing manually a low-dimensional RC model to describe an intricate geometrical transformation for the purpose of free-energy calculations and analyses remains a daunting challenge due to the inherent complexity of the conformational transitions at play. To solve this issue, remarkable progress has been made in employing machine-learning techniques, such as autoencoders, to extract the low-dimensional RC model from a large set of CVs. Implementation of the differentiable, nonlinear machine-learned CVs in common MD engines to perform free-energy calculations is, however, particularly cumbersome. To address this issue, we present here a user-friendly tool (called MLCV) that facilitates the use of machine-learned CVs in importance-sampling simulations through the popular \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Colvars\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " module. Our approach is critically probed with three case examples consisting of small peptides, showcasing that through hard-coded neural network in \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Colvars\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       ", deep-learning and enhanced-sampling can be effectively bridged with MD simulations. The MLCV code is versatile, applicable to all the CVs available in Colvars, and can be connected to any kind of dense neural networks. We believe that MLCV provides an effective, powerful, and user-friendly platform accessible to experts and nonexperts alike for machine-learning (ML)-guided CV discovery and enhanced-sampling simulations to unveil the molecular mechanisms underlying complex biochemical processes.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_20485059.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    EntF*\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " -D3<br>Molecular dynamics simulation of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    EntF*\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " peptide, residue 3 in D form -starting coordinates posre.gro energy minimized and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " + \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ions\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " equilibrated around peptide -topology topol.top force field \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    AMBER99SB-ILDB\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    TIP3P\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " protonation states at pH   3 (E,K protonated) [\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    NaCl\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "]   25 mM \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    disulfide\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " bond generated between residues 14 and 6 -md settings md.mdp time step 2 fs, temperature   \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    298 K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       ", pressure   1 bar \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    1 microsecond\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       " trajectory dry.xtc, dry.gro frame rate 10 ps, \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ions\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " removed, box centered on peptide -analysis files frame rate 1 ps radius of gyration, solvent accessible surface area (hydrophilic parts (backbone + hydrophilic side chains), hydrophobic parts (hydrophobic side chains))</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_3236015.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Simulations of 2D Lennard-Jones particles with obstacles<br>Simulations of 2D Lennard-Jones particles with confining obstacles mimicking aggregating \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    proteins\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (labeled 'a') and with non-confining obstacles corresponding to non-aggregating \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    proteins\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (labeled 'na'), as well as a simulation without obstacles (labeled 'free'). Simulations are performed with \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GROMACS 4.5.6\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       ", and all simulation outputs and inputs are provided. The simulation parameter file (mdp) is common for all systems. Simulation details and explanation of the data are provided in the related publication &quot;\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Protein\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Crowding in \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Lipid\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Bilayers Gives Rise to Non-Gaussian Anomalous Lateral Diffusion of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Phospholipids\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Proteins\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "&quot; by Jae-Hyung Jeon et al. at https://doi.org/10.1103/PhysRevX.6.021006</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_20334621.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Modeling \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Gd3+\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Complexes for Molecular Dynamics Simulations: Toward a Rational Optimization of MRI Contrast Agents<br>The correct parametrization of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    lanthanide\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " complexes is of the utmost importance for their characterization using computational tools such as molecular dynamics simulations. This allows the optimization of their properties for a wide range of applications, including medical imaging. Here we present a systematic study to establish the best strategies for the correct parametrization of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    lanthanide\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " complexes using [\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Gd (DOTA)\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "] as a reference, which is used as a contrast agent in MRI. We chose the bonded model to parametrize the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    lanthanide\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " complexes, which is especially important when considering the study of the complex as a whole (e.g., for the study of the dynamics of its interaction with proteins or membranes). We followed two strategies: a so-called heuristic approach employing strategies already published by other authors and another based on the more recent MCPB.py tool. Adjustment of the Lennard-Jones parameters of the metal was required. The final topologies obtained with both strategies were able to reproduce the experimental ion to oxygen distance, vibrational frequencies, and other structural properties. We report a new strategy to adjust the Lennard-Jones parameters of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    metal ion\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " in order to capture dynamic properties such as the residence time of the capping water ( m). For the first time, the correct assessment of the m value for \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Gd\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-based complexes was possible by recording the dissociative events over up to 10 s all-atom simulations. The MCPB.py tool allowed the accurate parametrization of [\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Gd (DOTA)\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "] in a simpler procedure, and in this case, the dynamics of the water molecules in the outer hydration sphere was also characterized. This sphere was divided into the first hydration layer, an intermediate region, and an outer hydration layer, with a residence time of 18, 10 and 19 ps, respectively, independent of the nonbonded parameters chosen for \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Gd3+\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". The Lennard-Jones parameters of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Gd3+\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " obtained here for [\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Gd (DOTA)\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "] may be used with similarly structured gadolinium MRI contrast agents. This allows the use of molecular dynamics simulations to characterize and optimize the contrast agent properties. The characterization of their interaction with membranes and proteins will permit the design of new targeted contrast agents with improved pharmacokinetics.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_3556404.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Molecular simulation of gas solubility in \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    nitrile butadiene\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " rubber - supporting information<br>Supporting information for paper published in Journal of Physical Chemistry B (Nov 16), &quot;Molecular simulation of gas solubility in \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    nitrile butadiene\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " rubber&quot;. Includes- Initial structures generated in Materials Studio, in \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    LAMMPS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " data file format- \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    LAMMPS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " scripts used to equilibrate structures, at STP and elevated conditions.- Free volume analysis code. Places probe at points on a cubic grid, looks for closest polymer atom, if atom is more than a critical radius away, then the point is free volume.- Modified 'tpi.c' from gromacs; allows the computation of the solubility using the free volume biased technique detailed in the paper. </div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_13116890.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Atomic force microscopy and atomistic molecular dynamics simulation data to resolve structures of negatively-supercoiled \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DNA\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " minicircles at base-pair resolution.<br>This data record consists 2 zipped folders: Full AFM raw data set.zip, and Source data .zip.The zipped folder Full AFM raw data set.zip contains all raw AFM data including repeats and experiments carried out in alternative conditionsThe primary subfolder names correspond to the method of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DNA\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " immobilisation:Nickel - use of 3 mM NiCl2 in Ph7.4 20 mM HEPES bufferPLLNaOAc - use of PLL and pH 5.4 50 mM NaOAc bufferHR images - high resolution images, obtained also using the nickel conditions.The secondary subfolder names correspond to the superhelical density as shown in figure 3 in the article, and these contain the raw AFM images as .spm isles, the sub folders within those are created by the program TopoStats, and are processed data from the raw AFM images. File formats included in the zipped folder: .spm, .tiff, .json, .txt and .pdf.The zipped folder Source data .zip comprises all relevant data, pdbs of all the structures depicted in the paper obtained from simulations and AFM. See below for details on each sub folder within Source data 2.zip. Each folder contains the data used to generate each figure ad supplementary figure in the article. Figure 1: AFM data: the AFM raw files for the high-resolution images shown in figure 1, and calculations of their aspect ratios as aspectratiomanual.xlsxAFM movie: the AFM raw files for the time-lapse images shown in figure 1.MD data: the MD images used for the high-resolution images shown in figure 1 and .tar files - the MD files used to generate the snapshots MD movie: the MD snapshots files for the time-lapse images shown in figure 1 and .tar files - the MD files used to generate the snapshotsFile formats included in the Figure 1 sub folder: 0## files where ## represent numbers, .gwy, .txt, .eps, .mpg and .xlsx.Figure 2: Kink and defect measurements - the measured bend angles shown in Fig 2 and an AFM image showing how the FAM bends were measuredMD Radgyr Writhe - measurements of radius of gyrations and writhe for each topoisomer.tar files - the MD files used to generate the snapshots in 2a.txt file - the profile shown in fig 2bFile formats included in the Figure 2 sub folder: .tiff, .txt and .datFigure 3: The subfolder names correspond to the superhelical density as shown in figure 3, and these contain the raw AFM images as '.spm' isles, the sub folders within those are created by the program TopoStats, and are processed data from the raw AFM images. The '.json' file contains the data used to make the plots shown in Figure 3File formats included in the Figure 3 sub folder: .spm, .tiff, .txt, .json and .pdfFigure 4: '.dat' files contain information from MD simulations used to create the subfigure they are labelled with.The '.spm' and '.037' files are the raw AFM images used in this figure.The .tar files are MD simulations data used to generate the snapshots shown in figure 4.File formats included in the Figure 4 sub folder: .spm, .txt, .pdf and .datFigure S1: Simulations data generated using the SerraLine program, showing the average and maximum deviations from planarity in relative and absolute numbers.Data were plotted suing the distributions plot.py script.File formats included in the Figure S1 sub folder: .csv, .pdf, and .txtFigure S2a: MD measurement of the writhe over time as a '.dat file' and snapshots as '.pdb' files. File formats included in the Figure S2a sub folder: .pdb and .dat.Figure S2b: MD measurement of the writhe over time as a '.dat file' and snapshots as '.pdb' files.File formats included in the Figure S2b sub folder: .pdb and .dat.Figure S3: The AFM and MD measurements of bending angles including all profiles for MD simulations, generated using Serraline A, FM images and measurements in the form '251angles' '339 angles'.File formats included in the Figure S3 sub folder: .tiff, .txt and .pdb.Figure S4: AFM length analysis of the position of the triplex on linearised minicircles. 'Csv' file contains the length data measured by hand using the IMOD software.Plots: plots of the data raw AFM data: AFM data files used in the analysisFile formats included in the Figure S4 sub folder: .csv, .xlsx, .pdf and 0## files where ## represent numbers.Figure S5: Surface plasmon resonance (SPR) data show the effect of ions on the affinity of the triplex for varying superhelical densities of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DNA\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " minicircles, plotted using the script 'sprplot'. '.pdf' files are the plots of the various excel files.File formats included in the Figure S5 sub folder: .json, .pdf and .xlsx.Figure S6: SPR data in showing the affinity of the triplex for varying superhelical densities of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DNA\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " minicircles, plotted using the script 'sprplot'. '.pdf' files are the plots of the various excel files.File formats included in the Figure S6 sub folder: .json, .pdf, .xlsx and .pdf.Figure S7: An MS '.tar' file containing the snapshots shown in figure S7File formats included in the Figure S7 sub folder: .pdbFigure S8: AFM data used in figure s8, the '.gwy' files are AFM images of the wide view, and each of the time-lapse images. The '.txt' files are the profiles taken in those images and plotted in the figure.File formats included in the Figure S8 sub folder: .gwy and .txt.Figure S9: Simulations data showing the difference between the OL$ and BSC1 forcefields.File formats included in the Figure S9 sub folder: .datSimulations: The simulations data File formats included in the Simulations sub folder: .gro and .xtcSupp videos: The supplementary videosFile formats included in the SuppVideods sub folder: .pdb and .mpgSoftware needed to access data: 20151103 251 NAT 17ng Ni 20mm 052DX.058 or AFM 339 TFO HR cs.037, spm files &amp; all files included in the Raw AFM data sub folder - Gwyddion, Nanoscope Analysiseps files - illustrator/ pdf software.mpg - any movie player.gro - \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    gromacs\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " files- GRO files may be viewed on a computer using a supporting HP calculator emulator, such as Emu48.xtc files - \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    gromacs\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " files- a suitable software like XTrkCADsee http://manual.gromacs.org/documentation/2018/user-guide/file-formats.html for more information on \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    gromacs\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " files.Study aims and methodology: In the cell, \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DNA\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " is arranged into highly-organised and topologically-constrained (supercoiled) structures. It remains unclear how this supercoiling affects the detailed double-helical structure of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DNA\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", largely because of limitations in spatial resolution of the available biophysical tools. In this study, the authors combined high-resolution atomic force microscopy (AFM) with molecular dynamics (MD) simulations to reveal how supercoiling affects global and local \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DNA\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " conformation, structure and dynamics in \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DNA\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " minicircles of length 250-340 bp. The following procedures are described in more detail in the related article: generation and purification of small \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DNA\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " circles, preparation and analysis of different topological species of minicircles, \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    S1 nuclease\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " digestions, atomic force microscopy, atomistic simulations and surface plasmon resonance.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_12046578.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Discovery of Self-Assembling Conjugated Peptides by Active Learning-Directed Coarse-Grained Molecular Simulation<br>Electronically active organic molecules have demonstrated great promise as novel soft materials for energy harvesting and transport. Self-assembled nanoaggregates formed from -conjugated oligopeptides composed of an aromatic core flanked by oligopeptide wings offer emergent optoelectronic properties within a \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-soluble and biocompatible substrate. Nanoaggregate properties can be controlled by tuning core chemistry and peptide composition, but the sequence structure function relations remain poorly characterized. In this work, we employ coarse-grained molecular dynamics simulations within an active learning protocol employing deep representational learning and Bayesian optimization to efficiently identify molecules capable of assembling pseudo-1D nanoaggregates with good stacking of the electronically active -cores. We consider the DXXX-OPV3-XXXD \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    oligopeptide\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " family, where D is an Asp residue and OPV3 is an \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    oligophenylenevinylene\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " oligomer (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    1,4-distyrylbenzene\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "), to identify the top performing XXX tripeptides within all 203   8000 possible sequences. By direct simulation of only 2.3% of this space, we identify molecules predicted to exhibit superior assembly relative to those reported in prior work. Spectral clustering of the top candidates reveals new design rules governing assembly. This work establishes new understanding of DXXX-OPV3-XXXD assembly, identifies promising new candidates for experimental testing, and presents a computational design platform that can be generically extended to other peptide-based and peptide-like systems.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_20473913.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Functional Loop Dynamics and Characterization of the Inactive State of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    NS2B-NS3 Dengue Protease\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " due to Allosteric Inhibitor Binding<br>Dengue virus, a flavivirus that causes dengue shock syndrome and dengue hemorrhagic fever, is currently prevalent worldwide. A two-component \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    protease\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    NS2B-NS3\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") is essential for maturation, representing an important target for designing anti-flavivirus drugs. Previously, consideration has been centered on developing active-site inhibitors of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    NS2B-NS3pro\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". However, the flat and charged nature of its active site renders difficulties in developing inhibitors, suggesting an alternative strategy for identifying allosteric inhibitors. The allosterically sensitive site of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    dengue protease\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " is located near Ala125, between the 120s loop and 150s loop. Using atomistic molecular dynamics simulations, we have explored the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    protease s\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " conformational dynamics upon binding of an allosteric inhibitor. Furthermore, characterization of the inherent flexible loops (71 75s loop, 120s loop, and 150s loop) is carried out for allosteric-inhibitor-bound wild-type and mutant A125C variants and a comparison is performed with its unbound state to extract the structural changes describing the inactive state of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    protease\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". Our study reveals that compared to the unliganded system, the inhibitor-bound system shows large structural changes in the 120s loop and 150s loop in contrast to the rigid 71 75s loop. The unliganded system shows a closed-state pocket in contrast to the open state for the wild-type complex that locks the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    protease\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " into the open and inactive-state conformations. However, the mutant complex fluctuates between open and closed states. Also, we tried to see how mutation and binding of an allosteric inhibitor perturb the connectivity in a protein structure network (PSN) at contact levels. Altogether, our study reveals the mechanism of conformational rearrangements of loops at the molecular level, locking the protein in an inactive conformation, which may be useful for developing allosteric inhibitors.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_53151.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Molecular dynamics simulation trajectory of a \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    cationic lipid\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " bilayer: 50/50 mol% \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DMTAP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "/\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DMPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " in 0.5 M \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    NaCl\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "<br>System: Symmetric bilayer of cationic \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DMTAP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    dimyristoyltrimethylammoniumpropane\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", 50 mol-%) and zwitterionic \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DMPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    dimyristoylphosphatidylcholine\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", 50 mol-%) lipids in 0.5 M \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    NaCl\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " solution. Number of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DMPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ": 64. Number of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DMTAP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ": 64. Number of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Na+\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " ions: 48. Number of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Cl-\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " ions: 112. Number of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    waters\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ": 5240. Lipid model: '\n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Berger\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       "' united-atom [Biophys. J. 72 2002 (1997)] with \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DMTAP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " modification by Gurtovenko et al. [Biophys. J. 86 3461 (2004)]. \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " model: \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    SPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " [In Intermolecular Forces, ed. Pullman. 331 (1981)]. Simulation engine: \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Gromacs 3.x\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " [https://removed Trajectory length: \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    179 ns\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       ". Previously equilibrated for: 21 ns. Sampling rate: every 10 ps. Time integration step: 2 fs. Thermodynamic ensemble: NpT. Temperature coupling: 'Berendsen' [J. Chem. Phys. 81 3684 (1984)] with lipids and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " coupled separately at T \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    323 K.\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       " Pressure coupling: 'Berendsen' [J. Chem. Phys. 81 3684 (1984)] with xy and z coupled separately at p 1.0 bar. Electrostatics: PME [J. Chem. Phys. 98 10089 (1993); J. Chem. Phys. 103 8577 (1995)], real-space cutoff at 1.0 nm. Van der Waals: Truncated at 1.0 nm. Constraints: Covalent bond lengths in lipids using \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    LINCS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " [J. Comput. Chem. 18 1463 (1997)], in \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " using \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    SETTLE\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " [J. Comput. Chem. 13 952 (1992)]. Used in publications: [1] Markus S. Miettinen, Andrey A. Gurtovenko, Ilpo Vattulainen, and Mikko Karttunen: &quot;Ion Dynamics in Cationic Lipid Bilayer Systems in Saline Solutions&quot;. J. Phys. Chem. B 113 9226 (2009). DOI: 10.1021/jp810233q. [2] Markus S. Miettinen: &quot;Computational Modeling of Cationic Lipid Bilayers in Saline Solutions&quot;. PhD Thesis. Aalto University School of Science and Technology, Helsinki, Finland. (2010). ISBN 978-952-60-3194-1.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_6974777.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Simulated structures of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ansamer amanitin\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " derivatives and precursors for atroposelective synthesis<br>\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    a-amanitin\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and its derivatives are eight-membered, cyclic peptides that are additionally bridged with a \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    tryptathionine\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " cross-link. Their synthesis sometimes yields isomers, which haven't been classified unambiguously so far. Until now, it was not clear whether these isomers differing in the positioning of the trpytathionine bridge indeed exist and if yes, under which conditions they are formed and how they shall be named. In our work, we therefore investigated the occurrence of these isomers. We have identified the structure and the dynamics of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    amanitin\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " derivative 4a and its isomer 4b. Investigating the impact of the macrocyclisation site on the occurrence of this isomer, we published an optimised strategy for atroposelective synthesis (including precursors 3b and 3c). Here, we present the simulated structures of the ansamers 4a and 4b, which both are in very good agreement to crystal data, together with the precursors 3b and 3c. 4a and 4b are the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    amanitin\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " derivatives, for whose classification we propose the term ansamers. For further instructions, please refer to 'README'. For the files required to reproduce the MD simulations, please refer to the Zenodo repository 'MD setup data for \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ansamer amanitin\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " derivatives and precursors for atroposelective synthesis' available under DOI: 10.5281/zenodo.7125315. </div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_55565.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Bug in CCFD stress profile calculation of \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GROMACS-LS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       "?<br>In \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Martini\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " simulations (\n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    15 μs\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       ", last 5 μs analyzed) of an asymmetric \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GM1\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " + \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " lipid bilayer (5+95 upper, 0+100 lower leaflet) strong stress fluctuations arising from the dihedral contributions of CCF decomposition are found. It is possible that these are due to a slight bug in the \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GROMACS-LS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " (http://mdstress.org) code, because: 1) The fluctuations do not show converge, when more data is added, see the attached plot effect of sampling rate.pdf . 2) The fluctuations (but no other features of the stress profiles) visibly depend on the chosen frame of reference, see effect of centering schemes.pdf . 3) The stress profiles calculated along the coordinates that lay in the membrane plane (x and y), are not constant in CCFD, see profiles along x and y.pdf . To assist debugging the code, this repository contains: 5to95 0to100 0.trr --- 1st 33.3% of a 5 μs trajectory with snapshots every 100 ps 5to95 0to100 0.trr --- 2nd 33.3% of a 5 μs trajectory with snapshots every 100 ps 5to95 0to100 0.trr --- 3rd 33.3% of a 5 μs trajectory with snapshots every 100 ps 5to95 0to100.tpr --- tpr used for production (\n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Gromacs 5.1.1\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       ") 5to95 0to100 rerun.tpr --- tpr used for stress analysis and centering 5to95 0to100.gro --- gro file after 15 μs (10 μs relaxation, 5 μs production) index.ndx --- index file used in production 5to95 0to100.ndx --- index file used in analysis system.top --- topology file \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    martini\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " v2.0 ions.itp --- \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    martini\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " FF file (system.top expects this to be in folder toppar/) \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    martini\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " v2.0 lipids all 201506.itp --- \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    martini\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " FF file (system.top expects this to be in folder toppar/) \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    martini\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " v2.2.itp --- \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    martini\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " FF file (system.top expects this to be in folder toppar/) \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    martini\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " straight \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GM1\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " saveVels.mdp --- run input file used for production (\n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Gromacs 5.1.1\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       ") \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    martini\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " straight \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GM1\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " rerunForP.mdp --- run input file used for analysis and centering effect of sampling rate.pdf --- plot of results showing that fluctuations in CCFD do not appear to converge when data is added profiles along x and y.pdf --- plot of results showing that in CCFD pressure profiles along the membrane directions are not constant effect of centering schemes.pdf --- plot of results showing that choice of center of mass (here three possibilities are shown: \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CoM\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GL1\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " beads, \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CoM\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPCs\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CoM\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " of all lipids) visibly affects CCFD fluctuations, but not other features of the stress profile effect of centering schemes 2.pdf --- plot of results (similar to 'effect of centering schemes.pdf', but the center of the bilayer is at 3.7 nm instead at 0.0 / 7.4 nm) individual components.pdf --- plot of results showing that the fluctuations in CCFD arise from dihedrals CCFD gridsize.pdf --- plot of results showing that decreasing gridsize has no clear effect on the CCFD fluctuations</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_6456372.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Trajectories for \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    EmrE\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (faRM)<br>These are the molecular dynamics trajectories for a refined \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    EmrE\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " model, as described by Vermaas et al. 2016. There are 25 trajectories total, each with 1001 frames. 5 trajectories are for the doubly deprotonated (apo) state, 5 have a proton on the &quot;A&quot; monomer (PA), 5 on the &quot;B&quot; monomer (PB), 5 are protonated on both \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    glutamate\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " residues (Protonated), and 5 are doubly deprotonated states with TPP+ bound. To load a trajectory in \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    VMD\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       ", one might do the following: mol new Apo-01.psf mol addfile Apo-01.dcd waitfor all For PyMol, the following is recommended: load Apo-01.pdb load traj Apo-01.dcd </div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_2154835.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Atomistic and Coarse Grain Topologies for the Cofactors Associated with the Photosystem II Core Complex<br>Electron transfers within and between protein complexes are core processes of the electron transport chains occurring in thylakoid (chloroplast), mitochondrial, and bacterial membranes. These electron transfers involve a number of cofactors. Here we describe the derivation of molecular mechanics parameters for the cofactors associated with the function of the photosystem II core complex: \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    plastoquinone\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    plastoquinol\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    heme b\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    chlorophyll A\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    pheophytin\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    carotene\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". Parameters were also obtained for \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ubiquinol\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ubiquinone\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", related cofactors involved in the respiratory chain. Parameters were derived at both atomistic and coarse grain (CG) resolutions, compatible with the building blocks of the \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GROMOS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " united-atom and \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Martini CG\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " force fields, respectively. Structural and thermodynamic properties of the cofactors were compared to experimental values when available. The topologies were further tested in molecular dynamics simulations of the cofactors in their physiological environment, e.g., either in a lipid membrane environment or in complex with the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    heme\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " binding protein \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    bacterioferritin\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ".</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_4625961.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Connexin 46\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    connexin 50\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " gap junction channel properties are shaped by structural and dynamic features of their N-terminal domains<br>Provided are reduced trajectories (.dcd) of the MD simulations -- each trajectory has 100 ps/frame with only protein and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ion\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " atoms remaining. Each set of trajectories are accompanied by a protein structure file (.psf) which is required to visualize the trajectories in \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    VMD\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       ". Additionally, the z-trajectories of each intracellular \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ion\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (2 ps/frame) are provided in zipped files. To re-create the potentials of mean force (PMF) in Yue &amp; Haddad et al., use the scripts provided with the paper (https://github.com/reichow-lab/Yue-Haddad et-al.JPhysiol2021): python3 GapJ Analysis.py &quot;Cx46 Ace Produc-1 POT *&quot; Choose a bin size in A (3) Choose an output name (Cx46 Ace) Choose option (M) Choose time (ps) / frame (2) Choose column from file (1) Choose binmin/binmax (auto) </div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_14932766.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Protein Dynamics and Substrate Protonation States Mediate the Catalytic Action of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    trans-4-Hydroxy l Proline\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Dehydratase\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "<br>The enzyme \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    trans-4-hydroxy-l-proline\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Hyp\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    dehydratase\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    HypD\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") is among the most abundant glycyl radical enzymes (GREs) in the healthy human gut microbiome and is considered a promising antibiotic target for the prominent antibiotic-resistant pathogen Clostridium difficile. Although an enzymatic mechanism has been proposed, the role of the greater \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    HypD\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " protein environment in mediating radical reactivity is not well understood. To fill this gap in understanding, we investigate \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    HypD\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " across multiple time- and length-scales using electronic structure modeling and classical molecular dynamics. We observe that the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Hyp\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " substrate protonation state significantly alters both its enzyme-free reactivity and its dynamics within the enzyme active site. Accurate coupled-cluster modeling suggests the deprotonated form of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Hyp\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " to be the most reactive protonation state for \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    C5 Hpro S\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " activation. In the protein environment, hydrophobic interactions modulate the positioning of the Cys434 radical to enhance the reactivity of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    C5 Hpro S\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " abstraction. Long-time dynamics reveal that changing \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Hyp\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " protonation states triggers the switching of a Leu643-gated \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " tunnel, a functional feature that has not yet been observed for members of the GRE superfamily.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_20485188.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    EntF*\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " -D5<br>Molecular dynamics simulation of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    EntF*\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " peptide, residue 5 in D form -starting coordinates posre.gro energy minimized and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " + \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ions\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " equilibrated around peptide -topology topol.top force field \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    AMBER99SB-ILDB\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    TIP3P\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " protonation states at pH   3 (E,K protonated) [\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    NaCl\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "]   25 mM \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    disulfide\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " bond generated between residues 14 and 6 -md settings md.mdp time step 2 fs, temperature   \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    298 K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       ", pressure   1 bar \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    1 microsecond\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       " trajectory dry.xtc, dry.gro frame rate 10 ps, \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ions\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " removed, box centered on peptide -analysis files frame rate 1 ps radius of gyration, solvent accessible surface area (hydrophilic parts (backbone + hydrophilic side chains), hydrophobic parts (hydrophobic side chains))</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_19115904.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Mechanistic Characterization of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Zeolite\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-Catalyzed Aromatic Electrophilic Substitution at Realistic Operating Conditions<br>\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Zeolite\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-catalyzed \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    benzene\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " ethylation is an important industrial reaction, as it is the first step in the production of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    styrene\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " for polymer manufacturing. Furthermore, it is a prototypical example of aromatic electrophilic substitution, a key reaction in the synthesis of many bulk and fine chemicals. Despite extensive research, the reaction mechanism and the nature of elusive intermediates at realistic operating conditions is not properly understood. More in detail, the existence of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    elusive arenium\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " ion (better known as Wheland complex) formed upon electrophilic attack on the aromatic ring is still a matter of debate. Temperature effects and the presence of protic guest molecules such as \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " are expected to impact the reaction mechanism and lifetime of the reaction intermediates. Herein, we used enhanced sampling ab initio molecular dynamics simulations to investigate the complete mechanism of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    benzene\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " ethylation with \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ethene\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ethanol\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " in the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    H-ZSM-5 zeolite\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". We show that both the stepwise and concerted mechanisms are active at reaction conditions and that the Wheland intermediate spontaneously appears as a shallow minimum in the free energy surface after the electrophilic attack on the benzene ring. Addition of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " enhances the protonation kinetics by about 1 order of magnitude at coverages of one \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " molecule per Br nsted acidic site. In the fully solvated regime, an overstabilization of the BAS as \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    hydronium\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " ion occurs and the rate enhancement disappears. The obtained results give critical atomistic insights in the role of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " to selectively tune the kinetics of protonation reactions in \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    zeolites\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ".</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_11704443.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Exploring \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Methamphetamine\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Nonenantioselectively Targeting \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Toll-like Receptor 4\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "/\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Myeloid\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Differentiation Protein 2 by in Silico Simulations and Wet-Lab Techniques<br>\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Methamphetamine\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    METH\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") is one of the highly addictive nonopioid psychostimulants, acting as a xenobiotic-associated molecular pattern (XAMP) to target \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    TLR4\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and activate microglia. However, the molecule recognition of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    METH\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " by innate immune receptor \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    TLR4\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "/\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    MD-2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " is not well-understood. \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    METH\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " exists in two enantiomeric forms, and it is unclear whether the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    TLR4\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " innate immune recognition with \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    METH\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " is stereoselective. Herein, molecular dynamics (MDs) simulations were performed to dissect the recognition of (+)- \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    METH\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and ( )- \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    METH\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " by \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    TLR4\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "/\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    MD-2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " at the atomic level. \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Amphetamine\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    AMPH\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "), which is an analogue of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    METH\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", was also investigated for comparison. Computational simulations indicate that \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    METH\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " binds into the interaction interface between \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    MD-2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " as well as \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    TLR4\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "* that is from the adjacent copy of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    TLR4\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    MD-2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", therefore stabilizing the active heterotetramer (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    TLR4\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "/\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    MD-2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " )2 complex. The calculated binding free energies and potential of mean force (PMF) values show that ( )- \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    METH\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and (+)- \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    METH\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " have similar \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    TLR4\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "/\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    MD-2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " binding affinity. Further dynamics analyses of bindings with \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    TLR4\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "/\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    MD-2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " indicate that ( )- \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    METH\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and (+)- \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    METH\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " behave similarly. Unlike the stereoselective neuron-stimulating activities of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    METH\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", no enantioselectivity was observed for \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    METH\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " interacting with \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    TLR4\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "/\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    MD-2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " complex as well as activating \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    TLR4\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " signaling. Compared to \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    METH\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    AMPH\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " showed much weaker interactions with \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    TLR4\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "/\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    MD-2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", indicating that the substituted methyl group is critical in the molecular recognition of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    METH\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " by \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    TLR4\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "/\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    MD-2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". In all, this study provides molecular insight into the innate immune recognition of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    METH\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", which demonstrates that \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    METH\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " could be nonenantioselectively sensed by \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    TLR4\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "/\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    MD-2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ".</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_5642866.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    6MSM\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CFTR\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " MD EQUILIBRATED MISSING R DOMAIN FRAGMENT MODELLED<br>A protein structure based on PDB ID \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    6MSM\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " with the unidentified fragment filled in based on results from molecular dynamics investigations and comparisons with \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    alphafold\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       ". In addition to the molecular dynamics workflow used to simulate it.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_20477204.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Improving All-Atom Force Field to Accurately Describe \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DNA G Quadruplex\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Loops<br>The \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DNA G-quadruplex\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GQ\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") displays structural polymorphisms, and interactions between its loops and flanking sequences critically determine which of the diverse \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GQ\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " conformers is adopted. All-atom molecular dynamics (MD) simulations of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GQs\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " are computationally challenging due to slow folding times and force field (ff) artifacts. In an earlier study, a direct folding simulation of the simplest \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DNA GQ\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    TBA15\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") was first reported using a modified version of the \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    AMBER bsc1\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " ff (\n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    bsc1 vdW\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " ff). Despite this successful folding simulation, it was later found that the \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    bsc1 vdW\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " ff is somewhat limited in terms of describing loop structures of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GQs\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", which is problematic because \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GQ\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " loop regions play key roles in ligand binding to modulate \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GQ\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " activities. In this study, we further modified the \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    bsc1 vdW\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " ff to enhance the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GQ\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " loop prediction by fine-tuning a limited number of van der Waals (vdW) parameters of the standard \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    AMBER bsc1\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " ff to improve the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GQ\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " loop distribution of a target \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GQ\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " system (three-layered antiparallel \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GQ\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "; mHtel21). Test simulations of this newly generated ff (\n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    bsc1 vdWL\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " ff) on \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DNA GQs\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " with diverse topologies (hybrid1, hybrid2, and parallel propeller) revealed that loop structures were predicted more accurately than by the \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    bsc1 vdW\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " ff. We consider that enhanced sampling MD simulation methods in combination with \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    bsc1 vdWL\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " provide useful simulation protocols for resolving outstanding issues of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DNA GQ\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " folding and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GQ\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "/ligand binding at the all-atom level.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_14919822.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Electrostatic Interactions Explain the Higher Binding Affinity of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CR3022\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Antibody for SARS-CoV 2 than the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    4A8\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Antibody<br>A structural understanding of the mechanism by which antibodies bind SARS-CoV-2 at the atomic level is highly desirable as it can tell the development of more effective antibodies to treat Covid-19. Here, we use steered molecular dynamics (SMD) and coarse-grained simulations to estimate the binding affinity of the monoclonal antibodies \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CR3022\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    4A8\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " to the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    SARS-CoV-2 receptor-binding domain\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    RBD\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    SARS-CoV-2 N-terminal domain\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    NTD\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "). Consistent with experiments, our SMD and coarse-grained simulations both indicate that \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CR3022\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " has a higher affinity for \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    SARS-CoV-2 RBD\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " than \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    4A8\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " for the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    NTD\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", and the coarse-grained simulations indicate the former binds three times stronger to its respective epitope. This finding shows that \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CR3022\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " is a candidate for Covid-19 therapy and is likely a better choice than \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    4A8\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". Energetic decomposition of the interaction energies between these two complexes reveals that electrostatic interactions explain the difference in the observed binding affinity between the two complexes. This result could lead to a new approach for developing anti-Covid-19 antibodies in which good candidates must contain charged amino acids in the area of contact with the virus.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_1034816.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">BMP (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    LBPA\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") topologies, parameters, configurations, and related scripts<br>Contents: 000README PDB files for \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    LBPA\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " configurations: 2-2R-RLBPA.pdb 2-2R-SLBPA.pdb 2-2S-SLBPA.pdb 3-3R-RLBPA.pdb 3-3R-SLBPA.pdb 3-3S-SLBPA.pdb Python script to check and convert R/S handedness: check chirals.py \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Slipids\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " topologies: Slipids 2-2LBPA.itp Slipids 3-3LBPA.itp \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Charmm36\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " topologies: top all36 2-2LBPA.rtf top all36 3-3LBPA.rtf Notes: 1) All bonded terms are already included in \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Slipids\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " and the charges are calculated based on the protocol used in \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Slipid\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " parametrization See the comments in the itp files. 2) One has to change the molecule name if one wants to simulate a mixture of 3-3 and 2-2 isomers.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_6144286.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Simulations for: Patched 1 regulates \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Smoothened\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " by controlling sterol binding to its extracellular cysteine-rich domain.<br>Atomistic simulations of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Smoothened\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ": Atomistic molecular dynamics simulations of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    glycosylated mouse Smoothened\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    mSMO\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    human Smoothened\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    hSMO\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") in distinct liganded states: - \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    mSMO\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " in an apo state (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    mSMO\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " apo) - \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    mSMO\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " with TMD \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    cholesterol\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " bound (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    mSMO\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " chol TMD) - \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    mSMO\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " with CRD \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    cholesterol\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " bound (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    mSMO\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " chol CRD) - \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    mSMO\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " with \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    cholesterol\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " bound to the CRD and TMD (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    mSMO\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " 2chol TMD CRD) - \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    mSMO\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " bound to CRD \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    cholesterol\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Smoothened\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " agonist (SAG) (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    mSMO\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " chol CRD SAG) - \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    mSMO\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " bound to CRD and TMD \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    cholesterols\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", SAG and an intracellular nanobody (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    mSMO\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " 2chol SAG nanobody) - \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    mSMO\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " bound to CRD and TMD \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    cholesterol\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and with an additional \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    cholesterol\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " bound at a proposed 'mid' position (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    mSMO\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " 3chol) - \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    hSMO\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " bound to CRD \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    cholesterol\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and SAG (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    hSMO\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " chol CRD SAG) All simulations were run for 5x \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    300 ns\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       ". The active \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    mSMO\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " conformation was obtained from the Protein Data Bank (PDB ID: \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    6O3C\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "). The \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    hSMO\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " structure was determined using X-ray crystallography as outlined within this study. File description for each system: - md fit firstframe.pdb : Initial frame used for atomistic simulations (replicate 1, each replicate equilibrated independently). - md fit X.xtc : \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Gromacs\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " trajectory file for each replicate (X replicate number). </div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_19533246.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">AIM: A Mapping Program for Infrared Spectroscopy of Proteins<br>Here, we present a new analysis program, AIM, that allows extracting the vibrational \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    amide-I\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Hamiltonian using molecular dynamics trajectories for protein infrared spectroscopy modeling. The constructed Hamiltonians can be used as input for spectral calculations allowing the calculation of infrared absorption spectra, vibrational circular dichroism, and two-dimensional infrared spectra. These spectroscopies allow the study of the structure and dynamics of proteins. We will explain the essence of how AIM works and give examples of the information and spectra that can be obtained with the program using the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Trypsin Inhibitor\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " as an example. AIM is freely available from GitHub, and the package contains a demonstration allowing easy introduction to the use of the program.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_22151355.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Hierarchical Aggregation in a Complex Fluid The Role of Isomeric Interconversion<br>There is an ever-increasing body of evidence that metallic complexes involving \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    amphiliphic ligands\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " do not form normal solutions in organic solvents. Instead, they form complex fluids with intricate structures. For example, the metallic complexes may aggregate into clusters, and these clusters themselves may aggregate into superclusters. To gain a deeper insight into the mechanisms at play, we have used an improved force field to conduct extensive molecular dynamics simulations of a system composed of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    zirconium nitrate\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    nitric acid\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    tri-n-butyl phosphate\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    n-octane\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". The important new finding is that a dynamic equilibrium between the cis and trans isomers of the metal complex is likely to play a key role in the aggregation behavior. The isolated cis and trans isomers have similar energies, but simulation indicates that the clusters consist predominantly of cis isomers. With increasing metal concentration, we hypothesize that more clustering occurs and the chemical equilibrium shifts toward the cis isomer. It is possible that such isomeric effects play a role in the liquid liquid extraction of other species and the inclusion of such effects in flow sheet modeling may lead to a better description of the process.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_6381427.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Structural basis of Omicron immune evasion: A comparative computational study of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Spike protein\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Antibody\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " interaction<br>The COVID-19 pandemic has caused more than 424 million infections and 5.9 million deaths so far. The vaccines used against SARS-COV-2 by now have been able to develop some neutralising \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    antibodies\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " in the vaccinated human population and slow down the infection rate. The effectiveness of the vaccines has been challenged by the emergence of the new strains with numerous mutations in the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    spike\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    S\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") protein of SARS-CoV-2. Since \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    S protein\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " is the major immunogenic protein of the virus and also contains \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Receptor Binding Domain\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    RBD\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") that interacts with the human \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Angiotensin-Converting Enzyme 2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ACE2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") receptors, any mutations in this region should affect the neutralisation potential of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    antibodies\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " leading to the immune evasion. Several variants of concern (VOC) of the virus have emerged so far. Among them, the most critical are Delta (B.1.617.2), and recently reported Omicron (B. 1.1.529) which have acquired a lot of mutations in the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    spike protein\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". We have mapped those mutations on the modelled \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    RBD\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and evaluated the binding affinities of various human \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    antibodies\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " with it. Docking and molecular dynamics simulation studies have been used to explore the effect of the mutations on the structure of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    RBD\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    RBD\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    antibody\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " interaction. The analysis shows that the mutations mostly at the interface of a nearby region lower the binding affinity of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    antibody\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " by ten to forty per cent, with a downfall in the number of interactions formed as a whole and therefore, it implies the generation of immune escape variants. Notable mutations and their effect was characterised by performing various analyses that explain the structural basis of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    antibody\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " efficacy in Delta and a compromised neutralisation effect for the Omicron variant. Our results pave the way for robust vaccine design that can be effective for many variants.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_5532214.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">\n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Microsecond\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       "-long molecular dynamics trajectory of an aggregate of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    elastin\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-like peptides in \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "<br>This binary file is provided as supplementary material for a research article on the structure and self-assembly of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    elastin\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-like peptides (1). It was generated by the molecular dynamics program \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GROMACS, version 4.6\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " (2). It contains the last microsecond of a \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    5-microsecond\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       " molecular dynamics trajectory of 27 (GVPGV)7 peptides in water. See ref. 1 for details. (1) S. Rauscher and R. Pome s. The liquid structure of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    elastin\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". eLife, in press (2017). DOI: 10.7554/eLife.26526 (2) Hess, B., Kutzner, C., van der Spoel, D. &amp; Lindahl, E. GROMACS 4: Algorithms for Highly Efficient, Load-Balanced, and Scalable Molecular Simulation. J. Chem. Theory Comput. 4, 435 447 (2008). DOI: 10.1021/ct700301q</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_4646862.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Addressing a Trapped High-Energy \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ": Design and Synthesis of Highly Potent \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Pyrimidoindole\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-based \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GSK-3 inhibitors\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ".<br>Raw concatenated trajectories of the molecular dynamics simulations and WaterMap-output files related to publication: Addressing a Trapped High-Energy \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ": Design and Synthesis of Highly Potent \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Pyrimidoindole\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-Based \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Glycogen Synthase Kinase-3 Inhibitors\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " J Med Chem (2021): https://doi.org/10.1021/acs.jmedchem.0c02146 Compounds (S)-5c and (S)-15 trajectories are provided in a single concatenated \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Desmond\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " trajectory file (Both: 1 s + 1 s + 1 s + 1 s + 1 s + 1 s + 1 s + 1 s + 1 s + 1 s   10 s in total) (S)-5c   Files named: M CL all-out.cms; M CL all trj/ (S)-15   Files named: M CC all-out.cms; M CC all trj/ WaterMap-output files (.zip) (S)-5c   watermap 5c wm.zip (S)-5m   watermap 5c wm.zip (S)-15   watermap 15 wm.zip; MD-derived (#1-5): MD derived watermap 15 ? wm.zip </div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_5975071.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Binding Modes of Ligands Using Enhanced Sampling (BLUES): Rapid Decorrelation of Ligand Binding Modes via Nonequilibrium Candidate Monte Carlo<br>Accurately predicting protein ligand binding affinities and binding modes is a major goal in computational chemistry, but even the prediction of ligand binding modes in proteins poses major challenges. Here, we focus on solving the binding mode prediction problem for rigid fragments. That is, we focus on computing the dominant placement, conformation, and orientations of a relatively rigid, fragment-like ligand in a receptor, and the populations of the multiple binding modes which may be relevant. This problem is important in its own right, but is even more timely given the recent success of alchemical free energy calculations. Alchemical calculations are increasingly used to predict binding free energies of ligands to receptors. However, the accuracy of these calculations is dependent on proper sampling of the relevant ligand binding modes. Unfortunately, ligand binding modes may often be uncertain, hard to predict, and/or slow to interconvert on simulation time scales, so proper sampling with current techniques can require prohibitively long simulations. We need new methods which dramatically improve sampling of ligand binding modes. Here, we develop and apply a nonequilibrium candidate Monte Carlo (NCMC) method to improve sampling of ligand binding modes. In this technique, the ligand is rotated and subsequently allowed to relax in its new position through alchemical perturbation before accepting or rejecting the rotation and relaxation as a nonequilibrium Monte Carlo move. When applied to a \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    T4 lysozyme\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " model binding system, this NCMC method shows over 2 orders of magnitude improvement in binding mode sampling efficiency compared to a brute force molecular dynamics simulation. This is a first step toward applying this methodology to pharmaceutically relevant binding of fragments and, eventually, drug-like molecules. We are making this approach available via our new Binding modes of ligands using enhanced sampling (BLUES) package which is freely available on GitHub.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_19858240.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Influence of the Greater Protein Environment on the Electrostatic Potential in \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Metalloenzyme\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Active Sites: The Case of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Formate Dehydrogenase\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "<br>The Mo/W-containing \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    metalloenzyme\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    formate dehydrogenase\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    FDH\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") is an efficient and selective natural catalyst that reversibly converts CO2 to \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    formate\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " under ambient conditions. In this study, we investigate the impact of the greater protein environment on the electrostatic potential (ESP) of the active site. To model the enzyme environment, we used a combination of classical molecular dynamics and multiscale quantum-mechanical (QM)/molecular-mechanical (MM) simulations. We leverage charge shift analysis to systematically construct QM regions and analyze the electronic environment of the active site by evaluating the degree of charge transfer between the core active site and the protein environment. The contribution of the terminal \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    chalcogen\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " ligand to the ESP of the metal center is substantial and dependent on the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    chalcogen\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " identity, with similar, less negative ESPs for Se and S terminal chalcogens in comparison to O regardless of whether the metal is Mo or W. The orientation of the side chains and conformations of the cofactor also affect the ESP, highlighting the importance of sampling dynamic fluctuations in the protein. Overall, our observations suggest that the terminal \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    chalcogen\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " ligand identity plays an important role in the enzymatic activity of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    FDH\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", suggesting opportunities for a rational bioinspired catalyst design.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_2182180.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Redox Potential Tuning through Differential \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Quinone\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Binding in the Photosynthetic Reaction Center of Rhodobacter sphaeroides<br>\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Ubiquinone\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " forms an integral part of the electron transport chain in cellular respiration and photosynthesis across a vast number of organisms. Prior experimental results have shown that the photosynthetic reaction center (RC) from Rhodobacter sphaeroides is only fully functional with a limited set of methoxy-bearing \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    quinones\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", suggesting that specific interactions with this substituent are required to drive electron transport and the formation of quinol. The nature of these interactions has yet to be determined. Through parameterization of a \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CHARMM\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       "-compatible \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    quinone\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " force field and subsequent molecular dynamics simulations of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    quinone\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-bound RC, we have investigated and characterized the interactions of the protein with the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    quinones\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " in the QA and QB sites using both equilibrium simulation and thermodynamic integration. In particular, we identify a specific interaction between the 2-methoxy group of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ubiquinone\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " in the QB site and the amide nitrogen of GlyL225 that we implicate in locking the orientation of the 2-methoxy group, thereby tuning the redox potential difference between the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    quinones\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " occupying the QA and QB sites. Disruption of this interaction leads to weaker binding in a \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ubiquinone\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " analogue that lacks a 2-methoxy group, a finding supported by reverse electron transfer electron paramagnetic resonance experiments of the QA QB biradical and competitive binding assays.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_20118543.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Adsorption and Diffusion of Oxygen on Pure and Partially Oxidized Metal Surfaces in Ultrahigh Resolution<br>The interaction of gas molecules with metal and oxide surfaces plays a critical role in corrosion, catalysis, sensing, and heterogeneous materials. However, insights into the dynamics of O2 from picoseconds to microseconds have remained unavailable to date. We obtained 3D potential energy surfaces for adsorption of O2 on 11 common pristine and partially oxidized (hkl) surfaces of Ni and Al in picometer resolution and high accuracy of 0.1 kcal/mol, identified binding sites, and surface mobility from 25 to 300 C. We explain relative oxidation rates and parameters for oxide growth. We employed over 150 000 molecular mechanics and molecular dynamics simulations with the interface force field (IFF) using structural data from X-ray diffraction (XRD) and low-energy electron diffraction (LEED). The methods reach 10 to 50 times higher accuracy than possible before and are suited to analyze gas interactions with metals up to the micrometer scale including defects and irregular nanostructures.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_9985349.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Foaming Dependence on the Interface Affinities of Surfactant-like Molecules<br>We have explored the relationship between physical properties and foaming behavior of 30 wt % aqueous \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    monoethanolamine\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    MEA\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") solutions with three different surfactant-like additives. The additives included one commercial surfactant, \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    cetyltrimethylammonium bromide\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CTAB\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "), and two \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CO2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " capture catalysts that behave as weak surfactants. The physical properties of interest were the additives affinity for the liquid vapor interface and the viscosity, surface tension, and surface elasticity of the solutions. The physical properties were explored using molecular dynamics simulations as well as experiments. The additives affinity for the liquid vapor interface played a key role in the foaming behavior by affecting the interfacial properties (surface tension and surface elasticity) of the aqueous \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    MEA\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " solutions. The solution containing \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CTAB\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " showed the largest surface tension depression and the largest surface elasticity and resulted in the largest amount of foaming. The two catalyst solutions had almost similar surface tensions but differed in their elasticity. The catalyst solution with higher surface elasticity showed more foaming due to higher foam stability.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_3865919.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Small molecule sequestration of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    amyloid\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " as a drug discovery strategy for Alzheimer's disease<br>These data describe the bound and unbound ensembles of a disordered peptide (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    amyloid beta\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") in the presence and absence of a small, drug-like molecule. The data were produced by metadynamic metainference simulations restrained with NMR chemical shift data. We used \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PLUMED version 2.6.0\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GROMACS 2018.3\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       ". All the data and PLUMED input files required to reproduce the metadynamic metainference results are available on PLUMED-NEST (www.plumed-nest.org), the public repository of the PLUMED consortium, as plumID:20.014. A Jupyter Notebook describing the analysis of these results is available from GitHub at https://github.com/vendruscolo-lab/amyloid-beta small mol/ in Metadynamic metainference/Analysis. These data support the findings of the manuscript entitled &quot;Small molecule sequestration of amyloid- as a drug discovery strategy for Alzheimer's disease&quot; by Heller et al. DOI: 10.1101/729392</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_1586671.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Structural Refinement of Proteins by Restrained Molecular Dynamics Simulations with Non-interacting Molecular Fragments<br>The knowledge of multiple conformational states is a prerequisite to understand the function of membrane transport proteins. Unfortunately, the determination of detailed atomic structures for all these functionally important conformational states with conventional high-resolution approaches is often difficult and unsuccessful. In some cases, biophysical and biochemical approaches can provide important complementary structural information that can be exploited with the help of advanced computational methods to derive structural models of specific conformational states. In particular, functional and spectroscopic measurements in combination with site-directed mutations constitute one important source of information to obtain these mixed-resolution structural models. A very common problem with this strategy, however, is the difficulty to simultaneously integrate all the information from multiple independent experiments involving different mutations or chemical labels to derive a unique structural model consistent with the data. To resolve this issue, a novel restrained molecular dynamics structural refinement method is developed to simultaneously incorporate multiple experimentally determined constraints (e.g., engineered metal bridges or spin-labels), each treated as an individual molecular fragment with all atomic details. The internal structure of each of the molecular fragments is treated realistically, while there is no interaction between different molecular fragments to avoid unphysical steric clashes. The information from all the molecular fragments is exploited simultaneously to constrain the backbone to refine a three-dimensional model of the conformational state of the protein. The method is illustrated by refining the structure of the voltage-sensing domain (VSD) of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Kv1.2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " potassium channel in the resting state and by exploring the distance histograms between spin-labels attached to \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    T4 lysozyme\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". The resulting VSD structures are in good agreement with the consensus model of the resting state VSD and the spin-spin distance histograms from ESR/DEER experiments on \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    T4 lysozyme\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " are accurately reproduced.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_7073366.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Morphology of a Bulk Heterojunction Photovoltaic Cell with Low Donor Concentration<br>Atomistic nonequilibrium molecular dynamics simulations have been used to model the morphology of small-molecule bulk heterojunction films formed by vapor deposition as used in organic photovoltaics. Films comprising C60 and 1, 5, 10, and 50 wt % of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    1,1-bis [4-bis (4-methylphenyl) aminophenyl] cyclohexane\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    TAPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") were compared to films of neat C60. The simulations suggest that if holes can hop between donor molecules separated by as little as 1.2 1.5 nm, then a \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    TAPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " concentration of 5 wt % is sufficient to form a percolating donor network and facilitate charge extraction. The results provide an explanation for why low donor content organic photovoltaics can still have high efficiencies. In addition, the roughness, porosity, and crystallinity of the films were found to decrease with increasing \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    TAPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " content.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_14602641.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Supporting Computational Data for publication: &quot;Engineering Improved \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Ethylene\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Production: Leveraging Systems Biology and Adaptive Laboratory Evolution&quot;<br>Supporting Computational Data for publication: &quot;Engineering Improved \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Ethylene\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Production: Leveraging Systems Biology and Adaptive Laboratory Evolution&quot;bySophie Vaud, Nicole Pearcy, Marko Hanz evac ki, Alexander M. W. Van Hagen, Salah Abdelrazig, Laudina Safo, Muhammad Ehsaan, Magdalene Jonczyk, Thomas Millat Sean Craig, Rajesh Reddy Bommareddy, Pierre-Yves Colin, Jamie Twycross, Paul Dalby, Nigel Minton, Christof M. Ja ger, Dong-Hyun Kim, Jianping Yu, Pin-Ching Maness, Sean Lynch, Carrie Eckert, Alex Conradie, and Samantha J. BryanForce Field parameter files and input files for MD simulations in manuscript.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_13416098.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Investigation of nanotubes as the smart carriers for targeted delivery of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    mercaptopurine\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " anticancer drug<br>\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Mercaptopurine\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    MCP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") is an anticancer drug that is used to treat acute lymphoblastic leukemia. The therapeutic effect of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    mercaptopurine\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " limits its severe side effects like other cytotoxic (anti-cancer) drugs. Nanostructures or nanoparticles can be widely used as potential drug carriers for diagnosis and treatment of cancer. In the current study, the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    boron nitride nanotube\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    BNNT\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    carbon nanotube\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CNT\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") were studied as the delivery carriers of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    MCP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " using the density functional theory (DFT) calculations and molecular dynamics (MDs) simulation studies. To further understand the electronic properties of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    mercaptopurine\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", the partial density of states (PDOS) was calculated. The PDOS results depicted that the electronic features of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    MCP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " do not change after the adsorption on the surface of the nanotubes. More stability of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    MCP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "/\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    BNNT\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " complexes may be attributed to the existence of the intermolecular hydrogen bonds (H-bonds) between the hydrogen atoms of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    MCP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " molecule and the N atoms of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    BNNT\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". The results of the quantum theory of atoms in molecules (QTAIM) confirmed the presence of H-bonds in these complexes. Moreover, MD simulation studies were done in the presence of five drug molecules. The results revealed that the strongest van der Waals (vdW) interaction energy was found between the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    BNNT\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    MCP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " among the studied nanotubes, indicating the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    BNNT\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " is a better nanocarrier than carbon nanotube for delivery of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    MCP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " drug within the biological systems. In general, the obtained results may present helpful information on the nature of the interactions between \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    mercaptopurine\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " anticancer drug with \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    BNNT\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and/or \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CNT\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". Communicated by Ramaswamy H. Sarma</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_3522090.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Ligand-induced Conformational Selection Predicts the Selectivity of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Cysteine Protease Inhibitors\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " - \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ICL\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    IKR\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " complexes MD<br>Supplementary data of &quot;Ligand-induced Conformational Selection Predicts the Selectivity of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Cysteine Protease Inhibitors\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "&quot; paper. This dataset consists of the molecular dynamics simulations trajectory and topology of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Cruzain\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Cathepsin K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Cathepsin L\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " enzymes in noncovalent and covalent complexes with selective ligand against \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Cruzain\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ICL\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "* and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CCL\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Cathepsin K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    IKR\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CKR\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "). We ran five replicate \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    100ns\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       " simulations on each complex, with randomized initial velocities. </div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_6580992.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">research data supporting &quot;Revealing the organization of catalytic sequence-defined oligomers via combined molecular dynamics simulations and network analysis&quot;<br>This repository contains all the data generated and analyzed including the starting structures, the input files, the trajectory files, the output data from cpptraj and network analyses, and in-house scripts used to prepare the network and module files shown in the paper &quot;Revealing the organization of catalytic sequence-defined oligomers via combined molecular dynamics simulations and network analysis&quot; published in Journal of Chemical Information and Modeling (DOI: 10.1021/acs.jcim.2c00101). </div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_12272015.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Glycosylated \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Swiss-model\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " molecular dynamics trajectory of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    SARS-CoV-2 spike\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    glycoprotein\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "<br>Molecular dynamics simulation trajectory. The system is the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    SARS-CoV-2 spike\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    glycoprotein\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " with \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    glycans\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " modeled on using \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GLYCAM-Web\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       ". \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    SARS-CoV2 spike\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    S\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") protein structure A 3D structure of the prefusion form of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    S\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " protein (RefSeq: YP 009724390.1, UniProt: \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    P0DTC2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    SPIKE SARS2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "), based on a Cryo-EM structure (PDB code \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    6VSB\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "), was obtained from the \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    SWISS-MODEL\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " server (swissmodel.expasy.org). The model has 95% coverage (residues 27 to 1146) of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    S\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " protein. The site specific \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    glycans\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " used to model a glycoform representative of the data obtained from the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    S\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    glycoprotein\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " expressed in HEK293 cells, are presented below: Hybrid: DManp 1-6[DManp 1-3]DManp 1-6[DGlcpNAc 1-2DManp 1-3]DManp 1-4DGlcpNAc 1-4DGlcpNAc 1-OH Glycosites: 657 \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    FA2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ": DGlcpNAc 1-2DManp 1-6[DGlcpNAc 1-2DManp 1-3]DManp 1-4DGlcpNAc 1-4[LFucp 1-6]DGlcpNAc 1-OH Glycosites: 149, 165, 331, 343, 616, 1134 A2: DGlcpNAc 1-2DManp 1-6[DGlcpNAc 1-2DManp 1-3]DManp 1-4DGlcpNAc 1-4DGlcpNAc 1-OH Glycosites: 1098 \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    FA2B\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ": DGlcpNAc 1-2DManp 1-6[DGlcpNAc 1-4][DGlcpNAc 1-2DManp 1-3]DManp 1-4DGlcpNAc 1-4[LFucp 1-6]DGlcpNAc 1-OH Glycosites: 74, 282 M5: DManp 1-6[DManp 1-3]DManp 1-6[DManp 1-3]DManp 1-4DGlcpNAc 1-4DGlcpNAc 1-OH Glycosites: 61, 122, 603, 709, 717, 801, 1074 M8: DManp 1-2DManp 1-6[DManp 1-2DManp 1-3]DManp 1-6[DManp 1-2DManp 1-3]DManp 1-4DGlcpNAc 1-4DGlcpNAc 1-OH Glycosites: 234 \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Glycan\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " naming system. In cases where the mass spectrometry data allows for multiple structures, such as \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    FA2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "/\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    FA1B\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    glycan\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " with the least ambiguity was selected. For example, \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    FA2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " was selected over \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    FA1B\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", as the position of the single antennae is ambiguous.Dataset supports the following preprint: 3D Models of glycosylated \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    SARS-CoV-2 spike\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    protein\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " suggest challenges and opportunities for vaccine development. Oliver C. Grant, David Montgomery, Keigo Ito, Robert J. Woods; bioRxiv 2020.04.07.030445; doi: https://doi.org/10.1101/2020.04.07.030445</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_9945170.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Unsupervised Machine Learning for Analysis of Phase Separation in Ternary Lipid Mixture<br>Phase separation in mixed lipid systems has been extensively studied both experimentally and theoretically because of its biological importance. A detailed description of such complex systems undoubtedly requires novel mathematical frameworks that are capable of decomposing and categorizing the evolution of thousands if not millions of lipids involved in the phenomenon. The interpretation and analysis of molecular dynamics (MD) simulations representing temporal and spatial changes in such systems are still a challenging task. Here, we present an unsupervised machine learning approach based on nonnegative matrix factorization called NMFk that successfully extracts latent (i.e., not directly observable) features from the second layer neighborhood profiles derived from coarse-grained MD simulations of a ternary lipid mixture. Our results demonstrate that NMFk extracts physically meaningful features that uniquely describe the phase separation such as locations and roles of different lipid types, formation of nanodomains, and timescales of lipid segregation.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_9788852.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">In-Cell Protein Protein Contacts: Transient Interactions in the Crowd<br>Proteins in vivo are immersed in a crowded environment of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", ions, metabolites, and macromolecules. In-cell experiments highlight how transient weak protein protein interactions promote (via functional quinary structure ) or hinder (via competitive binding or sticking ) complex formation. Computational models of the cytoplasm are expensive. We tackle this challenge with an all-atom model of a small volume of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    E. coli cytoplasm\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " to simulate protein protein contacts up to the \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    5 μs\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       " time scale on the special-purpose supercomputer Anton 2. We use three \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CHARMM\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       "-derived force fields: \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    C22*\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    C36m\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       ", and \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    C36mCU\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " (with CUFIX corrections). We find that both \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    C36m\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    C36mCU\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " form smaller contact surfaces than \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    C22*\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       ". Although CUFIX was developed to reduce protein protein sticking, larger contacts are observed with \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    C36mCU\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " than \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    C36m.\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " We show that the lifespan t of protein protein contacts obeys a power law distribution between 0.03 and 3 s, with 90% of all contacts lasting &lt;1 s (similar to the time scale for downhill folding).</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_12218135.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Nuclear Quantum Effects from the Analysis of Smoothed Trajectories: Pilot Study for \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "<br>Nuclear quantum effects have significant contributions to thermodynamic quantities and structural properties; furthermore, very expensive methods are necessary for their accurate computation. In most calculations, these effects, for instance, zero-point energies, are simply neglected or only taken into account within the quantum harmonic oscillator approximation. Herein, we present a new method, Generalized Smoothed Trajectory Analysis, to determine nuclear quantum effects from molecular dynamics simulations. The broad applicability is demonstrated with the examples of a harmonic oscillator and different states of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". Ab initio molecular dynamics simulations have been performed for ideal gas up to the temperature of \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    5000 K.\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       " Classical molecular dynamics have been carried out for hexagonal ice, liquid \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", and vapor at atmospheric pressure. With respect to the experimental heat capacity, our method outperforms previous calculations in the literature in a wide temperature range at lower computational cost than other alternatives. Dynamic and structural nuclear quantum effects of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " are also discussed.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_5357152.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Molecular Determinants and Pharmacological Analysis for a Class of Competitive Non-transported Bicyclic Inhibitors of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Betaine\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "/\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GABA Transporter BGT1\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ": Modeling Data<br>This archive contains the modeling data for the study &quot;Molecular Determinants and Pharmacological Analysis for a Class of Competitive Non-transported Bicyclic Inhibitors of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Betaine\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "/\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GABA Transporter BGT1\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "&quot; (doi: 10.3389/fchem.2021.736457). The following data sets are available: Induced fit docking results of all mentioned compounds in the study: ifd \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    hBGT1\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " occ clustering all compounds.zip MD simulations of bicyclo-\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GABA\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and compound 1 (\n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    100ns\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       ", 3 replica): MD simulation bicyclo-GABA run1.zip MD simulation bicyclo-GABA run2.zip MD simulation bicyclo-GABA run3.zip MD simulation cmd1 run1.zip MD simulation cmd1 run2.zip MD simulation cmd1 run3.zip A detailed description of the methods is available in the aforementioned publication. The compound numbering in the uploaded files differs from the compound numbering in the mentioned study: study / upload bicyclo-GABA / cmd4 1 / IIa 2 /8-2 3 / 8-3 4a / 7-1 4b / 7-2 4c / 7-3 </div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_7924394.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Molecular Level Understanding of the Free Energy Landscape in Early Stages of Metal Organic Framework Nucleation<br>The assembly mechanism of Metal-Organic Frameworks (MOFs) is controlled by the choice of solvent and the presence of spectator ions. In this paper, we apply enhanced sampling molecular dynamics methods to investigate the role of solvent and ions in the early stages of the synthesis of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    MIL-101\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Cr\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "). \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Microsecond\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       "-long well-tempered metadynamics simulations uncover a rich structural free energy landscape, with secondary building units (SBUs) adopting distinct crystal and noncrystal like configurations. In the presence of ions (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Na+\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    F\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " ), we observe a complex effect on the crystallinity of SBUs. By modulating the interactions between \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    terephthalate\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " linkers and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Cr\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " atoms, ions affect the abundance of crystal-like SBUs, consequently controlling the percentage of defects. Solvent effects are assessed by comparing \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " with \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    N,N-dimethylformamide\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", in which SBU adducts are appreciably more stable and compact. These results shed light on how solvent and ionic strength impact the free energy of the assembly phenomena that ultimately control material synthesis.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_4522359.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">All-atom \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Phospholipid\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " membranes of various sizes and compositions<br>Publication: Ripple-like instability in the simulated gel phase of finite size \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    phosphocholine\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " bilayers Published on: To be published Journal: To be submitted Description: Simulation files of different lipid bilayer made either of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DPPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DPPE\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " or \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DSPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". Different number of lipids per bilayers are provided (given in the name of the file). The average number of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " molecules per lipid is circa 160 \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " molecules per lipid. Complete simulations are provided for two of the systems Simulation details: Software: \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Gromacs (v. 2016.4)\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " Forcefield: \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Charmm36 (v. June 2015)\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " - \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ": \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    TIP3P\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " Thermostat: Nose-hoover (0.4ps, 2 groups) Barostat: Parrinello-Rahman semi-isotropic (2.0ps, 1.0 bar on each axis, 4.5e-5 bar-1) Duration: \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    50 ns\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       " </div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_6503359.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">FORECASTING MOLECULAR DYNAMICS SIMULATIONS OF POLYMER-LIPIDS IN SOLUTION WITH RNNs<br>Files and scripts pertaining to our work: \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GROMACS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " files for the topology (DSPE+PEG.top) and the initial structure of the aggregate (DSPE+PEG EA NPT.gro) \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GROMACS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " topology file for the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ethyl acetate\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " molecule: EA SI.top Scripts to submit the \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GROMACS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " utilities for calculation of the interaction energies are described in README.txt (Subset energy.sh , Interaction energies.sh) Scripts pertaining to PyTorch use and access of methods are described in README.txt (Multiple-run.sh. Job.sh, Pytorch train-model.py) Scripts pertaining to scikit learn access for the Expectation Maximization clustering are described in the README.txt (Job EM.sh, EM Clustering.py) Files with the time series of the potential energy (PE) and interaction energy (IE) of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DSPE\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PEG\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " aggregate with the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ethyl acetate\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " solvent. Series contain 500,000 snapshots taken every 10 fs along the NVT Molecular Dynamics trajectory at \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    300 K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       " and 906.3 kg/m3 density. The molecular solution is in a cubic box of edge length 13.76 nm, containing 16,000 \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ethyl acetate\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " molecules and one aggregate of 4 \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DSPE\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PEG\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    amide\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " macromolecules (224,000 atoms): Data Andrews etal \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DSPE\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PEG\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " 2022.zip ArXiv preprint: https://doi.org/10.48550/arXiv.2203.00151 (JAndrews etal arXiv-doi.pdf) </div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_51760.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Molecular dynamics simulation trajectory of a \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    cationic lipid\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " bilayer: 75/25 mol% \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DMTAP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "/\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DMPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " in 0.1 M \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    NaCl\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "<br>System: Symmetric bilayer of cationic \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DMTAP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    dimyristoyltrimethylammoniumpropane\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", 75 mol-%) and zwitterionic \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DMPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    dimyristoylphosphatidylcholine\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", 25 mol-%) lipids in 0.1 M \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    NaCl\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " solution. Number of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DMPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ": 32. Number of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DMTAP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ": 96. Number of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Na+\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " ions: 10. Number of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Cl-\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " ions: 106. Number of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    waters\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ": 5476. Lipid model: '\n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Berger\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       "' united-atom [Biophys. J. 72 2002 (1997)] with \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DMTAP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " modification by Gurtovenko et al. [Biophys. J. 86 3461 (2004)]. \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " model: \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    SPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " [In Intermolecular Forces, ed. Pullman. 331 (1981)]. Simulation engine: \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Gromacs 3.x\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " [https://removed Trajectory length: \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    110 ns\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       ". Previously equilibrated for: 11 ns. Sampling rate: every 10 ps. Time integration step: 2 fs. Thermodynamic ensemble: NpT. Temperature coupling: 'Berendsen' [J. Chem. Phys. 81 3684 (1984)] with lipids and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " coupled separately at T \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    323 K.\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       " Pressure coupling: 'Berendsen' [J. Chem. Phys. 81 3684 (1984)] with xy and z coupled separately at p 1.0 bar. Electrostatics: PME [J. Chem. Phys. 98 10089 (1993); J. Chem. Phys. 103 8577 (1995)], real-space cutoff at 1.0 nm. Van der Waals: Truncated at 1.0 nm. Constraints: Covalent bond lengths in lipids using \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    LINCS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " [J. Comput. Chem. 18 1463 (1997)], in \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " using \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    SETTLE\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " [J. Comput. Chem. 13 952 (1992)]. Used in publications: [1] Markus S. Miettinen, Andrey A. Gurtovenko, Ilpo Vattulainen, and Mikko Karttunen: &quot;Ion Dynamics in Cationic Lipid Bilayer Systems in Saline Solutions&quot;. J. Phys. Chem. B 113 9226 (2009). DOI: 10.1021/jp810233q. [2] Markus S. Miettinen: &quot;Computational Modeling of Cationic Lipid Bilayers in Saline Solutions&quot;. PhD Thesis. Aalto University School of Science and Technology, Helsinki, Finland. (2010). ISBN 978-952-60-3194-1.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_3901180.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">On calculating the bending modulus of lipid bilayer membranes from buckling simulations<br>Molecular dynamics simulations files that correspond to the buckling simulations method to calculate the bending modulus and the box area fluctuations method to calculate the area compressibility modulus. We performed all molecular dynamics (MD) simulations using the \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GROMACS software (v. 2016.4)\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " and the \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    MARTINI\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " coarse-grained (CG) force field (v. 2.2), using standard simulation parameters. Each file contains GRO, TOP, ITPs, and MDP files that correspond to the equilibration and the production runs. The name of each folder indicates the types of lipids. We simulate single-component lipid bilayers includes: \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DLPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", (14:0-14:0), \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DPPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (16:0-16:0), \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (16:0-18:1), \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DOPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (18:1-18:1), \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PUPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (16:0-22:6), \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DLiPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (18:2-18:2), \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DNPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (24:6-24:6), \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPG\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (16:0-18:1), \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (16:0-18:1), \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPE\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (16:0-18:1), and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DPSM\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (16:0-16:0), and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    lipid\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " mixtures includes : \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DOPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ":\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CHOL\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DPPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ":\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CHOL\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ":\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPE\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DPPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ":\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DLPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ":\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PUPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DNPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ":\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DLPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DPPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ":\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DLiPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ":\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CHOL\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", with different molar ratios specified in the folders names. </div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_3977034.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Divalent cations\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " bind to \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    phosphoinositides\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " to induce ion and isomer specific propensities for nano-cluster initiation in bilayer membranes<br>We report all-atom molecular dynamics simulations of physiologically composed asymmetric bilayers containing \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    phosphoinositides\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " in the presence of monovalent and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    divalent cations\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". We have characterized the molecular mechanism by which these \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    divalent cations\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " interact with \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    phosphoinositides\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Calcium\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " desolvates more readily, consistent with single-molecule calculations, and forms a network of ionic-like bonds that serve as a &quot;molecular glue'' that allows a single ion to coordinate with up to three \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    phosphatidylinositol- (4,5)-bisphosphate\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " lipids. The \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    phosphatidylinositol- (3,5)-bisphosphate\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " isomer shows no such effect and neither does PI (4,5)P2 in the presence of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Mg\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". The resulting network of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Ca\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-mediated lipid-lipid bonds grows to span the entire simulation space and therefore has implications for the lateral distribution of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    phosophoinositides\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " in the bilayer. We observe context-specific differences in lipid diffusion rates, lipid surface densities, and bilayer structure. The molecular-scale delineation of ion-lipid arrangements reported here provides insight into similar nanocluster formation induced by peripheral proteins to regulate the formation of functional signaling complexes on the membrane.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_2224288.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Short-Range Solvation Effects on Chiroptical Properties: A Time-Dependent Density Functional Theory and ab Initio Molecular Dynamics Computational Case Study on Austdiol<br>The description of solvation effects on the chiroptical properties of chiral molecules is still a difficult challenge in the field of computational spectroscopy; this issue is critical in stereochemical characterization, since a reliable assessment of absolute configuration requires high accuracy. The present case study reports the huge effect of solvation on the chiroptical properties of austdiol, a fungal metabolite of known stereochemistry. Standard protocols based on time-dependent density functional theory calculations failed to reproduce its experimental chiroptical properties in methanol. When short-range solvation effects are explicitly considered by means of ab initio molecular dynamics, the correlation between calculated and experimental data is greatly improved because of a better description of the chiral environment around the ketone chromophore, showing that the modeling of subtle solvent-induced perturbations may require the most accurate computational methods.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_7301759.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Data for: Excitatory and inhibitory \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    D-serine\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " binding to the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    NMDA receptor\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "<br>\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    N-methyl-D-aspartate receptors\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    NMDARs\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") uniquely require binding of two different neurotransmitter agonists for synaptic transmission. \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    D-serine\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    glycine\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " bind to one subunit, GluN1, while \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    glutamate\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " binds to the other, GluN2. These agonists bind to the receptor's bi-lobed ligand-binding domains (LBDs), which close around the agonist during receptor activation. To better understand the unexplored mechanisms by which \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    D-serine\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " contributes to receptor activation, we performed \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    multi-microsecond\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       " molecular dynamics simulations of the GluN1/GluN2A LBD dimer with free \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    D-serine\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    glutamate\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " agonists. Surprisingly, we observed \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    D-serine\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " binding to both GluN1 and GluN2A LBDs, suggesting that \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    D-serine\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " competes with \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    glutamate\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " for binding to GluN2A. This mechanism is confirmed by our electrophysiology experiments, which show that \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    D-serine\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " is indeed inhibitory at high concentrations. Although free energy calculations indicate that \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    D-serine\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " stabilizes the closed GluN2A LBD, its inhibitory behavior suggests that it either does not remain bound long enough or does not generate sufficient force for ion channel gating. We developed a workflow using pathway similarity analysis to identify groups of residues working together to promote binding. These conformation-dependent pathways were not significantly impacted by the presence of N-linked \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    glycans\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", which act primarily by interacting with the LBD bottom lobe to stabilize the closed LBD.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_5772140.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Inactive to active transition of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    human Thymidine Kinase 1\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " revealed by Molecular Dynamics simulations<br>The trajectories and input files for the manuscript Inactive to active transition of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    human Thymidine Kinase 1\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " revealed by Molecular Dynamics simulations (https://doi.org/10.1021/acs.jcim.1c01157) ABSTRACT Despite its importance for the nucleoside (and nucleoside prodrug) metabolism, the structure of the active conformation of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    human Thymidine Kinase 1\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    hTK1\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") remains elusive. We perform \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    microsecond\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       " molecular dynamics simulations of the inactive enzyme form bound to a bisubstrate inhibitor that was shown experimentally to activate another TK1-like kinase, \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Thermotoga maritima TK\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    TmTK\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "). Our results are in excellent agreement with the experimental findings for the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    TmTK\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " closed-to-open state transition. We show that the inhibitor induces an increase of the enzyme radius of gyration due to the expansion on one of the dimer interfaces; the structural changes observed, including the active site pocket volume increase, decrease in monomer-monomer buried surface area and of the number of hydrogen bonds (as compared to the inactive enzyme control simulation), show that the catalytically competent (open) conformation of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    hTK1\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " can be assumed in the presence of an activating ligand.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_3885771.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Barnase\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Barstar\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Binding Pathways<br>This is a collection of binding trajectories from the atomistic simulation of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Barnase\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Barstar\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " system presented in Protein protein binding pathways and calculations of rate constants using fully-continuous, explicit-solvent simulations by Saglam and Chong. There is one representative trajectory from each of the five pathways identified. The GRO file here is to be used as a &quot;topology&quot; file, so if your chosen MD viewer or analysis software includes the reference frame from the GRO file, be sure to delete the frame before running your analysis, etc. If you want to analyze just the first frame of a trajectory, load in the GRO/topology file, delete the frame from that file, load in the relevant XTC file and use the first frame from that. Please cite the Saglam and Chong paper if you use these data.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_3120376.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Molecular Dynamics of a \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-Absorbent Nanoscale Material Based on \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Chitosan\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "<br>Although hydrogels have been widely investigated for their use in materials science, nanotechnology, and novel pharmaceuticals, mechanistic details explaining their \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-absorbent features are not well understood. We performed an all-atom molecular dynamics study of the structural transformation of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    chitosan\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " nanohydrogels due to \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " absorption. We analyzed the conformation of dry, nanoscaled \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    chitosan\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", the structural modifications that emerge during the process of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " inclusion, and the dynamics of this biopolymer in the presence of nature s solvent. Two sets of nanoscaled, single-chained \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    chitosan\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " models were simulated: one to study the swelling dependence upon the degree of self-cross-linking and other to observe the response with respect to the degree of protonation. We verified that nanohydrogels keep their ability to absorb \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and grow, regardless of their degree of cross-linking. Noteworthy, we found that the swelling behavior of nanoscaled \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    chitosan\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " is pH-dependent, and it is considerably more limited than that of larger scale hydrogels. Thus, our study suggests that properties of nanohydrogels are significantly different from those of larger hydrogels. These findings might be important in the design of novel controlled-release and targeted drug-delivery systems based on \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    chitosan\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ".</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_7998374.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Complex Behavior of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Phosphatidylcholine Phosphatidic Acid\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Bilayers and Monolayers: Effect of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Acyl\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Chain Unsaturation<br>\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Phosphatidic acids\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PAs\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") have many biological functions in biomembranes, e.g., they are involved in the proliferation, differentiation, and transformation of cells. Despite decades of research, the molecular understanding of how \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PAs\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " affect the properties of biomembranes remains elusive. In this study, we explored the properties of lipid bilayers and monolayers composed of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PAs\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    phosphatidylcholines\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PCs\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") with various \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    acyl\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " chains. For this purpose, the Langmuir monolayer technique and atomistic molecular dynamics (MD) simulations were used to study the miscibility of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PA\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " lipids and the molecular organization of mixed bilayers. The monolayer experiments demonstrated that the miscibility of membrane components strongly depends on the structure of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    hydrocarbon\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " chains and thus on the overall lipid shape. Interactions between \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PA\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " molecules vary from repulsive, for systems containing lipids with saturated and unsaturated \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    acyl\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " tails (strongly positive values of the excess free energy of mixing), to attractive, for systems in which all lipid tails are saturated (negative values of the excess free energy of mixing). The MD simulations provided atomistic insight into polar interactions (formation of hydrogen bonds and charge pairs) in \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PA\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " systems. H-bonding between \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PA\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " monoanions and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PCs\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " in mixed bilayers is infrequent, and the lipid molecules interact mainly via electrostatic interactions. However, the number of charge pairs significantly decreases with the number of unsaturated lipid chains in the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PA\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " system. The \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PA\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " dianions weakly interact with the zwitterionic \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    lipids\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", but their headgroups are more hydrated as compared to the monoanionic form. The \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    acyl\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " chains in all \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PA\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " bilayers are more ordered compared to single-component \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " systems. In addition, depending on the combination of lipids, we observed a deeper location of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PA phosphate\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " groups compared to the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PC phosphate\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " groups, which can alter the presentation of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PAs\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " for the peripheral membrane proteins, affecting their accessibility for binding.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_5226209.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Molecular Dynamics Dataset - Dynamics of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GLP-1R\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " peptide agonist engagement are correlated with kinetics of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    G protein\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " activation<br>Molecular dynamics simulations of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    glucagon-like peptide receptor\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GLP-1R\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") in complex with \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Gs protein\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and four different agonist peptides: \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GLP-1\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    exendin-4\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    oxyntomodulin\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    exendin-P5\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". For each system, the .xtc file is the result of the merging of 4 replicas of \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    500 ns\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       " for a total of \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    2 microseconds\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    8 microseconds\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       " considering all four \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GLP-1R\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " complexes). The original MD trajectories have been stripped of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ions\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    phosphatidylcholine\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " atoms before the upload. The \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CHARMM36\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " force field was used.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_3983526.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Villin Headpiece\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Simulations<br>3 separate simulations each of \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    6µs\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       " concatenated together are contained herein. Each trajectory was run with \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ACEMD\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " using a \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CHARMM 27\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " force field. The \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    proteins\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    2RJY\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") were first solvated to 0.15 mol/L of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    NaCL\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and set in a \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " box modeled as a \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    TIP3P\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " model. \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Proteins\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " were then equilibrated (conjugate gradient minimization) and then heated from the crystal structure to unfold. Folding simulations were run after a subsequent equilibration. While the simulations do not find the native folded state, they sample many partially folded intermediates and offer insight into the folding pathway of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    villin headpiece\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". Simulations included here have had the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ions\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " removed for space considerations and they have been aligned to remove translation and rotation resulting from diffusion.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_20485017.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    EntF*\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " -L<br>Molecular dynamics simulation of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    EntF*\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " peptide, all residues in L form -starting coordinates posre.gro energy minimized and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " + \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ions\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " equilibrated around peptide -topology topol.top force field \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    AMBER99SB-ILDB\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    TIP3P\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " protonation states at pH   3 (E,K protonated) [\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    NaCl\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "]   25 mM \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    disulfide\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " bond generated between residues 14 and 6 -md settings md.mdp time step 2 fs, temperature   \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    298 K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       ", pressure   1 bar \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    1 microsecond\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       " trajectory dry.xtc, dry.gro frame rate 10 ps, \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ions\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " removed, box centered on peptide -analysis files frame rate 1 ps radius of gyration, solvent accessible surface area (hydrophilic parts (backbone + hydrophilic side chains), hydrophobic parts (hydrophobic side chains)) </div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_7305466.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Molecular dynamics simulations of the tripartite interface (Syt1 C2B SNARE Cpx Complex)<br>Synaptic transmission is mediated by an orchestra of presynaptic proteins that precisely control and trigger fusion between synaptic vesicles and the neuron terminal at the active zone upon an action potential. Critical to this process are the neuronal \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    SNAREs\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Soluble N-ethylmaleimide sensitive factor Attachment protein REceptor\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "), the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Ca2+-sensor synaptotagmin\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    activator/regulator complexin\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", and other factors. Here we present the data for molecular dynamics simulations of the tripartite interface bewteen these proteins. </div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_51750.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Molecular dynamics simulation trajectory of a cationic lipid bilayer: 75/25 mol% \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DMTAP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "/\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DMPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " in 1.0 M \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    NaCl\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "<br>System: Symmetric bilayer of cationic \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DMTAP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    dimyristoyltrimethylammoniumpropane\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", 75 mol-%) and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    zwitterionic DMPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    dimyristoylphosphatidylcholine\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", 25 mol-%) lipids in 1.0 M \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    NaCl\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " solution. Number of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DMPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ": 32. Number of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DMTAP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ": 96. Number of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Na+\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " ions: 96. Number of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Cl-\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " ions: 192. Number of waters: 5304. Lipid model: '\n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Berger\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       "' united-atom [Biophys. J. 72 2002 (1997)] with \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DMTAP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " modification by Gurtovenko et al. [Biophys. J. 86 3461 (2004)]. \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " model: \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    SPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " [In Intermolecular Forces, ed. Pullman. 331 (1981)]. Simulation engine: \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Gromacs 3.x\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " [https://removed Trajectory length: \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    110 ns.\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       " Previously equilibrated for: \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    11 ns\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       ". Sampling rate: every 10 ps. Time integration step: 2 fs. Thermodynamic ensemble: NpT. Temperature coupling: 'Berendsen' [J. Chem. Phys. 81 3684 (1984)] with lipids and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " coupled separately at T \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    323 K.\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       " Pressure coupling: 'Berendsen' [J. Chem. Phys. 81 3684 (1984)] with xy and z coupled separately at p 1.0 bar. Electrostatics: PME [J. Chem. Phys. 98 10089 (1993); J. Chem. Phys. 103 8577 (1995)], real-space cutoff at 1.0 nm. Van der Waals: Truncated at 1.0 nm. Constraints: Covalent bond lengths in \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    lipids\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " using \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    LINCS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " [J. Comput. Chem. 18 1463 (1997)], in water using \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    SETTLE\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " [J. Comput. Chem. 13 952 (1992)]. Used in publications: [1] Markus S. Miettinen, Andrey A. Gurtovenko, Ilpo Vattulainen, and Mikko Karttunen: &quot;Ion Dynamics in Cationic Lipid Bilayer Systems in Saline Solutions&quot;. J. Phys. Chem. B 113 9226 (2009). DOI: 10.1021/jp810233q. [2] Markus S. Miettinen: &quot;Computational Modeling of Cationic Lipid Bilayers in Saline Solutions&quot;. PhD Thesis. Aalto University School of Science and Technology, Helsinki, Finland. (2010). ISBN 978-952-60-3194-1.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_4196842.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Simulations of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " membranes of various shapes<br>Simulations of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " membranes in 1) budded shape with Gaussian curvature and 2) wave shape with mean curvature but no Gaussian curvature and with two wavelengths (&quot;long&quot; and &quot;short&quot;), and 3) a freely fluctuating membrane. Simulations were performed using \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GROMACS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " simulation engine (DOI: 10.1016/j.softx.2015.06.001 ) and the \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Martini 2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " force field ( DOI:10.1021/jp071097f ). The budded and wavy geometries generated using \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    BUMPy\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " ( DOI: 10.1021/acs.jctc.8b00765 ) are maintained using restrained dummy particles that repel lipid acyl chains All input and output files are provided for each simulation, and the cpt files allow to extend the simulations. Additionally, topologies (in the TOP.tar archive) and simulation parameters (md.mdp) are provided. Some of the simulations have been used in DOI: 10.1101/2020.09.23.310086 and DOI: 10.26434/chemrxiv-2021-6wksn</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_4971801.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Data from: Predicting allosteric mutants that increase activity of a major antibiotic resistance enzyme<br>The CTX-M family of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    beta lactamases\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " mediate broad-spectrum antibiotic resistance and present in the majority of drug-resistant gram-negative bacteria infections worldwide. Allosteric mutations that increase catalytic rates of these drug resistance enzymes have been identified in clinical isolates but are challenging to predict prospectively. We have used molecular dynamics simulations to predict allosteric mutants increasing CTX-M9 drug resistance, experimentally testing top mutants using multiple antibiotics. Purified enzymes show an increase in catalytic rate and efficiency, while mutant crystal structures show no detectable changes from wild-type CTX-M9. We hypothesize that increased drug resistance results from changes in the conformational ensemble of an \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    acyl\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " intermediate in hydrolysis. Machine-learning analyses on top-scoring mutants identify changes to the binding-pocket conformational ensemble by which these allosteric mutations transmit their effect. These findings show how molecular simulation can predict how allosteric mutations alter active-site conformational equilibria to increase catalytic rates and thus resistance against common clinically used antibiotics.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_7865144.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">The Necessity of d Thr in the New Antibiotic \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Teixobactin\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ": A Molecular Dynamics Study<br>Ever since the discovery of the new antibiotic \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    teixobactin\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", studies of its structure activity relationships have never ceased. Here we focus on the chirality of the threonine (Thr) residue, which belongs to the ring motif of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    teixobactin\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and plays an important role in the binding with its target, lipid II molecule. We study the structural propensity of the open and closed ring motifs with different chiral Thr residues as well as the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    teixobactin\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " lipid II complex with the help of molecular dynamics simulations. Our results suggest that different chiralities lead to different NH orientations of Thr with respect to the ring plane. Only in the closed ring motif with d-Thr is a favored binding cavity achievable with all four NH groups facing the same side of the ring plane. This study develops a deeper understanding of the binding mechanism of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    teixobactin\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and lipid II and is expected to be beneficial to new \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    teixobactin\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-based drug design.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_3688506.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Data Set to '\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Sodium\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-induced population shift drives activation of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    thrombin\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "'<br>The data set 10.5281/zenodo.3688506 contains the raw data used for the preparation of the manuscript '\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Sodium\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-induced population shift drives activation of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    thrombin\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "' (doi:10.1038/s41598-020-57822-0). To limit the required storage space, the trajectories are deposited only as coordinates of the analysed loop. The data set is separated into following parts: X-ray: Data used in the analysis of the PDB structures of thrombin, including a list of the PDB IDs, results of the PCA and values of the introduced features in the structures. cMD: Input parameters, topologies and starting coordinates of classical MD simulation based on different PDB structures (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    3bei\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    3lu9\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " with \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Na+\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " ions and without \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Na+\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " ions), resulting trajectories, projection on X-ray PCA and values of the introduced features during the simulations. TMD: Input parameters, topologies and starting coordinates of the targeted MD simulations, resulting trajectories, projection on X-ray PCA and RMSD values. Seeded-Simulations: Input parameters, topologies and starting coordinates of classical MD simulations that are started from cluster representatives of the TMD simulations and resulting trajectories (3*100 simulations, each lasting \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    200 ns\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       "), separated into simulations with \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Na+\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and without \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Na+\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", additionally values of the introduced features during the simulations. MSM: For both systems (with and without \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Na+\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "), files are provided that can be loaded with pyemma (TICA objects, kmeans clusterings, calculated implied timescales and MSMs); additionally TICs of the seeded simulations, psi-dihedrals of the seeded simulations, RMSD values of the E and E* states and representative structures of the E and E* states A more detailed description of the included files can be found in the README files in each folder.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_21263347.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Tilted State Population of Antimicrobial Peptide \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PGLa\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Is Coupled to the Transmembrane Potential<br>Cationic antimicrobial peptide \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PGLa\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " gets into close contact with the anionic bacterial cell membrane, facilitating cross-membrane transport phenomena and membrane disruption depending on the concentration. The mechanisms of action are closely associated with the tilted insertion geometry of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PGLa\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". Therefore, we aimed to understand the interaction between the transmembrane potential (TMP) and the orientation of the membrane-bound \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PGLa\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " helix. Molecular dynamics simulations were performed with TMP, and we found that the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PGLa\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " tilt angle relative to the membrane is coupled with the TMP. Elevated TMP increases the population of the tilted state. We observed positive feedback between the tilt angle and the TMP, which occurs due to the electrostatic interaction between the peptidic helix and the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Na+\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " cations at the membrane \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " interface. These TMP coupled phenomena can contribute to understanding the direct antimicrobial and adjuvant effects of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PGLa\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " in combination with regular antibiotics.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_21285264.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Supporting material for publication: The adaptability of the ion binding site by the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Ag\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (I) / \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Cu\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (I) \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    periplasmic chaperone\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    SilF\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "<br>Supporting material for publication: The adaptability of the ion binding site by the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Ag\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (I) / \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Cu\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (I) \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    periplasmic chaperone\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    SilF\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " by Ryan M. Lithgo, Marko Hanzevacki, Gemma Harris, Jos J. A. G. Kamps, Ellie Holden, Justin LP Benesch, Christof M. Ja ger, Anna Croft, Jonathan L. Hobman, Andrew Quigleybcd, Stephen B. Carr, David J. Scott Computational supporting data consisting of: - Molecular dynamics and QM/MM input, parameter, coordinates, and parameter files - Force field parameterisation files - run-scripts for setup and production runs - coordinate output files for QM/MM calcuations - analysis scripts and results</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_7107608.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Data set containing the energy landscapes for GPO and GPP \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    tropocollagen\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " models under pulling forces<br>Energy landscapes (databases of minima and transition states) for GPO and GPP repeat \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    collagen\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " models under constant pulling forces as explored with \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    OPTIM\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PATHSAMPLE\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " with an \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    AMBER\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " force field. The systems are seven GPO or GPP per chain capped with ACE and NME. The forces applied are 0 pN (F0), 10 pN (F1), 50 pN (F2), 100 pN (F3), 250 pN (F4), 500 pN (F5) and 750 pN (F6). The folders contains numerous analysis scripts and graphs. Most of these assume python with numpy and pandas, as well as cpptraj from \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    AMBERTools\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       ".</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_6980700.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Molecular dynamics simulation based analysis of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    celecoxib-polymer\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " interactions<br>This dataset contains scripts and coordinate files for running and analysing molecular dynamics simulations to investigate \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    celecoxib-polymer\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " interactions in aqueous solution. Trajectory files (stripped of water and ions) are also included. The associated study is described in: &quot; Comparative analysis of drug-salt-polymer interactions by experiment and molecular simulation improves biopharmaceutical performance&quot;, Sumit Mukesh, Goutam Mukherjee, Ridhima Singh, Nathan Steenbuck, Carolina Demidova, Prachi Joshi, Abhay T. Sangamwar, Rebecca C. Wade, submitted.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_12168798.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Dataflow programming for the analysis of molecular dynamics with \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    AViS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       ", an analysis and visualization software application<br>The study of molecular dynamics simulations is largely facilitated by analysis and visualization toolsets. However, these toolsets are often designed for specific use cases and those only, while scripting extensions to such toolsets is often exceedingly complicated. To overcome this problem, we designed a software application called \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    AViS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " which focuses on the extensibility of analysis. By utilizing the dataflow programming (DFP) paradigm, algorithms can be defined by execution graphs, and arbitrary data can be transferred between nodes using visual connectors. Extension nodes can be implemented in either Python, C++, and Fortran, and combined in the same algorithm. \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    AViS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " offers a comprehensive collection of nodes for sophisticated visualization state modifications, thus greatly simplifying the rules for writing extensions. Input files can also be read from the server automatically, and data is fetched automatically to improve memory usage. In addition, the visualization system of \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    AViS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " uses physically-based rendering techniques, improving the 3D perception of molecular structures for interactive visualization. By performing two case studies on complex molecular systems, we show that the DFP workflow offers a much higher level of flexibility and extensibility when compared to legacy workflows. The software source code and binaries for Windows, MacOS, and Linux are freely available at https://avis-md.github.io/.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_4012224.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Equilibration simulations of TM domains of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Bcl-2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " proteins \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Mcl-1\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Bok\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Bax\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "<br>Input files and final structures (after \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    1 microsecond\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       ") of simulations of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Mcl-1\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Bax\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Bok\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " transmembrane domains (TMDs) in a \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " membrane. \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CHARMM36\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " parameters are used for lipids and proteins and the \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CHARMM\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       "-specific \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    TIP3P\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " model for \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". The systems were set up using \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CHARMM-GUI\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       ", and simualted using \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GROMACS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " with the simulation parameters recommended for \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CHARMM\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " (see md.mdp). The simulations are described in detail in the paper [1]. Briefly, the final structures, given as .gro files here, were coarse-grained and used in high-throughput dimerization analyses. The dimer structures were clustered, and the most promising structures were fine-grained, and their stability was evaluated using atomistic simulations. [1] ADD</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_8292209.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Interdependent Electronic Structure, Protonation, and Solvatization of Aqueous \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    2 Thiopyridone\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "<br>\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    2-Thiopyridone\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    2-TP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "), a common model system for excited-state proton transfer, has been simulated in aqueous solution with ab initio molecular dynamics. The interplay of electronic structure, protonation, and solvatization is investigated by comparison of three differently protonated molecular forms and between the lowest singlet and triplet electronic states. An interdependence clearly manifests in the mixed-character T1 state for the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    2-TP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " form, systematic structural distortions of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    2-mercaptopyridine\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    2-MP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") form, and photobase protolysis of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    2-TP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " form, in the aqueous phase. In comparison, simplified continuum models for the solvatization are found to be significantly inaccurate for several of the species. To facilitate future computational studies, we therefore present a minimal representative solvatization complex for each stable form and electronic state. Our findings demonstrate the importance of explicit solvatization of the compound and sets the stage for including it also in future studies.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_6494975.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Using Intrinsic Surfaces To Calculate the Free-Energy Change When Nanoparticles Adsorb on Membranes<br>A reaction coordinate that can be used when investigating binding to dynamical surfaces with molecular dynamics is introduced. This coordinate measures the distance between the adsorbate and an isocontour in a density field. Furthermore, the coordinate is continuous so simulation biases that are a function of this coordinate can be added to the Hamiltonian to increase the rate of adsorption/desorption. The efficacy of this new coordinates is demonstrated by performing metadynamics simulations to measure the strength with which a hydrophilic nanoparticle binds to a lipid bilayer. An investigation of the binding mechanism that is performed using the coordinate demonstrates that the lipid bilayer undergoes a series of concerted changes in structure as the nanoparticle binds.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_5021365.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">MD trajectories for &quot;Communication Breakdown: Dissecting the COM Interfaces between the Subunits of Nonribosomal Peptide Synthetases&quot;<br>This dataset contains \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Amber\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " MD trajectories for the MD simulations described in the manuscript &quot;Communication Breakdown: Dissecting the COM Interfaces between the Subunits of Nonribosomal Peptide Synthetases&quot; by Christopher D. Fage, Simone Kosol, Matthew Jenner, Carl O ster, Angelo Gallo, Milda Kaniusaite, Roman Steinbach, Michael Staniforth, Vasilios G. Stavros, Mohamed A. Marahiel, Max J. Cryle, and Jo zef R. Lewandowski published in ACS Catalysis (https://doi.org/10.1021/acscatal.1c02113). If you use these data please cite the original manuscript (follow the manuscript DOI for the final citation, which was not available at the time of publishing this data set). To reduce their size the trajectories were stripped of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ions\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". Only frames every 1 ns or 5 ns were saved. Please see the Supporting Information of the source manuscript for the conditions for the simulations. </div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_3665677.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Project files provided as supporting information to the manuscript &quot;Ligand-protein interactions in \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    lysozyme\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " investigated through a dual-resolution model&quot;<br>README file for the project files provided as supporting information to the manuscript &quot;Ligand-protein interactions in \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    lysozyme\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " investigated through a dual-resolution model&quot; February 12, 2020 Authors: Raffaele Fiorentini, Kurt Kremer and Raffaello Potestio   Overview The dataset is organised in three (compressed) subfolders (see the tree diagrams in each section): - annihilation - decoupling - density The figure deltaG binding ann dec comparison.png shows the results of binding free energy calculations comparing the values obtained both for annihilation and decoupling. The figure deltaG binding annih \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    gromacs\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " espp.png displays the results for Binding FE, comparing the values obtained in \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GROMACS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ESPResSo++\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       ". The README.pdf file contains detailed information about these folders and their content.   The &quot;annihilation&quot; folder contains all results concerning the calculation of binding free energy in case of annihilation and it is divided in two parts: - complex - ligand In &quot;complex&quot; are reported the results of Ligand-Protein FE both in \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ESPResSo++\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GROMACS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       ". All simulations are fully-atomistic. In &quot;ligand&quot; are reported the results of ligand solvation free energy both in \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ESPResSo++\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GROMACS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       ". All simulations are fully-atomistic.   The &quot;decoupling&quot; folder contains all results concerning the calculation of binding free energy in case of decoupling and it is divided in three parts: - complex-DualRes - complex-FullyAT - ligand In &quot;complex-DualRes&quot; are reported the results of Ligand-Protein FE only in \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ESPResSo++\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GROMACS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " cannot do decoupling). The system is simulated in Dual-Resolution. It is possible to find the trajectory files in the sub-directories &quot;lambdaindex-0&quot; and &quot;lambdaindex-30&quot;. In &quot;complex-fullyAT&quot; are reported the results of Ligand-Protein FE only in \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ESPResSo++\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       ". The system simulated is fully-atomistic. It is possible to find the trajectory file in the sub-directories &quot;lambdaindex-0&quot; and &quot;lambdaindex-30&quot;. In &quot;ligand&quot; are reported the results of ligand solvation free energy only in \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ESPResSo++\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       ". All simulations are fully-atomistic. It is possible to find the trajectory file in the sub-directories &quot;lambdaindex-0&quot; and &quot;lambdaindex-20&quot;.   The &quot;density&quot; folder contains the data for the tuning of the c parameter of the steric repulsion among residues. This parameter is tuned so that the water density attains the value computed in all-atom simulations.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_6879091.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Molecular dynamics simulation trajectories of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    HIV protein\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    gp120\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " in complex with antibody \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    VRC01\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and 30 of its Ala mutants<br>This data set accompanies the publication by S. Conti, E. Lau, and V. Ovchinnikov entitled &quot;On the rapid calculation of binding affinities for antigen and antibody design and affinity maturation simulations&quot;, to be published in the MDPI journal Antibodies. It contains molecular dynamics simulation trajectories of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    HIV protein\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    gp120\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " in complex with antibody \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    VRC01\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and 30 of its Ala mutants, as described in the paper. The format of the trajectory files is \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CHARMM\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       "-compatible dcd. The files can be visualized with the program \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Visual Molecular Dynamics\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    VMD\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       ") (see paper by Humphrey et al. 1996, J. Molec. Graphics). The accompanying file &quot;view&quot; is a tcl-based script for \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    VMD\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " that can be executed in the Linux environment using: &quot;vmd -e view&quot;, which will display the trajectory of the mutant specified by editing the first noncomment line of the script. </div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_14498668.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">BFEE2: Automated, Streamlined, and Accurate Absolute Binding Free-Energy Calculations<br>Accurate absolute binding free-energy estimation in silico, following either an alchemical or a geometrical route, involves several subprocesses and requires the introduction of geometric restraints. Human intervention, for instance, to define the necessary collective variables, prepare the input files, monitor the simulation, and perform post-treatments is, however, tedious, cumbersome, and prone to errors. With the aim of automating and streamlining free-energy calculations, especially for nonexperts, version 2.0 of the binding free energy estimator (BFEE2) provides both standardized alchemical and geometrical workflows and obviates the need for extensive human intervention to guarantee complete reproducibility of the results. To achieve the largest gamut of protein ligand and, more generally, of host guest complexes, BFEE2 supports most academic force fields, such as \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CHARMM\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Amber\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    OPLS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       ", and \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GROMOS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       ". Configurational files are generated in the \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    NAMD\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Gromacs\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " formats, and all the post-treatments are performed in an automated fashion. Moreover, convergence of the free-energy calculation can be monitored from the intermediate files generated during the simulation. All in all, BFEE2 is a foolproof, versatile tool for accurate absolute binding free-energy calculations, assisting the end-user over a broad range of applications.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_5025392.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Data from: Antibiotic uptake across gram-negative outer membranes: better predictions towards better antibiotics<br>Crossing the gram-negative bacterial membrane poses a major barrier to antibiotic development, as many small molecules that can biochemically inhibit key bacterial processes are rendered microbiologically ineffective by their poor cellular uptake. The outer membrane is the major permeability barrier for many drug-like molecules, and the chemical properties that enable efficient uptake into mammalian cells fail to predict bacterial uptake. We have developed a computational method for accurate prospective prediction of outer-membrane uptake of drug-like molecules, which we combine with a new medium-throughput experimental assay of outer membrane vesicle swelling. Parallel molecular dynamics simulations of compound uptake through \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    E. coli OmpF\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " are used to successfully and quantitatively predict experimental permeabilities measured via either outer membrane swelling or via prior \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    liposome\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " swelling measurements. These simulations are analyzed using an inhomogeneous solubility-diffusion model to yield predictions of permeability. For most polar molecules we test, outer membrane permeability also correlates well with whole-cell uptake. The ability to accurately predict and measure outer-membrane uptake of a wide variety of small molecules will enable simpler determination of which molecular scaffolds and which derivatives are most promising prior to extensive chemical synthesis. It will also assist in formulating a more systematic understanding of the chemical determinants of outer-membrane permeability.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing osf_a43z2.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Large-scale motions of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    hGBP1\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "<br>\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Guanylate binding proteins\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GBPs\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") belong to the dynamin-related superfamily and exhibit various functions in the fight against infections. The functions of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    human guanylate binding protein 1\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    hGBP1\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") are tightly coupled to \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GTP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " hydrolysis and dimerization. Despite known crystal structures of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    hGBP1\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " monomer and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GTPase\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " domain dimer, little is known about the dynamics of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    hGBP1\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". To gain a mechanistic understanding of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    hGBP1\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", we performed sub-millisecond multi-resolution molecular dynamics (MD) simulations of both the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    hGBP1\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " monomer and dimer. We found that \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    hGBP1\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " is a highly flexible protein that undergoes a hinge motion similar to the movements observed for other dynamin-like proteins. Another large-scale motion was observed for the C-terminal helix 13, providing a molecular view for the 13 13 distances previously reported for the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    hGBP1\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " dimer. The MD simulations were performed using \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GROMACS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       ". The resulting trajectories along with the input files are provided. For the calculation of the Markov state models, PyEMMA was used; the Jupyter notebook that we developed in this context for the analysis of he \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    hGBP1\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " motions is supplied.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_5079480.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">The Shape of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " in \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Zeolites\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and its Impact on Epoxidation Catalysis<br>The Shape of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " in \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Zeolites\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and its Impact on Epoxidation Catalysis Daniel T. Bregante,1 Matthew Chan,1 Jun Zhi Tan,1 E. Zeynep Ayla,1 Christopher P. Nicholas,2,3 Diwakar Shukla,1 and David W. Flaherty1,* 1Department of Chemical and Biomolecular Engineering, University of Illinois at Urbana-Champaign, Urbana, IL 61802 2Exploratory Materials and Catalysis Research, Honeywell UOP, Des Plaines, IL 60016 3C2P Sciences L3C, Evanston, IL 60202 *Corresponding Author: dwflhrty@illinois.edu Molecular dynamics simulations for \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    zeolite\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " framework studied in &quot;The Shape of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " in \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Zeolites\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and its Impact on Epoxidation Catalysis&quot;. Repository contains the last \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    100 nanoseconds\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       " of classical molecular dynamics equilibration, ab initio molecular dynamics trajectories, initial and final simulated \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    zeolite\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " structures, and scripts used for analyzing MD trajectories. </div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_11808522.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Inclusion Complexation of Organic Micropollutants with \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Cyclodextrin\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "<br>Recently, \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    cyclodextrin\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " ( \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CD\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " )-based polymers with enhanced adsorption kinetics and high removal capacity of organic micropollutants (OMPs) and uptake rates have been synthesized and tested experimentally. Although the exact physical chemical mechanisms via which these polymers capture the various types of OMPs are not yet fully understood, it is suggested that the inclusion complex formation of OMPs with \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CD\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " is very important. In this study, the inclusion complex formation of OMPs with \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CD\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " in an aqueous solution is investigated by using the well-established attach pull release method in force field-based molecular dynamics simulations. A representative set of OMPs is selected based on the measured occurrences in surface and ground waters and the directives published by the European Union. To characterize the formation of the inclusion complex, the binding free energies, enthalpies, and entropies are computed and compared to experimental values. It is shown that computations using the \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    q4md-CD\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       "/\n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GAFF\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       "/\n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Bind3P\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " force field combination yield binding free energies that are in reasonable agreement with the experimental results for all OMPs studied. The binding enthalpies are decomposed into the main contributing interaction types. It is shown that, for all studied OMPs, the van der Waals interactions are favorable for the inclusion complexion and the hydrogen bond formation of the guest with the solvent and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CD\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " plays a crucial role in the binding mechanism. Our findings show that MD simulations can adequately describe the inclusion complex formation of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CD\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " with OMPs, which is the first step toward understanding the underlying mechanisms via which the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CD\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-based polymers capture OMPs.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_800228.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Insight into the Intermolecular Recognition Mechanism between \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Keap1\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    IKK\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Combining Homology Modelling, Protein-Protein Docking, Molecular Dynamics Simulations and Virtual Alanine Mutation<br>Degradation of certain proteins through the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ubiquitin\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    proteasome\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " pathway is a common strategy taken by the key modulators responsible for stress responses. \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Kelch-like ECH-associated protein-1\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Keap1\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "), a substrate adaptor component of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Cullin3\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Cul3\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") -based \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ubiquitin E3 ligase\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " complex, mediates the ubiquitination of two key modulators, \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    NF-E2-related factor 2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Nrf2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    I B kinase\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    IKK\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " ), which are involved in the redox control of gene transcription. However, compared to the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Keap1\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Nrf2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " protein-protein interaction (PPI), the intermolecular recognition mechanism of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Keap1\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    IKK\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " has been poorly investigated. In order to explore the binding pattern between \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Keap1\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    IKK\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " , the PPI model of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Keap1\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    IKK\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " was investigated. The structure of human \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    IKK\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " was constructed by means of the homology modeling method and using reported crystal structure of Xenopus laevis \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    IKK\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " as the template. A protein-protein docking method was applied to develop the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Keap1\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    IKK\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " complex model. After the refinement and visual analysis of docked proteins, the chosen pose was further optimized through molecular dynamics simulations. The resulting structure was utilized to conduct the virtual alanine mutation for the exploration of hot-spots significant for the intermolecular interaction. Overall, our results provided structural insights into the PPI model of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Keap1\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    IKK\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and suggest that the substrate specificity of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Keap1\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " depend on the interaction with the key tyrosines, namely Tyr525, Tyr574 and Tyr334. The study presented in the current project may be useful to design molecules that selectively modulate \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Keap1\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". The selective recognition mechanism of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Keap1\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " with \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    IKK\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " or \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Nrf2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " will be helpful to further know the crosstalk between \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    NF- B\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Nrf2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " signaling. </div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_51185.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " @ \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    310K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       ", 450 mM of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CaCl 2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Charmm36\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " with default \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Charmm\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " ions<br>Simulations of a \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " bilayer with 450 mM of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CaCl 2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". The goal was to study the effect of salt on the order parameters of the lipid head group and the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    glycerol\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " backbone for the NMRlipids project, see https://removed for more information. A \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " bilayer consisting of 200 lipids (100 per leaflet) is simulated in the presence of 450 mM \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CaCl 2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". The \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Charmm36\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " model [1] is employed for lipids, the \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Charmm\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " compatible variant of the \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    tip3p\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " model for \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", and the default \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Charmm\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " ion parameters (type CAL) for \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CaCl 2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". The \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Charmm36\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " force field parameters were obtained from https://removed ff.shtml The files are in \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GROMACS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " format. Trajectory (.xtc) is \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    100 ns\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       " long with data saved every 100 ps. These data were collected after 1900 ns of simulation Additionally, the initial structure (.gro), topology (.top), index file (.ndx), simulation paremeter file (.mdp), binary run input file for \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GROMACS v. 5.0 &gt;\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " (.tpr) and the energy output file (.edr) are provided. [1] Update of the \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CHARMM\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " All-Atom Additive Force Field for Lipids: Validation on Six Lipid Types Jeffery B. Klauda, Richard M. Venable, J. Alfredo Freites, Joseph W. O Connor, Douglas J. Tobias, Carlos Mondragon-Ramirez, Igor Vorobyov, Alexander D. MacKerell, Jr., and Richard W. Pastor The Journal of Physical Chemistry B 2010 114 (23), 7830-7843 DOI: 10.1021/jp101759q</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_6349893.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Molecular dynamics trajectories of protein folding<br>Molecular dynamics trajectories of protein folding are deposited for educational purposes. Currently, the following trajectories are available: \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Chignolin\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (five independent NVT simulations up to \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    1.5 micro-sec\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       "): 'movie.pse' is a \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PyMOL\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " session file of MD trajectories. 'movie.mp4' is a movie file that shows you how a protein folds during simulation. xtc files (\n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Gromacs\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " compressed format) and corresponding tpr files. gro files for movie making Computational setting \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Amber ff99SB-ILDN\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " for protein (Lindorff-Larsen, K. et al. Improved side-chain torsion potentials for the \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Amber ff99SB\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " protein force field. Proteins 78, 1950 1958 (2010)) \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    TIP3P\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " model 0.1 M salt concentration NVT ensemble at \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    300 K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       " with V-rescale thermostat (Bussi, G., Donadio, D. &amp; Parrinello, M. Canonical sampling through velocity rescaling. J. Chem. Phys. 126, 014101 (2007)) Time step : 2 fs \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    gromacs-2020.6\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " </div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_6521230.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Dataset for the Manuscript: Structural and mechanistic insights into the cleavage of clustered \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    O-glycan\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " patches-containing \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    glycoproteins\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " by \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    mucinases\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " of the human gut (in revision)<br>This dataset provides the classical and QM/MM MD simulation trajectory data to the manuscript: Structural and mechanistic insights into the cleavage of clustered \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    O-glycan\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " patches-containing \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    glycoproteins\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " by \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    mucinases\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " of the human gut The data set contains classical MD simulations of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    AM0627\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " with three substrate \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    peptides\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " P1, P2, P9, and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    BT4244\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " with \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    glycopeptides\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", as well as QM/MM metadynamics simulations for our manuscript. PDB files for Figures 4,5 and Figure S5-8,10 are also included.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_11808396.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">(PART 1 OF 5) 9 molecular dynamics simulations (\n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    500ps\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       " 3 x 3 replicates) of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    coronavirus 2019-nCoV protease\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " unrefined crystal structure in complex with 3 different conformations of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    lopinavir\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ".<br>Molecular dynamics simulations (\n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    500 ps\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       " MD at \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    310 K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       ") of the unrefined crystal structure by Prof. Yang group from ShanghaiTech of novel \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    coronavirus 2019-nCoV protease\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Mpro in complex with the ligand \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    lopinavir\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (see note below). The starting positions of the ligand (LP1, LP2, LP3) derive from docking experiments.The suffixes (a, b, c) on the filenames represent the replicates of the same MD simulation using random initial velocities.The archives contain also an heatmap representing the protein-ligand contact frequencies. Green-boxed occurrences in the heatmap represent hydrogen bond occurrences during the simulation.Note: The refined crystal structure of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    protease\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " of our partner Prof. Yang group from ShanghaiTech is oficially released: https://removed 1 of 5:XHD LC1aXHD LC1b+ images of the 3 docked starting positions</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_9956084.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Evidence of a Low High Density Turning Point in Liquid \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " at Ordinary Temperature under Pressure: A Molecular Dynamics Study<br>\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " has a fundamental role in important processes spanning a wide range of pressure and temperature conditions. Knowledge of structural, dynamic and thermodynamic properties of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " at nonstandard conditions is a primary concern since interest in astronomical, geological, and technological processes is continuously growing. Molecular dynamics simulations allow us to study thermodynamic conditions that require sophisticated techniques and instruments, while at the same time offering the interpretation of properties at the atomic level. It is established that the behavior of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " is strongly affected by the temperature and pressure conditions, determining the existence of low and high density regimes. For the first time, a thermodynamic property, isothermal compressibility, has been adopted to detect the low high density turning point at ambient temperature in liquid \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " due to pressure. Molecular dynamics simulations have been performed with five three-site models, allowing us to characterize the complexity of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " nature at these conditions at the atomic level.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_4541035.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Assessing the Efficacy of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Poly (N isopropylacrylamide)\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " for Drug Delivery Applications Using Molecular Dynamics Simulations<br>All-atom molecular dynamic simulations (AA-MD) are performed for aqueous solutions of hydrophobic drug molecules (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    phenytoin\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") with model polymer excipients, namely, (1) \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    N-isopropylacrylamide\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    pNIPAAm\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "), (2) \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    pNIPAAm-co-acrylamide\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Am\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "), and (3) \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    pNIPAAm-co-dimethylacrylamide\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DMA\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "). After validating the force field parameters using the well-known lower critical solution behavior of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    pNIPAAm\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", we simulate the polymer drug complex in \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and its behavior at temperatures below (\n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    295 K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       ") and above the LCST (\n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    310 K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       "). Using radial distribution functions, we find that there is an optimum comonomer molar fraction of around 20 30% \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DMA\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " at which interaction with \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    phenytoin\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " drug molecules is strongest, consistent with recent experimental findings. The results provide evidence that molecular simulations are able to provide guidance in the optimization of novel polymer excipients for drug release.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_5247880.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">\n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    OpenMM 7\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       ": Rapid development of high performance algorithms for molecular dynamics<br>\n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    OpenMM\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " is a molecular dynamics simulation toolkit with a unique focus on extensibility. It allows users to easily add new features, including forces with novel functional forms, new integration algorithms, and new simulation protocols. Those features automatically work on all supported hardware types (including both CPUs and GPUs) and perform well on all of them. In many cases they require minimal coding, just a mathematical description of the desired function. They also require no modification to \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    OpenMM\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " itself and can be distributed independently of \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    OpenMM\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       ". This makes it an ideal tool for researchers developing new simulation methods, and also allows those new methods to be immediately available to the larger community.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_1009607.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Set simulations small pure bilayers (72 lipids) using \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    charmm36\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " ff in \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    gromacs\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DPPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ")<br>Collection simulations of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DPPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " bilayers in \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    gromacs\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " using the \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    charmm36\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " force field. The list of systems describing their particular simulation conditions can be found below: 1) \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DPPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " 72 \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    325K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    500ns\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       ") 2) \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DPPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " 72 \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    310K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       " rmcomm leaflets (\n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    500ns\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       ") 3) \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DPPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " 72 \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    310K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       " rmcomm leaflets low hydration (\n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    500ns\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       ") 4) \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " 72 \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    310K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    500ns\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       ") 5) \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " 72 \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    310K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       " rmcomm leaflets (\n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    500ns\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       ") 6) \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " 72 \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    310K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       " rmcomm leaflets low hydration (\n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    550ns\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       ") 7) \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " 72 \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    303K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       " rmcomm leaflets low hydration (\n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    550ns\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       ") For further information read the Readme file provided for each simulation.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_3426170.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Combining Linear-Scaling DFT with Subsystem DFT in Born Oppenheimer and Ehrenfest Molecular Dynamics Simulations: From Molecules to a Virus in Solution<br>In this work, methods for the efficient simulation of large systems embedded in a molecular environment are presented. These methods combine linear-scaling (LS) Kohn Sham (KS) density functional theory (DFT) with subsystem (SS) DFT. LS DFT is efficient for large subsystems, while SS DFT is linear scaling with a smaller prefactor for large sets of small molecules. The combination of SS and LS, which is an embedding approach, can result in a 10-fold speedup over a pure LS simulation for large systems in aqueous solution. In addition to a ground-state Born Oppenheimer SS+LS implementation, a time-dependent density functional theory-based Ehrenfest molecular dynamics (EMD) using density matrix propagation is presented that allows for performing nonadiabatic dynamics. Density matrix-based EMD in the SS framework is naturally linear scaling and appears suitable to study the electronic dynamics of molecules in solution. In the LS framework, linear scaling results as long as the density matrix remains sparse during time propagation. However, we generally find a less than exponential decay of the density matrix after a sufficiently long EMD run, preventing LS EMD simulations with arbitrary accuracy. The methods are tested on various systems, including spectroscopy on dyes, the electronic structure of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    TiO2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " nanoparticles, electronic transport in carbon nanotubes, and the satellite tobacco mosaic virus in explicit solution.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_6541983.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Molecular Dynamics simulations of shear droplets<br>This dataset contains the results of non-equilibrium Molecular Dynamic simulations of 2-dimensional \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    SPC/E\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " nanodroplets confined between silica-like walls and under shear flow conditions, performed using \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Gromacs\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       ". The main purposes of these simulations are: a) to study the motion of three-phases contact lines over high-friction surfaces, b) to study the critical transition leading to droplet breakage and c) to test the modelling and prediction capabilities of continuous fluid dynamics simulation methods. The investigation of the points above is illustrated in an article, which has been digitally published on the Journal of Fluid Mechanics (doi:10.1017/jfm.2022.219, see references); please refer to the paper for a detailed description of the molecular simulations and of the tested CFD methods. The publication of this dataset not only grants the reproducibility of the results discussed in the article, but also serves as collection of benchmarks for the fellow researchers willing to test improved and/or alternative models to describe the motion of contact lines.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_19775703.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">ContactAngleCalculator: An Automated, Parametrized, and Flexible Code for Contact Angle Estimation in Visual Molecular Dynamics<br>Accurate, fast, and flexible approaches for contact angle estimation in molecular dynamics simulations are of great importance for characterization of surface wettability, especially for machine learning approaches which would usually require thousands of computational contact angle evaluations for training and prediction purposes. However, evaluation of the contact angle from molecular simulations is typically a human-intensive process, which hinders the required fast throughput. To address this challenge, here a flexible and automated contact angle estimation tool, ContactAngleCalculator, is developed to meet these new requirements. In contrast to the current widely used computational approaches that are laborious and human intensive, this code is based on the concepts of the coarse-graining technique and equivalent contact area and volume of the droplet. Once the parameters are determined for a target liquid, it can automatically estimate the contact angle of different time points of one case or multiple cases by only one click. This tool is targeted for integration with machine learning methods, in which it can substantially streamline and reduce human labor and time in a computational contact angle estimation.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_5190502.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Organic and Third Phase in \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    HNO3\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "/\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    TBP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "/\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    n-Dodecane\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " System: No Reverse Micelles<br>The composition and speciation of the organic and third phases in the system \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    HNO3\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "/\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    TBP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " ( \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    tri-n-butyl phosphate\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " )/ \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    n-dodecane\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " have been examined by a combination of gravimetric, Karl Fischer analysis, chemical analysis, FTIR, and 31P NMR spectroscopy, with particular emphasis on the transition from the two-phase to the three-phase region. Phase densities indicate that third-phase formation takes place for initial aqueous \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    HNO3\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " concentrations above 15 M, while the results from the stoichiometric analysis imply that the organic and third phases are characterized by two distinct species, namely the mono-solvate \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    TBP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    HNO3\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and the hemi-solvate \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    TBP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    2HNO3\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", respectively. Furthermore, the 31P NMR spectra of organic and third phase show no significant chemical differences at the phosphorus centers, suggesting that the second \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    HNO3\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " molecule in the third phase is bound to \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    HNO3\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " rather than \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    TBP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". The third-phase FTIR spectra reveal stronger vibrational absorption bands at 1028, 1310, 1653, and 3200 3500 cm 1, reflecting higher concentrations of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    H2O\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    HNO3\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    TBP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". The molecular dynamics simulation data predict structures in accord with the spectroscopically identified speciation, indicating inequivalent \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    HNO3\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " molecules in the third phase. The predicted structures of the organic and third phases are more akin to microemulsion networks rather than the distinct, reverse micelles assumed in previous studies. \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    H2O\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " appears to be present as a disordered hydrogen-bonded solvate stabilizing the polar \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    TBP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "/\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    HNO3\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " aggregates in the organic matrix, and not as a strongly bound hydrate species in aggregates with defined stoichiometry.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_7013022.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Using Molecular Dynamics Simulations to Interrogate \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    T Cell Receptor\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Non-Equilibrium Kinetics<br>Prinicipal component analysis was performed on \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    TCR-pMHC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " steered molecular dynamics simulation trajectories. These supplementary videos include rendered videos of those trajectories (named: mutant equilibrationtime pull.mp4), principal component trajectories (named: mutant equilibrationtime pcx.mp4), and the control trajectory with pull from termini (named: termini control.mp4). These are template names for the video trajectories (refer to README file for more details). Note, mutants refer to the in silico mutations made to the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    TCR-pMHC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " protein structures and are abbreviated \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    L1\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    MART1\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GVA\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", respectively. In addition, the starting configurations for all pull simulations have been provided free of charge.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_6791151.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Input files and structures for article &quot;\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Carbon dioxide\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " fixation in \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    RuBisCO\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " is protonation state dependent and irreversible&quot;<br>Pdb file and \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    amber\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " topology of the initial \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    RuBisCO\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " model together with parameter files for the MD simulations with \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    AMBER\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " to equilibrate the system. Structures of reactant, transition state and products in the enzyme \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    RuBisCo\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " used to correct the activation and reaction free energies. \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Molpro\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " input files used in the projector-based embedding calculations are provided. The same holds for the calculations involving the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Zn\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " cation. Optimized structures used to calculate descriptors with conceptual DFT. </div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_5592299.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Supplemental material for &quot;Computational study of the dissolution of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    cellulose\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " into single chains: the role of the solvent and agitation&quot;<br>Input-scripts and data-files from the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Cellulose\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " article &quot;Computational study of the dissolution of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    cellulose\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " into single chains: the role of the solvent and agitation&quot; by Bering, E., Torstensen, J., Lervik, A., Wijn, A. S. # Content The folder &quot;glycamstructures&quot; contains the output from \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GLYCAM\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    carbohydrate\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " builder with force-field parameters, coordinates and topology of a single chain of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    cellulose\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " composed of 4 \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    cellobiose\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " units. The folder &quot;intermol&quot; contains the output from the software \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Intermol\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " were the content of &quot;glycamstructures&quot; is translated into the lingo of \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    LAMMPS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       ". The folder &quot;packmol&quot; contains input files for the software \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Packmol\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       ", which was used for creating coordinate-files for solvated systems of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    cellulose\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " with water and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    cellulose\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " with the solvent mixture. (The bash-script clean.sh fixes the formatting of the .xyz-files such that \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    LAMMPS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " can read it.) The folder &quot;36bundle&quot; contains input files for creating the initial configuration of the bundle with 36 chains in a maze configuration, namely 36bundle.sh reads 1chain.xyz to make the coordinate file 36bundle.xyz. Further, nvt00.in is the initial \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    LAMMPS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " script for slowly starting up this system with the NVT-ensemble, which reads system information from the \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    LAMMPS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       "-datafile chain.36bundle, \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    cellulose\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " parameters from the \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    LAMMPS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       "-datafile data.cellulose nohybrid and coordinates from 36bundle.xyz. nvt01.in and nvt02.in continues sequentially, resulting in the configuration stored in the \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    LAMMPS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " data-file chain.nvt02. The folder &quot;36bundle mix&quot; continues from 36bundle by first adding the solvent mixture with coordinates from \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Packmol\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " in the file 36bm.xyz and slowly starting up in the NVT-ensemble with nvt00.in, with force-field parameters in data.cellulose.in, data.water spce.in, data.naoh.in and data.urea.in, with charges and connectivity of the solvent molecules in the files water spce.txt, naoh.txt and urea.txt. The \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    LAMMPS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " script npt00.in continues with the NPT-ensemble, which is continued with npt01.in etc., resulting in the configuration stored in the \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    LAMMPS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " data-file chain.npt06. These systems can be continued with length- or force-controlled oscillatory stretching/compression with osc length.in and osc force.in respectively, which was used to make the configurations stored in the \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    LAMMPS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " data-files chain.osc length4 and chain.osc force4. Similarly, the folder &quot;36bundle water&quot; continues from 36bundle by first adding the water with coordinates from \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Packmol\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " in the file 36bw.xyz and slowly starting up in the NVT-ensemble with nvt00.in, with force-field parameters in data.cellulose.in and data.water spce.in, with charges and connectivity of the water molecules in the file water spce.txt. Again, these systems can be continued with length- or force-controlled oscillatory stretching/compression with osc length.in and osc force.in respectively, which was used to make the configurations stored in the \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    LAMMPS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " data-files chain.osc length4 and chain.osc force4.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_13853.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">MD simulation trajectory and related files for \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " bilayer in low hydration (\n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GAFFlipid\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Gromacs 4.5\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       ")<br>Equilibrated \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " lipid bilayer simulation ran with \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Gromacs 4.5\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GAFFlipid\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " force field (https://removed in low hydration, \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    40ns\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       ", T \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    303K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       ", 126 \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " molecules, 896 \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " molecules. This data is ran for the nmrlipids.blospot.fi project. More details from nmrlipids.blospot.fi and https://removed If data is used, please cite nmrlipids.blogspot.fi project and the original publication of the parameters: Dickson et al. Soft Matter, 2012,8, 9617-9627 https://removed</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_6970327.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Automated protein-protein structure prediction of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    T cell receptor\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-peptide major histocompatibility complex<br>The \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    T Cell Receptor\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    TCR\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") recognition of a \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    peptide-major histocompatibility complex\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    pMHC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") is a crucial component of the adaptive immune response. The identification of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    TCR\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    pMHC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " pairs is a significant bottleneck in the implementation of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    TCR\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " immunotherapies and may be augmented by computational methodologies that accelerate the rate of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    TCR\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " discovery. The ability to computationally design \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    TCRs\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " to a target \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    pMHC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " will require an automated integration of next-generation sequencing, homology modeling, molecular dynamics (MD), and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    TCR\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " ranking. We present a generic pipeline to evaluate patient-specific, sequence-based \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    TCRs\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " to a target \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    pMHC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". The most expressed \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    TCRs\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " from 16 colorectal cancer patients are homology modeled to target the CEA peptide using \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Modeller\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ColabFold\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       ". Then, these \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    TCR\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    pMHC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " structures are compared by performing an automated molecular dynamics equilibration. We find that \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Colabfold\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " generates starting configurations that require, on average, an ~2.5X reduction in simulation time to equilibrate \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    TCR\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    pMHC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " structures compared to \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Modeller\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       ". In addition, there are differences between equilibrated structures generated by \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Modeller\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ColabFold\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       ". Moreover, we identify \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    TCR\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " ranking criteria that may be used to prioritize \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    TCRs\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " for evaluation of in vitro immunogenicity.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_3778112.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Mutated \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    EN2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (143-259) simulation with \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Amber ff03ws\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       "<br>MD simulation of a mutated protein \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    EN2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (143-259) with \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Amber ff03ws\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " and 40 mM \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    NaCl\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " as the solvent. The mutated sequence differs from the original sequence by amino acids \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    serine\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (148, 150, 153, 154, 155, 156) and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    aspartic acid\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (152,154) changed to \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    threonine\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    glutamate\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " respectively. The trajectory files contain the total \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    1080 ns\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       " simulation. The trajectory file without solvent has saving frequency 10 ps. The trajectory files with solvent have a saving frequency 100ps. Force field parameters for proteins and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    counter ions\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " are downloaded from https://removed fields, for \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    NaCl\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " from https://removed</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_34415.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    AMBER LIPID14\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CaCl2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " 035Mol<br>MD simulation trajectory and related files for fully hydrated \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " bilayer with 0.35M \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CaCl2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". The \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    LIPID14\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " force field was used with \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Gromacs 5.0.3.\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Ions\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " were described by \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    AMBER99SB-ILDN\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " force field. Conditions: T \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    298.15\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       ", 128 \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " molecules, 6400 \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    tip3p\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    waters\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    lipid\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "/\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " 1:50), 35 \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Ca\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", 70 \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Cl\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    200ns\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       " trajectory (preceded by 5ns NPT equillibration) (2 files of \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    100ns\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       "). THE TRAJECTORY &quot;035M \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CaCl2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " AMB 100 200ns.xtc&quot; IS CORRUPTED. FOR THE UNCORRUPTED FILE PLEASE FOLLOW THE LINK: https://removed This data is ran for the nmrlipids.blospot.fi project. More details from nmrlipids.blospot.fi and https://removed</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_3494177.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">General Force-Field Parametrization Scheme for Molecular Dynamics Simulations of Conjugated Materials in Solution<br>We describe a general scheme to obtain force-field parameters for classical molecular dynamics simulations of conjugated polymers. We identify a computationally inexpensive methodology for calculation of accurate intermonomer dihedral potentials and partial charges. Our findings indicate that the use of a two-step methodology of geometry optimization and single-point energy calculations using DFT methods produces potentials which compare favorably to high level theory calculation. We also report the effects of varying the conjugated backbone length and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    alkyl\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " side-chain lengths on the dihedral profiles and partial charge distributions and determine the existence of converged lengths above which convergence is achieved in the force-field parameter sets. We thus determine which calculations are required for accurate parametrization and the scope of a given parameter set for variations to a given molecule. We perform simulations of long oligomers of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    dioctylfluorene\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    hexylthiophene\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " in explicit solvent and find peristence lengths and end-length distributions consistent with experimental values.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_5570754.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Computational Insights into the Unfolding of a Destabilized \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Superoxide Dismutase 1\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Mutant<br>This data accompanies the article entitled Computational Insights into the Unfolding of a \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Destabilized Superoxide Dismutase 1\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Mutant and published in Biology. \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    SOD1\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " WT I35A REST2.zip: The zip archive includes REST2 trajectories for the two \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    SOD1\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " constructs and the two force fields investigated in the paper. The trajectories are saved in the \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GROMACS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " XTC file format, separately for each temperature (i 0,...,23). Owing to the considerable trajectory sizes, only protein coordinates are reported, and the output frequency is reduced to 100 ps. The initial geometry (in the \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Gromos87\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " GRO format) after a short relaxation is provided for each REST2 simulation (conf prot.gro). Furthermore, for each REST2 simulation, an xarray (https://removed dataset, saved in the netCDF file format, is included and contains the following observables: fraction of native contacts relative to the crystal structure, secondary-structure content (i.e., the fraction of protein residues found in an alpha-helix, beta-sheet, beta-bridge, or a turn), as well as the number of residues with the beta-sheet secondary structure per each beta-strand and beta-sheet of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    SOD1\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " barrel.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_9744458.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Comparative in silico study of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    congocidine\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " congeners as potential inhibitors of African swine fever virus<br>African swine fever virus (ASFV) infection is fatal in domesticated pigs, with a mortality rate approaching 100%. This may result in economic losses and threats to food security. Currently, there are no approved vaccines or antiviral therapies for ASFV. Therefore, in this study, we evaluated \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    congocidine\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " congeners and a \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    tris-benzimidazole\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " as potential inhibitors of ASFV transcription using an in silico approach. We applied redocking of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    congocidine\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and docking of its congeners and a \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    tris-benzimidazole\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " to a receptor containing \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    B-DNA\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " with AT-motifs as a target to mimic conserved ASFV late gene promoters. Subsequently, the binding scores of DNA-ligand docked complexes were evaluated and their binding affinity was estimated. Molecular dynamics (MD) simulation was then used to assess ligand behavior within the minor groove. From our results, it is evident the less toxic \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    congocidine\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " congeners and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    tris-benzimidazole\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " could dock to AT-rich regions significantly. Additionally, the predicted binding affinities had suitable values comparable to other experimentally determined minor groove binders, MD simulation of the docked DNA-ligand complexes and subsequent molecular trajectory visualization further showed that the ligands remained embedded in the minor groove during the time course of simulation, indicating that these ligands may have potential applications in abrogating ASFV transcription.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_3950029.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">212 \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DPPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Molecules bilayer in pure \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", simulated at temperatures ranging from \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    288K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       " to \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    358K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       "<br>Publication: A machine learning study of the two states model for lipid bilayer phase transitions Published on: 12 August 2020 Journal: Phys. Chem. Chem. Phys., 2020, DOI: 10.1039/D0CP02058C Description: Simulation files used to train our machine learning algorithm to identify the thermodynamic phase of individual lipid molecules in a bilayer, as well as the simulation files analysed by the machine learning models. Code source for the ML algorithm can be found on Github. The training files are named gel.gro and fluid.gro. They respectively correspond to the final frame of the systems simulated at \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    288K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    358K.\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       " All other files are the files analysed by the machine learning models. System composition: \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DPPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " molecules: 212 with 130 atoms each \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " molecules: 29,826 with 3 atoms each Simulation box dimensions (approx.): 8 x 8 x 20 nm Simulation details: Software: \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Gromacs (v. 2016.4)\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " Forcefield: \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Charmm36 (v. June 2015)\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " - \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ": \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    TIP3P\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " Thermostat: Nose-hoover (0.4ps, 2 groups) Barostat: Parrinello-Rahman semi-isotropic (2.0ps, 1.0 bar on each axis, 4.5e-5 bar-1) Duration: \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    50 ns\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       " </div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_2470129.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Experimental and Computational Studies Reveal an Alternative Supramolecular Structure for \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Fmoc-Dipeptide\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Self-Assembly<br>We have investigated the self-assembly of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    fluorenylmethoxycarbonyl\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-conjugated \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    dialanine\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Fmoc-AA\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") molecules using combined computational and experimental approaches. \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Fmoc-AA\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " gels were characterized using transmission electron microscopy (TEM), circular dichroism (CD), Fourier transform infrared (FTIR), and wide-angle X-ray scattering (WAXS). Computationally, we simulated the assembly of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Fmoc-AA\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " using molecular dynamics techniques. All simulations converged to a condensed fibril structure in which the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Fmoc\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " groups stack mostly within in the center of the fibril. However, the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Fmoc\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " groups are partially exposed to \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", creating an amphiphilic surface, which may be responsible for the aggregation of fibrils into nanoscale fibers observed in TEM. From the fibril models, radial distribution calculations agree with d-spacings observed in WAXS for the fibril diameter and -stacking interactions. Our analyses show that \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    dialanine\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", despite its short length, adopts a mainly extended \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    polyproline II\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " conformation. In contrast to previous hypotheses, these results indicate that -sheet-like hydrogen bonding is not prevalent. Rather, stacking of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Fmoc\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " groups, inter-residue hydrogen bonding, and hydrogen bonding with \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " play the important roles in stabilizing the fibril structure of supramolecular assemblies of short conjugated peptides.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_17698630.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Tetracycline\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Derivatives Inhibit \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Plasmodial Cysteine Protease\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Falcipain 2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " through Binding to a Distal Allosteric Site<br>Allosteric inhibitors regulate enzyme activity from remote and usually specific pockets. As they promise an avenue for less toxic and safer drugs, the identification and characterization of allosteric inhibitors has gained great academic and biomedical interest in recent years. Research on \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    falcipain-2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    FP-2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "), the major \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    papain\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-like \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    cysteine hemoglobinase\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " of Plasmodium falciparum, might benefit from this strategy to overcome the low selectivity against \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    human cathepsins\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " shown by active site-directed inhibitors. Encouraged by our previous finding that \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    methacycline\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " inhibits \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    FP-2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " noncompetitively, here we assessed other five \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    tetracycline\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " derivatives against this target and characterized their inhibition mechanism. As previously shown for \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    methacycline\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    tetracycline\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " derivatives inhibited \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    FP-2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " in a noncompetitive fashion, with Ki values ranging from 121 to 190 M. A possible binding to the S side of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    FP-2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " active site, similar to that described by X-ray crystallography (PDB: \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    6SSZ\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") for the noncompetitive inhibitor \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    E-chalcone 48\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    EC48\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "), was experimentally discarded by kinetic analysis using a large \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    peptidyl\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " substrate spanning the whole active site. By combining lengthy molecular dynamics (MD) simulations that allowed \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    methacycline\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " to diffuse from solution to different \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    FP-2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " surface regions and free energy calculations, we predicted the most likely binding mode of the ligand. Of note, the proposed binding pose explains the low differences in Ki values observed for the tested \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    tetracycline\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " derivatives and the calculated binding free energies match the experimental values. Overall, this study has implications for the design of novel allosteric inhibitors against \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    FP-2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and sets the basis for further optimization of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    tetracycline\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " scaffold to produce more potent and selective inhibitors.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_4993181.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Data from: Energy landscape for the insertion of amphiphilic nanoparticles into \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    lipid\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " membranes: a computational study<br>Amphiphilic, monolayer-protected \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    gold nanoparticles\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    NPs\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") have been shown to enter cells via a non-endocytic, non-disruptive pathway that could be valuable for biomedical applications. The same \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    NPs\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " were also found to insert into a series of model cell membranes as a precursor to cellular uptake, but the insertion mechanism remains unclear. Previous simulations have demonstrated that an amphiphilic \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    NP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " can insert into a single leaflet of a planar \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    lipid\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " bilayer, but in this configuration all charged end groups are localized to one side of the bilayer and it is unknown if further insertion is thermodynamically favorable. Here, we use atomistic molecular dynamics simulations to show that an \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    amphiphilic NP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " can reach the bilayer midplane non-disruptively if charged ligands iteratively ''flip'' across the bilayer. Ligand flipping is a favorable process that relaxes bilayer curvature, decreases the nonpolar solvent-accessible surface area of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    NP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " monolayer, and increases attractive ligand-\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    lipid\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " electrostatic interactions. Analysis of end group hydration further indicates that iterative ligand flipping can occur on experimentally relevant timescales. Supported by these results, we present a complete energy landscape for the non-disruptive insertion of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    amphiphilic NPs\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " into \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    lipid\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " bilayers. These findings will help guide the design of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    NPs\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " to enhance bilayer insertion and non-endocytic cellular uptake, and also provide physical insight into a possible pathway for the translocation of charged biomacromolecules.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_6817824.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Surfactant Proteins \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    SP-B\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    SP-C\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " in Pulmonary Surfactant Monolayers: Physical Properties Controlled by Specific Protein Lipid Interactions - Simulation dataset<br>Simulation data related to the article: Liekkinen, J., Olzynska, A., Cwiklik, L., Bernardino de la Serna, J., Vattulainen, I., &amp; Javanainen, M. Surfactant Proteins \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    SP-B\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    SP-C\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " in Pulmonary Surfactant Monolayers: Physical Properties Controlled by Specific Protein Lipid Interactions. bioRxiv, pp.2022-12. (Pre-print available at https://removed \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DPPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "/\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "/\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPG\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "/\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CHL1\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (60/20/10/10 mol-%) monolayers simulated in non-equilibrium conditions. The monolayer is compressed or expanded at a constant rate at different temperatures. Surfactant proteins \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    SP-B\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    SP-C\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " are included in the monolayers for comparison. The \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Charmm36\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " lipid model [1] is used together with the 4-point \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    OPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " model [2]. Each system consist of two monolayers with 169 lipids each. The monolayers are separated by a \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " slab, and surrounded by vacuum. All trajectories are simulated for \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    5000 ns\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       " with \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Gromacs 5.1.x\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " [3] using the default \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Charmm36\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " monolayer simulations parameters given in the mdp file. Topology (.top) and index (.ndx) files are also included. The topologies (.itp) for the lipids can be obtained from \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Charmm-GUI\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " and for the \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    OPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " model from https://removed [1] DOI: 10.1021/jp101759q [2] DOI: 10.1021/jz501780a [3] DOI: 10.1016/j.softx.2015.06.001</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_5642866.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Demystifying \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    P2Y1 Receptor\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Ligand Recognition through Docking and Molecular Dynamics Analyses<br>We performed a molecular modeling analysis of 100 nucleotide-like \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    bisphosphates\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and 46 non-nucleotide \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    arylurea derivatives\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " previously reported as \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    P2Y1R\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " binders using the recently solved \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    hP2Y1R\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " structures. We initially docked the compounds at the X-ray structures and identified the binding modes of representative compounds highlighting key patterns in the structure activity relationship (SAR). We subsequently subjected receptor complexes with selected key agonists (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    2MeSADP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    MRS2268\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") and antagonists (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    MRS2500\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    BPTU\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") to membrane molecular dynamics (MD) simulations (at least 200 ns run in triplicate, simulation time \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    0.6\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    1.6 μs\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       " per ligand system) while considering alternative protonation states of nucleotides. Comparing the temporal evolution of the ligand protein interaction patterns with available site-directed mutagenesis (SDM) data and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    P2Y1R\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " apo state simulation provided further SAR insights and suggested reasonable explanations for loss/gain of binding affinity as well as the most relevant charged species for nucleotide ligands. The MD analysis also predicted local conformational changes required for the receptor inactive state to accommodate nucleotide agonists.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_44622.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">MD simulation trajectory and related files for \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " bilayer with 30 mol% of deprotonated \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    pazePC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Berger\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Gromacs 4.5.\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       ")<br>Simulation trajectory and related files for \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " bilayer with 30mol% of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    pazePC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " in deprotonated form used in \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Acyl\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " chain disorder and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    azelaoyl\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " orientation in lipid membranes containing oxidised lipids by T. M. Ferreira, et al., https://removed 90 \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", 38 \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    pazePC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", 7250 \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and 38 \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    potassium\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " molecules. Simulation time \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    120ns\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       ". Force field for \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " is based on \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Berger\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " model and force field for \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    pazePC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " is from H. Khandelia and O. G. Mouritsen, Biophysical Journal, 2009, 96 (7), 2734 2743, https://removed If used, please cite approriate publications.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_6960098.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Predicting the mechanism and rate of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    H-NS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " binding to \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    AT-rich DNA\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "<br> Bacteria contain several nucleoid-associated proteins that organize their genomic \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DNA\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " into the nucleoid by bending, wrapping or bridging \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DNA\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". The \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Histone-like Nucleoid Structuring\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " protein \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    H-NS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " is a \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DNA\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " bridging \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    protein\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and can structure \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DNA\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " by binding to two separate \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DNA\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " duplexes or to adjacent sites on the same duplex, depending on external conditions. \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    H-NS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " occurs in enterobacteria and influences the expression of genes. Several nucleotide sequences have been identified to which \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    H-NS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " binds with high affinity, indicating \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    H-NS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " prefers \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    AT-rich DNA\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". To date, highly detailed structural information of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    H-NS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DNA\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " complex remains elusive. Molecular simulation can complement experiments by modelling structures and their time evolution in atomistic detail. In this paper we report an exploration of the different binding modes of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    H-NS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " to a high affinity nucleotide sequence and an estimate of the associated rate constant. By means of molecular dynamics simulations, we identified three types of binding for \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    H-NS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " to \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    AT-rich DNA\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". To further sample the transitions between these binding modes, we performed Replica Exchange Transition Interface Sampling, providing predictions of the mechanism and rate constant of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    H-NS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " binding to \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DNA\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    H-NS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " interacts with the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DNA\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " through a conserved QGR motif, aided by a conserved arginine at position 93. The QGR motif interacts first with \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    phosphate\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " groups, followed by the formation of hydrogen bonds between acceptors in the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DNA\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " minor groove and the sidechains of either Q112 or R114. After R114 inserts into the minor groove, the rest of the QGR motif follows. Full insertion of the QGR motif in the minor groove is stable over \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    several tens of nanoseconds\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       ", and involves hydrogen bonds between the bases and both backbone and sidechains of the QGR motif. The rate constant for the process of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    H-NS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " binding to \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    AT-rich\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " DNA resulting in full insertion of the QGR motif is in the order of 106 M 1s 1, which is rate limiting compared to the non-specific association of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    H-NS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " to the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DNA\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " backbone. </div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_7112198.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    acetonitrile\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    methanol\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " MD simulations driven by many-body ML potentials<br>Input, output, and trajectories of molecular dynamics (MD) simulations of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    acetonitrile\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    methanol\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". Simulations were driven by many-body machine learning (mbML) potentials including explicit 1-, 2-, and 3-body contributions. \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GDML\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GAP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       ", and \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    SchNet\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " models are provided in a separate repository. All simulations were performed in the atomic simulation environment (ASE). Analyses including radial distribution function (rdf) curves are provided here. Manifest The following simulations are included in this repository for each solvent. 1 ps hexamer NVE MD simulation driven by MP2/def2-TZVP (in ORCA v4.2.0), mbGDML, mbGAP, mbSchNet, and GFN2-xTB started with the same positions and velocities. Velocities were initialized at \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    298.15 K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       " with a Maxwell-Boltzmann distribution. Periodic NVT MD simulation at \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    298.15 K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       " for 10 or 30 ps with a 1 fs time step driven by mbGDML. These simulations contained 58 or 137 \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " molecules, 67 or 122 \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    acetonitrile\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " molecules, 61 \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    methanol\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " molecules. Systems that are not bolded were used for testing purposes.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_3786821.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Inputs for computational electrophysiology of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Glycine Receptor\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " with \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GROMACS 19\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       "<br>Inputs for computational electrophysiology of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Glycine Receptor\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (D&amp;B-open model, doi:10.5281/zenodo.3476169) with \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GROMACS 19\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " and the \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CHARMM36\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " force-field, using: 1- a single membrane system with the application of a constant electric field. 2- a double membrane system with the application of the charge imbalance protocol. The model of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    glycine receptor\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " was reduced to its transmembrane domain and simulated with atomic positional restraints. Related to the published article: &quot;On the functional annotation of open-channel structures in the glycine receptor&quot;.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_15019653.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Ensembles Are Required to Handle Aleatoric and Parametric Uncertainty in Molecular Dynamics Simulation<br>Classical molecular dynamics is a computer simulation technique that is in widespread use across many areas of science, from physics and chemistry to materials, biology, and medicine. The method continues to attract criticism due its oft-reported lack of reproducibility which is in part due to a failure to submit it to reliable uncertainty quantification (UQ). Here we show that the uncertainty arises from a combination of (i) the input parameters and (ii) the intrinsic stochasticity of the method controlled by the random seeds. To illustrate the situation, we make a systematic UQ analysis of a widely used molecular dynamics code (NAMD), applied to estimate binding free energy of a ligand-bound to a protein. In particular, we replace the usually fixed input parameters with random variables, systematically distributed about their mean values, and study the resulting distribution of the simulation output. We also perform a sensitivity analysis, which reveals that, out of a total of 175 parameters, just six dominate the variance in the code output. Furthermore, we show that binding energy calculations dampen the input uncertainty, in the sense that the variation around the mean output free energy is less than the variation around the mean of the assumed input distributions, if the output is ensemble-averaged over the random seeds. Without such ensemble averaging, the predicted free energy is five times more uncertain. The distribution of the predicted properties is thus strongly dependent upon the random seed. Owing to this substantial uncertainty, robust statistical measures of uncertainty in molecular dynamics simulation require the use of ensembles in all contexts.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_11771604.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Parametrization and Molecular Dynamics Simulations of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Nitrogen Oxyanions\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Oxyacids\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " for Applications in Atmospheric and Biomolecular Sciences<br>\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Nitrogen oxyanions\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    oxyacids\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " are important agents in atmospheric chemistry and medical biology. Although their chemical behavior in solution is relatively well understood, they may behave very differently at the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "/\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    air\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " interface of atmospheric aerosols or at the membrane/\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " interface of cells. Here, we developed a fully classical model for molecular dynamics simulations of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    NO3\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " , \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    NO2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " , \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    HNO3\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    HNO2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " in the framework of the \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GROMOS 53A6\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " and 54A7 force field versions. The model successfully accounted for the poorly structured solvation shell and ion pairing tendency of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    NO3\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " . Accurate pure-liquid properties and hydration free energies were obtained for the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    oxyacids\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". Simulations at the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "/\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    air\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " interface showed a local enrichment of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    HNO3\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and depletion of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    NO3\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " . The effect was discussed in light of earlier spectroscopic data and ab initio calculations, suggesting that \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    HNO3\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " behaves as a weaker acid at the surface of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". Our model will hopefully allow for efficient and accurate simulations of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    nitrogen oxyanions\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    oxyacids\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " in solution and at microheterogeneous interface environments.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_4300706.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">256 \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DPPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Molecules bilayer in pure \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", simulated at \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    288K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       " (gel) or \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    358K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       " (fluid)<br>Publication: MLLPA: A Machine Learning-assisted Python module to study phase-specific events in lipid membranes Published on: 08 April 2021 Journal: J Comp Chem, 2021, DOI: 10.1002/jcc.26508 Description: Simulation files used to train our Python module to identify the thermodynamic phase of individual lipid molecules in a bilayer, as well as the simulation files analysed by the machine learning models. More information on the module can be found on its website. The training files are named \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    dppc\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " gel.gro and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    dppc\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " fluid.gro. They respectively correspond to the final frame of the systems simulated at \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    288K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    358K.\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       " All other files are the files analysed by the module. System composition: \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DPPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " molecules: 256 with 130 atoms each \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " molecules: 42,492 with 3 atoms each Simulation box dimensions (approx.): 9 x 9 x 20 nm Simulation details: Software: \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Gromacs (v. 2020)\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " Forcefield: \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Charmm36 (v. June 2015)\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " - \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ": \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    TIP3P\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " Thermostat: Nose-hoover (0.4ps, 2 groups) Barostat: Parrinello-Rahman semi-isotropic (2.0ps, 1.0 bar on each axis, 4.5e-5 bar-1) Duration: \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    25 ns\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       " (after stabilisation) </div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_6394133.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Molecular Structure and Permeability at the Interface between Phase-Separated Membrane Domains<br>Phase-separated membrane domains, also known as lipid rafts, are believed to play an important role in cell function. Although most rafts are \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    sterol\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-enriched membrane regions, evidence suggests that living cells may also contain gel-like rafts. Interactions between gel and fluid domains have a large impact on membrane properties, as is the case with permeability. The membrane permeability may reach a peak at the main phase transition temperature, by far exceeding the values recorded at the fluid phase. It has been proposed that gel fluid interfaces are leaky, but the effect has not yet been demonstrated at the molecular level. Here, we performed atomistic molecular dynamics simulations of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    phospholipid\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " bilayers with coexisting gel-like and fluid domains. We found that the thickness mismatch between both phases, the membrane elasticity, and the lipid packing acted together to promote the formation of a thickness minimum at the gel fluid interface. Free energy calculations showed that pore-mediated ionic permeation was strongly facilitated at the constriction region, whereas \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " permeation by simple diffusion was only marginally affected. Long-lived, peristaltic undulations were recorded at the bulk fluid phase near the main transition temperature. They gave rise to thickness minima that, although shallower than the interface constrictions, could also enhance permeability. Finally, we demonstrated that an interface constriction was also formed at the boundaries of regular, \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    cholesterol\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-enriched lipid rafts. Our simulation results will hopefully contribute to a better understanding of biological processes such as transport, signaling, and cellular damage promoted by low temperature and dehydration.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_4245236.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Model data for Sequential Dynamics of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Stearoyl-CoA Desaturase\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " /Ligand Binding and Unbinding Mechanism: A Computational Study by Petroff et al. (submitted).<br>Model data for Sequential Dynamics of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Stearoyl-CoA Desaturase\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " /Ligand Binding and Unbinding Mechanism: A Computational Study by Petroff et al. (submitted). This folder contains the files needed to start each of the models described in the paper. The files were created using \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    MOE 2020\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " software made by Chemical Computing Group and run on \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    NAMD2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       ". The models identifiers in the paper correspond to the following terms in the code: Substrate: &quot;13 5 coa&quot; Product: &quot;13 5 coa desat fe3&quot; Apoprotein: &quot;13 5 no ligand&quot; Saturated Lipid: &quot;13 5 nocoa&quot; Desaturated Lipid: &quot;13 5 nocoa desat fe3&quot; CoA model: &quot;13 5 coa nolipid&quot; Substrate-waterbox model: &quot;13 5 coa waterbox&quot; Saturated Lipid-waterbox: &quot;13 5 nocoa waterbox&quot;</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_21217898.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Assessing the Activation of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Tyrosine Kinase\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    KIT\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " through Free Energy Calculations<br>\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    KIT\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " is a type 3 receptor \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    tyrosine kinase\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " that plays a crucial role in cellular growth and proliferation. Mutations in \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    KIT\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " can dysregulate its active inactive equilibrium. Activating mutations drive cancer growth, while deactivating mutations result in the loss of skin and hair pigmentation in a disease known as piebaldism. Here, we propose a method based on molecular dynamics and free energy calculations to predict the functional effect of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    KIT\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " mutations. Our calculations may have important clinical implications by defining the functional significance of previously uncharacterized \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    KIT\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " mutations and guiding targeted therapy.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_12661589.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Comprehensive Computational Analysis Exploring the Formation of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Caprolactam\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-Based Deep Eutectic Solvents and Their Applications in Natural Gas Desulfurization<br>Several deep eutectic solvents (DESs) have been recently developed for extraction of hydrogen sulfide from natural gas. Among these newly designed DESs, a combination of caprolactam and tetrabutylammonium halides at a molar ratio of 1:1, as the DESs, has the highest desulfurization efficiencies. In this work, we explored the formation of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    caprolactam\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-based DESs using molecular dynamics (MD) simulations and ab initio computations. The results, based on the time average of the equilibrated production run of MD simulations, revealed 15% decrease in the ionic interactions of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    tetrabutylammonium halides\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and, more interestingly, 92% decrease in the hydrogen bonds between \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    caprolactam\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", thereby explaining the strong depression in the melting point observed in experiments during the formation of the DES. Next, simulations of the DES with mixtures of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    methane\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    hydrogen sulfide\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " were performed to mimic the natural gas desulfurization process. Efficient absorption of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    hydrogen sulfide\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " from natural gas under different operating conditions (5000 and 10,000 ppm H2S, at \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    25\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    60 C\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       ", and at 1 and 10 bar) can be observed from the simulations. The results further unveiled strong interactions between the anions of the DESs and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    hydrogen sulfide\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", with interaction energies 10-fold higher than those of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    methane\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "/\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    hydrogen sulfide\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", explaining the mechanism of desulfurization by these DESs. Additionally, two different DESs composed of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    monoethanolamines\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "/\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    methyltriphenylphosphonium bromide\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    urea\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "/\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    choline chloride\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " were used to evaluate and compare their capacity to absorb \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    hydrogen sulfide\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "; however, we observed that the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    caprolactam\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-based DESs are highly efficient, particularly at low temperatures, low pressures, and low fuel/DES mole ratios. As the existing literature covers only experiments on the removal of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    hydrogen sulfide\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " via the DESs, the simulations presented herein provide insights into the mechanism of absorption, thereby paving a way forward for better design of the DESs.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_4336730.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Solvation Procedures Assessment of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Borohydride\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Reduction of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Carbon Dioxide\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "<br>Pathways, structures, gas-phase and solvation energies of aqueous borohydride reduction of carbon dioxide. Contents data Computational chemistry output files for gas-phase electronic energies, solvation energies, and QM/MM MD simulations are provided. They are organized by the method used to seek the reaction pathway. neb: contains computations involved with the g-SSNEB pathway from Groenenboom and Keith. gsm: contains computations either in preparation or execution of growing string method (GSM) calculations. The lego module of ABCluster was used to generate candidate starting structures. other: contains miscellaneous computations for additional analyses. scripts: contains all Python code used to generate Chemical JSON and CSV files. qmmm: contains GAMESS QM/MM MD trajectories and WHAM analyses. Note: the QM/MM MD data is in the zip with the &quot;qmmm&quot; suffix. Everything else is in the other zip. figures Contains Python scripts and figures made with matplotlib. Python files are named according to the data they use; for example, figure-neb.py is the code for figures that plot the various g-SSNEB pathways. Figures are organized according to where they appear: directly in the article (article/) or as supplemental information (si/). structures XYZ files relevant to this study organized by the chain-of-states method.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_21726720.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Perfluorinated Alcohols\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " at High Pressure: Experimental Liquid Density and Computer Simulations<br>The liquid density of five liquid 1H,1H-\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    perfluorinated alcohols\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (CF3 (CF2)n 1CH2OH n   2, 3, 4, 5, 6) was measured as a function of pressure (0.1 70 MPa) and temperature (\n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    293.15\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    313.15 K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       "). The corresponding isothermal compressibility and isobaric thermal expansivity coefficients were calculated from the experimental data. The results are compared with data from the literature for the equivalent \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    hydrogenated alcohols\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". Atomistic molecular dynamics simulations were also performed, providing molecular-level insight into the experimental results, in particular about the H-bond network of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    perfluorinated alcohols\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and the effect of pressure on the organization of the liquid.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_7291835.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">The Impact of Using Single Atomistic Long-Range Cutoff Schemes with the \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GROMOS 54A7\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " Force Field<br>With the recent increase in computing power, the molecular modeling community is now more focused on improving the accuracy and overall quality of biomolecular simulations. For the available simulation packages, force fields, and all other associated methods used, this relates to how well they describe the conformational space and thermodynamic properties of a biomolecular system. The parameter sets of \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GROMOS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " force fields have been parametrized and validated with the reaction field (RF) method using charge groups and a twin-range cutoff scheme (0.8/1.4 nm). However, the most recent versions of \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GROMACS (since v.2016)\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " discontinued the support for charge groups. To take full advantage of the newer and faster versions of this software package with \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GROMOS 54A7\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " and RF, we need to evaluate the impact of using a single cutoff scheme (vs twin-range) and of using the Verlet list update method (which is atomistic) compared to the group-based cutoff scheme. Our results show that the \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GROMOS 54A7\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " force field seems consistent with a single cutoff, since the resulting conformation and protonation ensembles were indistinguishable. The \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GROMOS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " parametrization procedure was also reproduced using an atomistic cutoff scheme, and we have observed that the hydration free energy values of small amino acid side-chain analogues were similar to the ones obtained with the group-based protocol. We do observe a small impact of the atomistic cutoff scheme in the conformational space of the model systems studied (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    G1-PAMAM\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DMPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "). However, since the structural properties of these systems are well converged for the cutoff range used (1.4 2.0 nm), unlike with the group-based cutoff schemes, we are confident that the atomistic cutoff can be adopted with RF for MD and constant-pH MD biomolecular simulations using the \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GROMOS 54A7\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " force field.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_2224255.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Relevant Interactions of Antimicrobial Iron Chelators and Membrane Models Revealed by Nuclear Magnetic Resonance and Molecular Dynamics Simulations<br>The dynamics and interaction of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    3-hydroxy-4-pyridinone\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " fluorescent iron chelators, exhibiting antimicrobial properties, with biological membranes were evaluated through NMR and molecular dynamics simulations. Both NMR and MD simulation results support a strong interaction of the chelators with the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    lipid\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " bilayers that seems to be strengthened for the rhodamine containing compounds, in particular for compounds that include \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ethyl\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " groups and a thiourea link. For the latter type of compounds the interaction reaches the hydrophobic core of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    lipid\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " bilayer. The molecular docking and MD simulations performed for the potential interaction of the chelators with \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DC-SIGN\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " receptors provide valuable information regarding the cellular uptake of these compounds since the results show that the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    fluorophore\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " fragment of the molecular framework is essential for an efficient binding. Putting together our previous and present results, we put forward the hypothesis that all the studied fluorescent chelators have access to the cell, their uptake occurs through different pathways and their permeation properties correlate with a better access to the cell and its compartments and, consequently, with the chelators antimicrobial properties.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_7952567.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Unfolding Pathways of Hen \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Egg-White Lysozyme\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " in \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Ethanol\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "<br>The aggregation of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    amyloid fibrils\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " can lead to various diseases including Alzheimer s, Parkinson s disease, and transmissible spongiform encephalopathy. \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Amyloid fibrils\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " can develop from a variety of proteins in the body as they misfold into a primarily -sheet structure and aggregate. \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Human lysozyme\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " has been shown to have far reaching effects in the human health a homologous enzyme, hen \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    egg-white lysozyme\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", has been shown to denature to a primarily -sheet structure at low pH and high alcohol content solution. We have studied these systems in atomic-level detail with a combination of constant pH and \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    microsecond\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       " long molecular dynamics simulation in explicit solvent, which cumulatively total over \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    10 μs\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       " of simulation time. These studies have allowed us to determine two potential unfolding pathways depending on the protonation state of a key glutamic acid residue as well as the effect of solution dynamics and pH on the unfolding process.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_14592.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">MD simulation trajectory and related files for \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DPPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " bilayer in full hydration (Poger \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GROMOS 53A6\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " L, \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Gromacs 4.0.7\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       ", Reaction Field, traj 1)<br>Equilibrated \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DPPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " lipid bilayer simulation ran with \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Gromacs 4.0.7\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " using Reaction Field for computing electrostatics, Poger \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GROMOS 53A6\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " L force field in full hydration (dx.doi.org/10.1002/jcc.21396), \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    100ns\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       ", T \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    323K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       ", 128 \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DPPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " molecules, 5841 \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " molecules. This data is ran for the nmrlipids.blospot.fi project. More details from nmrlipids.blospot.fi and https://removed If data is used, please cite nmrlipids.blogspot.fi project and the original publication of the parameters (dx.doi.org/10.1002/jcc.21396).</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_15054454.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Computation of Solid-State Vibrational Circular Dichroism in the Periodic Gauge<br>We introduce a new theoretical formalism to compute solid-state vibrational circular dichroism (VCD) spectra from molecular dynamics simulations. Having solved the origin-dependence problem of the periodic magnetic gauge, we present IR and VCD spectra of (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    1S,2S)-trans-1,2-cyclohexanediol\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " obtained from first-principles molecular dynamics calculations and nuclear velocity perturbation theory, along with the experimental results. Because the structure model imposes periodic boundary conditions, the common origin of the rotational strength has hitherto been ill-defined and was approximated by means of averaging multiple origins. The new formalism reconnects the periodic model with the finite physical system and restores gauge freedom. It nevertheless fully accounts for nonlocal spatial couplings from the gauge transport term. We show that even for small simulation cells the rich nature of solid-state VCD spectra found in experiments can be reproduced to a very satisfactory level.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_16604601.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Molecular View on the Dissociation Pathways and Transactivation Regulation Mechanism of Nonsteroidal \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GR\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Ligands<br>As a major drug target for anti-inflammatory therapy, the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    glucocorticoid receptor\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GR\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") regulates a wide range of physiological processes through transactivation (TA) or transrepression. \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GR\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " TA is involved in many adverse effects of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GR\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-targeting drugs, and therefore, the discovery of novel \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GR\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " ligands with lower TA activity and longer residence time is quite urgent. Undoubtedly, understanding the ligand dissociation mechanisms and the structural basis of the TA regulation is crucial for the development of novel \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GR\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-targeting drugs. Here, we used random accelerated molecular dynamics (RAMD) and funnel metadynamics (FM) simulations to explore the dissociation mechanisms of 5 classic \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    glucocorticoids\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and 6 nonsteroidal \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GR\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " ligands. Multiple ligand dissociation pathways were discovered. The classic glucocorticoids exhibit a strong preference for Path I, and most nonsteroidal ligands tend to dissociate along mixed pathways. We also find that the distinct unbinding preferences for AZD2906 and AZD9567, two representative nonsteroidal ligands with similar scaffolds but different TA activities, are primarily determined by their different polar interactions with the surrounding residues. Notably, the binding of AZD9567 poses a substantial impact on the conformation of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GR\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " homodimer interface, which provides a valuable clue to understand the mechanisms of the TA-related side effects induced by the adjustments of the homodimerization process. These findings are critical for the structure-based rational design of novel \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GR\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " ligands with more potent anti-inflammatory potency and reduced side effects.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_5756605.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">N-Glycosylation effects on \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    T cell receptor\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " kinetics<br>This methodology is an atomic-level investigation of the physiochemical effects of glycosylation on \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    TCR\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " dissociation kinetics. The force-dependent bond lifetime and bond strength of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DMF5\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    T Cell Receptor\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    TCR\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") to the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    MART1 peptide-Major Histocompatibility Complexes\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    pMHCs\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") were simulated using Steered Molecular Dynamics. N-glycsoylated and aglycosylated configurations were compared in triplicate. This data set contains atomic structure of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    TCR\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    pMHC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (.gro files), atomic interactions between the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    TCR\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    pMHC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (hmap &amp; cmap .csv files), and rendered trajecories of the pulling simulations (.mp4).</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_14591.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">MD simulation trajectory and related files for \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DPPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " bilayer in full hydration (\n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Poger GROMOS 53A6 L\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Gromacs 4.0.7\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       ", Reaction Field, traj 2)<br>Equilibrated \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DPPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " lipid bilayer simulation ran with \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Gromacs 4.0.7\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " using Reaction Field for computing electrostatics, \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Poger GROMOS 53A6 L\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " force field in full hydration (dx.doi.org/10.1002/jcc.21396), \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    100ns\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       ", T \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    323K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       ", 128 \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DPPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " molecules, 5841 \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " molecules. This data is ran for the nmrlipids.blospot.fi project. More details from nmrlipids.blospot.fi and https://removed If data is used, please cite nmrlipids.blogspot.fi project and the original publication of the parameters (dx.doi.org/10.1002/jcc.21396). </div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_3361945.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Structure and Stability of [4Fe-4S]-Maquettes with Non-coded Amino Acids<br>The dataset provides supporting information to a hypothesis of coded Cys being thermodynamically superior to non-coded homoCys and thioGly analogues. The results were first presented at the 30/80 Chemobrionics 2019 meeting in Granada, Spain (www.chemobrionics.eu) and the corresponding publication appears in the thematic issue Frontiers Interface RSC journal. The data set organized accordingly to the published figures and tables. Folders starting with numbers correspond to figures in the paper. The force field key files are to be used in combination of the amber99sb.prm force field, as provided by the Tinker 8.7 (dasher.wustl.edu/tinker/). The full secondary structure analysis was carried out using our in-house code (10.5281/zenodo.1442864). The folder 'trajectories' contains the peptides without the waterbath, while the 'waterbath' folder has the NPT equilibrated 6 nm waterbath in truncated octahedron geometry. The files starting with thermodynamics in the top folder has the numerical results used for calculating the cluster stability and ligand exchange reaction energetics. The folders 'reference cubane' and 'maquette with short peptides' contains the spin-polarized, ferro and antiferro-magnetically coupled electronic structure for all [4Fe-4S] cluster complexes. The key for the file extensions is as follows: FCHK: formatted \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Gaussian16\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " checkpoint file with spin polarized wave function KEY: Tinker control keyword file LOG: output file generated by a computer code LST: clear text file with the content of a compressed tar archive MSV: Discovery Studio Viewer Pro 5.0 binary file MD5: file checksum according to MD5 protocol ODS: Open Office electronic spreadsheet file PDW: PsiPlot data file PDB: Protein Databank Files PGW: PsiPlot graphics file PRM: Tinker force field parameter file TGZ: compressed tar archive TXT: clear text file or raw data set TXYZ: Tinker XYZ file with atomic, positions, atom types, and connectivity XYZ: Xmol file with atomic positions in Cartesian coordinates </div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_14428978.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Mechanism of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Glycans\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Modulating \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Cholesteryl Ester Transfer Protein\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ": Unveiled by Molecular Dynamics Simulation<br>Inhibition of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    cholesteryl ester transfer protein\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CETP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") has been considered as a promising way for the treatment of cardiovascular disease (CVD) for three decades. However, clinical trials of several \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CETP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " inhibitors with various potencies have been marginally successful at best, raising doubts on the target drugability of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CETP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". The in-depth understanding of the glycosylated \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CETP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " structure could be beneficial to more definitive descriptions of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CETP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " function and the underlying mechanism. In this work, large-scale molecular dynamics simulations were performed to thoroughly explore the mechanism of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    glycans\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " modulating \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CETP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". Here, the extensive simulation results intensely suggest that glycan88 tends to assist \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CETP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " in forming a continuous tunnel throughout interacting with the upper-right region of the N-barrel, while it also could prevent the formation of a continuous tunnel by swinging toward the right-rear of the N-barrel. Furthermore, glycan240 formed stable H-bonds with Helix-B and might further stabilize the central cavity of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CETP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". Furthermore, the nonspecific involvement of the hydroxyl groups from the various \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    glycans\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " with protein core interactions and the similar influence of different \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    glycans\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " trapped at similar regions on the protein structure suggest that physiological glycan may lead to a similar effect. This study would provide valuable insights into devising novel methods for CVD treatment targeting \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CETP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and functional studies about glycosylation for other systems.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_5875734.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Data for 'Dynamical origins of heat capacity changes in enzyme catalysed reactions'<br>Dynamical origins of heat capacity changes in enzyme catalysed reactionsMarc W van der Kamp, Erica J. Prentice, Kirsty L. Kraakmann, Michael Connolly, Adrian J. Mulholland &amp; Vickery L. ArcusSchool of Biochemistry, Biomedical Sciences Building, University Walk, University of Bristol, BS8 1TD, UK.Centre of Computational Chemistry, School of Chemistry, Cantock's Close, University of Bristol, BS8 1TS, UK.School of Science, University of Waikato, Hamilton, New Zealand.The data files contained in this repository are as follows: 1a. Input coordinates (starting structures) in PDB format are named as enzyme state.pdb. 1b. Input coordinates are named as enzyme state.inpcrd and are in the standard \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Amber\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " format (generated with \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Amber12\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       "). 2. Topology and parameter files are named as enzyme state.prmtop and are also in standard \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Amber\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " format (generated with \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Amber12\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       "). 3. Input files for the MD simulations are in the subdirectories KSI md input files and MalL md input files. Input files are included for the equilibration, heating and production runs of molecular dynamics, which were performed in this order: a. minsolv.i b. mdsolv.i c. min.i d. heat.i e. npt release1.i f. npt release2.i g. npt release3.i h. npt release4.i i. eq npt.i j. md10.i For KSI, a restraint was used in both RS and IS states, which is included in the file r7.RST. A restraint was also used in the MalL states (RS and TSA), which is included in the file d com.rst. All input files are in the standard \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Amber\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " format (see e.g. Amber16.pdf manual). </div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_19102457.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    LNA\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-induced dynamic stability in a therapeutic \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    aptamer\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ": insights from molecular dynamics simulations<br>Modulation of structural and thermodynamic properties of nucleic acids with synthetic modifications is a promising area of research with possible applications in nanotechnology and nanotherapeutics. \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Locked nucleic acid\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    LNA\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") is one such modification in which the C4 and O2 atoms of the sugar moiety are connected through a \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    methylene\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " bridge. The \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    LNA\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " modified \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DNA aptamer\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    RNV66\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", and its unmodified counterpart \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    V7t1\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", both of which target the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    vascular endothelial growth factor\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    VEGF\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") implicated in oncogenic angiogenesis, have a G-rich tract that can fold into G-quadruplex structures. However, it is not understood why \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    V7t1\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " has a polymorphic structure while its \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    LNA\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " modified counterpart \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    RNV66\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " has a unique quadruplex fold with higher nuclease resistance, thermal stability and greater binding affinity for \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    VEGF\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". In this work, we have performed extensive molecular dynamics simulations of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    RNV66\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    V7t1\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " to study and compare the structural and dynamic consequences of the insertion of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    LNAs\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". It was observed that the increase in dynamic stability was significant in the presence of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    LNA\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " residues and our protocol for combining different torsional parameters using \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    OL15\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " for the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DNA aptamer\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    parm99\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    LNA\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " along with \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    parmbsc0\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    OL15\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " for the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    LNAs\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " nicely reproduced the experimentally observed conformational features of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    RNV66\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". Our observations would help in further theoretical studies in understanding the lack of frustration in the folding of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    LNA\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " modified aptamer and its higher affinity for \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    VEGF\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". Communicated by Ramaswamy H. Sarma</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_7533587.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Molecular Dynamics Simulations of \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    HADDOCK\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       "-predicted Complex Structures of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    apoE2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Factor H\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "<br>Input and output data for the molecular dynamics simulations of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    FH5 7\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "/\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ApoE2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " complex. Initial structures generated with the \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    HADDOCK v2.4\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " web server with 3 nm distance restraints for lysine pairs that were experimentally found to be cross-linked with DSS. Five clusters and the four representative structures provided by \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    HADDOCK\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " were then used for atomistic molecular dynamics simulations. These structures were solvated and simulated with both \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CHARMM36m\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Amber FF14SB\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " force fields for \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    250 ns\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       " each using \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GROMACS 2021\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       ". The recommended simulation parameters were used for both force fields, and they are available in the mdp files. For each of these 5 (clusters) x 4 (structures per cluster) x 2 (force fields)   40 simulations, the outputs and inputs are provided; the trajectory (xtc), energy file (edr), final structure (gro), run parameter file (tpr), and continue point (cpt) are system-specific, whereas a single topology (top) and index file (ndx) is shared among all simulations with the same force field. The molecule definitions (itp) referred to in the topology are provided in the compressed files.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_19673947.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Computational Modeling of Conformer Stability in \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Benenodin-1\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", a Thermally Actuated Lasso Peptide Switch<br>\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Benenodin-1\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " is a thermally actuated lasso peptide rotaxane switch with two primary translational isomers that differ in the relative position of the residue Gln15. The conversion from one conformer to the other involves substantial enthalpy entropy compensation: one conformer is energetically favored and the other is entropically favored. Here, we take a multi-scale quantum mechanical (QM) and classical molecular dynamic (MD) approach to reveal residue-specific sources of these differences in stability. QM reveals that the two \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    benenodin-1\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " conformers involve distinct hydrogen bonding networks, with the enthalpically favored conformer having more intra-peptide hydrogen bonds between the Gln15 side chain and nearby residues. The evaluation of configurational entropy over the MD-sampled geometries reveals that the entropically favored conformer has enhanced conformational flexibility. By computing the by-residue-sum entropies, we identify the role of Gln15 and neighboring Glu14 in mediating the entropic variation during the switching process. These computational insights help explain the effects of Glu14Ala and Gln15Ala mutations on the conformational population of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    benenodin-1\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " observed experimentally.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_6797842.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Raw data for: Sublytic \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    gasdermin-D\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " pores captured in atomistic molecular simulations<br>This archive contains data and code related to &quot;Sublytic \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    gasdermin-D\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " pores captured in atomistic molecular simulations&quot; by Schaefer &amp; Hummer. Included are molecular dynamics parameter files, initial structures after system equilibration, final structures after production, subsampled production trajectories, and python scripts used for analysis. Publication doi: https://removed Contact information: Name: Stefan L. Schaefer Institution: Department of Theoretical Biophysics, Max Planck Institute of Biophysics Address: Max-von-Laue-Str. 3, 60438 Frankfurt am Main, Germany Email: stefan.schaefer@biophys.mpg.de</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_6380887.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Data from: Protein Conformational Space at the Edge of Allostery: Turning a Non-allosteric \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Malate Dehydrogenase\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " into an &quot;Allosterized&quot; Enzyme using Evolution Guided Punctual Mutations<br>This data accompanies the paper entitled Protein Conformational Space at the Edge of Allostery: Turning a Non-allosteric \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Malate Dehydrogenase\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " into an Allosterized Enzyme using Evolution Guided Punctual Mutations The zip archive contains the results of molecular dynamics simulations of the 4 systems investigated in the paper: wt of A. ful \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    MalDH\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and three mutants. Each system has been simulated at two temperatures, \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    300 K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    340 K.\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       " Starting configurations of the proteins after equilibration are provided for all the systems in GRO \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Gromos87\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " format. Trajectories with the positions of the proteins every 100 ps are provided for all the systems in XTC \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    gromacs\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " format.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_7192724.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Interaction of the inhibitory peptides \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ShK\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    HmK\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " with the voltage-gated potassium channel \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    KV1.3\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ": Role of conformational dynamics<br>ABSTRACT: Peptide toxins that adopt the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ShK\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " fold can inhibit the voltage-gated potassium channel \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    KV1.3\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " with IC50 values in the pM range, and are therefore potential leads for drugs targeting autoimmune and neuroinflammatory diseases. NMR relaxation measurements and pressure-dependent NMR have shown that, despite being cross-linked by disulfide bonds, \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ShK\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " itself is flexible in solution. This flexibility affects the local structure around the pharmacophore for \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    KV1.3\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " channel blockade and, in particular, the relative orientation of the key Lys and Tyr side chains (Lys22 and Tyr23 in \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ShK\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "), and has implications for the design of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    KV1.3\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " inhibitors. In this study, we have performed molecular dynamics (MD) simulations on \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ShK\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and a close homolog, \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    HmK\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", in order to probe the conformational space occupied by the Lys and Tyr residues, and docked the different conformations with a recently determined cryo-EM structure of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    KV1.3\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " channel. Although \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ShK\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    HmK\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " have 60% sequence identity, their dynamic behaviors are quite different, with \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ShK\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " sampling a broad range of conformations over the course of a \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    5 μs\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       " MD simulation, while \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    HmK\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " is relatively rigid. We also investigated the importance of conformational dynamics, in particular the distance between the side chains of the key dyad Lys22 and Tyr23, for binding to \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    KV1.3\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". Although these peptides have quite different dynamics, the dyad in both adopts a similar configuration upon binding, revealing a conformational selection upon binding to \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    KV1.3\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " in the case of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ShK\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " . Intriguingly, the more flexible peptide, \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ShK\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " , binds with nearly 300-fold higher affinity than \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    HmK\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " .</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_20017637.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">On the Possibility That Bond Strain Is the Mechanism of RING \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    E3\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Activation in the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    E2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-Catalyzed Ubiquitination Reaction<br>Ubiquitination is a type of post-translational modification wherein the small protein \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ubiquitin\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Ub\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") is covalently bound to a lysine on a target protein. Ubiquitination can signal for several regulatory pathways including protein degradation. Ubiquitination occurs by a series of reactions catalyzed by three types of enzymes: \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ubiquitin activating enzymes\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    E1\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "; \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ubiquitin conjugating enzymes\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    E2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "; and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ubiquitin ligases\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    E3\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    E2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " enzymes directly catalyze the transfer of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Ub\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " to the target protein the RING \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    E3\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " improves the efficiency. Prior to its transfer, \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Ub\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " is covalently linked to the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    E2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " via a thioester bond and the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Ub\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    E2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " conjugate forms a quaternary complex with the RING \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    E3\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". It is hypothesized that the RING \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    E3\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " improves the catalytic efficiency of ubiquitination by placing the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    E2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Ub\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " conjugate in a closed position, which tensions and weakens the thioester bond. We interrogate this hypothesis by analyzing the strain on the thioester during molecular dynamics simulations of both open and closed \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    E2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Ub\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "/\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    E3\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " complexes. Our data indicate that the thioester is strained when the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    E2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Ub\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " conjugate is in the closed position. We also show that the amount of strain is consistent with the experimental rate enhancement caused by the RING \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    E3\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". Finally, our simulations show that the closed configuration increases the populations of key hydrogen bonds in the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    E2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Ub\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " active site. This is consistent with another hypothesis stating that the RING \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    E3\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " enhances reaction rates by preorganizing the substrates.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_19657472.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Recognition and Binding of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    RsmE\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " to an \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    AGGAC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Motif of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    RsmZ\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ": Insights from Molecular Dynamics Simulations<br>\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CsrA\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "/\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    RsmE\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " is a post-transcriptional regulator protein widely distributed in bacteria. It impedes the expression of target \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    mRNAs\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " by attaching their 5 untranslated region. The translation is restored by small, noncoding \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    RNAs\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " that sequester \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CsrA\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "/\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    RsmE\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " acting as sponges. In both cases, the protein recognizes and attaches to specific \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    AGGAX\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    AXGGAX\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " motifs, where X refers to any nucleotide. \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    RsmZ\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " of Pseudomonas \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    protegens\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " is one of these small \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    RNAs\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". The structures of some of its complexes with \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    RsmE\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " were disclosed a few years ago. We have used umbrella sampling simulations to force the unbinding of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    RsmE\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " from the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    AGGAC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " motif located in the single-stranded region sited between stem loops 2 and 3 of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    RsmZ\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " . The calculations unveiled the identity of the main residues and nucleotides involved in the process. They also showed that the region adopts a hairpin-like conformation during the initial stages of the binding. The ability to acquire this conformation requires that the region has a length of at least nine nucleotides. Besides, we performed standard molecular dynamics simulations of the isolated fragments, analyzed their typical conformations, and characterized their movements. This analysis revealed that the free molecules oscillate along specific collective coordinates that facilitate the initial stages of the binding. The results strongly suggest that the flexibility of the single-stranded region of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    RsmZ\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " crucially affects the ability of its binding motif to catch \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    RsmE\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " .</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_21940502.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Molecular Dynamics with Chemical Accuracy \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Alkane\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Adsorption in \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Acidic Zeolites\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "<br>For the adsorption of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    methane\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " to \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    hexane\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " on \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    acidic zeolites\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " with varying pore sizes (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Socony Mobil-5\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    MFI\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "), \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    chabazite\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CHA\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "), \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    faujasite\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    FAU\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ")) and Br nsted acid site concentrations, heats of adsorption are predicted. The widely applied standard model of computational catalysis, density functional theory with some account of dispersion (DFT-D) and the harmonic approximation for local sampling of the potential energy surface (PES), leads to a large mean absolute deviation (MAD) from experiment of 17.2 kJ mol 1, far outside chemical accuracy limits ( 4 kJ mol 1). Passing either to molecular dynamics (MD) at the DFT-D level or to wave function-based electron correlation methods (second-order M ller Plesset perturbation theory MP2) for energies at local minimum structures reduces the MAD to 8.7 and 5.9 kJ mol 1, respectively, still outside the chemical accuracy range. We present MD simulations on an MP2 quality PES, which strongly reduces the MAD to 1.9 kJ mol 1. This has been achieved by finding two descriptors for the MP2 DFT-D energy differences, which reduces the total number of required MP2 calculations to only 36 and, hence, the computational demand by several orders of magnitude. The predicted heats of adsorption at reaction temperatures (650 K) support experimental results derived from spectroscopic measurements. They show that the observed decrease of experimental apparent barriers from \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    propane\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " to \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    pentane\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " for \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    alkane\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " cracking in \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    MFI\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (28 kJ mol 1) is largely due to increasing adsorption strengths (21 kJ mol 1) and to a much smaller extent to decreasing intrinsic barriers (7 kJ mol 1).</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_3362889.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Pyridinylimidazoles\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " as \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GSK3 inhibitors\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ": the impact of tautomerism on compound activity via water networks<br>Data related to publication: Heider et al.: \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Pyridinylimidazoles\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " as \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GSK3 inhibitors\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ": the impact of tautomerism on compound activity via water networks. The files include: 1) Output conformations of the QM Tautomer &amp; Conformation Predictor of \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Maestro\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " (Schro dinger, LLC, New York, NY, 2018) of the compounds 1, 3a, 3b, 3c, 3d, 3e, 3f, 3g, 3h, 3j, 3k, 3l, 3m, 6f and 6g [.mae and .sdf files]. 2) Movies of the MD simulations of compounds 3a, 3j, 4a, 4b, 6b, 6g (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    6GN1\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") and movies (* 2) of 3a, 3j, 6b, 6g (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    4PTC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") [.mpg files] . 3) Raw \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Desmond\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " trajectory files of the MD simulations of the compounds 3a, 3j, 4a, 4b, 6b, 6g [out.cms and the full trj files]. </div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_3975394.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Molecular dynamics trajectories for \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    SARS-CoV-2 Mpro\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " with 7 \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    HIV inhibitors\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "<br>Raw trajectory data (\n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GROMACS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " format) of all atom molecular dynamics simulation of COVID-19 related \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    SARS-CoV-2 dimeric main protease\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (based on PDB \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    6LU7\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") with 7 kinds of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    HIV inhibitors\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    darunavir\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    indinavir\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    lopinavir\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    nelfinavir\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ritonavir\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    saquinavir\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    tipranavir\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") were calculated on massively parallel supercomputer HOKUSAI Big Waterfall at RIKEN ISC, and a special-purpose computer, MDGRAPE-4A, at RIKEN BDR, JAPAN. For each ligand, \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    200ns\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       " length 28 trajectories were calculated. Some of these trajectories were calculated further longer. We can observe formation of encounter complex and investigate potential binding sites on the surface of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    dimeric protease\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". We hope that these raw data are valuable for further drug repurposing/development research targeting the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    SARS-CoV-2 main protease\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". We will submit analysis of these data to refereed journal. Molecular dynamics simulations were performed under NVT at \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    310K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       ", with the time step 2.5fs. The starting structure was prepared based on PDB \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    6LU7\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", with \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    amber14sb\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " force field in about 10nm cubic box with periodic boundary conditions. The ligands were initially placed apart from the active sites of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    dimeric main protease\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". We have also already deposited \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    10 microseconds\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       " trajectories of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    dimeric protease\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " without ligand (with \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    amber99sb-ildn\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " force field) in the repository https://data.mendeley.com/datasets/vpps4vhryg/1 (DOI:10.17632/vpps4vhryg.1). Files: LIG 28traj200ns every200ps.zip (28trajectories for each ligand) traj200ns every200ps/LIG/a/traj200ns every200ps LIG-a-n.xtc (trajectory in GROMACS XTC) traj200ns every200ps/LIG/a/conf.gro (initial condition in GROMACS GRO) traj200ns every200ps/LIG/topology/ (contains topology files) traj200ns every200ps/LIG/mdp/ (contains run paramter files) ZZZ 20traj1us every200ps.zip (20trajectories extended to 1microsecond) traj1us every200ps/traj1us every200ps LIG-a-n.xtc DAR-C-06, DAR-D-07 IND-C-05, IND-C-06, IND-D-06 LOP-A-02, LOP-D-03 NEL-B-01, NEL-C-07, NEL-D-02 RIT-B-07, RIT-C-07 SAQ-B-01, SAQ-C-04, SAQ-D-03 TPR-A-07, TPR-B-04, TPR-B-06, TPR-C-05, TPR-D-02 ZZZ 3traj6us every1ns.zip (3trajectories extended to \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    6microseconds\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       " or more) traj6us every1ns/traj6us every1ns LIG-a-n.xtc IND-D-06, NEL-B-01, TPR-B-04 ZZZ LigandBindingPosePDBs.zip (pickup 3 snapshots for each ligand) LigandBindingPosePDBs/LIG-a-n frame.pdb movies overlooking 28traj200ns.zip (7x2movies) movies 28traj200ns/movie overlooking LIG 28traj200ns-viewA.mp4 inspecting 28traj at once movies 28traj200ns/movie overlooking LIG 28traj200ns-viewB.mp4 from the opposite angle movies 1us.zip (17movies) movies 1us/movie LIG-a-n 1us.mp4 movies 6us.zip (3movies) movies 6us/movie LIG-a-n 6us.mp4 where LIG {DAR,IND,LOP,NEL,RIT,SAQ,TPR} DAR:darunavir IND:indinavir NEL:nelfinavir RIT:ritonavir SAQ:saquinavir TPR:tipranavir a {A,B,C,D} n {01,02,03,04,05,06,07} ZZZz 3traj1us every200ps unbinding.zip (3trajectories extended to 1microsecond exhibiting unbinding) ZZZz 56traj200ns every200ps negative control.zip (56trajectories as a negative control) ZZZz LigandBindingPosePDBsRevised.zip (pickuped 3 snapshots for each ligand) </div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_19534516.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">All-atom molecular simulation study of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    cellulose acetate\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ": amorphous structure and the dissolution of small molecule<br>All-atom molecular dynamics (MD) simulation was conducted for \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    cellulose acetate\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CA\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "). The intermolecular interactions were analyzed at the amorphous state with degrees of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    acetyl\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " substitution (DS) of 2, 2.5, and 3, and the energetics of dissolution was elucidated for H2O, CO2, and CH4. The MD simulation involved annealing procedures, and the free energy of dissolution was calculated by the energy-representation method of solvation. The setup files for MD and free-energy calculation are provided here.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_16410502.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Determining structure and action mechanism of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    LBF14\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " by molecular simulation<br>The recently discovered, membrane active peptide \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    LBF14\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " contains several non \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    proteinogenic amino acids\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and is able to transform vesicles into tubule networks. The exact membrane interaction mechanism and detailed secondary structure are yet to be determined. We performed molecular dynamics simulations of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    LBF14\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and let it fold de novo into its ensemble of native secondary structures. Histidine protonation state effects on secondary structure were investigated. An MD simulation of the peptide with a lipid bilayer was performed. Simulation results were compared to circular dichroism and electron paramagnetic resonance data of previous studies. \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    LBF14\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " contains a conserved helical section in an otherwise random structure. Helical stability is influenced by histidine protonation. The peptide localized to the polar layer of the membrane, consistent with experimental results. While the overall secondary structure is unaffected by membrane interaction, Ramachandran plot analysis yielded two distinct peptide conformations during membrane interaction. This conformational change was accompanied by residue repositioning within the membrane. \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    LBF14\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " only affected the local order in the membrane, and had no measurable effect on pressure. The simulation results are consistent with the previously proposed membrane interaction mechanism of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    LBF14\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and can additionally explain the local interaction mechanism. Communicated by Ramaswamy H. Sarma</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_13387534.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Understanding the Conformational Properties of Fluorinated Polypeptides: Molecular Modelling of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Unguisin A\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "<br>In this work, we investigate the conformational properties of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    unguisin A\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", a natural macrocyclic heptapeptide that incorporates a \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    aminobutyric acid\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Gaba\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "), and four of its difluorinated stereoisomers at the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Gaba\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " residue. According to nuclear magnetic resonance (NMR) experiments, their secondary structure depends dramatically on the stereochemistry of the fluorinated carbon atoms. However, many molecular details of the structure and flexibility of these systems remain unknown, so that a rationale of the conformational changes induced by the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    fluorine\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " atoms in the macrocycle is still missing. To fill this gap, we apply enhanced molecular dynamics (MD) techniques to explore the peptide conformational space in \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    dimethyl sulfoxide\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " solution followed by \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    4-8 μs\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       " of conventional MD simulations that provide extensive equilibrium sampling. The simulations, which compare reasonably well with the NMR-based observations, show that the secondary structure of the macrocycle is altered substantially upon fluorination, except for the (S,S) diastereomer. It also turns out that the conformations of the fluorinated peptides are visited during the enhanced MD simulation of natural \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    unguisin A\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", suggesting thus that conformations accessible to the unsubstituted macrocyclic peptide may be selected by fluorination. Therefore, computational characterization of the macrocyclic peptides could be helpful in the rational design of stereoselective fluorinated peptides with fine-tuned conformation and activity.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_6868243.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">On equilibrating non-periodic molecular dynamics samples for coupled particle-continuum simulations of amorphous polymers: dataset<br>Abstract: (from [1]) In the context of fracture simulations of polymers, the molecular mechanisms in the vicinity of the crack tip are of particular interest. Nevertheless, to keep the computational cost to a minimum, a coarser resolution must be used in the remaining regions of the numerical sample. For the specific case of amorphous polymers, the Capriccio method bridges the gap between the length and time scales involved at the different levels of resolution by concurrently coupling molecular dynamics (MD) with the finite element method (FEM). Within the scope of the Capriccio approach, the coupling to the molecular MD region introduces non-periodic, so-called stochastic boundary conditions (SBC). In similarity to typical simulations under periodic boundary conditions (PBC), the SBC MD simulations must reach an equilibrium state before mechanical loads are exerted on the coupled systems. In this contribution, we hence extensively study the equilibration properties of non-periodic MD samples using the Capriccio method. We demonstrate that the relaxation behavior of an MD-FE coupled MD domain utilizing non-periodic boundary conditions is rather insensitive to the specific coupling parameters of the method chosen to implement the boundary conditions. The behavior of an exemplary system equilibrated with the parameter set considered as optimal is further studied under uniaxial tension and we observe some peculiarities in view of creep and relaxation phenomena. This raises important questions to be addressed in the further development of the Capriccio method. Contact: Felix Weber Institute of Applied Mechanics Friedrich-Alexander-Universita t Erlangen-Nu rnberg Egerlandstr. 5 91058 Erlangen Germany Context: This dataset contains the results presented in [1] and related data. Content: Throughout this data set, \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Lammps\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " [2] real units are used. The following folders contain the results obtained under periodic boundary conditions: - biax PBC: biaxial loading - equil PBC: equilibration - ut PBC: uniaxial tension Each simulation directory contains: - input.prm: input parameters of the specific simulation (read by the input file) - job.out: simulation log file - meta.info: meta data of the specific simulation run - \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Lammps\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " input file (*.in) of the specific simulation - \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Lammps\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " data file (*.data, molecular style) of the investigated sample - \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    LAMMPS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " out: resulting \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Lammps\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " data and restart (*.rst) files and simulation results (\n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Lammps\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " thermo out) in tabulated form, an overview of the columns is given in the respective folders The following folders contain the results obtained under stochastic boundary conditions using the Capriccio method [3]: - best: equilibration with the parameter set considered to be most suitable for the MD-FE coupled equilibration - biax: biaxial loading - bridging: equilibration with different adaptivity levels of the bridging domain - descr obs: equilibration with a Lagrangian frame for the description of the observation region - dpd: equilibration with different thicknesses of the dissipative particle dynamics (DPD) region - friccoeff: equilibration with different friction coefficients applied in the dissipative particle dynamics region - fur: equilibration with different numbers of fur beads - gausspoints: equilibration with different numbers of quadrature points per direction - min: equilibration applying an initial, static energy minimization - nodes: equilibration with a higher number of finite element nodes - shifted: equilibration with particle systems obtained at different positions and points in time within the periodic master system - smdcub: equilibration with different remaining stiffness ratios for the cubic modified weighting factor - smdlin: equilibration with different remaining stiffness ratios for the linear modified weighting factor - timestepsize: equilibration with different molecular dynamics time step sizes - ut: uniaxial tension - ut friccoeff: uniaxial tension with different friction coefficients applied in the dissipative particle dynamics region - weighting: equilibration with different energy weighting functions - youngsmod: equilibration with different Young's moduli Each simulation directory contains: - input files: Abaqus [4] input file (*.inp) and \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Lammps\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " data file (*.data, molecular style) of the investigated sample - MD data: Results evaluated in the molecular dynamics region. MD data contains the following subfolders: anchorforces (dumped force components on the anchor points (AP) in kcal/mol, files anchorforce [load step] [MD-FE iteration].AF), data (resulting \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Lammps\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " data files *.[load step].[MD-FE iteration].data), Density (dumped mass density in the observation region in kg/m^3), Energy (dumped total (kinetic + potential), angle, bond, and pair energies in kcal/ (mol*Angstrom)), Strain (integral strains in the observation region in x-, y-, and z-direction calculated by means of the Matlab [5] script calc OBSstrain.m), Stress (stresses in the observation region in MPa), and Temperature (temperature in the observation region in K) - job.out: simulation log file - meta.info: meta data of the specific \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Lammps\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " simulation Information on the subfolders is given in the respective folders. References: [1] F. Weber, M. Ries, C. Bauer, C. R. Wick, S. Pfaller, &quot;On equilibrating non-periodic molecular dynamics samples for coupled particle-continuum simulations of amorphous polymers&quot;, Forces in Mechanics, 2023, 10, 100159. [2] A. P. Thompson, H. M. Aktulga, R. Berger, D. S. Bolintineanu, W. M. Brown, P. S. Crozier, P. J. in 't Veld, A. Kohlmeyer, S. G. Moore, T. D. Nguyen, R. Shan, M. J. Stevens, J. Tranchida, C. Trott, S. J. Plimpton, &quot;\n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    LAMMPS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " - a flexible simulation tool for particle-based materials modeling at the atomic, meso, and continuum scales&quot;, Computer Physics Communications, 2022, 271, 108171. [3] S. Pfaller, M. Rahimi, G. Possart, P. Steinmann, F. Mu ller-Plathe, M. C. Bo hm, &quot;An Arlequin-based method to couple molecular dynamics and finite element simulations of amorphous polymers and nanocomposites&quot;, Computer Methods in Applied Mechanics and Engineering, 2013, 260, 109-129. [4] Dassault Syste mes, &quot;Abaqus documentation&quot;, URL: https://abaqus-docs.mit.edu/2017/English/SIMACAEEXCRefMap/simaexc-c-docproc.htm. [5] The MathWorks, Inc, &quot;MATLAB. The Language of Technical Computing&quot;, URL: https://de.mathworks.com/help/matlab/. Funding: This research was funded by the Deutsche Forschungsgemeinschaft (DFG, German Research Foundation) - 377472739/GRK 2423/1-2019. The authors are very grateful for this support. Sebastian Pfaller is furthermore funded by the Deutsche Forschungsgemeinschaft (DFG, German Research Foundation) - 396414850 (Individual Research Grant Identifikation von Interphaseneigenschaften in Nanokompositen ).</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_1144128.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Atomistic trajectories from ab-initio molecular dynamics simulations of wetted \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    TiO2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " nanoparticle<br>This repository contains atomistic trajectories from ab-initio molecular dynamics simulations of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    TiO2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " nanoparticle described in the paper: E. G. Brandt, L. Agosta and A.P.Lyubartsev, &quot; Reactive wetting properties of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    TiO2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " nanoparticles predicted by ab initio molecular dynamics simulations&quot;, Nanoscale, 8, 13385-13398 (2016) DOI: 10.1039/c6nr02791a The trajectories are saved in the .xtc format, and initial structures with specification of atom types are given in the .pdb format. The name of each file contains brief information about the simulated system: \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    TiO2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " : composition of the nanoparticle n24 : number of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    TiO2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " units in the nanoparticle anatase/brookite/rutile : type of crystall structure - a number 0 - 30 : number of water molecules in the simulation 2fs - the time step For more details, see the referred paper</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_3813275.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">The Unfolding Journey of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Superoxide Dismutase 1\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Barrels Under Crowding: Atomistic Simulations Shed Light on Intermediate States and Their Interactions With Crowders<br>This data accompanies the article entitled The Unfolding Journey of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Superoxide Dismutase 1\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Barrels Under Crowding: Atomistic Simulations Shed Light on Intermediate States and Their Interactions With Crowders, published in J. Phys. Chem. Lett. (https://doi.org/10.1021/acs.jpclett.0c00699). 01_SOD1bar_unfolding_REST2.zip: The zip archive includes REST2 trajectories for the three systems investigated in the paper: 1:1 packing, 2:1 packing, and the dilute case. The trajectories are saved in the \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GROMACS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " XTC file format, separately for each temperature (i=0,...,23). Given the large trajectory sizes, only protein coordinates (SOD1bar + crowders) are reported, and the output frequency is reduced to 100 ps. A starting geometry (in the \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Gromos87\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " GRO format) after equilibration of the initial packing is provided for each REST2 simulation (conf_prot.gro). Moreover, for each REST2 simulation, an xarray (http://xarray.pydata.org) dataset, saved in the netCDF file format, is included with computed fraction of native contacts, secondary structure content, and the Calpha RMSD of the barrel core (beta sheets beta1 - beta8) with respect to the crystal structure. 02_SOD1bar_geometries_representative_unfolding.zip: Representative SOD1bar geometries along the unfolding pathway (presented in Figure 3 of the paper). 03_SOD1bar_geometries_loopVII.zip: SOD1bar geometries with varying loop VII conformation which were isolated from dilute REST2 and which are presented in Figure S9 of the paper.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_7279100.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Steered molecular dynamics simulations reveal critical residues for (un)binding of substrates, inhibitors and a product to the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    malarial M1 aminopeptidase\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "<br>Malaria is a life-threatening disease spread by mosquitoes. \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Plasmodium falciparum M1 alanyl aminopeptidase\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PfM1-AAP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") is a promising target for the treatment of malaria. The recently solved crystal structures of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PfM1-AAP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " revealed that the buried active site can be accessed through two channel openings: a short N-terminal channel with the length of 8 A and a long C-terminal channel with the length of 30 A . It is unclear, however, how substrates and inhibitors migrate to the active site and a product of cleavage leaves. Here, we study the molecular mechanism of substrate and inhibitor migration to the active site and the product release using steered molecular dynamics simulations. We identified a stepwise passage of substrates and inhibitors in the C-terminal channel of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PfM1-AAP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", involving (I) ligand recognition at the opening of the channel, (II) ionic translation to the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " reservoir , (III) ligand reorientation in the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " reservoir and (IV) passage in a suitable conformation into the active site. Endorsed by enzymatic analysis of functional recombinant \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PfM1-AAP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and mutagenesis studies, our novel ligand-residue binding network analysis has identified the functional residues controlling ligand migration within the C-terminal channel of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PfM1-AAP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". Furthermore, from unbinding simulations of the Arg product we propose a charge repulsion as the driving force to expel the product out from the N-terminal channel of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PfM1-AAP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". Our work paves the way towards the design of a novel class of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PfM1-AAP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " inhibitors based on preventing substrate entry to the active site.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_13352375.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Trajectory data generated by a virtual-system coupled canonical molecular dynamics simulation with a coarse-grained model of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    glutamine\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    octapeptide\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " dimer.<br>The trajectory data of the coarse-grained molecular dynamics simulation with the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    glutamine\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    octapeptide\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " dimer generated by \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    myPresto\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       "/\n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    omegagene\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       ". The details of simulation is reported in the manuscript &quot;\n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    myPresto\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       "/\n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    omegagene 2020\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       ": a molecular dynamics simulation engine for virtual-system coupled sampling&quot; (doi: 10.2142/biophysico.BSJ-2020013) The file &quot;md.trr&quot; is recorded in \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GROMACS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " .trr format, which can be visualized by using some standard molecular viewers such as \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    VMD\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " (https://www.ks.uiuc.edu/Research/vmd/) We recommend visualize the trajectory by using &quot;VDW&quot; or &quot;points&quot; representations for &quot;Drawing Method&quot;. The methods depending on connectivity information, e.g., &quot;NewCartoon&quot; and &quot;lines&quot;, are not working because of coarse graining. This file included the 10,000 snapshots analyzed in the &quot;Example&quot; section of the manuscript. As described in the manuscript, 10 independent runs of 10^7 steps of simulation were done as production runs. The snapshots were recorded at every 10,000 steps, and 1,000 snapshots were recorded for each run. The &quot;md.trr&quot; file was generated by concatenating these snapshots over 10 runs. The file &quot;q8dimer.pdb&quot; is the initial coordinates of the simulation model. Each atom represents a bead corresponding to one amino acid residue. The cell-dimension of cubic periodic boundary cell is 60 A .</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_4806544.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">MD simulations of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    P-glycoprotein\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " started from three different crystal strucures: \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    3G5U\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    4M1M\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    4KSB\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "<br>Molecular simulations of mouse \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    P-glycoprotein\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " started from three different crystal structures with PDB IDs corresponding to \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    3G5U\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    4KSB\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    4M1M\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " . The protein was transferred to \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "/\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    cholesterol\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " bilayer, followed by energy minimization and 10 ns equilibration period. All the simulations were performed in \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Gromacs 3.3.3\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " in conjunction with \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GROMOS54a7\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " force field at \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    300 K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       " and pressure of 1 bar. Each system was run in triplicates for \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    200 ns\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       ". The uploaded fileset contains: - MD protocol used to run simulations for \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    3G5U\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    4M1M\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    4KSB\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " systems; - Example input file for a production run for each system (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    3G5U\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " run1.mdp, \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    4M1M\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " run1.mdp, \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    4KSB\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " run1.mdp); - System topologies (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    3G5U\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " memb.top, \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    4M1M\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " memb.top, \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    4KSB\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " memb.top) and relevant parameter files (3g5u.itp, 4m1m.itp, 4ksb.itp, CLR.itp, POPC.itp, vdw OML CH3L.itp); - Set of three trajectories containing membrane-embedded \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    P-glycoprotein\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " with 1000 frames for each simulated system (snapshots printed every 200 ps); - Movies generated from each trajectory with a more detailed description provided in Movie Legends.pdf; - Model data calculated using principal component analysis method, including the covariance matrices, for each replica (3G5U1-3.eda.npz, 4KSB1-3.eda.npz, 4M1M1-3.eda.npz) and for the concatenated trajectory containing all 9 replicas (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    3G5U\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    4KSB\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " 4M1M.eda.npz). The files containing data for the transmembrane domain only have the tmd prefix. These models can be further explored in \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ProDy\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " package by using loadModel () function. - Figures representing \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    cholesterol\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " distributions in the upper (dark violet) and lower leaflets (light violet) in all three systems at the beginning and at the end of each simulation. </div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_19652739.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Expanding the Accessible Chemical Space of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    SIRT2 Inhibitors\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " through Exploration of Binding Pocket Dynamics<br>Considerations of binding pocket dynamics are one of the crucial aspects of the rational design of binders. Identification of alternative conformational states or cryptic subpockets could lead to the discovery of completely novel groups of the ligands. However, experimental characterization of pocket dynamics, besides being expensive, may not be able to elucidate all of the conformational states relevant for drug discovery projects. In this study, we propose the protocol for computational simulations of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    sirtuin 2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    SIRT2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") binding pocket dynamics and its integration into the structure-based virtual screening (SBVS) pipeline. Initially, unbiased molecular dynamics simulations of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    SIRT2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ":inhibitor complexes were performed using optimized force field parameters of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    SIRT2 inhibitors\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". Time-lagged independent component analysis (tICA) was used to design pocket-related collective variables (CVs) for enhanced sampling of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    SIRT2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " pocket dynamics. Metadynamics simulations in the tICA eigenvector space revealed alternative conformational states of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    SIRT2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " binding pocket and the existence of a cryptic subpocket. Newly identified \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    SIRT2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " conformational states outperformed experimentally resolved states in retrospective SBVS validation. After performing prospective SBVS, compounds from the under-represented portions of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    SIRT2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " inhibitor chemical space were selected for in vitro evaluation. Two compounds, NDJ18 and NDJ85, were identified as potent and selective \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    SIRT2 inhibitors\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", which validated the in silico protocol and opened up the possibility for generalization and broadening of its application. The anticancer effects of the most potent compound NDJ18 were examined on the triple-negative breast cancer cell line. Results indicated that NDJ18 represents a promising structure suitable for further evaluation.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_12986024.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Rigorous Computational Study Reveals What Docking Overlooks: Double Trouble from Membrane Association in \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Protein Kinase C\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Modulators<br>Increasing \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    protein kinase C\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PKC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") activity is of potential therapeutic value. Its activation involves an interaction between the C1 domain and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    diacylglycerol\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DAG\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") at intracellular membrane surfaces; \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DAG\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " mimetics hold promise as new drugs. We previously developed the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    isophthalate\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " derivative \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    HMI-1a3\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", an effective but highly lipophilic (clogP   6.46) \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DAG\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " mimetic. Although a less lipophilic \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    pyrimidine\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " analog, \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PYR-1gP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (clogP   3.30), gave positive results in computational docking, it unexpectedly presented greatly diminished binding to \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PKC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " in vitro. Through more rigorous computational molecular modeling, we reveal that, unlike \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    HMI-1a3\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PYR-1gP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " forms an intramolecular hydrogen bond, which both obstructs binding and reorients \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PYR-1gP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " in the membrane in a fashion that prevents it from correctly accessing the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PKC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " C1 domain. Our results highlight the great value of molecular dynamics simulations as a key component for the drug design process of ligands targeting weakly membrane-associated proteins, where simulation in the relevant membrane environment is crucial for obtaining biologically applicable results.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_167336.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Centered trajectories of NMRlipids II Collection of molecular dynamics simulation trajectories previously published in the NMRlipids community of Zenodo, and the related input files needed for their analysis. These centered (origin at the lipid bilayer center) trajectories were used in the NMRlipids II project. In addition to trajectories, files containing lipidwise \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    C-H\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " order parameters (for the beta and alpha segments of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PC-lipid\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " headgroup) for each frame are provided. Note that this upload is provided just for the ease of repetition of the analysis by the interested reader of the NMRlipids II publication Molecular electrometer and binding of cations to \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    phospholipid\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " bilayers (https://removed If you use any data provided in here for your own work, please cite the corresponding original uploads!</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_11764158.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">6 molecular dynamics simulations of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    coronavirus 2019-nCoV protease\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " model in complex with different conformations of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    lopinavir\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ".<br>Molecular dynamics simulations of comparative model of novel \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    coronavirus 2019-nCoV protease\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Mpro\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " in complex with 6 different conformations based on the Catalophore point-cloud alignment and (re)- docking of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    lopinavir\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " into the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    2019ncov virus protease\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " model. The docking experiment produced 8 clusters of possible conformations, we chose 6 out of 8 conformers and ran an all-atom 300 ps MD at \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    310 K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       " ( \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    36.85 C\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       " ).-The two images in the main folder refers to the docked structures before MD simulation.-The file all centroids.pse contains the frames representing the centroid of the subsequent MD simulation for each docking cluster.-Each archive contains the centroid in PDB format, the starting frame of the simulation in GRO format and the compressed trajectory in XTC format. In the directory &quot;other files&quot; there are other data generated during the simulation, i.e. heatmap representing the contact frequency between the ligand atoms and the ones belonging to the homology model.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_2134639.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Molecular Dynamics Simulations of the Initial-State Predict Product Distributions of Dediazoniation of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Aryldiazonium\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " in Binary Solvents<br>The dediazoniation of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    aryldiazonium\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " salts in mixed solvents proceeds by a borderline SN1 and SN2 pathway, and product distribution should be proportional to the composition of the solvation shell of the carbon attached to the N2 group (ipso carbon). The rates of dediazoniation of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    2,4,6-trimethylbenzenediazonium\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " in \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    methanol\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ethanol\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    propanol\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    acetonitrile\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " were similar, but measured product distributions were noticeably dependent on the nature of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "/cosolvent mixture. Here we demonstrated that solvent distribution in the first solvation shell of the ipso carbon, calculated from classical molecular dynamics simulations, is equal to the measured product distribution. Furthermore, we showed that regardless of the charge distribution of the initial state, i.e., whether the positive charge is smeared over the molecule or localized on \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    phenyl\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " moiety, the solvent distribution around the reaction center is nearly the same.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_14791452.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Datasets and input for &quot;Revisiting the structure of calcined and hydrated \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    AlPO-11\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " with DFT-based molecular dynamics calculations&quot;<br>This dataset contains data as well as sample input files associated with the study Revisiting the structure of calcined and hydrated \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    AlPO-11\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " with DFT-based molecular dynamics calculations. The following files are included in the dataset: CP2K inputs.zip: Sample CP2K input files\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "     AlPO-1\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "1 structures CIF.zip: DFT-optimised structures (CIF format)\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "     AlPO-1\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "1 calcined MD avpos.zip: AIMD average structures of calcined\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "     AlPO-1\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "1 (CIF format) AlPO-11 calcined MD trajectories.zip: Production part of AIMD trajectories, calcine\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    d AlPO-\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "11 (PDB format) AlPO-11 8H2O MD avpos.zip: AIMD average structures of hydrat\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ed AlPO\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-11 (CIF format) AlPO-11 8H2O MD trajectories.zip: Production part of AIMD trajectories, hydra\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ted AlP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "O-11 (PDB format)</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_19656195.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">DataSheet2 A Computational Model for the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PLP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-Dependent Enzyme \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Methionine\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " -Lyase .zip<br>\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Pyridoxal-5 -phosphate\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PLP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") is a cofactor in the reactions of over 160 enzymes, several of which are implicated in diseases. \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Methionine -lyase\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    MGL\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") is of interest as a therapeutic protein for cancer treatment. It binds \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PLP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " covalently through a Schiff base linkage and digests methionine, whose depletion is damaging for cancer cells but not normal cells. To improve \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    MGL\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " activity, it is important to understand and engineer its \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PLP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " binding. We develop a simulation model for \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    MGL\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", starting with force field parameters for \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PLP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " in four main states: two \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    phosphate\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " protonation states and two tautomeric states, keto or enol for the Schiff base moiety. We used the force field to simulate \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    MGL\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " complexes with each form, and showed that those with a fully-deprotonated \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PLP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    phosphate\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", especially keto, led to the best agreement with \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    MGL\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " structures in the PDB. We then confirmed this result through alchemical free energy simulations that compared the keto and enol forms, confirming a moderate keto preference, and the fully-deprotonated and singly-protonated \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    phosphate\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " forms. Extensive simulations were needed to adequately sample conformational space, and care was needed to extrapolate the protonation free energy to the thermodynamic limit of a macroscopic, dilute protein solution. The computed \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    phosphate\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " pKa was 5.7, confirming that the deprotonated, 2 form is predominant. The \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PLP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " force field and the simulation methods can be applied to all \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PLP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " enzymes and used, as here, to reveal fine details of structure and dynamics in the active site.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_10287393.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Cholesteryl Hemisuccinate\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Is Not a Good Replacement for \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Cholesterol\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " in Lipid Nanodiscs<br>Nanodiscs are suitable tools for studies of membrane proteins (MPs) due to their ability to mimic native biological membranes, and several MP structures are solved in nanodiscs. Among the various cell membrane components, \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    cholesterol\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CHL\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") is known to regulate protein function and its concentration can reach up to 50 mol %. However, studies comprising \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    cholesterol\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " are challenging due to its hydrophobic nature, hence, nanodiscs with only a low \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    cholesterol\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " concentration have been studied. To overcome the problem, \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    cholesterol\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " analogs with high solubility in polar solutions are often used, and one of them is \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    cholesteryl hemisuccinate\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CHS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "). Nevertheless, in molecular dynamics (MD) simulation, this is not an obstacle. In this study, we performed MD simulations of nanodiscs containing neutral \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    phosphatidylcholine\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") lipids, negatively charged \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    phosphatidylglycerol\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPG\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") lipids, \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CHL\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", or negatively charged \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    cholesterol\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " analog, \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CHS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". Our simulations show that \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CHS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " increases the order of lipids in nanodiscs; the effect is, however, weaker than \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CHL\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and even smaller in nanodiscs. Furthermore, \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CHS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " gathered around scaffold proteins while \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    cholesterol\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " was uniformly distributed in the nanodiscs. Thus, nanodiscs with \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CHS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " are heterogeneous and not equivalent to nanodiscs with \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CHL\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". Finally, we also observed the increased concentration of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPG\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " near the scaffold proteins, driven by electrostatic interactions. The MD results are experimentally validated using electron paramagnetic resonance spectroscopy. These results show that nanodiscs are, in fact, complex structures not easily comparable with planar \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    lipid\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " bilayers.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_8258741.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">\n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    AMOEBA+\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " Classical Potential for Modeling Molecular Interactions<br>Classical potentials based on isotropic and additive atomic charges have been widely used to model molecules in computers for the past few decades. The crude approximations in the underlying physics are hindering both their accuracy and transferability across chemical and physical environments. Here we present a new classical potential, \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    AMOEBA+\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       ", to capture essential intermolecular forces, including permanent electrostatics, repulsion, dispersion, many-body polarization, short-range charge penetration, and charge transfer, by extending the polarizable multipole-based \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    AMOEBA\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Atomic Multipole Optimized Energetics for Biomolecular Applications\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       ") model. For a set of common organic molecules, we show that \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    AMOEBA+\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " with general parameters can reproduce both quantum mechanical interactions and energy decompositions according to Symmetry-Adapted Perturbation Theory (SAPT). Additionally, a new water model based on the \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    AMOEBA+\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " framework captures various liquid-phase properties in molecular dynamics simulations while remaining consistent with SAPT energy decompositions, utilizing both ab initio data and experimental liquid properties. Our results demonstrate that it is possible to improve the physical basis of classical force fields to advance their accuracy and general applicability.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_20812257.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Capturing the Recognition Dynamics of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    para-Sulfonato-calix[4]arenes\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " by \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Cytochrome c\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ": Toward a Quantitative Free Energy Assessment<br>\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Calix[n]arenes\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " selective recognition of protein surfaces covers a broad range of timely applications, from controlling protein assembly and crystallization to trapping partially disordered proteins. Here, the interaction of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    para-sulfonated calix-[4]-arenes\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " with \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    cytochrome c\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " is investigated through all-atom, explicit \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " molecular dynamics simulations which allow characterization of two binding sites in quantitative agreement with experimental evidence. Free energy calculations based on the MM-PBSA and the attach-pull-release (APR) methods highlight key residues implicated in the recognition process and provide binding free energy results in quantitative agreement with isothermal titration calorimetry. Our study emphasizes the role of MD simulations to capture and describe the walk of sulfonated \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    calix-[4]-arenes\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " on the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    cytochrome c\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " surface, with the arginine R13 as a pivotal interacting residue. Our MD investigation allows, through the quasi-harmonic multibasin (QHMB) method, probing an allosteric reinforcement of several per-residue interactions upon calixarene binding, which suggests a more complex mode of action of these supramolecular auxiliaries.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_6498021.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Molecular simulations matching denaturation experiments for \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    N6-methyladenosine\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "<br>Dataset relative to Molecular dynamics simulation performed for the work &quot;Molecular simulations matching denaturation experiments for \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    N6-methyladenosine\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "&quot;. The dataset contains data of 73 alchemical simulations and is subdivided in 10 zip files, one for each of the systems considered in this work Folders are named following the scheme: system configuration forcefield, or 1M system configuration forcefield for simulation performed at 1 M salt concentrations Each folder contains: - topology files (topol.top, topol RNA chain A.itp, topol RNA chain B.itp) - .mdp files for each replica - index files needed to reconstruct the demuxed trajectories (replica index.xvg , replica index.xvg) - 20 .edr files corresponding to energies for the concatenated trajectory computed for 20 set of DeQs drawn from a gaussian distribution - 16 folders, one for each replica (lam0 ... lam15), containing: - final configuration (confout.gro) - log file (md.log) - dhdl.xvg files needed for computing Free Energies via &quot;gmx bar&quot; command - energies relative to the single replica (ener.edr) - energies for the concatenated trajectories recomputed for the realtive replica hamiltonian (ener trj conc.edr). </div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_19405967.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Specificity and Reactivity of Mycobacterium tuberculosis \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Serine/Threonine Kinases\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PknG\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PknB\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "<br>Mycobacterium tuberculosis (Mtb), the causative agent of Tuberculosis, has 11 eukaryotic-like \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    serine/threonine protein kinases\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", which play essential roles in cell growth, signal transduction, and pathogenesis. \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Protein kinase G\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PknG\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") regulates the carbon and nitrogen metabolism by phosphorylation of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    glycogen accumulation regulator\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GarA\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") protein at Thr21. \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Protein kinase B\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PknB\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") is involved in cell wall synthesis and cell shape, as well as phosphorylates \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GarA\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " but at Thr22. While \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PknG\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " seems to be constitutively activated and recognition of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GarA\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " requires phosphorylation in its unstructured tail, \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PknB\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " activation is triggered by phosphorylation of its activation loop, which allows binding of the forkhead-associated domain of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GarA\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "  In the present work, we used molecular dynamics and quantum-mechanics/molecular mechanics simulations of the catalytically competent complex and kinase activity assays to understand \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PknG\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "/\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PknB\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " specificity and reactivity toward \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GarA\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "  Two hydrophobic residues in \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GarA\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", Val24 and Phe25, seem essential for \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PknG\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " binding and allow specificity for Thr21 phosphorylation. On the other hand, phosphorylated residues in \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PknB\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " bind Arg26 in \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GarA\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and regulate its specificity for Thr22. We also provide a detailed analysis of the free energy profile for the phospho-transfer reaction and show why \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PknG\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " has a constitutively active conformation not requiring priming phosphorylation in contrast to PknB . Our results provide new insights into these two key enzymes relevant for Mtb and the mechanisms of serine/threonine phosphorylation in bacteria.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_6965759.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Reactive \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Oxygen\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Nitrogen\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Species at \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Phospholipid\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Bilayers: \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Peroxynitrous Acid\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and Its Homolysis Products<br>\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Peroxynitrite\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " is a powerful and long-lived oxidant generated in vivo. \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Peroxynitrous acid\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ONOOH\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "), its protonated form, may penetrate into \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    phospholipid\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " bilayers and undergo homolytic cleavage to \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    nitrogen dioxide\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " ( \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    NO2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    hydroxyl\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " radicals ( \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    OH\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "), causing severe nitro-oxidative damage. The membrane environment is thought to influence \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ONOOH\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " reactions, but the mechanisms remain speculative. Most experimental techniques lack the level of resolution required to keep track of the motion of very reactive species and their interactions with the membrane. Here, we performed molecular dynamics simulations of the permeation, interactions, and dynamics of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ONOOH\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and its homolysis products in the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    phospholipid\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " membrane environment. We started by developing an \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ONOOH\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " model that successfully accounted for its conformational equilibria and solvation energies. Membrane permeation of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ONOOH\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " was accompanied by conformational changes. \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ONOOH\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " exhibited a strong tendency to bind to and accumulate at the membrane headgroup region. There, \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ONOOH\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " homolysis led to \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    NO2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " radicals, which in turn partitioned to the membrane interior. About one-third of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    OH\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " radicals readily escaped to the aqueous phase within 1 ns. However, a significant number of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    OH\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " radicals became trapped at the lipid headgroup region for a longer period. The possible implications for membrane-based nitration and oxidation processes were discussed.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_7523635.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Comparative performance of coarse-grained IDP models at different resolutions<br>Files for simulations. \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Molsim\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " start scripts and final outputs, \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    MARTINI\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " Stark start scripts (including force field files, and some intermediate data). Not all intermediate data available, but all data necessary to reproduce simulations should be included. \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    MARTINI\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " simulations follow the procedure &quot;Dry energy minimization&quot; -&gt; &quot;Solvation&quot; -&gt; &quot;Salting&quot; -&gt; &quot;Energy minimization&quot; -&gt; &quot;NVT equilibration&quot; -&gt; &quot;NPT equilibration&quot; -&gt; &quot;Production Run&quot;, where the output from the previous step is the input into the next step. For some folders, the output from previous step has been copied, but not always. In case of another simulation, but using a bigger box, some files from the simulation with smaller box were re-used. Thus, if a file seems to be missing, check the folder with corresponding simulation using a smaller box. Some &quot;analysis files&quot; are included, e.g. the radius of gyration for each time step, in order to allow for the reproduction of graphs in upcoming publication, and further exploration.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_2653735.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Atomistic simulations of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DAPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "/\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    cholesterol\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " bilayers with one \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    cholesterol\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " restrained to the membrane core<br>Membranes consisting of 260 \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DAPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and 28 (10 mol%) \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    cholesterol\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " molecules were simulated at various temperatures of \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    285\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    298\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    310\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    320\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       ", and \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    333 K.\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       " Notably, one \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    cholesterol\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " molecule was restrained to the membrane core. The \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CHARMM36\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " force field [1] was used and the membranes were generated using \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CHARMM-GUI\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " [2]. For simulations without a restrained \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    cholesterol\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", see the other upload at DOI: The simulations were run for \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    2 microseconds\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       " using the \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GROMACS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " simulation suite [3]. Simulation parameters are found in the common mdp file (note that the temperature varies between simulations). The upload contains simulation inputs and outputs that allows the replication, extension, or analysis of the simulation data: Topology files (top) and molecular definitions (itp) Index files (ndx) A common run parameter file (mdp) A run input file (tpr) Trajectory file (xtc) written every 100 ps Energy file (edr) Log file (log) Final structure file (gro) Continue point file (cpt) The files are named LLLL_AA_CHOLXX_CORE_TTT.FFF, where LLLL is the type of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    phospholipid\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " AA stands for all atom (Coarse-grained data in a separate upload) CHOLXX stands for the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    cholesterol\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " concentration (CHOL10 for 10 mol%) TTT is the temperature FFF is the tile type (see above) Note that topologies/index files are the same regardless of temperature, and hence their file names do not have the TTT section. [1] DOI: 10.1021/jp101759q [2] DOI: 10.1021/acs.jctc.5b00935 [3] DOI: 10.1016/j.softx.2015.06.001</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_5018433.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Project files provided as supporting information to the manuscript &quot;How Communication Pathways Bridge Local and Global Conformations in an \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    IgG4 Antibody\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ": a Molecular Dynamics Study&quot;<br>June 23, 2021 Thomas Tarenzi, Marta Rigoli and Raffaello Potestio   The dataset contains the following folders: - contact area binding site: files with the computed surface area, used for the calculation of the contact area between PD-1 and the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    antibody Fab\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (Fig. S41). - hbonds ab-pd1: number of hydrogen bonds between the antigen and the antibody, for each holo cluster (Fig. S41). - mutual information: matrices with the computed mutual information, for each pair of residues (Fig. S34, S35, S46). The folder contains also the generalized correlation coefficients (Fig. S43) and the correlation scores (Fig. 4, S44, S45), computed from the mutual informations. - networks: communities - for each cluster, each residue is assigned to a community within the interaction network (Fig. S32, S33). betweenness - the values of edge betweenness for each cluster (S30, S31). - output clustering: each frame of the apo and holo simulations is assigned a cluster index, on the basis of the structural similarity (Fig. S4). - PAD: per-residue values of PAD parameter, for apo and holo systems (Fig. 3, S39). - PCA: principal component analysis for each conformational cluster (Section S2.1). - representative structures: representative structures for each conformational cluster (Fig. 2). - r gyr antibody: radii of gyration of the antibody, for each cluster (Fig. 2). - r gyr hinge: radii of gyration of the sole hinge segment, for each cluster (Fig. S36). - RMSD antibody: distributions of the root-mean-square deviation of the antibody, for each cluster (Fig. S5). - RMSD antigen: root-mean-square deviation of the antigen PD-1, for each cluster (Fig. S42). - RMSD binding site: distributions of the root-mean-square deviation of the residues belonging to the paratope, for each cluster (Fig. S40, S47). - RMSD matrix: root-mean-square deviation between structures belonging to different pairs of clusters (Fig. S9). - rmsf antigen: root-mean-square fluctuation of the antigen PD-1, for each cluster (Fig. S42). - rmsf hinge: difference between the total root-mean-square fluctuations of the two hinge segments, for each cluster (Fig. S38). - salt bridge: distribution of distances between residues R979 and D1377 (Fig. 4). - sasa domains: contact area between Fab and Fc antibody domains, for each cluster (Fig. S8). - sasa hinge: solvent accessible surface area of each hinge segment, for each cluster (Fig. S37).</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_19621175.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Novel US-CpHMD Protocol to Study the Protonation-Dependent Mechanism of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ATP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "/\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ADP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Carrier<br>We have designed a protocol combining constant-pH molecular dynamics (CpHMD) simulations with an umbrella sampling (US) scheme (US-CpHMD) to study the mechanism of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ADP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "/\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ATP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " transport (import and export) by their inner mitochondrial membrane carrier protein [\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ADP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "/\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ATP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " carrier (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    AAC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ")]. The US scheme helped overcome the limitations of sampling the slow kinetics involved in these substrates transport, while CpHMD simulations provided an unprecedented realism by correctly capturing the associated protonation changes. The import of anionic substrates along the mitochondrial membrane has a strong energetic disadvantage due to a smaller substrate concentration and an unfavorable membrane potential. These limitations may have created an evolutionary pressure on \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    AAC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " to develop specific features benefiting the import of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ADP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". In our work, the potential of mean force profiles showed a clear selectivity in the import of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ADP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " compared to \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ATP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", while in the export, no selectivity was observed. We also observed that \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    AAC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " sequestered both substrates at longer distances in the import compared to the export process. Furthermore, only in the import process do we observe transient protonation of both substrates when going through the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    AAC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " cavity, which is an important advantage to counteract the unfavorable mitochondrial membrane potential. Finally, we observed a substrate-induced disruption of the matrix salt-bridge network, which can promote the conformational transition (from the C- to M-state) required to complete the import process. This work unraveled several important structural features where the complex electrostatic interactions were pivotal to interpreting the protein function and illustrated the potential of applying the US-CpHMD protocol to other transport processes involving membrane proteins.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_17021821.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Reconsidering the Roles of Noncovalent Intramolecular Locks in Conjugated Molecules<br>The extrinsic properties of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    organic semiconductors\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    OSC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") are connected both to local and long-range thin-film morphologies. While it is desirable to synthetically regulate \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    OSC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " solid-state structure, there remains limited understanding of the rich and complex relationships among the molecular structure of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    chromophores\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " that comprise the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    OSC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and the functional parameters of the processing environment (e.g., solvent, solution composition, and temperature), each of which will impact the final \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    OSC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " structure and characteristics. Here, we are interested in exploring how \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    chromophore\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " chemistry and the processing environment impact the structures of oligomers comprised of electron-rich donor and electron-deficient acceptor moieties, as conformational variations among these groups can impact \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    OSC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " formation. Specifically, we make use of quantum-chemical calculations and molecular dynamics (MD) simulations to systematically investigate how variations in molecular design and processing chemistry influence the structure, dynamics, and aggregation tendencies of donor acceptor (D A) oligomers in solution. The investigation reveals preferential rotational isomer populations as a function of the oligomer chemistry, solvent environment, and oligomer concentration. Notably, questions are brought forward concerning the current emphasis on the roles of noncovalent intramolecular interactions in the design of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    OSC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " building blocks. Overall, the results provide an in-depth molecular-scale foundation to allow for thermodynamic and kinetic control of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    OSC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " morphology development through \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    chromophore\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " design and solvent optimization.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_19971077.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Impact of Inert Crowders on Host Guest Recognition Process<br>Biological environments typically contain high concentrations (300 400 mg/mL) of different macromolecules at volume fractions as large as 30% 40%. Biomolecular recognition processes, a ubiquitous biological phenomena, occurring in such crowded heterogeneous media would differ significantly compared to the dilute buffer solutions. Here we quantify the potential impact of inert crowders on prototypical host guest recognition process by explicit-solvent molecular dynamics (MD) simulations in atomic resolution. We demonstrate that the crowders, when smaller in size, would facilitate the binding process of the guest molecule by decreasing the free energy barrier for binding via excluded volume effect and desolvation of the host receptor. However, the extent of crowder-induced stabilization of a host guest complex is found to be significantly higher when the guest molecule is sterically constricted to approach the host along a centrosymmetric direction, compared to its unrestricted, freely diffusive movement. A kinetic analysis of the recognition process reveals that the origin of a relatively stronger crowder impact during constricted movement of guest molecule lies in the relatively enhanced residence time of the guest inside the host by crowders. Together, our results suggest that the extent of impact of crowding on recognition processes would be contingent upon the presence or absence of constriction on ligand movement.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_5060102.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Molecular dynamics simulation of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    tricaproin\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " in gas phase using machine-learning potential \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ANI2x\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       "<br>\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Tricaproin\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Glycerol trihexanoate\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") is an example of a \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    triglyceride\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " molecule with very short \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    alkyl\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " tails attached to the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    glycerol\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " moiety, and this deposit contains a \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    10 ns\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       " long simulation of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    tricaproin\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " in a gas phase. The model chemistry (a.k.a. interaction potential or force field) is the machine-learning potential \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ANI2x\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " implemented in python package \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    torchANI\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       ", which has a close-to-DFT accuracy, yet low cost compared to DFT or other electronic structure theories. Molecular dynamics were run using \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ASE\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " with Langevin integrator at a constant temperature of \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    310 K.\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       " The resulting trajectory was written every 1 ps (&quot;traj.h5&quot;, can be viewed in \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ASEgui\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       "), and gathered every 10 ps in a XTC format (&quot;traj.xtc&quot;, open in \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    MDAnalysis\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    VMD\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    UnityMol\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GROMACS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " tools ...). Detailed simulation settings are in the python script. This simulation was performed for the purpose of building a coarse grained \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Martini 3\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " model of this molecule. ANI2x https://doi.org/10.26434/chemrxiv.11819268.v1 torchANI https://aiqm.github.io/torchani/index.html ASE https://wiki.fysik.dtu.dk/ase/tutorials/md/md.html#constant-temperature-md</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_7672195.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Classification of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GTP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-dependent \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    K-Ras4B\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " active and inactive conformational states<br>Dataset for the paper: Classification of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GTP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-dependent \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    K-Ras4B\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " active and inactive conformational states. Cite as: J. Chem. Phys. 158, 000000 (2023); DOI: 10.1063/5.0139181 Submitted: 18 December 2022; Accepted: 13 February 2023; Published Online: 13 February 2023. All molecular dynamics and molecular docking data presented, analyzed, and discussed in this paper are available at reasonable requests submitted to the corresponding author. The following data are also available online: (i) file KRas4B pdbs.zip, a compressed archive including coordinate (.pdb) and structure (.psf) files for \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    KRas-4B\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " WT and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    D33E\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    proteins\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", (ii) KRas4B WT traj.zip, a compressed archive including trajectory files (.trr) for the WT \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    KRas-4B\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " runs, including 120 &quot;*.trr&quot;-formatted trajectories each corresponding to \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    40 ns\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       " of MD simulation time, and (iii) a sample Python script to generate a free energy plot as shown in Fig. 2.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_14766557.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Atomistic Modeling of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PEDOT:PSS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Complexes II: Force Field Parameterization<br>The conductive polymer complex poly (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    3,4-ethylenedioxythiophene\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    polystyrene sulfonate\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PEDOT:PSS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") is integral to many next-generation polymeric devices. All-atom molecular dynamics (MD) is increasingly employed to probe film microstructure, but the commonly used \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Generalized AMBER Force Field\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GAFF\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       ") parameters remain unvalidated for this system. Here, we develop all-atom force fields for both undoped and highly doped \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PEDOT\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " compatible with \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GAFF\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       ". Molecular geometries, vibrational energies, torsional profiles, and conformational energetics calculated with density functional theory comprise the training data. The optimized force fields improve upon \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GAFF\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " for all the training data, including notable improvements in torsional barriers, and capture the finite size of charge carriers in doped \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PEDOT\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". Although \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GAFF\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " predicts similar behavior between undoped and doped polymers, our models identify meaningful differences in ground-state geometry, vibrational spectra, and torsional barriers between the two. Furthermore, vacuum MD simulations show that the neutral polymer is more flexible than the highly doped polymer, primarily due to the weaker inter-monomer dihedral potentials. The foregoing trends are directly attributable to the aromatic-to-quinoid transition \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PEDOT\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " experiences upon doping. This coupling between doping and molecular flexibility highlights the importance of charge geometry interactions in atomistic modeling and should be considered while modeling conjugated polymers beyond \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PEDOT\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ".</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_4445375.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Ternary lipid composition in a curved geometry<br>Simulations of ternary \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DLiPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "/\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DPPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "/\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    cholesterol\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " membranes in a budded geometry with Gaussian curvature. Five replicas. Simulations were performed using \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GROMACS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " simulation engine (DOI: 10.1016/j.softx.2015.06.001 ) and the \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Martini 2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " force field ( DOI:10.1021/jp071097f ). The geometries, generated using \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    BUMPy\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " ( DOI: 10.1021/acs.jctc.8b00765 ), are maintained using restrained dummy particles that repel \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    lipid acyl\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " chains All input and output files are provided for each simulation, and the cpt files allow to extend the simulations. The top, itp (in TOP.tar archive), mdp, and ndx files are common for each simulation. The simulations have been used in DOI: 10.1101/2020.09.23.310086.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_22203473.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Extension of the \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    TraPPE\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " Force Field for Battery Electrolyte Solvents<br>Optimizing electrolyte formulations is key to improving performance of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Li-/Na-ion\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " batteries, where transport properties (diffusion coefficient, viscosity) and permittivity need to be predicted as functions of temperature, salt concentration and solvent composition. More efficient and reliable simulation models are urgently needed, owing to the high cost of experimental methods and the lack of united-atom molecular dynamics force fields validated for electrolyte solvents. Here the computationally efficient \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    TraPPE\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " united-atom force field is extended to be compatible with carbonate solvents, optimizing the charges and dihedral potential. Computing the properties of electrolyte solvents, \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ethylene carbonate\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    EC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "), \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    propylene carbonate\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "), \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    dimethyl carbonate\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DMC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "), \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    diethyl carbonate\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DEC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "), and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    dimethoxyethane\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DME\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "), we observe that the average absolute errors in the density, self-diffusion coefficient, permittivity, viscosity, and surface tension are approximately 15% of the corresponding experimental values. Results compare favorably to all-atom \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CHARMM\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    OPLS-AA\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " force fields, offering computational performance improvement of at least 80%. We further use \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    TraPPE\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " to predict the structure and properties of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    LiPF6\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " salt in these solvents and their mixtures. \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    EC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " form complete solvation shells around \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Li+\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " ions, while the salt in \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DMC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " forms chain-like structures. In the poorest solvent, \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DME\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    LiPF6\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " forms globular clusters despite \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DME\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " s higher permittivity than \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DMC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ".</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_1146536.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">On Calculation of the Electrostatic Potential of a \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Phosphatidylinositol Phosphate\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-Containing \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Phosphatidylcholine\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Lipid Membrane Accounting for Membrane Dynamics<br>Many signaling events require the binding of cytoplasmic proteins to cell membranes by recognition of specific charged lipids, such as \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    phosphoinositol\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    phosphates\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". As a model for a protein-membrane binding site, we consider one charged \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    phosphoinositol phosphate\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PtdIns (3)P\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") embedded in a \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    phosphatidylcholine\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " bilayer. As the protein-membrane binding is driven by electrostatic interactions, continuum solvent models require an accurate representation of the electrostatic potential of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    phosphoinositol phosphate\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-containing membrane. We computed and analyzed the electrostatic potentials of snapshots taken at regular intervals from molecular dynamics simulations of the bilayer. We observe considerable variation in the electrostatic potential of the bilayer both along a single simulation and between simulations performed with the \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GAFF\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " or \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CHARMM c36\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " force fields. However, we find that the choice of \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GAFF\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " or \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CHARMM c36\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " parameters has little effect on the electrostatic potential of a given configuration of the bilayer with a \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PtdIns (3)P\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " embedded in it. From our results, we propose a remedian averaging method for calculating the electrostatic potential of a membrane system that is suitable for simulations of protein-membrane binding with a continuum solvent model.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_2257141.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">A Unified \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    AMBER\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       "-Compatible Molecular Mechanics Force Field for \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Thiolate\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-Protected Gold Nanoclusters<br>We present transferable \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    AMBER\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       "-compatible force field parameters for \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    thiolate\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-protected gold nanoclusters. Five different sized clusters containing both organo-soluble and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-soluble \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    thiolate\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " ligands served as test systems in MD simulations, and parameters were validated against DFT and experimental results. The cluster geometries remain intact during the MD simulations in various solvents, and structural fluctuations and energetics showed agreement with DFT calculations. Experimental diffusion coefficients and crystal structures were also reproduced with sufficient accuracy. The presented parameter set contains the minimum number of cluster-specific parameters enabling the use of these parameters for several different gold nanoclusters. The parameterization of ligands can also be extended to different types of ligands.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_2332498.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Molecular Dynamics Studies of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Liposomes\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " as Carriers for Photosensitizing Drugs: Development, Validation, and Simulations with a Coarse-Grained Model<br>\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Liposomes\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " are proposed as drug delivery systems and can in principle be designed so as to cohere with specific tissue types or local environments. However, little detail is known about the exact mechanisms for drug delivery and the distributions of drug molecules inside the lipid carrier. In the current work, a coarse-grained (CG) \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    liposome\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " model is developed, consisting of over 2500 lipids, with varying degrees of drug loading. For the drug molecule, we chose \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    hypericin\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", a natural compound proposed for use in photodynamic therapy, for which a CG model was derived and benchmarked against corresponding atomistic membrane bilayer model simulations. \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Liposomes\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " with 21 84 \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    hypericin\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " molecules were generated and subjected to \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    10 microsecond\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       " simulations. Distribution of the hypericins, their orientations within the lipid bilayer, and the potential of mean force for transferring a \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    hypericin\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " molecule from the interior aqueous droplet through the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    liposome\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " bilayer are reported herein.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_1009027.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Simulations \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DPPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " bilayers (512 lipids) using \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    charmm36\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " ff in \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    gromacs\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       "<br>Collection simulations of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DPPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (512 lipids) bilayers in \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    gromacs\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " using the \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    charmm36\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " force field. Several temperatures between \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    315\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    338 K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       " are included. The list of systems can be found below where the several parameter are: 1) \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DPPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " 512 \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    NaCl\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " 150mM \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    315K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       " v-rescale (\n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    500ns\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       ") 2) \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DPPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " 512 \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    NaCl\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " 150mM \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    320K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    700ns\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       ") 3) \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DPPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " 512 \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    NaCl\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " 150mM \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    320K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       " v-rescale (\n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    500ns\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       ") 4) \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DPPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " 512 \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    NaCl\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " 150mM \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    322K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       " v-rescale (\n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    700ns\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       ") 5) \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DPPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " 512 \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    NaCl\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " 150mM \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    325K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    500ns\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       ") 6) \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DPPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " 512 \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    NaCl\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " 150mM \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    325K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       " v-rescale (\n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    500ns\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       ") 7) \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DPPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " 512 \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    NaCl\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " 150mM \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    325K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       " cutoff09 (\n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    500ns\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       ") 8) \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DPPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " 512 \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    NaCl\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " 150mM \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    325K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       " MEMB \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    338K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    500ns\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       ") 9) \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DPPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " 512 \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    NaCl\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " 150mM \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    338K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    500ns\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       ") For further information read the Readme file provided for each simulation.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_20376757.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Role of Enzyme and Active Site Conformational Dynamics in the Catalysis by \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Amylase\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Explored with QM/MM Molecular Dynamics<br>We assessed enzyme:substrate conformational dynamics and the rate-limiting glycosylation step of a human pancreatic \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    amylase\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ":\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    maltopentose\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " complex. \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Microsecond\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       " molecular dynamics simulations suggested that the distance of the catalytic Asp197 nucleophile to the anomeric carbon of the buried \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    glucoside\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " is responsible for most of the enzyme active site fluctuations and that both Asp197 and Asp300 interact the most with the buried \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    glucoside\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " unit. The buried \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    glucoside\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " binds either in a 4C1 chair or 2SO skew conformations, both of which can change to TS-like conformations characteristic of retaining \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    glucosidases\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". Starting from four distinct enzyme:substrate complexes, umbrella sampling quantum mechanics/molecular mechanics simulations (converged within less than 1 kcal mol 1 within a total simulation time of \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    1.6 ns\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       ") indicated that the reaction occurrs with a Gibbs barrier of 13.9 kcal mol 1, in one asynchronous concerted step encompassing an acid base reaction with Glu233 followed by a loose SN2-like nucleophilic substitution by the Asp197. The transition state is characterized by a 2H3 half-chair conformation of the buried \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    glucoside\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " that quickly changes to the E3 envelope conformation preceding the attack of the anomeric carbon by the Asp197 nucleophile. Thermodynamic analysis of the reaction supported that a \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " molecule tightly hydrogen bonded to the glycosidic oxygen of the substrate at the reactant state ( 1.6 A ) forms a short hydrogen bond with Glu233 at the transition state ( 1.7 A ) and lowers the Gibbs barrier in over 5 kcal mol 1. The resulting Asp197-\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    glycosyl\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " was mostly found in the 4C1 conformation, although the more endergonic B3,O conformation was also observed. Altogether, the combination of short distances for the acid base reaction with the Glu233 and for the nucleophilic attack by the Asp197 nucleophile and the availability of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " within hydrogen bonding distance of the glycosidic oxygen provides a reliable criteria to identify reactive conformations of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    amylase\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " complexes.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_7066551.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Datasets for &quot;Resolving the microscopic hydrodynamics at the moving contact line&quot;<br>Datasets for the article: &quot;Resolving the microscopic hydrodynamics at the moving contact line&quot; Amal K. Giri, Paolo Malgaretti, Dirk Peschka, and Marcello Sega Phys. Rev. Fluids 7, L102001 DOI: 10.1103/PhysRevFluids.7.L102001 Includes: \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GROMACS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " input files Modifications to the \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GROMACS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " source code thermostat as described in the article Instructions on how to invoke the patched version of \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GROMACS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " with decoupled directions Matlab datafiles with FE solutions and scripts to analyse and compare them to MD velocity field (also included) See also: https://github.com/Marcello-Sega/pytim https://github.com/dpeschka/stokes-free-boundary</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_2586559.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Interaction of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    N-3-oxododecanoyl homoserine lactone\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " with \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    transcriptional regulator\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    LasR\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " of Pseudomonas aeruginosa: Insights from molecular docking and dynamics simulations<br>Dataset and supplementary files of the research: Interaction of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    N-3-oxododecanoyl homoserine lactone\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " with \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    transcriptional regulator\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    LasR\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " of Pseudomonas aeruginosa: Insights from molecular docking and dynamics simulations (https://doi.org/10.1101/121681) - Supporting Information - Input: Parameters and initial structures - Output: Trajectories, Docking poses \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Gromacs\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " (multi-core with CUDA) was used for the simulations. \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Autodock Vina\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    FlexAid\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    rDock\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " were used for molecular docking.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_9941741.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Absolute Free Energy of Binding Calculations for \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Macrophage Migration Inhibitory Factor\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " in Complex with a Druglike Inhibitor<br>Calculation of the absolute free energy of binding ( Gbind) for a complex in solution is challenging owing to the need for adequate configurational sampling and an accurate energetic description, typically with a force field (FF). In this study, Monte Carlo (MC) simulations with improved side-chain and backbone sampling are used to assess Gbind for the complex of a druglike inhibitor (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    MIF180\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") with the protein \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    macrophage migration inhibitory factor\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    MIF\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") using free energy perturbation (FEP) calculations. For comparison, molecular dynamics (MD) simulations were employed as an alternative sampling method for the same system. With the \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    OPLS-AA/M\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " FF and CM5 atomic charges for the inhibitor, the Gbind results from the MC/FEP and MD/FEP simulations, 8.80 0.74 and 8.46 0.85 kcal/mol, agree well with each other and with the experimental value of 8.98 0.28 kcal/mol. The convergence of the results and analysis of the trajectories indicate that sufficient sampling was achieved for both approaches. Repeating the MD/FEP calculations using current versions of the \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CHARMM\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    AMBER\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " FFs led to a 6 kcal/mol range of computed Gbind. These results show that calculation of accurate Gbind for large ligands is both feasible and numerically equivalent, within error limits, using either methodology.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_19949518.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Understanding the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Mannose\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Transfer Mechanism of Mycobacterial \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Phosphatidyl-myo-inositol Mannosyltransferase A\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " from Molecular Dynamics Simulations<br>\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Glycolipids\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " like \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    phosphatidylinositol hexamannosides\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PIM6\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    lipoglycans\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", such as \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    lipomannan\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    LM\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    lipoarabinomannan\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    LAM\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "), play crucial roles in virulence, survival, and antibiotic resistance of various mycobacterial species. \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Phosphatidyl-myo-inositol mannosyltransferase A\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PimA\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") catalyzes the transfer of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    mannose\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " moiety (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    M\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") from \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GDP-mannose\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GDPM\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") to \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    phosphatidyl-myo-inositol\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PI\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") to synthesize \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GDP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    phosphatidyl-myo-inositol monomannoside\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PIM\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "). This \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PIM\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " is mannosylated, acylated, and further modified to give rise to the higher \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PIMs\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    LM\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    LAM\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". It is yet to be known how \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PI\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PIM\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PI\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GDPM\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PIM\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GDP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " interact with \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PimA.\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Here, we report the docked structures of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PI\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PIM\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " to understand how the substrates and the products interact with \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PimA.\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Using molecular dynamics (MD) simulations for \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    300 ns\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       ", we have investigated how various ligand-bound conformations change the dynamics of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PimA.\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Our studies demonstrated the open to closed motions of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PimA.\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " We observed that \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PimA\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " is least dynamic when bound to both \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GDPM\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PI\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". MD simulations indicated that the loop residues 59 70 and the -helical residues 73 86 of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PimA\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " play important roles while interacting with both \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PI\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PIM\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". MD analyses also suggested that the residues Y9, P59, R68, L69, N97, R196, R201, K202, and R228 of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PimA\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " play significant roles in the mannose transfer reaction. Overall, docking studies and MD simulations provide crucial insights to design future therapeutic drugs against mycobacterial \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PimA.\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_6955175.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Nonproductive Binding Modes as a Prominent Feature of A 40 Fiber Elongation: Insights from Molecular Dynamics Simulation<br>The formation of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    amyloid\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " fibers has been implicated in a number of neurodegenerative diseases. The growth of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    amyloid\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " fibers is strongly thermodynamically favorable, but kinetic traps exist where the incoming monomer binds in an incompatible conformation that blocks further elongation. Unfortunately, this process is difficult to follow experimentally at the atomic level. It is also too complex to simulate in full detail and to date has been explored either through coarse-grained simulations, which may miss many important interactions, or full atomic simulations, in which the incoming peptide is constrained to be near the ideal fiber geometry. Here we use an alternate approach starting from a docked complex in which the monomer is from an experimental NMR structure of one of the major conformations in the unbound ensemble, a largely unstructured peptide with the central hydrophobic region in a 310 helix. A \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    1000 ns\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       " full atomic simulation in explicit solvent shows the formation of a metastable intermediate by sequential, concerted movements of both the fiber and the monomer. A Markov state model shows that the unfolded monomer is trapped at the end of the fiber in a set of interconverting antiparallel -hairpin conformations. The simulation here may serve as a model for the binding of other non- -sheet conformations to \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    amyloid\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " fibers.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_21568583.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Computational Exploration and Characterization of Potential \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Calcium\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Sensitizing Mutations in \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Cardiac Troponin C\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "<br>\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Calcium\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-dependent heart muscle contraction is regulated by the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    cardiac troponin protein\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " complex (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    cTn\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") and specifically by the N-terminal domain of its \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    calcium\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " binding subunit (cNTnC). cNTnC contains one \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    calcium\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " binding site (site II), and altered \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    calcium\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " binding in this site has been studied for decades. It has been previously shown that cNTnC mutants, which increase \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    calcium\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " sensitization may have therapeutic benefits, such as restoring cardiac muscle contractility and functionality post-myocardial infarction events. Here, we computationally characterized eight mutations for their potential effects on \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    calcium\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " binding affinity in site II of cNTnC. We utilized two distinct methods to estimate \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    calcium\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " binding: adaptive steered molecular dynamics (ASMD) and thermodynamic integration (TI). We observed a sensitizing trend for all mutations based on the employed ASMD methodology. The TI results showed excellent agreement with experimentally known \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    calcium\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " binding affinities in wild-type cNTnC. Based on the TI results, five mutants were predicted to increase \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    calcium\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " sensitivity in site II. This study presents an interesting comparison of the two computational methods, which have both been shown to be valuable tools in characterizing the impacts of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    calcium\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " sensitivity in mutant cNTnC systems.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_6616150.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Uncovering the molecular basis for the better \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Gefitinib\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " sensitivity of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    EGFR\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " with complex mutations over single rare mutation: insights from molecular simulations<br>Molecular Dynamics trajectories and topology files of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    APO\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Gefitinib\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (IRE)-bound \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Epidermal growth factor receptor\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    EGFR\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") mutants: rare (G719S), rare + rare (G719S/L861Q) and rare + classical (G719S/L858R). The trajectories presented in this work were analyzed and presented in &quot;Uncovering the molecular basis for the better \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Gefitinib\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " sensitivity of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    EGFR\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " with complex mutations over single rare mutation: insights from molecular simulations&quot;.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_21806756.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">\n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    LassoHTP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       ": A High-Throughput Computational Tool for \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Lasso Peptide\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Structure Construction and Modeling<br>\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Lasso peptides\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " are a subclass of ribosomally synthesized and post-translationally modified peptides with a slipknot conformation. With superior thermal stability, protease resistance, and antimicrobial activity, \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    lasso peptides\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " are promising candidates for bioengineering and pharmaceutical applications. To enable high-throughput computational prediction and design of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    lasso peptides\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", we developed a software, \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    LassoHTP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       ", for automatic \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    lasso peptide\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " structure construction and modeling. \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    LassoHTP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " consists of three modules, including the scaffold constructor, mutant generator, and molecular dynamics (MD) simulator. With a user-provided sequence and conformational annotation, \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    LassoHTP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " can either generate the structure and conformational ensemble as is or conduct random mutagenesis. We used \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    LassoHTP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " to construct eight known \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    lasso peptide\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " structures de novo and to simulate their conformational ensembles for 100 ns MD simulations. For benchmarking, we calculated the root mean square deviation (RMSD) of these ensembles with reference to their experimental crystal or NMR PDB structures; we also compared these RMSD values against those of the MD ensembles that are initiated from the PDB structures. Dihedral principal component analysis was also conducted. The results show that the \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    LassoHTP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       "-initiated ensembles are similar to those of the PDB-initiated ensembles. \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    LassoHTP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " offers a computational platform to develop strategies for \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    lasso peptide\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " prediction and design.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_247386.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "/\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Cholesterol\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (50:50) lipid membrane, \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    303K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Charmm36\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " force field, simulation files and \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    100 ns\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       " trajectory for \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GROMACS simulation engine v5\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       "<br>All runs were performed with \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GROMACS simulation engine v5\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CHARMM36\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " additive force field parameters obtained from MacKerell lab website (https://removed ff.shtml, also available at https://removed Conditions: T \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    303\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       ", 80 \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and 80 \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Cholesterol\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " molecules, 7200 \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    tip3p\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    waters\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    100ns\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       " trajectory (preceded with equilibration). These data were originally obtained for the nmrlipids.blospot.fi project. Find more details at nmrlipids.blospot.fi and https://removed</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_3613573.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Simulations of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ":\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPG\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " 1:1 membranes with varying levels of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CaCl 2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " using the \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Slipids\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " force field<br>Simulations of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "/\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPG\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " lipid bilayers consisting of a total of 256 lipids (128 \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " + 128 \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPG\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") spread equally between the two leaflets. The initial \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CaCl 2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " concentration in the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " phase was either 0, 100, 200, 500, or 1000 mM. \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Na+\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " was used as the counterion for the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPG\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " charges. The simuatlions were performed using the \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Slipids\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " model, available at https://removed . The Dang ion parameters were used, and they were obtained from https://removed . The common simulation parameters are given in the md.mdp file. The simulations are \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    1500 ns\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       " long (except CA0, which is \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    500 ns\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       " long), and the &quot;extend&quot; files contain the last \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    1000 ns\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       " of the outputs. The &quot;extend&quot; continue points should also be used to extend the simulations beyond \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    1500 ns\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       ". All relevant input and output files are provided to repeat or extend the simulations.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_3634769.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">MD snapshots: \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Potassium\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " channel blocker crafted by -hairpinin scaffold engineering<br>Snapshots of molecular dynamics (MD) simulations of 25 complexes of Kv1.3 channel with a series of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Tk-hefu\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " peptide analogues (25 different peptides). Each system include solvated lipid bilayer, the channel and the peptide at 0 (before MD) and \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    110 ns\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       " (after MD). Files are in *.gro format. Additionally, this data set includes two excel tables (*.xlsx) with analysis of intermolecular contacts between the channel and the peptides for all systems in dynamics. These are the same supplementary tables that are uploaded as Supplementary information to Frontiers in chemistry website.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_2615170.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Small Molecule Solvation Free Energy: Enhanced Conformational Sampling Using Expanded Ensemble Molecular Dynamics Simulation<br>We present an efficient expanded ensemble molecular dynamics method to calculate the solvation free energy (or residual chemical potential) of small molecules with complex topologies. The methodology is validated by computing the solvation free energy of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ibuprofen\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " in \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    methanol\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ethanol\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " at \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    300 K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       " and 1 bar and comparing to reference simulation results using Bennett s acceptance ratio method. Difficulties with \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ibuprofen\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " using conventional molecular dynamics methods stem from an inadequate sampling of the carboxylic acid functional group, which, for the present study, is subject to free energy barriers of rotation of 14 20 kBT. While several advances have been made to overcome such weaknesses, we demonstrate how this shortcoming is easily overcome by using an expanded ensemble methodology to facilitate conformational sampling. Not only does the method enhance conformational sampling but it also boosts the rate of exploration of the configurational phase space and requires only a single simulation to calculate the solvation free energy. Agreement between the expanded ensemble and the reference calculations is good for all three solvents, with the reported uncertainties of the expanded ensemble being comparable to the uncertainties of the reference calculations, while requiring less simulation time; the reduced simulation time demonstrates the improved performance gained from the expanded ensemble method.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_11343020.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Molecular Dynamic Simulation data of compensatory mutations in \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Bruton's Tyrosine Kinase\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "<br>The research aims to find compensatory mutations that rescue \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Bruton s tyrosine kinase\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Btk\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") activity in the presence of severe \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    X-linked agammaglobulinemia\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    XLA\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") causing mutation in the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    kinase\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " domain. The data set includes the molecular dynamic simulation trajectories of native, with R641H deleterious mutation and with the R641H-K420R compensatory mutations structures separately. The trajectories describe the conformational dynamics changes and show that the compensatory mutation site was able to rescue the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Btk\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " linker-\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    kinase\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " domain dynamics in the presence of R641H.This MD simulation data set supports the research article: &quot;Identifying a Compensating Mutation to Rescue Function in \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Bruton s Tyrosine Kinase\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " with an \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    XLA\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Mutant&quot;.The multiple sequence alignment data for calculating high order dependence is also included in the data set (PF07714.fa). The links for all the software and scripts used in the research are available in the README file.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_20359967.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Mebendazole\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " s Conformational Space and Its Predicted Binding to Human \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Heat-Shock Protein 90\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "<br>Recent experimental evidence suggests that \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    mebendazole\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", a popular antiparasitic drug, binds to \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    heat shock protein 90\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Hsp90\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") and inhibits acute myeloid leukemia cell growth. In this study we use quantum mechanics (QM), molecular similarity, and molecular dynamics (MD) calculations to predict possible binding poses of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    mebendazole\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " to the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    adenosine triphosphate\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ATP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") binding site of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Hsp90\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". Extensive conformational searches and minimization of the five \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    mebendazole\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " tautomers using the MP2/aug-cc-pVTZ theory level resulted in 152 minima. \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Mebendazole\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Hsp90\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " complex models were subsequently created using the QM optimized conformations and protein coordinates obtained from experimental crystal structures that were chosen through similarity calculations. Nine different poses were identified from a total of \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    600 ns\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       " of explicit solvent, all-atom MD simulations using two different force fields. All simulations support the hypothesis that \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    mebendazole\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " is able to bind to the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ATP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " binding site of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Hsp90\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ".</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_3940377.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Molecular Dynamics of &quot;Fuzzy&quot; Transcriptional Activator-Coactivator Interactions<br>Transcriptional activation domains (ADs) are generally thought to be intrinsically unstructured, but capable of adopting limited secondary structure upon interaction with a coactivator surface. The indeterminate nature of this interface made it hitherto difficult to study structure/function relationships of such contacts. Here we used atomistic accelerated molecular dynamics (aMD) simulations to study the conformational changes of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GCN4\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " AD and variants thereof, either free in solution, or bound to the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GAL11 coactivator\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " surface. We show that the AD-coactivator interactions are highly dynamic while obeying distinct rules. The data provide insights into the constant and variable aspects of orientation of ADs relative to the coactivator, changes in secondary structure and energetic contributions stabilizing the various conformers at different time points. We also demonstrate that a prediction of -helical propensity correlates directly with the experimentally measured transactivation potential of a large set of mutagenized ADs. The link between -helical propensity and the stimulatory activity of ADs has fundamental practical and theoretical implications concerning the recruitment of ADs to coactivators.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_6957888.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Inferring global dynamics from local structure in liquid electrolytes.<br>Molecular dynamics simulation data (obtained using the \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    HOOMD-blue\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " package) used to generate results in &quot;Inferring global dynamics from local structure in liquid electrolytes&quot;. 108 bulk electrolytic systems were simulated with twelve concentrations ranging from 0.0005 3 to 0.05 3 and nine Bjerrum lengths in the range 2.5 to 10.0 . Each simulation consisted of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    cations\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    anions\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", and solvent, all of which are modelled as beads of diameter (the Lennard-Jones unit of distance) and unit mass. If the size of each bead is mapped to the size of a \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " molecule (2.75 A ), the concentration range of 0.0005 3 to 0.05 3 approximately corresponds to 0.04 M to 4 M in real units. For example, the system with concentration 0.01 3 and Bjerrum length 4.0 has the following relevant file names: &quot;conc0.01 lb4.0.dcd&quot; contains the MD trajectory, and &quot;initial config conc0.01.gsd&quot; contains the initial configuration. The jupyter notebook &quot;sample data analysis.ipynb&quot; illustrates how to load these trajectories as well as the computed molar conductivities for each system. &quot;transport coefficients.zip&quot; contains the computed Onsager transport coefficients for each of the simulated systems. </div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_30894.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Ulmschneider\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    OPLS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    NaCl\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " 1Mol<br>MD simulation trajectory and related files for fully hydrated \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " bilayer with 1M \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    NaCl\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". The \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Ulmschneider\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " force field for \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " was used with \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Gromacs 5.0.3\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " [1,2]. \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Ions\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " were described by \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    OPLS-AA\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " force field (\n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Gromacs 5.0.3\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       "). Conditions: T \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    298.15\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       ", 128 \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " molecules, 5120 \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    tip3p\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    waters\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", 77 \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Na\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", 77 \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Cl\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    200ns\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       " trajectory (preceded by 5ns NPT equillibration). This data is ran for the nmrlipids.blospot.fi project. More details from nmrlipids.blospot.fi and https://removed [1] J.P. Ulmschneider &amp; M.B. Ulmschneider, United Atom Lipid Parameters for Combination with the Optimized Potentials for Liquid Simulations All-Atom Force Field, JCTC 2009, 5 (7), 1803 1813 [2] https://removed</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_13135976.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Hundred nanosecond molecular dynamics simulation of the G domain of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    K-Ras4B\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "<br> (This is a public copy of https://figshare.com/s/453b1b215cf2f9270769)This is a one hundred nanosecond simulation of the G domain of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    K-Ras4B\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (UNIPROT:P01116-2, residues 1-166). The simulation was generated using \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GROMACS v.4.6.5\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " with the \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    AMBER99SB-ILDN\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " forcefield (altered following doi:10.1002/jcc.10262, see References), \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    TIP3P\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and 0.15 M salt. The leap-frog integrator with 2 fs timestep was used. A v-rescale thermostat was used at \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    310 K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       ", with two energy groups (protein + \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GTP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " + \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    MG\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " + salt) and a time constant of 0.1 ps. PME was used with a tolerance of 1e-5 for the coulomb interactions and a cutoff was used for the VdW interactions, both with a cutoff of 1.1 nm. A Parrinello-rahman barostat was applied with a reference pressure of 1 bar and a time constant of 1 ps. For more information take a look at md.mdp. Coordinates were saved every 1 nanosecond and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    waters\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " were stripped, resulting in a trajectory of 101 snapshots. The initial model was made using the G-domain of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GppNHp\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " bound \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    H-Ras\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (PDB ID: 4EFL). The protein was converted using \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    MODELLER v9.16\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " (doi:10.1002/cpbi.3 , see References) and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GppNHp\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " was converted to \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GTP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " by hand.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_2481151.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Molecular Dynamics Simulations of the Interactions of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DMSO\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " with \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DPPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DOPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Phospholipid Membranes<br>Molecular dynamics simulations have been used to investigate the effect of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DMSO\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " on \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    1,2-dipalmitoyl-sn-glycero-3-phosphatidylcholine\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DPPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    1,2-dioleoyl-sn-glycero-3-phosphocholine\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DOPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") phospholipid bilayers. The concentration of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DMSO\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " was varied between 0 and 25.0 mol %. For both lipids, \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DMSO\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " causes the membrane to expand in the plane of the membrane while thinning normal to that plane. Above a critical concentration, pores in the membrane form spontaneously, and if the concentration is increased further, then the bilayer structure is destroyed. Even at concentrations below those required to induce pores, \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DMSO\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " readily diffuses across the bilayers. The free-energy profile associated with the diffusion of a \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DMSO\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " molecules across the membrane has been calculated. The simulations suggest that the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DOPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " bilayer is more resistant to the deleterious effects of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DMSO\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", both increasing the stability of the membranes and decreasing the rate at which \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DMSO\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " diffuses across the membrane. In this way, the work highlights the importance of investigating the lipid composition of cell membranes when characterizing the effects of cryosolvents.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_2271034.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">First-Principles Simulations of the Initial Phase of Self-Aggregation of a \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Cyanine\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Dye: Structure and Optical Spectra<br>Using first-principles simulations, we investigated the initial steps of the self-aggregation of the dye \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    pseudoisocyanine\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PIC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") in \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". First, we performed molecular dynamics (MD) simulations of the self-aggregation process, in which pile-of-coins oligomers ranging from dimers to stacks of about 20 molecules formed. The oligomer structures were found to be very flexible, with the dimers entering a weakly coupled state and then returning to a stable stacked conformation on a \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    nanosecond\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       " time scale. The structural information from the MD simulations was combined with quantum chemical calculations to generate a time-dependent Frenkel exciton Hamiltonian for monomers, dimers, and trimers, which included vibronic coupling. This Hamiltonian, in turn, was used to calculate the absorption spectra for these systems. The simulated dimer spectrum compared well to experiment, validating the face-to-face stacked dimer arrangement found in our MD simulations. Comparison of the simulated trimer spectrum to experiment suggested that oligomers larger than the dimer cannot be abundant at the onset of J-aggregation. Finally, the conformation of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PIC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " J-aggregate was investigated by testing the stability of several possible conformations in our MD simulations; none of the tested structures was found to be stable.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_2227162.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Reparameterized United Atom Model for Molecular Dynamics Simulations of Gel and Fluid \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Phosphatidylcholine\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Bilayers<br>A new united atom parametrization of diacyl lipids like \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    dipalmitoylphosphatidylcholine\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DPPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") and the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    dimyristoylphosphatidylcholine\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DMPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") has been constructed based on ab initio calculations to obtain fractional charges and the dihedral potential of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    hydrocarbon\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " chains, while the Lennard-Jones parameters of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    acyl\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " chains were fitted to reproduce the properties of liquid \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    hydrocarbons\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". The results have been validated against published experimental X-ray and neutron scattering data for fluid and gel phase \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DPPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". The derived charges of the lipid \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    phosphatidylcholine\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") headgroup are shown to yield dipole components in the range suggested by experiments. The aim has been to construct a new force field that retains and improves the good agreement for the fluid phase and at the same time produces a gel phase at low temperatures, with properties coherent with experimental findings. The gel phase of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    diacyl-PC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " lipids forms a regular triangular lattice in the hydrocarbon region. The global bilayer tilt obtains an \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    azimuthal\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " value of 31 and is aligned between lattice vectors in the bilayer plane. We also show that the model yields a correct heat of melting as well as decent heat capacities in the fluid and gel phase of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DPPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ".</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_7586417.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Molecular modelling of HPV \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    E6\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " protein reveal new potential inhibitors against high-risk Human Papillomavirus: Supporting Information<br>Abstract: High-risk strains of human papillomavirus (HPV) have been identified as the etiologic agent of some anogenital tract, head, and neck cancers. Although prophylactic HPV vaccines have been approved; it is still necessary a drug-based treatment against the infection and its oncogenic effects. The \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    E6\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    oncoprotein\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " is one of the most studied therapeutic targets of HPV, it has been identified as a key factor in cell immortalization and tumor progression in HPV-positive cells. \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    E6\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " can promote the degradation of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    p53\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", a tumor suppressor protein, through the interaction with the cellular \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ubiquitin ligase\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    E6AP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". Therefore, preventing the formation of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    E6\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    E6AP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " complex is one of the main strategies to inhibit the viability and proliferation of infected cells. Herein, we propose an in silico pipeline to identify small-molecule inhibitors of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    E6\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    E6AP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " interaction. Virtual screening was carried out by predicting the ADME properties of the molecules and performing ensemble-based docking simulations to \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    E6\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " protein followed by binding free energy estimation through MM/PB (GB)SA methods. Finally, the top-three compounds were selected, and their stability in the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    E6\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " docked complex and their effect in the inhibition of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    E6\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    E6AP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " interaction was corroborated by molecular dynamics simulation. Therefore, this pipeline and the identified molecules represent a new starting point in the development of anti-HPV drugs.This fileset contains raw data from molecular dynamics simulations of HPV-16 \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    E6\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " protein, including initial files and backbone trajectories. Also includes the initial compound library, ADME properties predictions, molecular docking results and hit compounds in mol2 format.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_5362218.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">trajectories for: Membrane-binding mechanism of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    EEA1\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    FYVE\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " domain revealed by multi-scale molecular dynamics simulations<br>Coarse-grained trajectories produced and analysed for publication: ---------------------- Membrane-binding mechanism of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    EEA1\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    FYVE\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " domain revealed by multi-scale molecular dynamics simulations Andreas Haahr Larsen*, Lilya Tata*, Laura John &amp; Mark S.P. Sansom Department of Biochemistry, University of Oxford, Oxford, United Kingdom, OX1 3QU PLOS comp biol (in press) ------------------------- ** file overview** md X.xtc: (X 0..14) 15 repeated CG simulations (\n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    1500 ns\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       " each) of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    FYVE\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " domain from \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    EEA1\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " binding to \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ":\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POP1\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " bilayer. The repeats differ in the rotation of the initial frame. final cg2at aligned.pdb: initial frame for AT (after CG2AT) prod cym cent repX.xtc (X 1,2,3) 3 repeated AT sims (\n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    500 ns\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       " each) of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    FYVE\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " domain from \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    EEA1\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " binding to \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ":\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POP1\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " bilayer. ** scripts for reproduction at GitHub** scripts and files for reproduction are available at: https://github.com/andreashlarsen/Larsen-Tata2021-FYVE</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_13378605.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Insight into the Dual Inhibition Mechanism of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Corilagin\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " against \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    MRSA\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Serine/Threonine Phosphatase\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Stp1\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") by Molecular Modeling<br>\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Serine/threonine phosphatase\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Stp1\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") is known to be involved in the regulation of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    cysteine\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " phosphorylation levels in many different pathways, such as virulence factor regulation in \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    methicillin-resistant Staphylococcus aureus\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    MRSA\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "). Therefore, \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Stp1\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " can be used as a potential target for inhibiting \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    MRSA\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " infection. In this study, using virtual screening, we found that corilagin, a natural compound, was screened as a potential \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Stp1\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " inhibitor. Then, the phosphatase assay exhibited high inhibitory activity against \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Stp1\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". On the basis of the enzyme kinetics experiment, we found that \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    corilagin\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " exhibited a dual inhibitory mechanism of competitive and allosteric inhibition. To further elucidate the mechanism of interaction between \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    corilagin\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Stp1\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", molecular dynamics (MD) simulations were performed on the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Stp1\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    corilagin\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " complex. Consistent with the mutagenesis assays and fluorescence quenching assays results, the competitive and allosteric binding sites of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    corilagin\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " with \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Stp1\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " were identified. In the competitive binding site of Stp1, Asn162, Ile164, Tyr199, and Lys232 were found to play a key role in this complex. In the allosteric binding site, hydrophobic interaction was the main binding force. The Asn142, Val145, Leu146, Pro152, and Phe179 residues of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Stp1\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " were found to play a critical role in the binding of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    corilagin\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " with \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Stp1\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". In this study, we used MD simulation to reveal the ligand protein interactions, providing a theoretical basis. This research work, thus, lays down the foundation for the development of new \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Stp1\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " inhibitors to be utilized in the future.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_11808321.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Quantum Mechanics/Molecular Mechanics Density Functional Theory Simulations of the Optical Properties Fingerprinting the Ligand-Binding of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Pentameric Formyl Thiophene Acetic Acid\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " in \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Amyloid- (1 42)\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "<br>The binding pocket proposed by Ko nig [Chem. Commun. 2018, 54, 3030 3033] for the biomarker \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    pentameric formyl thiophene acetic acid\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    p-FTAA\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") in the fibrillar structure of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    amyloid- (1 42)\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " has been put to the test by the comparison of theoretical and experimental optical absorption and fluorescence spectra obtained in a \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " environment and inside the protein scaffold. The optical absorption/emission properties of this luminescent conjugated \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    oligothiophene\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " were studied by means of classical force field molecular dynamics simulations to account for the sampling of configuration space in conjunction with polarizable embedding time-dependent density functional theory calculations of spectra. The nuclear motions of residues in the -sheet were found to be modest, and the time dependence of embedding parameters was shown to be negligible so that a time-independent parameter set could be derived and used for all 300 snapshots considered in the spectrum averaging. In regard to linear absorption spectra, the calculated red shift due to protein binding for the dominant S1 S0 transition in \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    p-FTAA\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " was found to be equal to 23 nm (0.17 eV), which is in excellent agreement with the corresponding experimental result of 18 nm and taken as corroborating evidence for having correctly identified the binding pocket of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    p-FTAA\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " in the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    amyloid\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". The underlying mechanisms for the calculated red shift were disentangled, and it is shown that some 20 nm (0.15 eV) of the total 23 nm (0.17 eV) is associated with increased planarity of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    p-FTAA\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " in the binding pocket, whereas a mere 3 nm (0.02 eV) is associated with changes in the environment. In regard to emission spectra, we demonstrate that intersystem crossing from the excited S1 state to the triplet manifold of states is a less likely event for \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    p-FTAA\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " in the binding pocket as compared to in the aqueous solution, and we thereby partly explain the much higher quantum yield of fluorescence for the more rigid \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    p-FTAA\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " in the binding pocket. Two-photon absorption in \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    p-FTAA\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " is shown to predominantly occur to an overall symmetric excited state and be more than twice as strong for the biomarker in the binding pocket as compared to in \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". The corresponding red shift, on the other hand, is very small. Earlier experimental two-photon fluorescence imaging using \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    p-FTAA\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " is shown not to target the dominant two-photon state, and ways to reach a higher image quality (lower signal-to-noise ratio) are proposed in terms of tuning the laser wavelength toward the region of 600 nm or the synthesis of asymmetric ligands with S1 states that are both one- and two-photon allowed.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_5912821.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Choline acetyltransferase\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ChAT\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") - \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Amyloid beta\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " peptides complex Molecular dynamics Trajectories<br>In silico molecular dynamics study was performed for the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    choline acetyltransferase\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ChAT\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") - \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Abeta\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " peptides complex. The molecular docking of A 40 and A 42 on \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ChAT\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " suggested three most probable binding clusters for both the A peptides. Thus generating \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ChAT\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-A 40 Cluster-0, \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ChAT\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-A 40 Cluster-1, \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ChAT\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-A 40 Cluster-2 for A 40 peptide on \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ChAT\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", likewise \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ChAT\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-A 42 Cluster-0, \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ChAT\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-A 42 Cluster-1, \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ChAT\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-A 42 Cluster-2 were generated for A 42 peptide on \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ChAT\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". Each of the folders contains the topology file with a .gro extension and a trajectory file with .xtc extension generated from the \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    100 ns\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       " molecular dynamics performed for each of the clusters mentioned above that were generated from the molecular docking. The folders are named as follows: \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ChAT\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " AB40 Cluster 0: Containing the topology file (ab40 0.gro) and the trajectory file (ab40 0.xtc) \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ChAT\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " AB40 Cluster 1: Containing the topology file (ab40 1.gro) and the trajectory file (ab40 1.xtc) \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ChAT\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " AB40 Cluster 2: Containing the topology file (ab40 2.gro) and the trajectory file (ab40 2.xtc) \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ChAT\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " AB42 Cluster 0: Containing the topology file (ab42 0.gro) and the trajectory file (ab42 0.xtc) \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ChAT\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " AB42 Cluster 1: Containing the topology file (ab42 1.gro) and the trajectory file (ab42 1.xtc) \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ChAT\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " AB42 Cluster 2: Containing the topology file (ab42 2.gro) and the trajectory file (ab42 2.xtc </div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_15550.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">MD simulation trajectory and related files for \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DPPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " bilayer (\n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GAFFlipid\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Gromacs 4.5\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       ")<br>Equilibrated \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DPPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " lipid bilayer simulation ran with \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Gromacs 4.5\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GAFFlipid\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " force field (https://removed \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    50ns\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       ", T \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    323K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       ", 72 \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DPPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " molecules, 2197 \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " molecules. This data is ran for the nmrlipids.blospot.fi project. More details from nmrlipids.blospot.fi and https://removed If data is used, please cite nmrlipids.blogspot.fi project and the original publication of the parameters: Dickson et al. Soft Matter, 2012,8, 9617-9627 https://removed</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_3760965.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Computational insights into voltage dependence of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    polyamine\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " block in a strong inwardly rectifying \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    K+ channel\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "<br>Supplemental Material to the study &quot;Computational insights into voltage dependence of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    polyamine\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " block in a strong inwardly rectifying \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    K+ channel\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "&quot;, to be published in Frontiers in Pharmacology. The upload contains (1) a representative trajectory (run4), and (2) two frames of the SPM inward-pulling simulations: (1) Free MD run4 with outward driving force, containing the compressed .xtc, the .tpr, and the final step as .gro file (2) steered MD, containing the starting frame (SPM in the central cavity below the Selectivity Filter) and a representative frame of SPM interacting with the CTD-binding site. </div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_57131.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Aldeghi et al. Files for absolute free energy calculations in \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    gromacs\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       ".<br>These are the files for performing absolute free energy calculations using \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    gromacs\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " as reported in &quot;Accurate calculation of the absolute free energy of binding for drug molecules. Aldeghi M, Heifetz A, Bodkin MJ, Knapp S, Biggin PC. Chem Sci. 2016 Jan 14;7 (1):207-218. DOI: 10.1039/C5SC02678D&quot; The files should prove useful for anyone wishing to try out their own methodology for comparison purposes or even just to repeat the work on a known dataset. The data is presented as a zip archive that should unpack into a directory called &quot;Aldeghi-et-al-chemical-science-2016&quot;. There are four sub-directories in there and a README.txt file which should explain the details of the data. </div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_14594.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">MD simulation trajectory and related files for \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DPPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " bilayer in full hydration (\n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Poger GROMOS 53A6 L\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Gromacs 4.0.7\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       ", PME, traj 1)<br>Equilibrated \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DPPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " lipid bilayer simulation ran with \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Gromacs 4.0.7\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " using PME (Particle Mesh Ewald) for computing electrostatics, \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Poger GROMOS 53A6 L\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " force field in full hydration (dx.doi.org/10.1002/jcc.21396), \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    100ns\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       ", T \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    323K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       ", 128 \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DPPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " molecules, 5841 \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " molecules. This data is ran for the nmrlipids.blospot.fi project. More details from nmrlipids.blospot.fi and https://removed If data is used, please cite nmrlipids.blogspot.fi project and the original publication of the parameters (dx.doi.org/10.1002/jcc.21396).</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_21175362.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Scalable Constant pH Molecular Dynamics in \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GROMACS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       "<br>Molecular dynamics (MD) computer simulations are used routinely to compute atomistic trajectories of complex systems. Systems are simulated in various ensembles, depending on the experimental conditions one aims to mimic. While constant energy, temperature, volume, and pressure are rather straightforward to model, pH, which is an equally important parameter in experiments, is more difficult to account for in simulations. Although a constant pH algorithm based on the -dynamics approach by Brooks and co-workers [Kong, X.; Brooks III, C. L. J. Chem. Phys.1996, 105, 2414 2423] was implemented in a fork of the \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GROMACS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " molecular dynamics program, uptake has been rather limited, presumably due to the poor scaling of that code with respect to the number of titratable sites. To overcome this limitation, we implemented an alternative scheme for interpolating the Hamiltonians of the protonation states that makes the constant pH molecular dynamics simulations almost as fast as a normal MD simulation with \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GROMACS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       ". In addition, we implemented a simpler scheme, called multisite representation, for modeling side chains with multiple titratable sites, such as \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    imidazole\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " rings. This scheme, which is based on constraining the sum of the -coordinates, not only reduces the complexity associated with parametrizing the intramolecular interactions between the sites but also is easily extendable to other molecules with multiple titratable sites. With the combination of a more efficient interpolation scheme and multisite representation of titratable groups, we anticipate a rapid uptake of constant pH molecular dynamics simulations within the \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GROMACS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " user community.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_7370684.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Stereospecific 1,3 H Transfer of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Indenols\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Proceeds via Persistent Ion-Pairs Anchored by NH Interactions<br>The base-catalyzed rearrangement of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    arylindenols\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " is a rare example of a suprafacial [1,3]- hydrogen atom transfer. The mechanism has been proposed to proceed via sequential [1,5]-sigmatropic shifts, which occur in a selective sense and avoid an achiral intermediate. A computational analysis using quantum chemistry casts serious doubt on these suggestions: These pathways have enormous activation barriers, and in constrast to what is observed experimentally, they overwhelmingly favor a racemic product. Instead we propose that a suprafacial [1,3]-prototopic shift occurs in a two-step deprotonation/reprotonation sequence. This mechanism is favored by 15 kcal mol 1 over that previously proposed. Most importantly, this is also consistent with stereospecificity since reprotonation occurs rapidly on the same -face. We have used explicitly solvated molecular dynamics studies to study the persistence and condensed-phase dynamics of the intermediate ion-pair formed in this reaction. Chirality transfer is the result of a particularly resilient contact ion-pair, held together by electrostatic attraction and a critical NH interaction which ensures that this species has an appreciable lifetime even in polar solvents such as \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DMSO\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    MeOH\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ".</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_13301090.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Data Sheet 2 Spontaneous Membrane Nanodomain Formation in the Absence or Presence of the Neurotransmitter \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Serotonin\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " .ZIP<br>Detailed knowledge on the formation of biomembrane domains, their structure, composition, and physical characteristics is scarce. Despite its frequently discussed importance in signaling, e.g., in obtaining localized non-homogeneous receptor compositions in the plasma membrane, the nanometer size as well as the dynamic and transient nature of domains impede their experimental characterization. In turn, atomistic molecular dynamics (MD) simulations combine both, high spatial and high temporal resolution. Here, using \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    microsecond\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       " atomistic MD simulations, we characterize the spontaneous and unbiased formation of nano-domains in a plasma membrane model containing \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    phosphatidylcholine\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "), \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    palmitoyl-sphingomyelin\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PSM\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "), and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    cholesterol\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Chol\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") in the presence or absence of the neurotransmitter \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    serotonin\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " at different temperatures. In the ternary mixture, highly ordered and highly disordered domains of similar composition coexist at 303 K. The distinction of domains by lipid acyl chain order gets lost at lower temperatures of 298 and 294 K, suggesting a phase transition at ambient temperature. By comparison of domain ordering and composition, we demonstrate how the domain-specific binding of the neurotransmitter \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    serotonin\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " results in a modified domain lipid composition and a substantial downward shift of the phase transition temperature. Our simulations thus suggest a novel mode of action of neurotransmitters possibly of importance in neuronal signal transmission.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_4743637.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Calcium Assists \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Dopamine\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Release by Preventing Aggregation on the Inner Leaflet of Presynaptic Vesicles<br>In this study, the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    dopamine\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " lipid bilayer interactions were probed with three physiologically relevant ion compositions using atomistic molecular dynamics simulations and free energy calculations. The in silico results indicate that calcium is able to decrease significantly the binding of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    dopamine\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " to a neutral (zwitterionic) \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    phosphatidylcholine\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " lipid bilayer model mimicking the inner leaflet of a presynaptic vesicle. We argue that the observed calcium-induced effect is likely in crucial role in the neurotransmitter release from the presynaptic vesicles docked in the active zone of nerve terminals. The inner leaflets of presynaptic vesicles, which are responsible for releasing neurotransmitters into the synaptic cleft, are mainly composed of neutral lipids such as \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    phosphatidylcholine\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    phosphatidylethanolamine\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". The neutrality of the lipid head group region, enhanced by a low pH level, should limit membrane aggregation of transmitters. In addition, the simulations suggest that the high calcium levels inside presynaptic vesicles prevent even the most lipophilic transmitters such as \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    dopamine\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " from adhering to the inner leaflet surface, thus rendering unhindered neurotransmitter release feasible.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_11697906.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Biosynthetic Mechanism of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Lanosterol\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ": A Completed Story<br>The remarkable biochemical conversion of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    acyclic 2,3-oxidosqualene\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " to \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    tetracyclic lanosterol\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (the common precursor to \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    cholesterol\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and its relatives) with seven stereocenters constructed, catalyzed by \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    oxidosqualene cyclase\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    OSC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "), has fascinated chemists for over a half century. Although many experimental and theoretical efforts have been reported, most of the studies focused on the initial cyclization while the subsequent cascade carbocation rearrangement and deprotonation were ignored, which determine the regio- and stereochemistry of the final product \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    lanosterol\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " associated with the function-related structure domains of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    cholesterol\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and its relatives. Herein we continue our previous works on the cyclization (Angew. Chem., Int. Ed. 2015, 54, 8693 8696); the mechanistic details of the remaining part of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    lanosterol\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " biosynthesis, involving three hydride shifts, two methyl shift and one deprotonation, were investigated by quantum mechanical/molecular mechanical (QM/MM) molecular dynamics (MD) simulations. We obtained the complete free energy profile for \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    lanosterol\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " biosynthesis, which is thermodynamically and kinetically reasonable for the sole formation of the desired product with byproducts effectively avoided. We identified some key factors such as electrostatic interactions and CH interactions that facilitate migration of the carbocation. We also identified the direct deprotonation precursor, intermediate I, which has been controversial in previous studies. More importantly, we found that the enzyme mediates the energy transfer from the initial cyclization to the subsequent rearrangements through the electric field of the active pocket, which guarantees the fidelity of the enzyme catalysis.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_5651101.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Structural analysis of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    human glycoprotein butyrylcholinesterase\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " using atomistic molecular dynamics: The importance of glycosylation site ASN241<br>\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Human butyrylcholinesterase\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    BChE\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") is a glycoprotein capable of bioscavenging toxic compounds such as organophosphorus (OP) nerve agents. For commercial production of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    BChE\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", it is practical to synthesize \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    BChE\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " in non human expression systems, such as plants or animals. However, the glycosylation profile in these systems is significantly different from the human glycosylation profile, which could result in changes in \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    BChE\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " s structure and function. From our investigation, we found that the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    glycan\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " attached to ASN241 is both structurally and functionally important due to its close proximity to the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    BChE\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " tetramerization domain and the active site gorge. To investigate the effects of populating glycosylation site ASN241, monomeric human \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    BChE\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " glycoforms were simulated with and without site ASN241 glycosylated. Our simulations indicate that the structure and function of human \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    BChE\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " are significantly affected by the absence of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    glycan\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " 241.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_16772812.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Code and Data for &quot;Physics-driven coarse-grained model for biomolecular phase separation with near-quantitative accuracy&quot;<br>This file is part of the supporting data for the manuscript 'Physics-driven coarse-grained model for biomolecular phase separation with near-quantitative accuracy', and contains scripts and code for running all-atom and coarse-grained simulations of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    proteins\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " described in the manuscript.The included README file outlines the structure of the archive and contains \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Lammps\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " installation instructions for running the remaining code supplied.There are five directories in this archive. Four of these correspond to figures in the results section of the manuscript, and each one contains a separate README file detailing its contents. They include \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Gromacs\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Lammps\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " scripts with example simulation set-ups to run simulations corresponding to figures 2 [potentials of mean force of all-atom models], 4 [radius of gyration simulations of coarse-grained models], 5 [direct-coexistence simulations used to determine the phase diagrams] and 6 [multiphase equilibria] of the manuscript. The directories contain input scripts and corresponding parameter and simulation configuration files, as well as example simulation output to benchmark against.The final directory, 'all-model-parameters', contains \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Lammps\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " parameter files not only for the \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Mpipi\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " potential, but also for all the other models we have benchmarked in the paper. These parameter files can be used instead of the \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Mpipi\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " parameters in the simulations provided in the other directories.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_9905063.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Acceptance and Kinetic Resolution of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Methyl\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-Substituted \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Aldehydes\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " by \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Norcoclaurine Synthases\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "<br>\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Norcoclaurine synthase\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    NCS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") catalyzes a stereoselective Pictet Spengler reaction to give the key intermediate, \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    (S)-norcoclaurine\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " in \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    benzylisoquinoline\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    alkaloid\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " biosynthesis. This family of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    alkaloids\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " contains many bioactive molecules including \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    morphine\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    berberine\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". Recently, \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    NCS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " has been demonstrated to accept a variety of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    aldehydes\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and some \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ketones\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " as substrates, leading to a range of chiral \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    tetrahydroisoquinoline\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    THIQ\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") products. Here, we report the unusual acceptance of -substituted \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    aldehydes\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", in particular \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    methyl\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-substituted aldehydes, by wild-type \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Thalictrum flavum NCS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " ( \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    33TfNCS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") to give \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    THIQ\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " products. Moreover, the kinetic resolution of several -substituted aldehydes to give \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    THIQs\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " with two defined chiral centers in a single step with high conversions was achieved. Several \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    dopamine\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " analogues were also accepted as substrates, and reactions were amenable to scale up. Active site mutants of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    TfNCS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " were then used, which demonstrated the potential to enhance the stereoselectivities in the reaction and improve yields. The rationale for the acceptance of these substrates and improved activity with different mutants has been gained from a co-crystallized structure of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    33TfNCS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " with a nonproductive mimic of a reaction intermediate bound in the active site. Finally, molecular dynamics simulations were performed to study the binding of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    dopamine\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and an -substituted \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    aldehyde\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and provided further insight into the reaction with these substrates.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_12064266.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">On Use of the \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Amber\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " Potential with the Langevin Dipole Method<br>Inclusion of solvent effects in biomolecular simulations is most ideally done using explicit methods, as they are able to capture the heterogeneous environment typical of biomolecules and systems involving them (e.g., proteins at solid interfaces). Common explicit methods based on molecular solvent models (e.g., \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    TIP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    SPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " models) and molecular dynamic or Monte Carlo simulation are computationally expensive and are, therefore, not well-suited to situations where many simulations are required (e.g., in the ab initio structure prediction or design contexts). In such cases, more coarse-grained explicit approaches such as the Langevin dipole (LD) method of Warshel and co-workers are more appropriate. The recent incarnations of the LD method appear to produce good solvation free energy estimates. These incarnations use charges and solute structures obtained from high-level quantum mechanics simulations. As such an approach is clearly not possible for larger solutes or when many structures are to be considered, an alternative must be sought. One possibility is to use structures and charges derived from an existing analytical potential model we report on such a coupling here with the \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Amber\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " potential model. The accuracy and computational performance of this hybrid approach, which we term LD \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Amber\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " to distinguish it from previous incarnations of the LD method, was assessed by comparing results obtained from the approach with those from experiment and other theoretical methods for the solvation of 18 \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    amino acid\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " analogues and the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    alanine dipeptide\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". This comparison shows that the LD \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Amber\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " approach can yield results in line with experiment both qualitatively and quantitatively and is as accurate as other explicit methods while being computationally much cheaper.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_14669430.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Scaffold Hopping Transformations Using Auxiliary Restraints for Calculating Accurate Relative Binding Free Energies<br>In silico screening of drug target interactions is a key part of the drug discovery process. Changes in the drug scaffold via contraction or expansion of rings, the breaking of rings, and the introduction of cyclic structures from acyclic structures are commonly applied by medicinal chemists to improve binding affinity and enhance favorable properties of candidate compounds. These processes, commonly referred to as scaffold hopping, are challenging to model computationally. Although relative binding free energy (RBFE) calculations have shown success in predicting binding affinity changes caused by perturbing R-groups attached to a common scaffold, applications of RBFE calculations to modeling scaffold hopping are relatively limited. Scaffold hopping inevitably involves breaking and forming bond interactions of quadratic functional forms, which is highly challenging. A novel method for handling ring opening/closure/contraction/expansion and linker contraction/expansion is presented here. To the best of our knowledge, RBFE calculations on linker contraction/expansion have not been previously reported. The method uses auxiliary restraints to hold the atoms at the ends of a bond in place during the breaking and forming of the bonds. The broad applicability of the method was demonstrated by examining perturbations involving small-molecule macrocycles and mutations of proline in proteins. High accuracy was obtained using the method for most of the perturbations studied. The rigor of the method was isolated from the force field by validating the method using relative and absolute hydration free energy calculations compared to standard simulation results. Unlike other methods that rely on -dependent functional forms for bond interactions, the method presented here can be employed using modern molecular dynamics software without modification of codes or force field functions.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_3780206.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Molecular dynamics simulation data of regulatory \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ACT\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " domain dimer of human \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    phenylalanine hydroxylase\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PAH\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ")<br>Raw data of molecular dynamics simulations of regulatory \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ACT\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " domain dimer. binding.zip: simulation starting from 21 dimer conformations with 19 \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Phe\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " ligand bound.zip: simulation starting from dimer with bound \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Phe\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " ligand dimer.zip: simulation starting from 21 dimer conformations simulation Simulation setup files are also included in each folder. Details can be found in this paper: Yunhui Ge, Elias Borne, Shannon Stewart, Michael R. Hansen, Emilia C. Arturo, Eileen K. Jaffe and Vincent A. Voelz. Simulation of the regulatory \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ACT\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " domain of human \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PAH\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " unveil the mechanism of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    phenylalanine\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " binding. J. Biol. Chem., 2018, 293 (51), pp 19532-19543</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_9985175.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">\n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Martini\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " Coarse-Grained Model for Clay Polymer Nanocomposites<br>We have developed a coarse-grained (CG) model of a polymer clay system consisting of organically modified \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    montmorillonite\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (oMMT) nanoclay as the nanoparticle in accordance with the \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    MARTINI\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " force field. We have used mechanical properties and cleavage free energy of the clay particle to respectively parameterize bonded and nonbonded interaction parameters for an oMMT clay particle, where intergallery \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Na+\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " ions are replaced by \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    tetramethylammonium\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    TMA\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") ions. The mechanical properties were determined from the slope of the stress strain curve and cleavage free energy was determined by allowing for full surface reconstruction corresponding to a slow equilibrium cleavage process. Individual dispersive and polar contributions to oMMT cleavage energy were used for determination of appropriate \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    MARTINI\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " bead types for the CG oMMT sheet. The self-consistency of the developed \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    MARTINI\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " FF parameters for the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    TMA\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    montmorillonite\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " polymer system was verified by comparing estimates for select structural, thermodynamic, and dynamic properties obtained in all-atomistic simulations with that obtained in CG simulations. We have determined the influence of clay particles on properties of three polymer melts (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    polyethylene\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    polypropylene\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    polystyrene\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") at two temperatures to establish transferability of the developed parameters. We have also shown that the effect of clay polymer interactions on structure property relationships in the polymer clay nanocomposite system is well captured by Rosenfeld s excess entropy scaling.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_7024651.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">A quantitative interphase model for polymer nanocomposites: Verification, validation, and consequences regarding size effects: dataset<br>Abstract: (from [1]) The enhanced mechanical behavior of polymer nanocomposites with spherical filler particles is attributed to the formation of matrix-filler interphases. The nano-scale leads to particularly high interphase volume fractions while rendering experimental investigations extremely difficult. Previously, we introduced a molecular dynamics-based interphase model capturing the crucial spatial profiles of elastic and inelastic properties inside the interphase. This contribution demonstrates that our model captures polymer nanocomposites essential characteristics reported from experiments. To this end, we thoroughly verify and validate the model before discussing the resulting local plastic strain distribution. Furthermore, we obtain a reinforcement in terms of the overall stiffness for smaller particles and higher filler contents, while the influence of particle spacing seems negligible, matching experimental observations in the literature. This paper proposes a methodology to unravel the underlying complex mechanical behavior of polymer nanocomposites and to translate the findings into engineering quantities accessible to a broader audience and technical applications. Contact: Maximilian Ries Institute of Applied Mechanics Friedrich-Alexander-Universita t Erlangen-Nu rnberg Egerlandstr. 5 91058 Erlangen Software: \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Abaqus version R2018\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " License: Creative Commons Attribution 4.0 International Context: Data set supplementing journal paper: [1] Ries, M.; Weber, F.; Possart, G.; Steinmann, P. &amp; Pfaller, S., A quantitative interphase model for polymer nanocomposites: Verification, validation, and consequences regarding size effects , Composites Part A: Applied Science and Manufacturing, 2022, 107094. This dataset contains the results presented in [1] and the necessary data to obtain those. Content: simulation folder denotation ( - used instead of decimal points): distance particles radius particle thickness ip num ip length box factor el length fraction box length switch mat ip with distance particles: center distance of the nanoparticles in nm radius particle: radius of the nanoparticles in nm thickness ip: thickness of the interphase layers in nm num ip: number of interphase layers length box: box edge length in nm factor el length: factor scaling the element length on the arcs of the interphase layers (element length   factor el length * thickness ip) fraction box length: matrix element length   length box / fraction box length switch mat ip: if   0: interphases are assigned their actual material properties, if   1: interphases are assigned the material properties of the bulk each simulation folder contains the following file types: .cae: Abaqus model database, containing parts, meshes, loads, etc. .dat: Printed output from the analysis input file processor, as well as printed output of selected results written during the analysis .inp: Analysis input file .log: Log file, which contains start and end times for modules run by the current execution procedure .msg: Diagnostic or informative messages about the progress of the solution .odb: Output database containing all results data from an Abaqus analysis .sta: Status file with increment summaries folder structure: Standard case: simulation folders of the standard close (particle center distance: 5.1776 nm) and distant (particle center distance: 7.9481 nm) cases (particle radius: 2 nm, filler content 0.054 vol.%, number of interphase layers: 4, factor el length: 1.0) and further particle center distances Layers: simulation folders with different numbers of interphase layers, i.e., different values for num ip, based on the standard close and distant cases Close case Distant case Mesh: simulation folders with different mesh qualities, i.e., different values for factor el length, based on the standard close and distant cases Close case Distant case Particle size: simulation folders with different particle sizes 2 nm: simulation folders with particle surface distance 2 nm vol ratio 0-00054: simulation folders with filler content 0.054 vol.% vol ratio 0-0075: simulation folders with filler content 0.75 vol.% 4 nm: simulation folders with particle surface distance 4 nm vol ratio 0-00054: simulation folders with filler content 0.054 vol.% vol ratio 0-0075: simulation folders with filler content 0.75 vol.% 8 nm: simulation folders with particle surface distance 8 nm vol ratio 0-00054: simulation folders with filler content 0.054 vol.% vol ratio 0-0075: simulation folders with filler content 0.75 vol.% </div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_5573728.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Exploring the interaction of a \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    curcumin azobioisostere\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " with \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Abeta42\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " dimers using replica exchange molecular dynamics simulations<br>Structural data and parameters relative to the evaluation of the interaction of an anti aggregating \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    azobioisostere\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " compound with the full-length A 42 peptide by replica-exchange molecular dynamics (REMD) simulations. Two different force fields (\n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Amber\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CHARMM\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       ") were used to simulate the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    azobioisostere-Abeta42\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    AZ-Ab42\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") complex in a monomeric and dimeric assembly. A brief description of the shared output data is reported below: 1. \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Amber\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CHARMM\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       "-adapted parameters for the simulated \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    azobioisostere\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    AZ\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") compound. 2. Modified version of the \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CHARMM36m\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " FF - (\n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CHARMM36mW\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       ") 3. Starting (equilibrated) structures (first 5 T-replicas) for REMD on each of the following three systems (PDB): \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Amber\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       ": AZ-Ab42 (monomeric ensemble): Repl.0 (\n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    315.0 K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       "), Repl.1 (\n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    316.7 K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       "), Repl.2 (\n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    318.4 K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       "), Repl.3 (\n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    320.1 K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       "), Repl.4 (\n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    321.8 K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       ") \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Amber\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       ": AZ-Ab42 (dimeric ensemble): Repl.0 (\n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    315.0 K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       "), Repl.1 (\n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    316.0 K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       "), Repl.2 (\n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    317.0 K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       "), Repl.3 (\n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    318.0 K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       "), Repl.4 (\n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    319.1 K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       ") \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CHARMM\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       ": AZ-Ab42 (dimeric ensemble): Repl.0 (\n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    315.0 K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       "), Repl.1 (\n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    316.0 K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       "), Repl.2 (\n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    317.0 K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       "), Repl.3 (\n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    318.0 K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       "), Repl.4 (\n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    319.1 K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       ") 4. Most populated clusters for the three simulated systems (PDB): \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Amber\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " AZ-Ab42 (monomeric ensemble): 7 clusters (Cl.0: 12.4%, Cl.1: 10.1%, Cl.2: 9.8%, Cl.3: 5.4%, Cl.4: 3.7%, Cl.5: 3.6%, Cl.6: 2.0%) \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Amber\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " AZ-Ab42 (dimeric ensemble): 5 clusters (Cl.0: 7.1%, Cl.1: 4.3%, Cl.2: 3.5%, Cl.3: 3.2%, Cl.4: 2.4%) \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CHARMM\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       ": AZ-Ab42 (dimeric ensemble): 3 clusters (Cl.0: 4.7%, Cl.1: 4.6%, Cl.2: 2.5%) </div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_5153261.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">MD simulations of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    SARS-CoV-2 Spike\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Protein under static electric fields This dataset contains trajectories corresponding to all-atom MD simulations of segments of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    SARS-CoV-2 Spike\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Protein, and in-silico mutations, under the influence of moderate external electric fields. The final structures of some of the simulations were used to perform in-silico docking with \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ACE2 receptor\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " to evaluate the effect of comformational changes (docking was perform with \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PyDOCK\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       "). The file trajectories 6vsb dt1ns.zip contains trajectories of simulations that were performed on a segment of the Protein Data Bank ID \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    6VSB\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " comprising RBD, SD1 and SD2. The file trajectories 6m0j dt1ns.zip correspond to the RBD in \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Protein\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Data Bank ID 6M0J. The file trajectories in-silico mutations dt1ns.zip correspond to simulations performed on in-silico generated mutations following the mutations corresponding to WHO Variants of Concern UK, South Africa and Brazil. In all cases, simulations were performed at different electric field intensities ranging between 104 V/m and 107 V/m, with an extra short simulation under very high intensity (109 V/m). The file docked structures 6m0j.zip contains the 100 best scored docked structures for each case as the output of \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PyDOCK\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       ". Trajectories are stored in \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GROMACS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " compressed trajectory file format (.xtc), downsampled to a 1ns timestep. Individual trajectories length are between \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    300 nanoseconds\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    1 microsecond\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       ". In-silico docked structures are in PDB format. See linked preprint for more details.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_3623150.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Function and evolution of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    B-Raf\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " loop dynamics relevant to cancer recurrence under drug inhibition<br>Oncogenic mutations in the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    kinase\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " domain of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    B-Raf\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " protein have long been associated with cancers involving the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    RAF\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    MEK\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ERK\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " pathway. One constitutive \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ERK\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " activating mutation in \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    B-Raf\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", the V600E (valine to glutamate) replacement occurring adjacent to a site of threonine phosphorylation (T599) occurs in many types of cancer, and in a large percentage of certain cancers, such as melanoma. Because natural \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ATP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " binding activity and the V600E mutation are both known to alter the physical behavior of the activation loop in the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ATP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " binding domain, this system is especially amenable to comparative functional analyses using molecular dynamics of drug classes and genetic variants at all-atom resolution. Here, we employ machine learning enabled identification of functionally conserved protein dynamics to compare how disease pertinent molecular variations impact conserved loop dynamics during the binding of four \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ATP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " competitive \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    B-Raf inhibitors\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    sorafenib\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    vemurafenib\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    dabrafenib\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PLX7904\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") that differ in their effectiveness in preventing cancer recurrence. We demonstrate that drug development targeting \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    B-Raf\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " has progressively moved towards \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ATP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " competitive inhibitors that demonstrate less tendency to mimic the functional dynamic shifts associated with \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ATP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " binding in this domain. We argue this functional dynamic mimicry in first generation \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    B-Raf\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " inhibitors increases the side effect of hyperactivation (i.e. inducing \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    MAPK\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " activation in non-tumorous cells in the absence of secondary mutation). Within the context of the binding interaction of each inhibitor, we compare the functional dynamic impacts of V600E and other sensitizing and drug resistance causing mutations in the activation and P-loops, confirming sites of low mutational tolerance in these two functional regions. Lastly, we investigate V600E sensitivity of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    B-Raf\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " loop dynamics in an evolutionary context, demonstrating that while it probably shares partial origin with its early evolution in primitive eukaryotes, the functional sensitivity to V600E was secondarily increased during early jawed vertebrate evolution.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_7607222.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">A \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GROMOS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " Force Field for \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Furanose\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-Based \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Carbohydrates\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "<br>The article describes a \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GROMOS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " force field parameter set for molecular dynamics simulations of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    furanose\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    carbohydrates\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". The proposed united-atom force field is designed and validated with respect to the conformational properties of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    furanose\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " mono-, di-, oligo-, and polymers in aqueous solvent. The set accounts for the possibility of arbitrary glycosidic linkage connectivity between units, O-alkylation, as well as of different anomery. The compatibility with the already existing, \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    pyranose\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-dedicated \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GROMOS 56A6CARBO/CARBO R\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " set allows one to use the presently proposed extension for studying more diverse and biologically relevant \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    carbohydrates\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " that exploit both \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    pyranose\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    furanose\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " units. The validation performed against the quantum-mechanical and experimental data concerning the structural and conformational features shows that the newly developed set is capable to reproduce conformational equilibrium within the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    furanose\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " ring, relative free energies of anomers, \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    hydroxymethyl\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " rotamers, and glycosidic linkage conformers. Additionally, the results concerning the conformation of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    furanose\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " ring with relation to the two-state model as well as other conformational features of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    furanose\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-containing \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    saccharides\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " are discussed.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_4673703.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Data from &quot;Phase equilibrium of liquid \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and hexagonal ice from enhanced sampling molecular dynamics simulations&quot;<br>This repository contains the input files to reproduce the results of the paper mentioned above. Brief summary of the contents of each folder: Environments : This folder has the reference environments that will be used for the definition of the CV Equilibration : Input and output files for the equilibration of liquid \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and ice Ih. Distributions of the kernel and equilibrium box sizes for production runs. BiasedSimulations : Input and output files for the biased simulations (production runs). Most of the analysis of the simulations is described in a Jupyter Notebook in BiasedSimulations/Analysis.ipynb.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_7857659.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Determinants of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Orexin Receptor\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Binding and Activation A Molecular Dynamics Study<br>We assess the stability of two previously suggested binding modes for the neuropeptide \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    orexin-A\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " in the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    OX2 receptor\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " through extensive molecular dynamics simulations. As the activation determinants of the receptor remain unknown, we simulated an unliganded receptor and two small-molecular ligands, the antagonist suvorexant and the agonist \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Nag26\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " for comparison. Each system was simulated in pure \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " membrane as well as in the 25% \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    cholesterol\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " membrane. In total, we carried out \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    36 μs\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       " of simulations. Through this set of simulations, we report a stable binding mode for the C-terminus of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    orexin-A\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " . In addition, we suggest interactions that would promote \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    orexin receptor\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " activation, as well as others that would stabilize the inactive state.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_6870476.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Dataset for: Conformational and Thermodynamic Differences Underlying Wild-type and Mutant YEATS Domain Specificity for Epigenetic Marks<br>The set of molecular dynamics trajectories generated and analyzed in the paper &quot;Conformational and Thermodynamic Differences Underlying Wild-type and Mutant YEATS Domain Specificity for Epigenetic Marks.&quot; Citations should refer directly to the manuscript. The preprint version is available at: https://doi.org/10.1101/2022.07.14.500085 The uploaded zip file &quot;trajectory data.zip&quot; contains trajectories for the (1) \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    apo\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (2) \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    h3k27ac\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and (3) \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    h3k9ac\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " systems. In each directory is a subdirectory corresponding to the wild-type, deletion, and insertion mutations. In each of those subdirectories are 10 XTC files, which correspond to the trajectories for the 10 different simulations performed for each system, along with an \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    AMBER\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       "-formated \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PRMTOP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " file. Solvent molecules were removed from each file, and trajectories were strided such that there is one frame per 500 ps. </div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_11569452.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Rationalizing the Activity of an Artificial Diels-Alderase : Establishing Efficient and Accurate Protocols for Calculating Supramolecular Catalysis<br>Self-assembled cages have emerged as novel platforms to explore bioinspired catalysis. While many different size and shape supramolecular structures are now readily accessible, only a few are known to accelerate chemical reactions under substoichiometric conditions. These limited examples point to a poor understanding of cage catalysis in general, limiting the ability to design new systems. Here we show that a simple and efficient density-functional-theory-based methodology, informed by explicitly solvated molecular dynamics and coupled cluster calculations, is sufficient to accurately reproduce experimental guest binding affinities (MAD   1.9 kcal mol 1) and identify the catalytic Diels Alder proficiencies (&gt;80% accuracy) of two homologous \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Pd2L4\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " metallocages with a variety of substrates. This analysis reveals how subtle structural differences in the cage framework affect binding and catalysis. These effects manifest in a smaller distortion and more favorable interaction energy for the catalytic cage compared to the inactive structure. This study gives detailed insight that would otherwise be difficult to obtain from experiments, providing new opportunities in the design of catalytically active supramolecular cages.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_7572990.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">MD simulation input files and results for ,,Atomistic MD simulations of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    n-alkanes\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " in a \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    phospholipid\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " bilayer: \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CHARMM36\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " versus \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Slipids\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       "<br>Input files and trajectories of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    n-alkane\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "/lipid systems used in the article: ,,Atomistic MD simulations of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    n-alkanes\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " in a \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    phospholipid\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " bilayer: \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CHARMM36\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " versus \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Slipids\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       ". Equilibrated starting configurations were created using \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CHARMM36\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       ". Otherwise, filenames specifiy the force field used (C36 or SL), Temperature and system composition. .xtc trajectories were created using gmx trjconv with options -pbc nojump -dt 10. run2 equals simulations from \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    100\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       "-\n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    200ns\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       ", run3 equals \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    200\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       "-\n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    300ns\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       ". The first 100ns were excluded from the analysis and are not included in this dataset. Exemplary .mdp files have been included for both force-fields. Please refer to the manuscript for the force field sources and additional information.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_51748.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Molecular dynamics simulation trajectory of a cationic \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    lipid\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " bilayer: 50/50 mol% \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DMTAP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "/\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DMPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " System: Symmetric bilayer of cationic \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DMTAP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    dimyristoyltrimethylammoniumpropane\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", 50 mol-%) and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    zwitterionic\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DMPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    dimyristoylphosphatidylcholine\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", 50 mol-%) \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    lipids\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " in water. Number of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DMPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ": 64. Number of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DMTAP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ": 64. Number of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Cl\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "--\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ions\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ": 64. Number of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    waters\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ": 5336. \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Lipid\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " model: \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    'Berger\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       "' united-atom [Biophys. J. 72 2002 (1997)] with \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DMTAP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " modification by Gurtovenko et al. [Biophys. J. 86 3461 (2004)]. \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " model: \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    SPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " [In Intermolecular Forces, ed. Pullman. 331 (1981)]. Simulation engine: \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Gromacs 3.x\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " [https://removed Trajectory length: \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    149 ns.\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       " Previously equilibrated for: 41 ns. Sampling rate: every 10 ps. Time integration step: 2 fs. Thermodynamic ensemble: NpT. Temperature coupling: 'Berendsen' [J. Chem. Phys. 81 3684 (1984)] with lipids and water coupled separately at T 323 K. Pressure coupling: 'Berendsen' [J. Chem. Phys. 81 3684 (1984)] with xy and z coupled separately at p 1.0 bar. Electrostatics: PME [J. Chem. Phys. 98 10089 (1993); J. Chem. Phys. 103 8577 (1995)], real-space cutoff at 1.0 nm. Van der Waals: Truncated at 1.0 nm. Constraints: Covalent bond lengths in lipids using LINCS [J. Comput. Chem. 18 1463 (1997)], in water using SETTLE [J. Comput. Chem. 13 952 (1992)]. Used in publications: [1] Markus S. Miettinen, Andrey A. Gurtovenko, Ilpo Vattulainen, and Mikko Karttunen: &quot;Ion Dynamics in Cationic Lipid Bilayer Systems in Saline Solutions&quot;. J. Phys. Chem. B 113 9226 (2009). DOI: 10.1021/jp810233q. [2] Markus S. Miettinen: &quot;Computational Modeling of Cationic Lipid Bilayers in Saline Solutions&quot;. PhD Thesis. Aalto University School of Science and Technology, Helsinki, Finland. (2010). ISBN 978-952-60-3194-1.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_7691966.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Addressing Polarization Phenomena in Molecular Machines Containing Transition \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Metal Ions\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " with an Additive Force Field<br>Modeling transition metals in supramolecular assemblies, in general, is extremely challenging due to polarization and charge transfer. In this work, we demonstrate that the inherent shortcomings of additive force fields in modeling \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Cu+\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ether-O\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Cu+\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    olefin-C\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " interactions are rooted in the Lorentz Berthelot rules. A general method for investigating transition-metal-containing molecular assays using classical force fields is, therefore, proposed. In this strategy, QM/MM calculations have been performed to determine the potential of mean force (PMF) describing the interaction of a cation and a specific functional group. van der Waals parameters for the corresponding pairs of particles have then been optimized using the NBFIX feature of the \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CHARMM\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " force field to fit the QM/MM PMF. This method has been applied to decipher the mechanism underlying the dialing of a molecular machine controlled by \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Li+\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Cu+\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " cations, indicating that the process is controlled by the competition between cation \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ether-O\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and cation \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    olefin-C\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " interactions.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_2542912.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    BPTF\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    AU1\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " conformation space network<br>This dataset contains: allstates.gexf: a graph file describing the nodes and edges of the configuration space network. Nodes are clusters of molecular conformations, and edges are the connections between them, derived from molecular dynamics trajectories. This file also contains properties for each node: minimum ligand-\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    protein\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " distance (mind), ligand solvent accessible surface area (sasa), number of observations from A-initialized simulations (na), number of observations from B-initialized simulations (nb), total number of observations (ntot), free energy prediction (eig wt final sym). This file can be opened in network analysis programs (e.g. Gephi). \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    bptf\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " au1.pdb: a PDB file containing the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    BPTF\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " protein and the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    AU1\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " molecule clust90.dcd: a trajectory composed of randomly selected structures from cluster 90, which was predicted to be the most stable cluster according to the free energy allclusts.tgz: an archive file containing a (randomly selected) PDB structure for every cluster in the network </div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_1545562.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Longbow: A Lightweight Remote Job Submission Tool<br>We present Longbow, a lightweight console-based remote job submission tool and library. Longbow allows the user to quickly and simply run jobs on high performance computing facilities without leaving their familiar desktop environment. Not only does Longbow greatly simplify the management of compute-intensive jobs for experienced researchers, it also lowers the technical barriers surrounding high performance computation for the next generation of scientists and engineers. Longbow has already been used to remotely submit jobs in a number of projects and has the potential to redefine the manner in which high performance computers are used. Longbow can be installed by executing: pip install longbow The code is also contained within Longbow.zip however. In LongbowHosts.zip is an example hosts configuration file which provides the details Longbow needs of the remote resource on which the job will run. Example molecular dynamics jobs including input files are in LongbowExamples.zip. For full details on how to install and use Longbow see the documentation at http://www.hecbiosim.ac.uk/wikis/index.php/Longbow</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_1381865.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Molecular Dynamics Simulations of Sheet-like Replicators<br>This dataset contains 3D structures, MD trajectories and powder X-ray diffractograms (simulated and experimental) of tetrameric \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    carboxynaphthalene disulfide\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". The MD simulations were performed in \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " with periodic boundary conditions. Either \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    AMBER\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " or \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GROMACS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " packages were used. For detailed description of the simulations see the linked paper (the link will appear upon acceptance). PDB files are snapshots from the MD simulations; they can be viewed using any 3D structure viewer (e.g. Avogadro). The trajectories (*.xtc - \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GROMACS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       ", *.mdcrd - \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    AMBER\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       "/NetCDF) can be visualized using e.g. \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    VMD\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       ". Partial charges for the naphthalene building blocks are provided within *.mol2 files (N-0 for the neutral and N-1 for the deprotonated subunit). The parameter/topology files have .top (\n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GROMACS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       ") and .prmtop (\n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    AMBER\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       ") extensions. Files labeled &quot;sheet1&quot; contain results of simulations of 16 repeats of tetramer IV showed in Figure 4 and S13 of the paper. The suffix &quot;-exp box&quot; refers to structures which were constrained to the box determined experimentally (&quot;min&quot; - after energy minimization, &quot;equil&quot; - after equilibration). Files labeled &quot;sheet2&quot; contain results of simulations of 12 repeats of tetramer IV showed in Figure S14 of the paper. Files labeled &quot;sheet3&quot; contain results of simulations of 16 repeats of tetramer IV showed in Figure S15 of the paper. The assembly was unstable, the initial structure started dissociating from the very beginning of the simulation. The trajectory is in NetCDF format. Files labeled &quot;tetramerIV-2&quot; contain results of simulations of isolated, partially deprotonated tetramer IV in \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (solvent was removed from the trajectory). Link to the article will be added upon its acceptance.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_3988469.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">In silico prediction of ARB resistance: A first step in creating personalized ARB therapy<br>\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    AT1R\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Model preparation The crystal structure of human \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    AT1R\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " bound to \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    olmesartan\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (PDB: \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    4ZUD\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") was downloaded from the RCSB Protein Data Bank. \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    4ZUD\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " contains \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    apocytochrome\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    b562RIL\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " fused to the amino terminus, and many of the flexible regions, as well as helix 8, are not resolved. In order to generate an appropriate starting structure, \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    olmesartan\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    apocytochrome\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    b562RIL\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " fusion were removed from \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    4ZUD\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", and the missing regions were added to the protein with \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    MOE\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " software (Chemical Computing Group ULC, Montreal, Canada). Specifically, the N-Terminus (residues 1 to 25), intracellular loop 2 (residues 134 to 140), extracellular loop 2 (residues 186 to 188), intracellular loop 3 (residues 223 to 234), and helix 8 (residues 305 to 316) were added to the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    AT1R\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " in accordance to the human \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    AT1R\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " sequence and PDB: \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    4YAY\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". The remaining carboxyl-tail of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    AT1R\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (residues 317 to 359) was not modeled. The \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    AT1R\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " model then underwent an energy minimization within \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    MOE\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " using the \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Amber10: Extended Huckel Theory (EHT)\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " force field. Molecular dynamic (MD) simulations and analysis The \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    MOE\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " minimized \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    AT1R\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " was loaded into \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CHARMM-GUI\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       ". An 80 A by 80 A lipid bi-layer composed of 13% \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    cholesterol\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and 87% \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Phosphatidylcholine\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") was generated around the receptor. \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " was packed 17.5 A above and below the lipid bi-layer, and 150 mM \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Na+\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Cl-\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " ions were added to the system via Monte-Carlo ion placing. The all-atom \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CHARMM C36\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " force field for proteins and ions, and the \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CHARMM\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    TIP3P\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " force field for \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " were selected. A hard non-bonded cutoff of 8.0 angstroms was utilized. All molecular dynamics simulations were performed using the \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PMEMD\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " module of the \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    AMBER16\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " package with support for MPI multi-process control and GPU acceleration code. Orthorhombic periodic boundary conditions with a constant pressure of 1 atm was set via the NPT ensemble and temperature was set to \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    310.15 K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    37 C\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       ") using Langevin dynamics. The \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    SHAKE\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " algorithm was used to constrain bonds containing hydrogens. The dynamics were propagated using Langevin dynamics with Langevin damping coefficient of 1 ps-1 and a time step of 2 fs. Before the production run, the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    AT1R\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " model was minimized for 5000 steps using the steepest descent method and then equilibrated for 600 ps. The protein coordinates were saved in 10 ps intervals. The production run lasted \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    150 ns\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       ", at which point all three replicas were stable for at least the last 20 ns.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_7273800.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Block chemistry for accurate modeling of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    epoxy resins\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "<br>## Abstract from [1]: Accurate molecular modelling of the physical and chemical behavior of highly cross-linked \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    epoxy resins\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " at the atomistic scale is important for the design of new property-optimized materials. However, a systematic approach to parametrizing and characterizing these systems in molecular dynamics is missing. We, therefore, present a unified scheme to derive atomic charges for amine-based \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    epoxy resins\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", in agreement with the \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    AMBER\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " force field, based on defining reactive fragments blocks building the network. The approach is applicable to all stages of curing, from pure liquid, to gelation, to fully cured glass. We utilize this approach to study DGEBA/DDS epoxy systems, incorporating dynamic topology changes into atomistic Molecular Dynamics simulations of the curing reaction with 127,000 atoms. We study size effects in our simulations and predict the gel point utilizing rigorous percolation theory to accurately recover the experimental data. Furthermore, we observe excellent agreement between the estimated and the experimentally determined glass transition temperatures as a function of curing rate. Finally, we demonstrate the quality of our model by the prediction of the elastic modulus, based on uniaxial tensile tests. The presented scheme paves the way for a broadly consistent approach for modelling and characterizing all amine-based epoxy resins. ## Contact Ana-Sunc ana Smith and Christian R. Wick Friedrich-Alexander-University Erlangen-Nu rnberg (FAU), Faculty of Science, Department of Physics, PULS Group, Interdisciplinary Center for Nanostructured Films (IZNF), Cauerstrasse 3, 91058, Germany ## License Creative Commons Attribution 4.0 International ## Context Dataset to paper: [1] All input scripts and force field files presented in [1] ## Folder structure amberLib (cf. page 9-12 of [1]) Amber lib and frcmod files for all the Amber residues necessary to reconstruct the charge fragmentation scheme for the DGEBA \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    epoxy resin\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and the amine hardeners DDS, DICY and IPTA. Examples containing leap input scripts to construct DGEBA molecules of length n. lammps data (cf. page 12 of [1]) The Lammps data files for the five DGEBA/DDS MD systems, straight out of assembling by \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Packmol\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " (no minimization performed yet). fixBondReact (cf. page 18 of [1]) Pre- and post-reaction topology and map files to run the curing simulations. Compatible with \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Lammps Stable Dec. 2018\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       ". inputScripts minimizationEquilibration (cf. page 13 of [1]) The Lammps scripts for initial minimization and equilibration. crossLinking (cf. page 18 of [1]) Input scripts to run curing simulations and their corresponding annealing cycles. tg (cf. page 23 of [1]) Input scripts to run the annealing procedure required to collect density data for the measurement of glass transition temperature. The procedure is split into three parts due to cluster walltime. tensileTests (cf. page 28 of [1]) Input script to run the uniaxial deformation with isotropic barostat in the lateral directions. postCuringAnnealing (cf. page 28 of [1]) Input scripts to run the post-curing annealing protocol. lammps noCharge (cf. page 18 of [1]) Example Lammps input/data required to simulate cross-linking in the absence of Coulomb interactions. curingReaction (cf. page 13 of [1]) optimized structures for reactants, transition structures and products for uncatalysed concerted and step wise reactions and for the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    alcohol\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " catalysed step wise reactions at wB97XD/def2-TZVP level of theory ## Software DFT Calcualtions: \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Gaussian 16 Rev B01\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " [2] Simulations: \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    LAMMPS (Dec 2018, stable)\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " ## Funding This research was funded by the Deutsche Forschungsgemeinschaft (DFG, German Research Foundation) - 377472739/GRK 2423/1-2019 FRASCAL ## References [1] M. Livraghi, S. Pahi, Piotr Nowakowski, D. M. Smith, C. R. Wick, A.-S. Smith, &quot;Block chemistry for accurate modeling of epoxy resins&quot;. [2] Frisch, M. J.; Trucks, G. W.; Schlegel, H. B.; Scuseria, G. E.; Robb, M. A.; Cheeseman, J. R.; Scalmani, G.; Barone, V.; Petersson, G. A.; Nakatsuji, H.; et al. Gaussian 16 Rev. B.01, 2016. [3] Plimpton, S. Fast Parallel Algorithms for Short-Range Molecular Dynamics. Journal of Computational Physics 1995, 117 (1), 1 19.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_6874760.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Machine Learning Adaptive Basis Sets for Efficient Large Scale Density Functional Theory Simulation<br>It is chemically intuitive that an optimal atom centered basis set must adapt to its atomic environment, for example by polarizing toward nearby atoms. Adaptive basis sets of small size can be significantly more accurate than traditional atom centered basis sets of the same size. The small size and well conditioned nature of these basis sets leads to large saving in computational cost, in particular in a linear scaling framework. Here, it is shown that machine learning can be used to predict such adaptive basis sets using local geometrical information only. As a result, various properties of standard DFT calculations can be easily obtained at much lower costs, including nuclear gradients. In our approach, a rotationally invariant parametrization of the basis is obtained by employing a potential anchored on neighboring atoms to ultimately construct a rotation matrix that turns a traditional atom centered basis set into a suitable adaptive basis set. The method is demonstrated using MD simulations of liquid \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", where it is shown that minimal basis sets yield structural properties in fair agreement with basis set converged results, while reducing the computational cost in the best case by a factor of 200 and the required flops by 4 orders of magnitude. Already a very small training set yields satisfactory results as the variational nature of the method provides robustness.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_6272471.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Force Field for \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " over Pt (111): Development, Assessment, and Comparison<br>\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Metal\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "/\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " interfaces are key in many natural and industrial processes, such as corrosion, atmospheric, or environmental chemistry. Even today, the only practical approach to simulate large interfaces between a \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    metal\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " is to perform force-field simulations. In this work, we propose a novel force field, \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GAL17\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       ", to describe the interaction of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and a Pt (111) surface. \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GAL17\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " builds on three terms: (i) a standard Lennard-Jones potential for the bonding interaction between the surface and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", (ii) a Gaussian term to improve the surface corrugation, and (iii) two terms describing the angular dependence of the interaction energy. The 12 parameters of this force field are fitted against a set of 210 adsorption geometries of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " on Pt (111). The performance of \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GAL17\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " is compared to several other approaches that have not been validated against extensive first-principles computations yet. Their respective accuracy is evaluated on an extended set of 802 adsorption geometries of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    H2O\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " on Pt (111), 52 geometries derived from icelike layers, and an MD simulation of an interface between a c (4 6) Pt (111) surface and a \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " layer of 14 A thickness. The newly developed \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GAL17\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " force field provides a significant improvement over previously existing force fields for Pt (111)/ \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    H2O\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " interactions. Its well-balanced performance suggests that it is an ideal candidate to generate relevant geometries for the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    metal\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "/\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " interface, paving the way to a representative sampling of the equilibrium distribution at the interface and to predict solvation free energies at the solid/liquid interface.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_11688372.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">\n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    AMBER\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " Force Field Parameters for \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Cobalt\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-Containing Biological Systems: A Systematic Derivation Study<br>In the present work, the parameterization of a set of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    cobalt\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-containing systems has been performed to create a comprehensive library for bonding parameters of biological Co-containing systems. A standard process for the extraction and validation of parameters was employed, which could be used to create force field parameters for the other \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    metal\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-containing systems. All protein data banks were searched to extract common chemical groups in bonding with \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    cobalt\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", and finally, 16 structures were designed to represent the binding model of the chemical moieties with \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    cobalt\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". The Hessian matrix of each structure was computed at the B3LYP/6-311++G (2d,2p) level of theory and the Seminario method was employed to compute \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    cobalt\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " bond stretching and angle bending parameters. Validation of the derived parameters was performed using structural minimization and molecular dynamics (MD) simulations of four models. Further validation was performed using an extensive MD simulation on \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    carbonic anhydrase II\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " as a common \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    cobalt\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-containing metalloprotein. The results demonstrated that among models, the bonded model in combination with the RESP charges can produce the most reliable and accurate structural conformations for the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    metal\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " site of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    cobalt\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-containing systems.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_6592231.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Modulation of Multidrug Resistance Protein 1-mediated transport processes by the antiretroviral drug \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ritonavir\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "<br>We here provide the structure files, trajectories and topology files for analyzing and reproducing simulations performed in the publication: &quot;Modulation of Multidrug Resistance Protein 1-mediated transport processes by the antiretroviral drug \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ritonavir\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "&quot;. Classical molecular dynamics simulation of rat Mrp1 have been performed in complex with native substrates like \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GSH\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GSSG\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " or drug molecules like \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ritonavir\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", to study binding modes of different combinations. Structure and trajectory files (all \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    1 µs\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       " long) as well as recorded movies of the time progression are provided for the following simulation systems: 1. \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GSH\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " bound to rat Mrp1 2. \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GSSG\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " bound to rat Mrp1 3. \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Ritonavir\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " bound to rat Mrp1 4. \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GSH\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " + \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Ritonavir\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " bound to rat Mrp1 5. \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GSSG\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " + \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Ritonavir\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " bound to rat Mrp1 Additionally itp files (\n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GROMACS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " format) containing parameter information about the ligands \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GSH\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GSSG\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Ritonavir\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " are provided for the force field \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CHARMM36m.\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " Raw data of the homology modeling of rat Mrp1 by I-Tasser as well as Docking results produced via \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    AutoDock4\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " are uploaded as zip directories. </div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_1198171.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Simulation data for \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CHARMM36\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " bilayer, 100 lipids/leaflet, 450 mM \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CaCl2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (NB-Fix used), \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    310K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GROMACS 5.1.4\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       "<br>Simulations of a \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " bilayer with 450 mM of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CaCl 2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". The second from the set of 6 simulations. The goal was to study the effect of scaling the \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CHARMM\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " FF on the ion binding. Done for the NMRlipids project, see https://removed for more information. A \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " bilayer consisting of 200 lipids (100 per leaflet) is simulated in the presence of 450 mM \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CaCl 2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". The \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Charmm36\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " model is employed for lipids, the \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Charmm\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " compatible variant of the \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    tip3p\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " model for \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", and the default \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Charmm\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ion\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " parameters (type CAL) for \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CaCl 2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". The new extra nonbonded parameters were used for trating the calcium bonding (NB-Fix) The \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Charmm36\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " force field parameters were obtained from https://removed The files are in \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GROMACS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " format. Trajectory (.xtc) is \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    360 ns\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       " long with data saved every 100 ps. the initial structure (.gro), topology (.top), index file (.ndx), simulation paremeter file (.mdp), binary run input file for \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GROMACS v. 5.1 &gt;\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " (.tpr) and the energy output file (.edr) are provided.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_6973476.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Outputs of molecular dynamics simulations of two \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    NS1 ZIKV\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " variants in the membrane presence<br>Files corresponding to outputs obtained through Molecular Dynamics (MD) simulations of two \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Non-structural (NS) proteins 1\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " of the Zika virus from Uganda (ZIKV-UG) and Brazil (ZIKV-BR). Simulations were performed using \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GROMACS 5.1.5\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " or later versions. Systems were built based on atomistic models (https://zenodo.org/record/5608521#.YvDNZTlBzJw) and converted to a coarse-grained representation employing \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    MARTINI 2.2p\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " ElNeDyn. It was assumed to be \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    NS1\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " systems in apo and holo forms (i.e., in the absence and presence of a lipid bilayer, respectively). The membrane model tries to reproduce a lipid concentration of an endoplasmic reticulum lipid bilayer. Holo and apo systems were simulated until reached \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    20\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    10 µs\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       ", respectively. Trajectories do not include \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " molecules. For the specific case of holo systems, frames were skipped every 5 frames, which means that processed trajectories are equivalent to simulations when it is recorded every 1000 ps. More details can be found at https://www.biorxiv.org/content/10.1101/2022.04.08.487413v1.full. Note: Some topology and index files important for MD analysis are also present.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_1198158.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Simulation data for \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CHARMM36\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " bilayer, 100 lipids/leaflet, \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    310K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GROMACS 5.1.4\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " <br>Simulations of a \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " bilayer without ions with \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CHARMM36\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " FF from the \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CHARMM-Gui\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       ". The first from the set of 6 simulations. The goal was to study the effect of scaling the \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CHARMM\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " FF on the ion binding. Done for the NMRlipids project, see https://removed for more information. A \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " bilayer consisting of 200 lipids (100 per leaflet) is simulated. The \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Charmm36\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " model is employed for lipids, the \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Charmm\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " compatible variant of the \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    tip3p\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " model for \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". The \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Charmm36\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " force field parameters were obtained from https://removed The files are in \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GROMACS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " format. Trajectory (.xtc) is \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    380 ns\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       " long with data saved every 100 ps. the initial structure (.gro), topology (.top), index file (.ndx), simulation paremeter file (.mdp), binary run input file for \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GROMACS v. 5.1 &gt;\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " (.tpr) and the energy output file (.edr) are provided. </div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_7069778.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Anomalous Effects of Velocity Rescaling Algorithms: The Flying Ice Cube Effect Revisited<br>The flying ice cube effect is a molecular dynamics simulation artifact in which the use of velocity rescaling thermostats sometimes causes violation of the equipartition theorem, affecting both structural and dynamic properties. The reason for this artifact and the conditions under which it occurs have not been fully understood. Since the flying ice cube effect was first demonstrated, a new velocity rescaling algorithm (the CSVR thermostat) has been developed and become popular without its effects on the equipartition theorem being truly known. Meanwhile, the use of simple velocity rescaling and Berendsen (weak coupling) thermostat algorithms has not abated but has actually continued to grow. Here, we have calculated the partitioning of the kinetic energy between translational, rotational, and vibrational modes in simulations of diatomic molecules to explicitly determine whether the equipartition theorem is violated under different thermostats and while rescaling velocities to different kinetic energy distributions. We have found that the underlying cause of the flying ice cube effect is a violation of balance leading to systematic redistributions of kinetic energy under simple velocity rescaling and the Berendsen thermostat. When velocities are instead rescaled to the canonical ensemble s kinetic energy distribution, as is done with the CSVR thermostat, the equipartition theorem is not violated, and we show that the CSVR thermostat satisfies detailed balance. The critical necessity for molecular dynamics practitioners to abandon the use of popular yet incorrect velocity rescaling algorithms is underscored with an example demonstrating that the main result of a highly cited study is entirely due to artifacts resulting from the study s use of the Berendsen thermostat.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_6270413.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Hydration-Dependent Dynamical Modes in \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Xyloglucan\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " from Molecular Dynamics Simulation of 13C NMR Relaxation Times and Their Distributions<br>Macromolecular dynamics in biological systems, which play a crucial role for biomolecular function and activity at ambient temperature, depend strongly on moisture content. Yet, a generally accepted quantitative model of hydration-dependent phenomena based on local relaxation and diffusive dynamics of both polymer and its adsorbed \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " is still missing. In this work, atomistic-scale spatial distributions of motional modes are calculated using molecular dynamics simulations of hydrated \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    xyloglucan\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (XG). These are shown to reproduce experimental hydration-dependent 13C NMR longitudinal relaxation times (T1) at room temperature, and relevant features of their broad distributions, which are indicative of locally heterogeneous polymer reorientational dynamics. At low hydration, the self-diffusion behavior of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " shows that \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " molecules are confined to particular locations in the randomly aggregated XG network while the average polymer segmental mobility remains low. Upon increasing \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " content, the hydration network becomes mobile and fully accessible for individual \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " molecules, and the motion of hydrated XG segments becomes faster. Yet, the polymer network retains a heterogeneous gel-like structure even at the highest level of hydration. We show that the observed distribution of relaxations times arises from the spatial heterogeneity of chain mobility that in turn is a result of heterogeneous distribution of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " chain and chain chain interactions. Our findings contribute to the picture of hydration-dependent dynamics in other macromolecules such as proteins, DNA, and synthetic polymers, and hold important implications for the mechanical properties of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    polysaccharide\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " matrixes in plants and plant-based materials.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_21158960.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">How the Layer Alignment in Two-Dimensional Nanoporous Covalent Organic Frameworks Impacts Its Electronic Properties<br>Two-dimensional nanoporous covalent organic frameworks (2D COFs) have gathered significant interest due to their wide range of applications. Due to the lack of strong covalent interlayer interactions, their layers can be stacked in countless ways, each resulting in unique nanoscale characteristics impacting the structural, chemical, and electronic properties. To characterize and understand the layer stacking in 2D COFs and its effect on the structural and electronic properties, we carried out a detailed density functional theory investigation on four materials, CTF-1, COF-1, COF-5, and Pc-PBBA. This entailed an in-depth evaluation of the potential energy as a function of the interlayer distance and offset, the powder X-ray diffraction (PXRD) pattern, and the electronic properties. From the potential energy surfaces, the typical slipped AA-stacking configuration was confirmed as optimal for each of the 2D COFs, with a slight offset from a perfect alignment of the layers. The statically calculated PXRD patterns based on these optimized stacking configurations showed discrepancies when compared to experimental data. Instead, when properly accounting for dynamic fluctuations by calculating the average diffraction pattern over the course of a molecular dynamics simulation, a better agreement with the experiment is obtained. Different stacking configurations also profoundly affected the electronic band structure of COFs as the interlayer interactions are significantly impacted by the layer offset. Evidently, with decreasing layer offsets, the interactions increase due to the layer alignment, leading to a decrease in the band gap and an increase in interlayer charge mobility. Our study highlights the need for accurate modeling of the stacking configuration in 2D COFs as a small-scale deviation in the adjacent layer position can significantly affect the structural and electronic properties.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_3893244.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Q Band Electron-Nuclear Double Resonance Reveals Out-of-Plane Hydrogen Bonds Stabilize an \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Anionic Ubisemiquinone\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " in \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Cytochrome bo3\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " from Escherichia coli<br>The respiratory \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    cytochrome bo3 ubiquinol oxidase\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " from Escherichia coli has a high-affinity \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ubiquinone\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " binding site that stabilizes the one-electron reduced \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ubisemiquinone\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (SQH), which is a transient intermediate during the electron-mediated reduction of O2 to \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". It is known that SQH is stabilized by two strong hydrogen bonds from R71 and D75 to \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ubiquinone\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    carbonyl oxygen\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " O1 and weak hydrogen bonds from H98 and Q101 to O4. In this work, SQH was investigated with orientation-selective Q-band ( 34 GHz) pulsed 1H electron nuclear double resonance (ENDOR) spectroscopy on fully deuterated \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    cytochrome\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    cyt\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") bo3 in a \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    H2O\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " solvent so that only exchangeable protons contribute to the observed ENDOR spectra. Simulations of the experimental ENDOR spectra provided the principal values and directions of the hyperfine (hfi) tensors for the two strongly coupled H-bond protons (H1 and H2). For H1, the largest principal component of the proton anisotropic hfi tensor Tz   11.8 MHz, whereas for H2, Tz   8.6 MHz. Remarkably, the data show that the direction of the H1 H-bond is nearly perpendicular to the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    quinone\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " plane ( 70 out of plane). The orientation of the second strong hydrogen bond, H2, is out of plane by 25 . Equilibrium molecular dynamics simulations on a membrane-embedded model of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    cyt bo3\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " QH site show that these H-bond orientations are plausible but do not distinguish which H-bond, from R71 or D75, is nearly perpendicular to the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    quinone\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " ring. Density functional theory calculations support the idea that the distances and geometries of the H-bonds to the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ubiquinone\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    carbonyl oxygens\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", along with the measured proton anisotropic hfi couplings, are most compatible with an anionic (deprotonated) \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ubisemiquinone\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ".</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_3951861.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Connexin-46/50\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " in a dynamic lipid environment resolved by CryoEM at 1.9 A <br>These are the molecular dynamics (MD) data that are analyzed, in Flores et al. 2020. Each trajectory file (.dcd) has an associated structure file (.psf), which can be analyzed in \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    VMD\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       ". Each gap junction system, \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Cx46\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " &amp; \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Cx50\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", were simulated with either \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    KCl\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " or \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    NaCl\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " in the intracellular space: \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Cx46\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    KCl\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "/\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    NaCl\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Cx50\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    KCl\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "/\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    NaCl\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". All four systems were equilibrated for 30ns and then followed up with 2 separate \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    100ns\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       " production runs. For MD lipid-densities the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Cx50\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    KCl\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " system was used as the representative dataset. Lipid densities from all other systems can be calculated with the provided TCL script calc-density.tcl. In \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    VMD\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " TK-Console: &gt; mol new &lt;system&gt;.psf &gt; mol addfile &lt;system&gt;.dcd waitfor all &gt; source /path/to/scripts/LipNetwork.tcl &gt; align &gt; source /path/to/scripts/calc-density.tcl &gt; dmpcdensity &lt;outname&gt; [output] outname ltailden.dx Convert from .dx to .mrc using UCSF-Chimera Volume Viewer plugin. [output] outname ltailden.mrc Using Relion: $ relion image handler --i outname ltailden.mrc --o outname ltailden D6-Sym.mrc --sym D6 [output] outname ltailden D6-Sym.mrc All trajectory files (.dcd) are \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    100ns\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       " longs (1,000 frames x 100ps/frame). For questions regarding MD-data analysis and full trajectory files (2ps/frame), please contact Dr. Steve Reichow (reichow@pdx.edu).</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_15059757.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Data for &quot;Emergence of Coupled Rotor Dynamics in Metal Organic Frameworks via Tuned Steric Interactions&quot;<br>Raw data for &quot;Emergence of Coupled Rotor Dynamics in Metal Organic Frameworks via Tuned Steric Interactions&quot; by Adrian Gonzalez-Nelson, Srinidhi Mula, Mantas S ime nas, Sergejus Balc iu nas, Adam R. Altenhof, Cameron S. Vojvodin, Stefano Canossa, Ju ras Banys, Robert W. Schurko, Franc ois-Xavier Coudert, and Monique A. van der VeenDataset includes NMR, BDS, DFT and AIMD data used for analysis and plotting. Representative input files for DFT and Molecular dynamics (AIMD) calculations are also included. Python code for calculating the rotational angle between the benzene ring and reference plane from the CP2K MD trajectories ( *.xyz) is deposited on Github at https://github.com/srinidhimula/supplementary-data </div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_6988344.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Etidocaine\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " binding to \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " membrane, \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CHARMM36\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       "-\n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ParamChem\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       ", 34 mM, \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    298 K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       "<br>System info in README.yaml, including basic analysis - headgroup OP, averaged APL and repead distance, analysis performed only from eqilibrated part of the simulation, marked as BINDINGEQ, here: FILES: xtc: NAME: [name] SIZE: [MB] MODIFIED: [yyyy-mm-dd time] LENGTH: [ps] SAVING FREQUENCY: [ps] BEGIN: [ps] - how much of the simulation was simulated before and is excluded FILES: tpr: GMX VERSION: \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    gromacs\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " version used for gmx grompp creation of *.tpr file at local computer log: GMX VERSION: \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    gromacs\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " version used for gmx mdrun at cluster and read out from *.log file</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_18095690.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">MD simulation trajectories used in the paper &quot;ABCA1 is an extracellular \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    phospholipid translocase\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "&quot;<br>MD simulation trajectories used in the paper &quot;ABCA1 is an extracellular \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    phospholipid translocase\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "&quot; 1. 10 us coarse-grained MD simulation files of ABCA1 (WT, K568A, 11CmutA), (abca1 cgmd.zip) 1-1. wt.gro 1-2. wt-set1-10us.trr 1-3. wt-set2-10us.trr 1-4. wt-set3-10us.trr 1-5. k568A.gro 1-6. k568A-set1-10us.trr 1-7. k568A-set2-10us.trr 1-8. k568A-set3-10us.trr 1-9. 11CmutA.gro 1-10. 11CmutA set1 10us.trr 1-11. 11CmutA set2 10us.trr 1-12. 11CmutA set3 10us.trr 1-13. readme.txt 2. \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    30 ns\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       " steered molecular dynamic simulation file (slow pulling) (abca1 smd.zip) 2-1. smd.psf 2-2. 30ns.dcd 2-3. readme.txt 3. \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    0.79 ns\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       " steered molecular dynamic simulation file (fast pulling) (abca1 smd2.zip) 3-1. smd2.psf 3-2. 0.79ns.dcd 3-3. readme.txt 4. custom tcl scripts to analyze the salt bridge formation and to analyze the extraction of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " in the Gateway (tcl scripts.zip) 4-1. count popc.vmd 4-2. saltbr-cg.vmd 4-3. readme.txt </div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_1488094.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Molecular dynamics simulations of lipid bilayers containing \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (various mixtures) with \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ECC-lipids\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " force field, and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Na+\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    K+\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") counterions<br>Classical molecular dynamics simulations of various mixtures of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ":\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " lipid bilayers in \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " solution with only \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Na+\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " counterions (or with \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    K+\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " counterions when noted with &quot; KCl&quot; suffix). \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ECC-lipids\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " force field parameters used for lipids, \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    SPC/E\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " model and \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ECC-ions\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       ", all parameters available at https://removed lipids simulations performed with \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Gromacs 2018.0\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " (*.xtc files) simulation length \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    1000 ns\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    1 microsecond\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       " temperature \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    298 K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Gromacs\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " simulation setting is in the file npt lipid bilayer.mdp</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_2267653.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Structural Correspondence of Solution, Liquid Crystal, and Crystalline Phases of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Chromonic Mesogen\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Sunset Yellow<br>The azo dye, sunset yellow, is a prototypical, chromonic liquid crystal in which assembly in aqueous solution at high volume fraction leads to lyotropic mesophases with a package of properties distinct in almost every aspect (Lydon, J. Curr. Opin. Colloid Interface Sci. 2004, 8, 480). In particular, the isotropic to nematic transition in such phases, the consequence of stacking of dye molecules in chains, is difficult to bring into correspondence with athermal theories for rigid rods as well as modifications that consider chain interactions with one another. Chromonic mesogens, small molecules that stack to form lyotropic liquid crystals, prompt structural questions that have yet to be answered; a full understanding of structure should inform colligative properties. Herein, the single crystal structure of a guanidinium salt of the sunset yellow dianion, a known \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    chromonic mesogen\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", is reported. The compound crystallizes as a \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    dihydrate\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    tetrahydrofuran\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " solvate in the orthorhombic space group Pnna, with a   6.8426 (5) A , b   20.048 (1) A , c   21.466 (2) A . The sunset yellow molecules, point group approximately Cs, are disordered about a crystallographic diad axis. The structure is informative because pairwise interactions in the disordered crystal structure show a remarkable correspondence with the stereochemistry of sunset yellow molecules in solution and in the liquid crystal phase. The solution structure is here simulated by the combination of molecular dynamics, metadynamics, and quantum chemical computations. The comparable disorder in the fluid and solid states suggests the possibility that stacked aggregates adhere to growing crystals intact. Computations were used to evaluate proposals that stacking faults and branching points lower the X-ray correlation lengths while preserving extended structures. Evidence is found for stacking faults but not branches. The solution stereochemistry and stereodynamics has implications for the geometry of long rods, for which understanding is a prerequisite for reckoning properties of vexing chromonic mesophases.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_7440399.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">An in silico approach to determine inter-subunit affinities in human \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    septin\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " complexes<br>The \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    septins\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " are a conserved family of filament-forming guanine nucleotide binding proteins, often named the fourth component of the cytoskeleton. Correctly assembled \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    septin\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " structures are required for essential intracellular processes such as cytokinesis, vesicular transport, polarity establishment, and cellular adhesion. \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Septins\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " structurally belong to the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    P-Loop NTPases\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " but they do not mediate signals to effectors through \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GTP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " binding and hydrolysis. \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GTP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " binding and hydrolysis are believed to contribute to \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    septin\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " complex integrity, but biochemical approaches addressing this topic are hampered by the stability of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    septin\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " complexes after recombinant expression and the lack of nucleotide-depleted complexes. To complement this limitation, we used a molecular dynamics based approach to determine inter-subunit free binding energies in available human \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    septin\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " dimer structures and in their apo forms, which we generated in silico. The \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    nucleotide\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " in the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GTPase\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " active subunits \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    SEPT2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    SEPT7\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", but not in \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    SEPT6\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", was identified as a stabilizing element in the G interface as it is coordinated at its ribose ring to conserved amino acids. Removal of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GDP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " from \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    SEPT2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    SEPT7\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " results in flipping of a conserved Arg residue and disruption of an extensive hydrogen bond network in the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    septin\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " unique element, concomitant with a decreased inter-subunit affinity.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_5546158.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Protein dynamics and structural \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    waters\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " in bromodomains<br>Bromodomains are epigenetic readers of acetylated lysines that are integral parts of histone tails. The 61 bromodomains in humans are structurally highly conserved but specifically bind to widely varying recognition motifs, suggesting that dynamic rather than static factors are responsible for recognition selectivity. To test this hypothesis, the dynamics of the binding sites and structural \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " molecules of four bromodomains (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ATAD2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    BAZ2B\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    BRD2 (1)\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CREBBP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") representing four different subtypes is studied with \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    1 μs\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       " MD simulations using the \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    RSFF2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " force field. The different dynamics of the ZA-loops and BC-loops between the four bromodomains leads to distinct patterns for the opening and closing of the binding pocket. This in turn determines the structural and energetic properties of the structural \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    waters\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " in the binding pocket, suggesting that these \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    waters\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " are not only important for the recognition itself, as has been proposed previously, but also contribute to the selectivity of different bromodomains.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_21121417.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Best Practices in Constant pH MD Simulations: Accuracy and Sampling<br>Various approaches have been proposed to include the effect of pH in molecular dynamics (MD) simulations. Among these, the -dynamics approach proposed by Brooks and co-workers [Kong, X.; Brooks III, C. L. J. Chem. Phys. 1996, 105, 2414 2423] can be performed with little computational overhead and hfor each typeence be used to routinely perform MD simulations at \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    microsecond\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       " time scales, as shown in the accompanying paper [Aho, N. et al. J. Chem. Theory Comput. 2022, DOI: 10.1021/acs.jctc.2c00516]. At such time scales, however, the accuracy of the molecular mechanics force field and the parametrization becomes critical. Here, we address these issues and provide the community with guidelines on how to set up and perform long time scale constant pH MD simulations. We found that barriers associated with the torsions of side chains in the \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CHARMM36m\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " force field are too high for reaching convergence in constant pH MD simulations on \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    microsecond\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       " time scales. To avoid the high computational cost of extending the sampling, we propose small modifications to the force field to selectively reduce the torsional barriers. We demonstrate that with such modifications we obtain converged distributions of both protonation and torsional degrees of freedom and hence consistent pKa estimates, while the sampling of the overall configurational space accessible to proteins is unaffected as compared to normal MD simulations. We also show that the results of constant pH MD depend on the accuracy of the correction potentials. While these potentials are typically obtained by fitting a low-order polynomial to calculated free energy profiles, we find that higher order fits are essential to provide accurate and consistent results. By resolving problems in accuracy and sampling, the work described in this and the accompanying paper paves the way to the widespread application of constant pH MD beyond pKa prediction.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_3470993.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Structural Characterization of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Alkylsilane\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Fluoroalkylsilane\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Self-Assembled Monolayers on \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    SiO2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " by Molecular Dynamics Simulations<br>We present molecular dynamics simulations of self-assembled monolayers (SAMs) chemisorbed on an atomically flat amorphous \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    silicon dioxide substrate\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". We model two prototypical SAM-forming \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    alkylsilanes\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    octadecyl trichlorosilane\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    OTS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    1H,1H,2H,2H-perfluoro decyl trichlorosilane\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    FDTS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "), that find widespread use in organic electronic applications. Crucially, our model does not rely on an explicit bonding between the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    alkylsilane\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and the substrate, thus allowing for the spontaneous organization of molecules into regular structures, which we studied as a function of coverage. By comparing the calculated tilt angle, film thickness, and lattice parameters with experiments, we conclude that the simulated morphologies are quantitatively consistent with the experimental evidence, demonstrating the accuracy of the simulation methodology. We take advantage of the atomistic resolution of the calculations for carrying out a detailed one-to-one comparison between the structure and the electronic properties of the two SAMs. In particular, we find that \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    OTS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " molecules show a coverage-dependent tilt, while \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    FDTS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " molecules are always vertically oriented, regardless of the coverage. More importantly for organic electronic applications, we observe that \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    OTS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " SAMs do not alter the electrostatic potential of silica, while \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    FDTS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " SAMs induce a negative voltage shift which increases with coverage and saturates at about 2 V.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_2018634.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Derivation and Systematic Validation of a Refined All-Atom Force Field for \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Phosphatidylcholine\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Lipids<br>An all-atomistic force field (FF) has been developed for fully saturated phospholipids. The parametrization has been largely based on high-level ab initio calculations in order to keep the empirical input to a minimum. Parameters for the lipid chains have been developed based on knowledge about bulk \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    alkane\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " liquids, for which thermodynamic and dynamic data are excellently reproduced. The FFs ability to simulate lipid bilayers in the liquid crystalline phase in a tensionless ensemble was tested in simulations of three lipids: \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    1,2-diauroyl-sn-glycero-3-phospocholine\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DLPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "), \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    1,2-dimyristoyl-sn-glycero-3-phosphocholine\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DMPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "), and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    1,2-dipalmitoyl-sn-glycero-3-phospcholine\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DPPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "). Computed areas and volumes per lipid, and three different kinds of bilayer thicknesses, have been investigated. Most importantly NMR order parameters and scattering form factors agree in an excellent manner with experimental data under a range of temperatures. Further, the compatibility with the \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    AMBER\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " FF for biomolecules as well as the ability to simulate bilayers in gel phase was demonstrated. Overall, the FF presented here provides the important balance between the hydrophilic and hydrophobic forces present in lipid bilayers and therefore can be used for more complicated studies of realistic biological membranes with protein insertions.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_13392.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " @ \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    310K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       ", Model by Ulmschneider and Ulmschneider<br>Files required for a simulation of a \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " bilayer and the resulting trajectories. The \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " model is that by Ulmschneider and Ulmschneider [1]. The structure and topology files are downloaded from \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Lipidbook\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " [2]. The files (xtc and mdp) with ' dc' suffix are simulated with dispersion correction applied to both energy and pressure whereas the files without this suffix do not use this correction. This data is used in the project Matching lipid force fields with NMR data, see: https://removed [1] J.P. Ulmschneider &amp; M.B. Ulmschneider, United Atom Lipid Parameters for Combination with the Optimized Potentials for Liquid Simulations All-Atom Force Field, JCTC 2009, 5 (7), 1803 1813 [2] https://removed</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_11569377.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Promising Route for the Development of a Computational Framework for Self-Assembly and Phase Behavior Prediction of Ionic \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Surfactants\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Using \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    MARTINI\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       "<br>\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Surfactants\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " are amphiphilic molecules with multiple uses and industrial applications as detergents, wetting agents, emulsifiers, and so forth. They can be divided into three main categories: nonionic, ionic, and zwitterionic. The development of a universal computational framework able to predict key properties such as their critical \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    micelle\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " concentration (cmc) and the size of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    micelles\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " they form and to ultimately extract phase diagrams for their aqueous solutions, possibly in the presence of salts and oils, using their chemical constitution as input, would provide valuable information for the design and the production of these materials. In this work, we focus on ionic \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    surfactants\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and investigate a possible route toward the development of such a framework based on coarse-grained simulations using the \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    MARTINI\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " forcefield in two versions: its implicit solvent version, called Dry \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    MARTINI\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       ", and its explicit solvent version, called Wet \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    MARTINI\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       ". The \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    surfactants\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " considered in our efforts are the anionic \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    sodium dodecyl sulfate\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    SDS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") and the three cationic \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    cetyl\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    dodecyl\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    octyl trimethyl ammonium bromide\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CTAB\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DTAB\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    OTAB\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", respectively). First, we choose their mapping onto coarse-grained \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    MARTINI\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " beads. Next, we estimate their cmc s, their peak aggregation numbers, Nagg, and in the case of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    SDS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", its small angle neutron scattering pattern at low concentrations, applying the Dry \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    MARTINI\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " forcefield. With a single modification to the Lennard-Jones energy parameter between hydrophobic beads and invoking Ewald summation with a physically meaningful dielectric constant for electrostatic interactions, our estimates are in very good agreement with experimental results. Furthermore, we predict the phase behavior of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    SDS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "/\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CTAB\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "/\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " binary solutions using Wet \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    MARTINI\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " and find semiquantitative agreement with experimental phase diagrams. We conclude that the \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    MARTINI\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " forcefield, with careful treatment of electrostatic interactions and appropriate modification of parameters for some key functional groups, can be a powerful ally in the quest for a universal computational framework for the design of new \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    surfactants\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " with improved properties.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_8291933.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Effect of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Triclosan\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Chloroxylenol\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " on Bacterial Membranes<br>\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Triclosan\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    chloroxylenol\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " are broad-spectrum \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    biocides\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " used extensively in healthcare and consumer products. They have been suggested to perturb the structure of bacterial membranes, but studies so far have not considered that most bacterial membranes contain large amounts of branched-chain lipids. Here, molecular dynamics simulation is used to examine the effect of the two \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    biocides\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " on membranes consisting of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    lipids\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " with methyl-branched chains, cyclopropanated chains, and nonbranched chains. It is shown that \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    triclosan\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    chloroxylenol\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " induced a phase transition in membranes from a liquid-crystalline to a liquid-ordered phase irrespective of the presence and nature of branching groups. At high concentration, \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    chloroxylenol\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " promoted chain interdigitation. Our results suggest that \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    triclosan\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    chloroxylenol\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " decrease the degree of fluidity of membranes and that this effect is more pronounced in bacterial membranes. As a result, their biocidal activity could be associated with a change in the function of membrane proteins.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_16574000.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">All-atom molecular dynamics simulation of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    peptides\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "<br>The initial structure was built using the LEaP module of \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    AmberTools21\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       ", and we will run a total of 4000 minimization steps before starting to run molecular dynamics; 2000 is the steepest decline, followed by the conjugate gradient of 2000. That should be enough to clean up our structures. Then heat and equilibrate in three 250 ps stages: from heat 100 K to production temperature \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    300K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       ", heavy atom position limiting, force constant decreasing from 10.0 to 1.0, then decreasing to 0.1 kcal/mol/A 2. The step size was set to 0.002 and the unconstrained production simulation was performed in an implicit generalized Bonn solvent (igb 8) in the canonical ensemble (NVT). The temperature was controlled by a Langevin thermostat (ntt 3) and molecular dynamics was performed for \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    200ns\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       " at a collision frequency of   0.01 ps-1 (For implicit solvent, setting collision frequency to 0.01ps 1 can lead to significant, up to 100-fold in some cases, speedup of conformational sampling). Finally, the cpptraj module of \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    AmberTools21\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " was used for combined clustering, RMSD, and other molecular dynamics trajectory analysis.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_19780550.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">A Bottom-Up Coarse-Grained Model for \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Nucleosome\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Nucleosome\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Interactions with Explicit Ions<br>The \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    nucleosome core particle\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    NCP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") is a large complex of 145 147 base pairs of DNA and eight \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    histone\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " proteins and is the basic building block of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    chromatin\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " that forms the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    chromosomes\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". Here, we develop a coarse-grained (CG) model of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    NCP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " derived through a systematic bottom-up approach based on underlying all-atom MD simulations to compute the necessary CG interactions. The model produces excellent agreement with known structural features of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    NCP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and gives a realistic description of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    nucleosome\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    nucleosome\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " attraction in the presence of multivalent \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    cations\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Mg (H2O)62+\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " or \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Co (NH3)63+\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") for systems comprising 20 \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    NCPs\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". The results of the simulations reveal structural details of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    NCP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    NCP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " interactions unavailable from experimental approaches, and this model opens the prospect for the rigorous modeling of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    chromatin\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " fibers.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_6575438.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Models and trajectories of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GLUTAMINE SYNTHETASE\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " with \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    palmitoyl-coenzyme A\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "<br>These are the models and molecular dynamics trajectories obtained for the paper Role of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    glutamine synthetase\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " in angiogenesis beyond \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    glutamine\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " synthesis [G. Eelen et al. Nature 2018 vol. 561 (7721) pp. 63-69].The PDB files (human.pdb and bacterial.pdb) contain representative structures extracted from the molecular dynamics simulations of the S. Typhimurium (bacterial.pdb) and Human \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Glutamine Synthetase\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (human.pdb). For the latter, models of the two most relevant binding modes of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Palmitoyl-CoA\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " are included in the file.The trajectories included are the FINAL \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    450 ns\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       " of 2 fully atomistic molecular dynamics simulations of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Glutamine Synthetase\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") performed with \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GROMACS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " and the \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Amber ff14SB\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " force field. The Human GS.x is the trajectory of human \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (reference PDB 2OJW) with \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Palmitoyl-CoA.\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " In this trajecory the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    palmitate\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " tail points upwards (which was the most interesting orientation found). &quot;Bacterial \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "&quot; is the bacterial \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " trajectory (reference starting PDB 1FPY) with \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Palmitoyl-CoA\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " - \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    palmitate\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " tail points downwards. The trajectories are in \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GROMACS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " trr format and have been saved every 200ps. \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " molecules and the initial 50 ns (relaxation) are not included in the files.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_14225919.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Benchmarking of Molecular Dynamics Force Fields for Solid Liquid and Solid Solid Phase Transitions in \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Alkanes\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "<br>Accurate prediction of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    alkane\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " phase transitions involving solids is needed to prevent catastrophic pipeline blockages, improve lubrication formulations, smart insulation, and energy storage, as well as bring fundamental understanding to processes such as artificial morphogenesis. However, simulation of these transitions is challenging and therefore often omitted in force field development. Here, we perform a series of benchmarks on seven representative molecular dynamics models (\n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    TraPPE\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PYS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CHARMM36\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    L-OPLS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    COMPASS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Williams\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       ", and the newly optimized \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Williams 7B\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       "), comparing with experimental data for liquid properties, liquid solid, and solid solid phase transitions of two prototypical \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    alkanes\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    n-pentadecane\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    C15\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    n-hexadecane\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    C16\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "). We find that existing models overestimate the melting points by up to 34 K, with \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PYS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Williams 7B\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " yielding the most accurate results deviating only 2 and 3 K from the experiment. We specially design order parameters to identify crystal rotator phase transitions in \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    alkanes\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". United-atom models could only produce a rotator phase with complete rotational disorder, whereas all-atom models using a 12-6 Lennard-Jones potential show no rotator phase even when superheated above the melting point. In contrast, \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Williams\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " (Buckingham potential) and \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    COMPASS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " (9-6 Lennard-Jones) reproduce the crystal-to-rotator phase transition, with the optimized \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Williams 7B\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " model having the most accurate crystal rotator transition temperature of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    C15\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ".</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_4962493.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Data from: De novo active sites for resurrected \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Precambrian enzymes\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "<br>Protein engineering studies often suggest the emergence of completely new enzyme functionalities to be highly improbable. However, enzymes likely catalysed many different reactions already in the last universal common ancestor. Mechanisms for the emergence of completely new active sites must therefore either plausibly exist or at least have existed at the primordial protein stage. Here, we use resurrected \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Precambrian proteins\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " as scaffolds for protein engineering and demonstrate that a new active site can be generated through a single hydrophobic-to-ionizable amino acid replacement that generates a partially buried group with perturbed physico-chemical properties. We provide experimental and computational evidence that conformational flexibility can assist the emergence and subsequent evolution of new active sites by improving substrate and transition-state binding, through the sampling of many potentially productive conformations. Our results suggest a mechanism for the emergence of primordial enzymes and highlight the potential of ancestral reconstruction as a tool for protein engineering.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_3248612.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Simulation \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CER160\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " 128 \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " 512 \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    NaCl\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " 150mM \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    310K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       " rep1 membrane<br>Title publication: A \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Ceramide\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-Regulated Element in the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Late Endosomal Protein\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    LAPTM4B\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Controls Amino Acid Transporter Interaction Year publication: 2018 DOI publication: 10.1021/acscentsci.7b00582 Description: Simulation 512 \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPCs\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " membrane containing 128 \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ceramides\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CER160\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") using \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    charmm36\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " at \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    310K.\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       " MD engine: \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    gromacs 2016.1\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " Force field: \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    charmm36\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " Temperature: \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    310 K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       " Pressure: 1 bar Simulation time: \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    3000 ns\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       " Saving frequency: 100 ps Molecular content: \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " 512 \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CER160\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " 128 \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    SOL\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " 32000 \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    NA\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " 86 \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CL\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " 86 Additional information:</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_7239968.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">SAMSET - Model<br>\n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    LEAP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " models for SAMSET project.Urbanisation rates in Africa are the highest in the world, and in most Sub-Saharan countries service delivery is inadequate to keep up with the needs. African populations remain amongst the poorest in the world, and efforts to achieve the energy-related dimensions of the Millennium Development Goals s have in most cases not had significant impact on urban populations.The situation can be summarised as one where much urban energy transformation research does not understand the detailed organisational dynamics and constraints in cities and therefore is often of limited use; where there is a gap between policy and implementation; where capacity within local/national government departments involved in energy and urban development is inadequate in the face of increasing challenges; and where modes of knowledge transfer are not effective in facilitating sustainable energy transitions in cities.SAMSET seeks to develop a knowledge exchange framework for supporting local and national bodies involved in municipal energy planning in the effective transition to sustainable energy use in urban areas. Through close partnering with six cities in three African countries (Ghana, Uganda and South Africa), the project aims to develop an information base from which to support cities, undertake direct support for cities around strategy development and priority initiatives, and facilitate knowledge exchange and capacity building.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_2267512.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Growth and Characterization of Molecular Crystals of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    para-Sexiphenyl\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " by All-Atom Computer Simulations<br>We present all-atom molecular dynamics computer simulations of molecular crystals of the conjugated organic molecule \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    para-sexiphenyl\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    p-6P\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "), which constitutes a popular basic molecule for optoelectronic applications. After validating single-molecule properties with ab initio calculations, we demonstrate that gradually performed simulated temperature annealing leads to the spontaneous self-assembly of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    p-6P\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " molecules from the fully isotropic state into the correct room-temperature solid crystal, with only a few percent deviation from the experimental unit-cell structure. A detailed investigation of the single crystal in anisotropic \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Gibbs\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " ensemble simulations yields experimentally consistent structures and solid to liquid-crystal phase behavior over a wide temperature range, providing molecular insight into nanometer-scale structural and dynamic properties of self-assembled \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    p-6P\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " crystals. This study thus paves the way for future investigations of the computational description of nucleation and growth mechanisms of novel \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    p-polyphenylene\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " derivatives in the bulk as well as at functional interfaces or heterojunctions.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_1118682.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Simulations of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " lipid bilayer in \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " solution with various molar fractions of cationic surfactant \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    dihexadecylammonium\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " using \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ECC-POPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " force field<br>Classical molecular dynamics simulations of a \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " lipid bilayer in \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " solution with various molar fractions of cationic surfactant \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    dihexadecylammonium\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " using \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ECC-POPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " force field parameters, \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    SPC/E\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " model and \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ECC-ions.\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " Simulation at pure \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " is in a separate Zenodo deposit https://removed file names report molar fraction of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    cations\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (i.e. not bulk concentrations) simulations performed with \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Gromacs 5.1.4\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " (*.xtc files) and \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    openMM 7\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " (*.dcd files) simulation length \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    200 ns\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       " temperature \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    313 K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       " (otherwise noted)</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_3610470.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">GMX lipid17.ff: \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Gromacs\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " Port of the \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    amber LIPID17\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " force field<br>This is a \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Gromacs\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " port of the \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    amber LIPID17\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " force field. To use this force field, the user can construct the lipid bilayer using \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Charmm-GUI\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " and convert the atom names to the \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    amber\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " atom names using charmmlipid2amber.py. This port has also retained the modular feature of the \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    LIPID17\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " force field, where the user can customise the head group or \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    acryl\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " chain and use pdb2gmx to construct the topology. The coordinate files for the \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    amber\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " lipids can also be obtained from the 'gro' folder. The force field 'lipid17.ff', itp file 'lipid17.itp' and a custom PI head group are all included in the attached compressed file. For the details of the generation and validation protocol, please consult the relevant \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Github\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " page.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_3540691.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Determination and Evaluation of the Non-Additivity in Wetting of Molecularly Heterogeneous Surfaces<br>All the data relevant to the paper &quot;Determination and Evaluation of the Non-Additivity in Wetting of Molecularly Heterogeneous Surfaces&quot; (PNAS, 2019) are available here. The details about the data collection and analysis are available in the paper. The data are divided into two zipped folders: one of them contains all the experimental data (experiments.zip) and the other one all the files related to the molecular dynamics simulations (MD.zip). Each of the two folders contains a README.txt file, with details about the folder and its structure.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_3878218.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Molecular recognition and dynamics of linear \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    poly-ubiquitins\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ": integrating coarse-grain simulations and experiments<br>\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Poly-ubiquitin\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " chains are flexible multidomain proteins, whose conformational dynamics enable their molecular recognition by a large number of partners in multiple biological pathways. By using alternative linkage, it is possible to obtain \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    poly-ubiquitin\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " molecules with different dynamical properties. This flexibility is further increased by the possibility to tune the length of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    poly-ubiquitin\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " chains. Characterizing the dynamics of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    poly-ubiquitins\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " as a function of their length is thus relevant to understand their biology. Structural characterization of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    poly-ubiquitin\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " conformational dynamics is challenging both experimentally and computationally due to increasing system size and conformational variability. Here, by developing highly efficient and accurate small-angle X-ray scattering driven \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Martini\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " coarse-grain simulations, we characterize the dynamics of linear M1-linked \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    di-, tri- and tetra-ubiquitin\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " chains. Our data show that the behavior of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    di-ubiquitin\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " subunits is independent of the presence of additional \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ubiquitin\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " modules. We propose that the conformational space sampled by linear \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    poly-ubiquitins\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", in general, may follow a simple self-avoiding polymer model. These results, combined with experimental data from small angle X-ray scattering, biophysical techniques and additional simulations show that binding of \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    NEMO\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       ", a central regulator in the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    NF-κB\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " pathway, to linear \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    poly-ubiquitin\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " obeys a 2:1 (\n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    NEMO\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       ":\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    poly-ubiquitin\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") stoichiometry in solution, even in the context of four \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ubiquitin\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " units. Eventually, we show how the conformational properties of long \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    poly-ubiquitins\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " may modulate the binding with their partners in a length-dependent manner.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_3908769.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">\n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CHARMM27\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " dynamics simulation trajectories of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    α-conotoxin LsIA\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and its C-terminal carboxylated analogue bound at \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    α3β2 nAChR\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "<br>The whole simulation trajectories (28 individual trajectories with \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    27ns\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       " for each) contain the coordinates and parameters of atoms with time for \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    α-conotoxin LsIA\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and its C-terminal carboxylated analogue anchored to rat \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    α3β2 nAChR\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", respectively. The \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GROMACS 4.6.5\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " with the \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CHARMM27\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " force field is used for the simulation. The trajectory (.xtc) files are saved every 100ps time for each protein complex only. The portable binary run input (.tpr) files are also uploaded with the data.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_17207477.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Markovian Weighted Ensemble Milestoning (M-WEM): Long-Time Kinetics from Short Trajectories<br>We introduce a rare-event sampling scheme, named Markovian Weighted Ensemble Milestoning (M-WEM), which inlays a weighted ensemble framework within a Markovian milestoning theory to efficiently calculate thermodynamic and kinetic properties of long-time-scale biomolecular processes from short atomistic molecular dynamics simulations. M-WEM is tested on the Mu ller Brown potential model, the conformational switching in alanine dipeptide, and the millisecond time-scale protein ligand unbinding in a \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    trypsin benzamidine\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " complex. Not only can M-WEM predict the kinetics of these processes with quantitative accuracy but it also allows for a scheme to reconstruct a multidimensional free-energy landscape along additional degrees of freedom, which are not part of the milestoning progress coordinate. For the ligand receptor system, the experimental residence time, association and dissociation kinetics, and binding free energy could be reproduced using M-WEM within a simulation time of a few hundreds of nanoseconds, which is a fraction of the computational cost of other currently available methods, and close to 4 orders of magnitude less than the experimental residence time. Due to the high accuracy and low computational cost, the M-WEM approach can find potential applications in kinetics and free-energy-based computational drug design.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_259443.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Simulations of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Na+\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Ca2+\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " binding to \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    phospholipid\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " membranes<br>Simulations of various bilayers together with ions. A \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DOPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " bilayer in the liquid disordered phase (\n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    310 K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       ") and a \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DPPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " bilayer in the gel phase (\n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    293 K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       "), both formed of 200 lipids, were simulated with the presence of either 130 mM of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    NaCl\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " or 450 mM of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CaCl_2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". The \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Slipids\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " model [1–3] is employed for lipids, \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    tip3p\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " model for \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", the ion parameters by Smith and Dang for \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    NaCl\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " [4] and the ion parameters by Kohagen et al. for \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CaCl_2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " [5]. Trajectories with \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    NaCl\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " are \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    200 ns\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       " long, while with \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CaCl_2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " they were either \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    500 ns\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DOPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") or \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    800 ns\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DPPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") long. Additionally, a ternary mixture of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DPPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DOPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    cholesterol\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " was simulated with the presence of 150 mM or 1 M of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    NaCl\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " [4] at \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    310 K.\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       " The membrane was pre-assembled to contain Ld/Lo coexistence with a well-defined boundary. The Lo side consisted of 205 \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DPPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and 52 \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    cholesterol\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " molecules, while the Ld side consisted of 166 \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DOPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " molecules. Trajectories are \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    200 ns\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       " long. The \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Slipids\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " force field parameters are available at http://www.fos.su.se/~sasha/SLipids/ and the ion parameters are available at https://bitbucket.org/hseara/ions/ –––––––––––––––––––––––––––––––––––––––––––––––––––––– The files are in \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GROMACS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " format. Trajectories (.xtc) are saved every 100 ps. Additionally, the final structure (.gro), topology (.top), index file (.ndx), energy output file (.edr), and binary run input files (for \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Gromacs 4.6-&gt;\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       ") (.tpr) are provided for each system. The common simulation parameter file except for the varying temperatures (md.mdp) is also provided. –––––––––––––––––––––––––––––––––––––––––––––––––––––– [1] Derivation and Systematic Validation of a Refined All-Atom Force Field for Phosphatidylcholine Lipids. Joakim P. M. Jämbeck and Alexander P. Lyubartsev, The Journal of Physical Chemistry B 2012 116 (10), 3164-3179, DOI: 10.1021/jp212503e [2] An Extension and Further Validation of an All-Atomistic Force Field for Biological Membranes. Joakim P. M. Jämbeck and Alexander P. Lyubartsev, Journal of Chemical Theory and Computation 2012 8 (8), 2938-2948, DOI: 10.1021/ct300342n [3] Another Piece of the Membrane Puzzle: Extending Slipids Further. Joakim P. M. Jämbeck and Alexander P. Lyubartsev, Journal of Chemical Theory and Computation 2013 9 (1), 774-784, DOI: 10.1021/ct300777p [4] Computer simulations of NaCl association in polarizable water. David E. Smith and Liem X. Dang, The Journal of Chemical Physics 1994 100, 3757-3766, DOI: 10.1063/1.466363 [5] Accurate Description of Calcium Solvation in Concentrated Aqueous Solutions. Miriam Kohagen, Philip E. Mason, and Pavel Jungwirth, The Journal of Physical Chemistry B 2014 118 (28), 7902-7909, DOI: 10.1021/jp5005693</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_17319679.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Free-Energy Simulations Support a Lipophilic Binding Route for \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Melatonin Receptors\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "<br>The effects of the neurohormone melatonin are mediated by the activation of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GPCRs\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    MT1\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    MT2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " in a variety of tissues. Crystal structures suggest ligand access to the orthosteric binding site of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    MT1\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    MT2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " receptors through a lateral channel between transmembrane (TM) helices IV and V. We investigated the feasibility of this lipophilic entry route for \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    2-iodomelatonin\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", a nonselective agonist with a slower dissociation rate from the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    MT2 receptor\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", applying enhanced sampling simulations and free-energy calculations. \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    2-Iodomelatonin\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " unbinding was investigated with steered molecular dynamics simulations which revealed different trajectories passing through the gap between TM helices IV and V for both receptors. For one of these unbinding trajectories from the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    MT1 receptor\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", an umbrella-sampling protocol with path-collective variables provided a calculated energy barrier consistent with the experimental dissociation rate. The side-chain flexibility of Tyr5.38 was significantly different in the two receptor subtypes, as assessed by metadynamics simulations, and during ligand unbinding it frequently assumes an open conformation in the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    MT1\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " but not in the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    MT2 receptor\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", favoring \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    2-iodomelatonin\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " egress. Taken together, our simulations are consistent with the possibility that the gap between TM IV and V is a way of connecting the orthosteric binding site and the membrane core for lipophilic melatonin receptor ligands. Our simulations also suggest that the open state of Tyr5.38 generates a small pocket on the surface of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    MT1 receptor\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", which could participate in the recognition of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    MT1\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-selective ligands and may be exploited in the design of new selective compounds.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_8431.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Molecular dynamics derived side chain order parameters for Asp, Glu, Asn, and Gln residues in ribonucleases H, and molecular dynamics trajectories for \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    E. coli ribonuclease H\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "<br>This dataset contains tab-delimited text files reporting the complete calculated S2 side chain order parameters for \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    carboxyl\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " - and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    carbonyl\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " -containing residues (Asp, Glu, Asn, Gln) in all \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    RNase H\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " homologs studied, as well as complete simulation trajectory data for the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    E. coli apo\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", site A, and site B simulations described in the text. We hope that easy access to the complete trajectory data for the experimentally best-characterized homolog will facilitate future comparison between simulation and experiments. order parameters magnesium.dat contains the calculated S2 side chain order parameters for \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    carboxyl\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " - and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    carbonyl\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " -containing residues in each of the three E. coli trajectories supplied in this dataset. These data are sufficient to reproduce Figure 6. 2RN2.apo.pdb, 2RN2.MG.siteA.pdb, and 2RN2.MG.siteB.pdb define the equilibrated structure from which the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    E. coli apo\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", site A, and site B trajectories were initiated. All protein atoms, including hydrogens, are included, as well as \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Mg\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " ions. Explicit \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    waters\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    chloride counterions\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " have been removed. 2RN2.apo.dcd, 2RN2.MG.siteA.dcd, and 2RN2.MG.siteB.dcd contain \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    100ns\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       " trajectories initiated from the starting structures in the corresponding pdb files. Frames were written every 4.5ps. Trajectories have been aligned to the initial structure of the apo trajectory to facilitate visualization. The remaining *.S2.CO.dat files contain the calculated S2 side chain order parameters for all \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    apo RNases H\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " simulated in this study. Each file is named for the PDB ID of its corresponding structure (see Table 1). These data are sufficient to reproduce Figure 8 (which reports on only the conserved active-site residues).</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_12821617.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">MRP.py: A Parametrizer of Post-Translationally Modified Residues<br>MRP.py is a Python-based parametrization program for covalently modified amino acid residues for molecular dynamics simulations. Charge derivation is performed via an RESP charge fit, and force constants are obtained through rewriting of either protein or \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GAFF\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " database parameters. This allows for the description of interfacial interactions between the modifed residue and protein. MRP.py is capable of working with a variety of protein databases. MRP.py s highly general and systematic method of obtaining parameters allows the user to circumvent the process of parametrizing the modified residue protein interface. Two examples, a covalently bound inhibitor and covalent adduct consisting of modified residues, are provided in the Supporting Information.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_3567651.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Peptidomimetics-based identification of FDA approved compounds inhibiting \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    IRE1\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " activity: Simulation Data<br>Protocols and data sets for docking and MD simulations of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Methotrexate\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Cefoperazone\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Folinic\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " acd and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Folic\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " acid interaction with \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    IRE1\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". All docking data based on \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Glide\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " (Schro dinger), and all MD simulations are based on the \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Gromacs\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " software. 2D interaction maps from the initial docking of the four molecules to the four Lysine sites, and from snapshots every 100 ns of the \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    500 ns\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       " MD simulations of the molecules interacting with Lys599 (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    kinase\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " site) and Lys907 (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    RNase\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " site). 2D interaction maps produced using Schro dinger. </div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_4451274.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">The \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    SARS-CoV-2 Spike\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Variant D614G Favors an Open Conformational State<br>The COVID-19 pandemic underwent a rapid transition with the emergence of a dominant viral variant (from the D-form to the G-form ) that carried an amino acid substitution D614G in its Spike protein. The G-form is more infectious in vitro and associated with increased viral loads. To gain insight into the molecular-level underpinnings of these characteristics, we employed \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    microsecond\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       " all-atom simulations. Here we show that changes in the protein energetics favor a higher population of infection-capable (open) states through release of hydrogen bonds of an asymmetry present in the D-form but not the G-form. Thus, the increased infectivity of the G-form is likely due to a higher rate of profitable binding encounters with the host receptor. It is also predicted to be more neutralization sensitive due to enhanced exposure of the receptor binding domain, a key target region for neutralizing antibodies. These results are significant for vaccine design. The Molecular Dynamics Simulations datasets generated in this study, namely that of the soluble form \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    SARS-CoV-2 Spike\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " protein are made available here. Details of the trajectory and supporting files are given in the README.txt file.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_20080728.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Insights into the Atomistic Mechanisms of Phosphorylation in Disrupting Liquid Liquid Phase Separation and Aggregation of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    FUS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Low-Complexity Domain<br>\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Fused in sarcoma\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    FUS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "), a nuclear RNA binding protein, can not only undergo liquid liquid phase separation (LLPS) to form dynamic biomolecular condensates but also aggregate into solid amyloid fibrils which are associated with the pathology of amyotrophic lateral sclerosis and frontotemporal lobar degeneration diseases. Phosphorylation in the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    FUS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " low-complexity domain (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    FUS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-LC) inhibits \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    FUS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " LLPS and aggregation. However, it remains largely elusive what are the underlying atomistic mechanisms of this inhibitory effect and whether phosphorylation can disrupt preformed \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    FUS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " fibrils, reversing the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    FUS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " gel/solid phase toward the liquid phase. Herein, we systematically investigate the impacts of phosphorylation on the conformational ensemble of the FUS37 97 monomer and dimer and the structure of the FUS37 97 fibril by performing extensive all-atom molecular dynamics simulations. Our simulations reveal three key findings: (1) phosphorylation shifts the conformations of FUS37 97 from the -rich, fibril-competent state toward a helix-rich, fibril-incompetent state; (2) phosphorylation significantly weakens protein protein interactions and enhances protein \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " interactions, which disfavor \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    FUS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-LC LLPS as well as aggregation and facilitate the dissolution of the preformed \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    FUS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-LC fibril; and (3) the FUS37 97 peptide displays a high -strand probability in the region spanning residues 52 67, and phosphorylation at S54 and S61 residues located in this region is crucial for the disruption of LLPS and aggregation of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    FUS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-LC. This study may pave the way for ameliorating phase-separation-related pathologies via site-specific phosphorylation.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_51747.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Molecular dynamics simulation trajectory of a \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    cationic lipid\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " bilayer: 75/25 mol% \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DMTAP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "/\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DMPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "<br>System: Symmetric bilayer of cationic \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DMTAP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    dimyristoyltrimethylammoniumpropane\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", 75 mol-%) and zwitterionic \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DMPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    dimyristoylphosphatidylcholine\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", 25 mol-%) lipids in \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". Number of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DMPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ": 32. Number of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DMTAP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ": 96. Number of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Cl\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "--ions: 96. Number of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    waters\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ": 5496. Lipid model: '\n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Berger\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       "' united-atom [Biophys. J. 72 2002 (1997)] with \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DMTAP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " modification by Gurtovenko et al. [Biophys. J. 86 3461 (2004)]. \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " model: \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    SPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " [In Intermolecular Forces, ed. Pullman. 331 (1981)]. Simulation engine: \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Gromacs 3.x\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " [https://removed Trajectory length: \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    109 ns\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       ". Previously equilibrated for: 31 ns. Sampling rate: every 10 ps. Time integration step: 2 fs. Thermodynamic ensemble: NpT. Temperature coupling: 'Berendsen' [J. Chem. Phys. 81 3684 (1984)] with lipids and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " coupled separately at T \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    323 K.\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       " Pressure coupling: 'Berendsen' [J. Chem. Phys. 81 3684 (1984)] with xy and z coupled separately at p 1.0 bar. Electrostatics: PME [J. Chem. Phys. 98 10089 (1993); J. Chem. Phys. 103 8577 (1995)], real-space cutoff at 1.0 nm. Van der Waals: Truncated at 1.0 nm. Constraints: Covalent bond lengths in lipids using \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    LINCS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " [J. Comput. Chem. 18 1463 (1997)], in \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " using \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    SETTLE\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " [J. Comput. Chem. 13 952 (1992)]. Used in publications: [1] Markus S. Miettinen, Andrey A. Gurtovenko, Ilpo Vattulainen, and Mikko Karttunen: Ion Dynamics in Cationic Lipid Bilayer Systems in Saline Solutions. J. Phys. Chem. B 113 9226 (2009). DOI: 10.1021/jp810233q. [2] Markus S. Miettinen: Computational Modeling of Cationic Lipid Bilayers in Saline Solutions. PhD Thesis. Aalto University School of Science and Technology, Helsinki, Finland. (2010). ISBN 978-952-60-3194-1.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_53212.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Molecular dynamics simulation trajectory of a \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    cationic lipid\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " bilayer: 50/50 mol% \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DMTAP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "/\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DMPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " in 0.1 M \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    NaCl\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "<br>System: Symmetric bilayer of cationic \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DMTAP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    dimyristoyltrimethylammoniumpropane\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", 50 mol-%) and zwitterionic \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DMPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    dimyristoylphosphatidylcholine\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", 50 mol-%) lipids in 0.1 M \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    NaCl\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " solution. Number of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DMPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ": 64. Number of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DMTAP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ": 64. Number of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Na+\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " ions: 10. Number of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Cl-\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " ions: 74. Number of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    waters\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ": 5316. Lipid model: '\n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Berger\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       "' united-atom [Biophys. J. 72 2002 (1997)] with \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    DMTAP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " modification by Gurtovenko et al. [Biophys. J. 86 3461 (2004)]. \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " model: \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    SPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " [In Intermolecular Forces, ed. Pullman. 331 (1981)]. Simulation engine: \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Gromacs 3.x\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " [https://removed Trajectory length: \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    190 ns.\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       " Previously equilibrated for: 21 ns. Sampling rate: every 10 ps. Time integration step: 2 fs. Thermodynamic ensemble: NpT. Temperature coupling: 'Berendsen' [J. Chem. Phys. 81 3684 (1984)] with lipids and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " coupled separately at T \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    323 K.\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       " Pressure coupling: 'Berendsen' [J. Chem. Phys. 81 3684 (1984)] with xy and z coupled separately at p 1.0 bar. Electrostatics: PME [J. Chem. Phys. 98 10089 (1993); J. Chem. Phys. 103 8577 (1995)], real-space cutoff at 1.0 nm. Van der Waals: Truncated at 1.0 nm. Constraints: Covalent bond lengths in lipids using \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    LINCS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " [J. Comput. Chem. 18 1463 (1997)], in \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " using \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    SETTLE\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " [J. Comput. Chem. 13 952 (1992)]. Used in publications: [1] Markus S. Miettinen, Andrey A. Gurtovenko, Ilpo Vattulainen, and Mikko Karttunen: &quot;Ion Dynamics in Cationic Lipid Bilayer Systems in Saline Solutions&quot;. J. Phys. Chem. B 113 9226 (2009). DOI: 10.1021/jp810233q. [2] Markus S. Miettinen: &quot;Computational Modeling of Cationic Lipid Bilayers in Saline Solutions&quot;. PhD Thesis. Aalto University School of Science and Technology, Helsinki, Finland. (2010). ISBN 978-952-60-3194-1.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_2532337.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Are Protein Force Fields Getting Better? A Systematic Benchmark on 524 Diverse NMR Measurements<br>Recent hardware and software advances have enabled simulation studies of protein systems on biophysically relevant time scales, often revealing the need for improved force fields. Although early force field development was limited by the lack of direct comparisons between simulation and experiment, recent work from several laboratories has demonstrated direct calculation of NMR observables from protein simulations. Here, we quantitatively evaluate 11 recent molecular dynamics force fields in combination with 5 solvent models against a suite of 524 chemical shift and J coupling (3JHNH , 3JHNC , 3JH C , 3JHNC , and 3JH N) measurements on \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    dipeptides\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    tripeptides\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    tetra-alanine\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ubiquitin\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". Of the force fields examined (\n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ff96\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ff99\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ff03\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ff03*\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ff03w\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ff99sb*\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ff99sb-ildn\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ff99sb-ildn-phi\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ff99sb-ildn-NMR\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CHARMM27\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       ", and \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    OPLS-AA\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       "), two force fields (\n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ff99sb-ildn-phi\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ff99sb-ildn-NMR\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       ") combining recent side chain and backbone torsion modifications achieved high accuracy in our benchmark. For the two optimal force fields, the calculation error is comparable to the uncertainty in the experimental comparison. This observation suggests that extracting additional force field improvements from NMR data may require increased accuracy in J coupling and chemical shift prediction. To further investigate the limitations of current force fields, we also consider conformational populations of dipeptides, which were recently estimated using vibrational spectroscopy.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_238943.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Umbrella sampling data of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    AuNP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " interacting with a \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " bilayer<br>Files in \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Gromacs (v4.6)\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " format used to generate the PMF profile shown in Fig. 3 in [1]. The nanoparticle is pulled to the EC leaflet in the 'genstart' simulations. The initial structures for the umbrella sampling windows are extracted from this trajectory. The files obtained from umbrella sampling windows are named 'pullX.*', where X is the distance between the centers of mass of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    AuNP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and the EC leaflet phosphori. Trajectories (.xtc) are saved every 100ps. Additionally, the initial and final structures (.gro), the run input files (.tpr), energy files (.edr), log files (.log), checkpoint files (.cpt), simulation parameter files (.mdp), and the files generated by the pull code (pull forces and pull distances, .xvg) are given for each umbrella window. Additionally, an index file (index.ndx) and a topology file (topol.top) common for all systems is provided. TOP.tar contains the molecule definitions (.itp). 'pullf-files.dat' and 'tpr-files.dat' are used by g_wham to extract the profile shown in Fig. 3 (except for the choice of the zero level) by 'g_wham -it tpr-files.dat -if pullf-files.dat -b 40000 -nBootstrap 100' [1] Heikkilä et al., Cationic Au Nanoparticle Binding with Plasma Membrane-like Lipid Bilayers: Potential Mechanism for Spontaneous Permeation to Cells Revealed by Atomistic Simulations, J. Phys. Chem. C, 2014, 118 (20), pp 11131–11141, DOI: 10.1021/jp5024026</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_7327525.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Raw data for: Cell-free biosynthesis combined with deep learning accelerates de novo-development of antimicrobial peptides<br>This repository contains data related to &quot;Cell-free biosynthesis combined with deep learning accelerates de novo-development of antimicrobial peptides&quot; by Pandi et al. Included are molecular dynamics parameter files, initial structures after system equilibration and production trajectories. For simulations of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    AMPs\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " with membranes, trajectories are subsampled with 1 frame every 5 ns and final structures after \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    1 μs\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       " of production simulation are included. Contact information: Name: Stefan L. Schaefer Institution: Department of Theoretical Biophysics, Max Planck Institute of Biophysics Address: Max-von-Laue-Str. 3, 60438 Frankfurt am Main, Germany Email: stefan.schaefer@biophys.mpg.de</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_3862992.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Data from: A multiscale biophysical model for the recruitment of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    actin\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " nucleating proteins at the membrane interface<br>The dynamics and organization of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    actin\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " cytoskeleton are crucial to many cellular events such as motility, polarization, cell shaping, and cell division. The intracellular and extracellular signaling associated with this cytoskeletal network is communicated through cell membranes. Hence the organization of membrane macromolecules and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    actin\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " filament assembly are highly interdependent. Although the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    actin\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-membrane linkage is known to happen through many routes, the major class of interactions is through the direct interaction of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    actin\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-binding proteins with the lipid class containing \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    poly-phosphatidylinositols\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PPIs\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "). Among the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PPIs\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    phosphatidylinositol bisphosphate\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PI (4,5)P2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") acts as a significant factor controlling \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    actin\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " polymerization in the proximity of the membrane by binding to \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    actin\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-associated proteins. The molecular interactions between these \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    actin\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-binding proteins and the membrane \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    lipids\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " remain elusive. Here, using molecular modeling, analytical theory, and experimental methods, we investigate the binding of three different \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    actin\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-binding proteins, \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    mDia2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    NWASP\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    gelsolin\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", to membranes containing \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PI (4,5)P2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " lipids. We perform molecular dynamics simulations on the protein-bilayer system and analyze the membrane binding in the form of hydrogen bonds and salt bridges at various \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PI (4,5)P2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    cholesterol\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " concentrations. Our experimental study with \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PI (4,5)P2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-containing large unilamellar vesicles mimics the computational experiments. Using the multivalencies of the proteins obtained in molecular simulations and the cooperative binding mechanisms of the proteins, we also propose a multivalent binding model that predicts the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    actin\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " filament distributions at various \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PI (4,5)P2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and protein concentrations.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_121241.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Modeling of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Arylamide\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Helix Mimetics in the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    p53\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Peptide Binding Site of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    hDM2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Suggests Parallel and Anti-Parallel Conformations Are Both Stable<br>The design of novel -helix mimetic inhibitors of protein-protein interactions is of interest to pharmaceuticals and chemical genetics researchers as these inhibitors provide a chemical scaffold presenting side chains in the same geometry as an -helix. This conformational arrangement allows the design of high affinity inhibitors mimicking known peptide sequences binding specific protein substrates. We show that \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GAFF\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    AutoDock\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " potentials do not properly capture the conformational preferences of -helix mimetics based on \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    arylamide\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " oligomers and identify alternate parameters matching solution NMR data and suitable for molecular dynamics simulation of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    arylamide\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " compounds. Results from both docking and molecular dynamics simulations are consistent with the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    arylamides\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " binding in the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    p53\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " peptide binding pocket. Simulations of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    arylamides\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " in the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    p53\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " binding pocket of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    hDM2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " are consistent with binding, exhibiting similar structural dynamics in the pocket as simulations of known \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    hDM2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " binders \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Nutlin-2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and a \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    benzodiazepinedione\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " compound. \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Arylamide\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " conformations converge towards the same region of the binding pocket on the 20 ns time scale, and most, though not all dihedrals in the binding pocket are well sampled on this timescale. We show that there are two putative classes of binding modes for \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    arylamide\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " compounds supported equally by the modeling evidence. In the first, the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    arylamide\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " compound lies parallel to the observed \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    p53\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " helix. In the second class, not previously identified or proposed, the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    arylamide\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " compound lies anti-parallel to the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    p53\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " helix. </div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_13655516.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GerAB\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "<br>Bacillus subtilis forms dormant spores upon nutrient depletion. \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Germinant receptors\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GRs\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") in spore s inner membrane respond to ligands such as \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    L-alanine\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", and trigger spore germination. In B. subtilis spores, \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GerA\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " is the major \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GR\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", and has three subunits, \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GerAA\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GerAB\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GerAC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    L-Alanine\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " activation of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GerA\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " requires all three subunits, but which binds \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    L-alanine\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " is unknown. To date, how \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GRs\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " trigger germination is unknown, in particular due to lack of detailed structural information about B subunits. Using homology modelling with molecular dynamics (MD) simulations, we present structural predictions for the integral membrane protein \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GerAB\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". These predictions indicate that \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GerAB\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " is an alpha-helical transmembrane protein containing a water channel. The MD simulations with free \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    L-alanine\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " show that alanine binds transiently to specific sites on \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GerAB\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". These results provide a starting point for unraveling the mechanism of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    L-alanine\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " mediated signaling by \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GerAB\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", which may facilitate early events in spore germination.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_2276194.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Free Energy Calculation of Nanodiamond-Membrane Association The Effect of Shape and Surface Functionalization<br>\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Nanodiamonds\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    NDs\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") are nanoscale diamond particles with broad applications in biosensing, drug delivery, and long-term tracking. Their interactions with a membrane dictate both the endocytosis process and subsequent intracellular fate of the nanoparticles. However, details of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ND\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-membrane association and the energetics of this process remain largely unknown. In this work, we use all-atom molecular dynamics (MD) simulations to determine the free energy profile and molecular details of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ND\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-membrane association, with a focus on the impact of shape and surface functionalization. Through altogether 6.5 s umbrella sampling on six atomistic \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ND\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " models of different shapes (spherical or pyramidal) and surface functionalization (5%, 35%, and 55%), we show that \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    nanodiamonds\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " associate favorably with the membrane, which is largely driven by \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ND\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " lipid interactions. During its membrane association, the shape of a nanodiamond plays a key role in determining the location of the free energy minimum, while its surface functionalization modulates the depth of the minimum. Of the six models studied here, all spherical \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    NDs\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " adhere to the bilayer surface, whereas pyramidal \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    NDs\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", with the exception of the most functionalized P55, anchor inside the membrane. Shape also dominates the height of the free energy barrier: the sharp pyramidal \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    NDs\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " have much lower barriers against penetrating a \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " bilayer than spherical ones. Our all-atom \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ND\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " models and their bilayer association strength determined here can be combined with future coarse-grained or continuum models to further explore \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ND\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " membrane interactions on larger length scales.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_7882718.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Rapid interpretation of small-angle X-ray scattering data<br>The fundamental aim of structural analyses in biophysics is to reveal a mutual relation between a molecule s dynamic structure and its physiological function. Small-angle X-ray scattering (SAXS) is an experimental technique for structural characterization of macromolecules in solution and enables time-resolved analysis of conformational changes under physiological conditions. As such experiments measure spatially averaged low-resolution scattering intensities only, the sparse information obtained is not sufficient to uniquely reconstruct a three-dimensional atomistic model. Here, we integrate the information from SAXS into molecular dynamics simulations using computationally efficient native structure-based models. Dynamically fitting an initial structure towards a scattering intensity, such simulations produce atomistic models in agreement with the target data. In this way, SAXS data can be rapidly interpreted while retaining physico-chemical knowledge and sampling power of the underlying force field. We demonstrate our method s performance using the example of three protein systems. Simulations are faster than full molecular dynamics approaches by more than two orders of magnitude and consistently achieve comparable accuracy. Computational demands are reduced sufficiently to run the simulations on commodity desktop computers instead of high-performance computing systems. These results underline that scattering-guided structure-based simulations provide a suitable framework for rapid early-stage refinement of structures towards SAXS data with particular focus on minimal computational resources and time.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_3467309.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Structural Studies of Supramolecular G Quadruplexes Formed from \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    8 Aryl-2 -deoxyguanosine\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Derivatives<br>Self-assembly is a powerful tool for the construction of complex nanostructures. Despite advances in the field, the development of precise self-assembled structures remains a challenge. We have shown that, in the presence of suitably sized cations like \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    K+\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    8-aryl-2 -deoxyguanosine\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    8ArG\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") derivatives self-assemble into sets of coaxially stacked planar tetramers, which we term supramolecular G-quadruplexes (SGQs). Previously, we reported that, when the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    8-aryl\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " group is a \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    phenyl\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " ring with a meta-carbonyl group, the resulting supramolecule is a hexadecamer, which is remarkably robust as illustrated by its isostructural assembly in both organic and aqueous environments. We report here a detailed three-dimensional structure of the SGQs formed by lipophilic, and hydrophilic, \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    8ArG\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " derivatives with either \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    8- (meta-acetylphenyl)\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    8- (para-acetylphenyl)\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", or \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    8- (meta-ethoxycarbonylphenyl)\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " groups. The chirality and close contacts between the subunits impose different levels of steric and electrostatic constraints on opposite sides of the tetrads, which determine their preferred relative orientation. The balance between attractive noncovalent interactions juxtaposed with repulsive steric and electrostatic interactions explains the high cooperativity, fidelity, and stability of these SGQs. These structural studies, together with titration experiments and molecular dynamics simulations, provide insight into the mechanism of formation of these SGQs.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_3696970.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Probing the Electrical Double Layer by Operando X-ray Photoelectron Spectroscopy through a \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Graphene\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Carbon\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Nanotube Composite Window<br>Supporting information for Molecular Dynamics \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    NAMD 2.0\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " simulations of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    BPY1,4-TFSI-Li\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " ionic liquid mixture encased in cuboid \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    graphene\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " box for \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    graphene\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " window double layer calculations presented in submitted manuscript to Eco Mat journal. The files contain new developed parameters for \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    BPY1,3-TFSI\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " force field based on \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Gaussian09\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " DFT calculations and relevant PDB and PSF and configuration files for \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    NAMD\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " runs at 0, 1 and 3Volt. Abstract: The electrical double layer is known to spontaneously form at the electrode-electrolyte interface, impacting many important chemical and physical processes as well as applications including electrocatalysis, electroorganic synthesis, nanomaterial preparation, energy storage, and even emulsion stabilization. However, it has been challenging to study this fundamental phenomenon at the molecular level because the electrical double layer is deeply buried by the bulk \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    electrolyte\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " solution. Here, we report a quantitative probing of the electrical double layer of ionic liquids from the solid side of a photoelectron-transparent \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    graphene\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    carbon\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " nanotube hybrid membrane electrode using X-ray photoelectron spectroscopy. The membrane window is ultrathin (~1.5 nm), large (~1 cm2), and robust, enabling a tight seal of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    electrolyte\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and quantitative measurement with excellent photoelectron signals. By operandomonitoring the population changes of cations and anions in response to the applied electrical potentials, we experimentally resolve the chemical structure and dynamics of the electrical double layer, which corroborate results from molecular dynamics simulations.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_3207154.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Accurate Structure Prediction and Conformational Analysis of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Cyclic Peptides\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " with Residue-Specific Force Fields<br>\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Cyclic peptides\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CPs\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") are promising candidates for drugs, chemical biology tools, and self-assembling nanomaterials. However, the development of reliable and accurate computational methods for their structure prediction has been challenging. Here, 20 all-trans \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CPs\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " of 5 12 residues selected from Cambridge Structure Database have been simulated using replica-exchange molecular dynamics with four different force fields. Our recently developed residue-specific force fields \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    RSFF1\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    RSFF2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " can correctly identify the crystal-like conformations of more than half \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CPs\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " as the most populated conformation. The \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    RSFF2\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " performs the best, which consistently predicts the crystal structures of 17 out of 20 \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CPs\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " with rmsd &lt; 1.1 A . We also compared the backbone ( , ) sampling of residues in \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CPs\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " with those in short linear peptides and in globular proteins. In general, unlike linear peptides, \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CPs\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " have local conformational free energies and entropies quite similar to globular proteins.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_1198454.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Simulation data for \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CHARMM36\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " bilayer, 100 lipids/leaflet, 940 mM \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    NaCl\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    310K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GROMACS 5.1.4\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       "<br>Simulations of a \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " bilayer with 940 mM of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    NaCl\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". The fifth from the set of 6 simulations. The goal was to study the effect of scaling the \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CHARMM\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " FF on the ion binding. Done for the NMRlipids project, see https://removed for more information. A \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " bilayer consisting of 200 lipids (100 per leaflet) is simulated in the presence of 940 mM \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    NaCl\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". The \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Charmm36\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " model is employed for lipids, the \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Charmm\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " compatible variant of the \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    tip3p\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " model for \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", and the default \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Charmm\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " ion parameters (type SOD) for \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    NaCl\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". NB-Fix used for sodium. The \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Charmm36\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " force field parameters were obtained from https://removed The files are in \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GROMACS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " format. Trajectory (.xtc) is \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    370 ns\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       " long with data saved every 100 ps. the initial structure (.gro), topology (.top), index file (.ndx), simulation paremeter file (.mdp), binary run input file for \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GROMACS v. 5.1 &gt;\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " (.tpr) and the energy output file (.edr) are provided. </div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_1167532.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Simulation of a \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " bilayer at \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    298K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    lipid\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " model by Maciejewski and Rog<br>Simulation of a \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " bilayer containing 128 \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    lipids\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " with 40 \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " molecules per \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    lipid\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and in the absence of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ions\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " at \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    298 K.\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       " The \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    lipid\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " model by Maciejewski and Rog [1,2] was used. Topologies (.itp) were obtained from [2]. \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    TIP3P\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " was used, and the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ions\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " were modelled using default \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    OPLS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ion\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " parameters. The simulation is \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    200 ns\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       " long with trajectory saved every 100 ps. Simulations were performed with \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Gromacs 2016.3\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " [3] The trajectory (.xtc), energy file (.edr), checkpoint file (.cpt), run input file (.tpr), index file (.ndx), topology file (.top), final structure (gro), and the simulation parameter file (.mdp) are provided. [1] Maciejewski et al., J. Phys Chem. B 118, 2014, pp. 4571 4581, DOI: 10.1021/jp5016627 [2] Kulig et al., Data in Brief 5, 2015, pp. 333 336, DOI: 10.1016/j.dib.2015.09.013 [3] Abraham et al., SoftwareX 1 2, 2015, pp. 19 25, DOI: 10.1016/j.softx.2015.06.001</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_5496910.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">\n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    doGlycans Tools\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " for Preparing \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Carbohydrate\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Structures for Atomistic Simulations of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Glycoproteins\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Glycolipids\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Carbohydrate\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Polymers for \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GROMACS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       "<br>\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Carbohydrates\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " constitute a structurally and functionally diverse group of biological molecules and macromolecules. In cells they are involved in, e.g., energy storage, signaling, and cell cell recognition. All of these phenomena take place in atomistic scales, thus atomistic simulation would be the method of choice to explore how \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    carbohydrates\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " function. However, the progress in the field is limited by the lack of appropriate tools for preparing \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    carbohydrate\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " structures and related topology files for the simulation models. Here we present tools that fill this gap. Applications where the tools discussed in this paper are particularly useful include, among others, the preparation of structures for \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    glycolipids\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    nanocellulose\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    glycans\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " linked to \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    glycoproteins\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". The molecular structures and simulation files generated by the tools are compatible with \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GROMACS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       ".</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_2478628.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Adsorption Behavior of Hydrophobin Proteins on \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Polydimethylsiloxane\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Substrates<br>The design of a bioactive surface with appropriate wettability for effective protein immobilization has attracted much attention. Previous experiments showed that the adsorption of hydrophobic protein HFBI onto a \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    polydimethylsiloxane\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PDMS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") substrate surface can reverse the inherent hydrophobicity of the surface, hence making it suitable for immobilization of a secondary protein. In this study, atomistic molecular dynamics simulations have been conducted to elucidate the adsorption mechanism of HFBI on the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PDMS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " substrate in an aqueous environment. Nine independent simulations starting from three representative initial orientations of HFBI toward the solid surface were performed, resulting in different adsorption modes. The main secondary structures of the protein in each mode are found to be preserved in the entire course of adsorption due to the four disulfide bonds. The relative binding free energies of the different adsorption modes were calculated, showing that the mode, in which the binding residues of HFBI fully come from its hydrophobic patch, is most energetically favored. In this favorable binding mode, the hydrophilic region of HFBI is fully exposed to water, leading to a high hydrophilicity of the modified \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PDMS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " surface, consistent with experiments. Furthermore, a set of residues consisting of Leu12, Leu24, Leu26, Ile27, Ala66, and Leu68 were found to play an important role in the adsorption of HFBI on different hydrophobic substrates, irrespective of the structural features of the substrates.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_13164486.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Self-Assembled Solute Networks in Crowded Electrolyte Solutions and Nanoconfinement of Charged Redoxmer Molecules<br>Redoxmers are electrochemically active organic molecules storing charge and energy in electrolyte fluids circulating through redox flow batteries (RFBs). Such molecules typically have solvent-repelling cores and solvent-attracting pendant groups introduced to increase solubility in liquid electrolytes. These two features can facilitate nanoscale aggregation of the redoxmer molecules in crowded solutions. In some cases, this aggregation leads to the emergence of continuous networks of solute molecules in contact, and the solution becomes microscopically heterogeneous. Here, we use small-angle X-ray scattering (SAXS) and molecular dynamics modeling to demonstrate formation of such networks and examine structural factors controlling this self-assembly. We also show that salt ions become excluded from these solute aggregates into small pockets of electrolytes, where these ions strongly associate. This confinement by exclusion is also likely to occur to charged redoxmer molecules in a sea of neutral precursors coexisting in the same solution. Here, we demonstrate that the decay lifetime of the confined charged molecules in such solutions can increase several fold compared to dilute solutions. We attribute this behavior to a microreactor effect on reverse reactions of the confined species during their decomposition.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_1293762.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">\n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GROMOS 43A1-S3\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPE\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Simulations (versions 1 and 2) \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    313 K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       " (NOTE: anisotropic pressure coupling)<br>Two \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GROMOS 43A1-S3\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPE\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " bilayer simulations performed using \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GROMACS 4.0.7\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " for \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    200 ns\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       " with different starting velocities. Simulations were performed with the standard 43A1-S3 settings: a 1.0 nm cut-off with PME for the Coulombic interactions and a twin-range 1.0/1.6 nm cut-off for the van der Waals interactions. These simulations were performed at \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    313 K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       " with a 128 lipid bilayer and used anisotropic pressure coupling. The full trajectories are provided bar the initial \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    100 ns\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       ". The starting structure was made through the conversion of an equilibrated \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    GROMOS 43A1-S3\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " membrane.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_12987701.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Melting Points of \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    OPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    OPC3\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Models<br>A recently introduced family of globally optimal \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " models, \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    OPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       ", has shown promise in a variety of biomolecular simulations, but properties of these \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " models outside of the liquid phase remain mostly unexplored. Here, we contribute to filling the gap by reporting melting temperatures of ice Ih of \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    OPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    OPC3\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " models. Through the direct coexistence method, which we make available in the \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    AMBER\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " package, the melting points of \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    OPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    OPC3\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " are estimated as \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    242\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    210 K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       ", similar to \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    TIP4P-Ew\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    SPC/E\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " models, respectively, and appreciably below the experimental value of \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    273.15 K\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       " under 1 bar pressure. \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " models of the \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    OPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " family were optimized to best reproduce \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    water\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " properties in the liquid phase where these models offer noteworthy accuracy advantages over many models of previous generations. It is not surprising that the accuracy of \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    OPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " models in describing the phase transition to the solid state does not appear to offer similar improvements. The new anisotropic barostat option implemented in \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    AMBER\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " may benefit system preparation and simulation outside of the direct coexistence applications, such as modeling of membranes or very long DNA strands.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_12312368.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Small Changes in Hydration Determine \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Cephalosporinase\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Activity of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    OXA-48 β‑Lactamases\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "<br>\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    β-Lactamase\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " mediated antibiotic resistance threatens treatment of bacterial infections. OXA-48 enzymes are clinically important class D \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    serine β-lactamases\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    SBLs\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") that confer resistance to most \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    β-lactam\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " antibiotics, including \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    carbapenems\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". However, OXA-48 and related enzymes vary widely in their activity toward different substrates: OXA-48 primarily hydrolyzes \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    carbapenems\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", whereas the OXA-163 variant is a \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    cephalosporinase\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " with minimal \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    carbapenemase\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " activity. The basis of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    cephalosporinase\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " activity in OXA-163 remains elusive. Here we use QM/MM reaction simulations (umbrella sampling molecular dynamics) to study the breakdown of the \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    cephalosporin\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " antibiotic \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ceftazidime\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", a key antibiotic for healthcare-associated infections, by selected OXA-48 variants. Calculated free energy barriers for \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ceftazidime\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " deacylation correctly capture the differing catalytic efficiencies of the studied enzymes and identify the catalytically competent orientation for bound \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ceftazidime\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". Additionally, we show that high flexibility of the Ω loop bordering the active site, a determinant of specificity in many \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    SBLs\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ", is not required for efficient deacylation. Based on our simulations, cephalosporin breakdown in OXA-163 is efficient due to subtle control of active site solvation, which requires a particular orientation of Leu158 in the Ω loop. Our simulations further predict that a single mutation in the OXA-48 β5-β6 loop (Arg214Ser) will increase the efficiency of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ceftazidime\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " deacylation to that of OXA-163. The finding that the hydration of the general base in the active site determines deacylation efficiency is possibly important in other class D \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    β-lactamases\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ".</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_12120159.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Defining the mobility range of a hinge-type connection using molecular dynamics and metadynamics<br>A designed \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    disulfide\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-rich -hairpin peptide that dimerizes spontaneously served as a hinge-type connection between proteins. Here, we analyze the range of dynamics of this hinge dimer with the aim of proposing new applications for the DNA-encodable peptide and establishing guidelines for the computational analysis of other \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    disulfide\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " hinges. A recent structural analysis based on nuclear magnetic resonance spectroscopy and ion mobility spectrometry revealed an averaged conformation in the hinge region which motivated us to investigate the dynamic behavior using a combination of molecular dynamics simulation, metadynamics and free energy surface analysis to characterize the conformational space available to the hinge. Principal component analysis uncovered two slow modes of the peptide, namely, the opening and closing motion and twisting of the two -hairpins assembling the hinge. Applying a collective variable (CV) that mimics the first dominating mode, led to a major expansion of the conformational space. The description of the dynamics could be achieved by analysis of the opening angle and the twisting of the -hairpins and, thus, offers a methodology that can also be transferred to other derivatives. It has been demonstrated that the hinge peptide s lowest energy conformation consists of a large opening angle and strong twist but is separated by small energy barriers and can, thus, adopt a closed and untwisted structure. With the aim of proposing further applications for the hinge peptide, we simulated its behavior in the sterically congested environment of a four-helix bundle. Preliminary investigations show that one helix is pushed out and a three-helix bundle forms. The insights gained into the dynamics of the tetra-\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    disulfide\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " peptide and analytical guidelines developed in this study may contribute to the understanding of the structure and function of more complex hinge-type proteins, such as the IgG antibody family.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing zenodo_30904.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " Ulmschneider \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    OPLS\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " Verlet Group<br>MD simulation trajectory and related files for fully hydrated \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " bilayer run with Verlet and Group schemes. The Ulmschneider force field for \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " was used with \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Gromacs 5.0.3\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " [1,2]. Conditions: T \n",
       "<mark class=\"entity\" style=\"background: #ffb3ba; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    298.15\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">TEMP</span>\n",
       "</mark>\n",
       ", 128 \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    POPC\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " molecules, 5120 \n",
       "<mark class=\"entity\" style=\"background: #cdb4db; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    tip3p\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">FFM</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    waters\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". \n",
       "<mark class=\"entity\" style=\"background: #baffc9; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    200ns\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">STIME</span>\n",
       "</mark>\n",
       " trajectory (preceded by 5ns NPT equillibration). Starting structure was obtained from \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    CHARMM-GUI\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SOFT</span>\n",
       "</mark>\n",
       " [3]. This data is ran for the nmrlipids.blospot.fi project. More details from nmrlipids.blospot.fi and https://removed [1] J.P. Ulmschneider &amp; M.B. Ulmschneider, United Atom Lipid Parameters for Combination with the Optimized Potentials for Liquid Simulations All-Atom Force Field, JCTC 2009, 5 (7), 1803 1813 [2] https://removed [3] https://removed</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing figshare_9805616.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Solvent-Mediated Affinity of Polymer-Wrapped Single-Walled Carbon Nanotubes for Chemically Modified Surfaces<br>Semiconducting single-walled carbon nanotube (s-CNT) arrays are being explored for next-generation semiconductor electronics. Even with the multitude of alignment and spatially localized s-CNT deposition methods designed to control s-CNT deposition, fundamental understanding of the driving forces for s-CNT deposition is still lacking. The individual roles of the dispersant, solvent, target substrate composition, and the s-CNT itself are not completely understood because it is difficult to decouple deposition parameters. Here, we study \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    poly [ (9,9-dioctylfluorenyl-2,7-diyl)-alt-co- (6,6 -[2,2 -{bipyridine}])]\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " (\n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PFO-BPy\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ") -wrapped s-CNT deposition from solution onto a chemically modified substrate. We fabricate various self-assembled monolayers (SAMs) to gain a greater understanding of substrate effects on \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PFO-BPy\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-wrapped s-CNT deposition. We observe that s-CNT deposition is dependent on both the target substrate and s-CNT dispersion solvent. To complement the experiments, molecular dynamics simulations of \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    PFO-BPy\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       "-wrapped s-CNT deposition on two different SAMs are performed to obtain mechanistic insights into the effect of the substrate and solvent on s-CNT deposition. We find that the global free-energy minimum associated with favorable s-CNT adsorption occurs for a configuration in which the minimum of the solvent density around the s-CNT coincides with the minimum of the solvent density above a SAM-grafted surface, indicating that solvent structure near a SAM-grafted surface determines the adsorption free-energy landscape driving s-CNT deposition. Our results will help guide informative substrate design for s-CNT array fabrication in semiconductor devices.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n",
      "Visualizing osf_82n73.json...\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">Brown Lab, Public<br>We are a collaborative molecular modeling and bioinformatics lab at Virginia Tech. We work with individuals all over campus and at other universities to provide computational support to their research, as well as pursuing our own work on \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    amyloid\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       " and \n",
       "<mark class=\"entity\" style=\"background: #bae1ff; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    IDPs\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">MOL</span>\n",
       "</mark>\n",
       ". With this OSF, we give you information on our most commonly used techniques and example methods for generating a shared group, multi-project OSF. For information about our current projects or to view our tutorials, head to the Wiki page. To see some of our published papers and images, along with scripts and files to help you with your own molecular modeling, check our Files. Like what we are doing? Feel free to use our templates to make your research more transparent and open. If you have any questions, please contact Dr. Anne M. Brown at ambrown7@vt.edu.</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "=================================================================\n",
      "\n"
     ]
    }
   ],
   "source": [
    "folder = \"../annotations\"\n",
    "\n",
    "for filename in os.listdir(folder):\n",
    "    # if ends with json and contains one '_' only\n",
    "    if filename.endswith(\".json\") and filename.count(\"_\") == 1:\n",
    "        file_path = os.path.join(folder, filename)\n",
    "        converted_data = convert_annotations(file_path)\n",
    "        colors = {\n",
    "            \"TEMP\": \"#ffb3ba\",\n",
    "            \"SOFTNAME\": \"#ffffba\",\n",
    "            \"SOFTVERS\": \"#orange\",\n",
    "            \"STIME\": \"#baffc9\",\n",
    "            \"MOL\": \"#bae1ff\",\n",
    "            \"FFM\": \"#cdb4db\",\n",
    "        }\n",
    "        options = {\"colors\": colors}\n",
    "        print(f\"Visualizing {filename}...\")\n",
    "        displacy.render(converted_data, style=\"ent\", manual=True, options=options)\n",
    "        print(\"\\n=================================================================\\n\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [],
   "source": [
    "from pathlib import Path\n",
    "from typing import Dict, Any, List\n",
    "\n",
    "\n",
    "def convert_label_annotations(file_path: str | Path) -> Dict[str, Any]:\n",
    "    \"\"\"\n",
    "    Convert one custom-format JSON file to the structure required by\n",
    "    spaCy's displaCy manual visualiser.\n",
    "\n",
    "    Returns\n",
    "    -------\n",
    "    {\n",
    "        \"text\": \"<full text>\",\n",
    "        \"ents\": [ {\"start\": 0, \"end\": 5, \"label\": \"TAG\"}, ... ]\n",
    "    }\n",
    "    \"\"\"\n",
    "    file_path = Path(file_path)\n",
    "\n",
    "    with file_path.open(encoding=\"utf-8\") as f:\n",
    "        data = json.load(f)\n",
    "\n",
    "    ents: List[Dict[str, Any]] = [\n",
    "        {\"start\": int(s), \"end\": int(e), \"label\": tag} for s, e, tag in data[\"label\"]\n",
    "    ]\n",
    "\n",
    "    return {\"text\": data[\"text\"], \"ents\": ents}"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {},
   "outputs": [
    {
     "ename": "NameError",
     "evalue": "name 'os' is not defined",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mNameError\u001b[0m                                 Traceback (most recent call last)",
      "Cell \u001b[0;32mIn[17], line 4\u001b[0m\n\u001b[1;32m      1\u001b[0m original_annotations \u001b[38;5;241m=\u001b[39m \u001b[38;5;124m\"\u001b[39m\u001b[38;5;124m../annotations/\u001b[39m\u001b[38;5;124m\"\u001b[39m\n\u001b[1;32m      2\u001b[0m converted_annotations \u001b[38;5;241m=\u001b[39m \u001b[38;5;124m\"\u001b[39m\u001b[38;5;124m../testing/annotations_attempts/\u001b[39m\u001b[38;5;124m\"\u001b[39m\n\u001b[0;32m----> 4\u001b[0m \u001b[38;5;28;01mfor\u001b[39;00m filename \u001b[38;5;129;01min\u001b[39;00m \u001b[43mos\u001b[49m\u001b[38;5;241m.\u001b[39mlistdir(original_annotations):\n\u001b[1;32m      5\u001b[0m     \u001b[38;5;66;03m# if ends with json and contains one '_' only\u001b[39;00m\n\u001b[1;32m      6\u001b[0m     \u001b[38;5;28;01mif\u001b[39;00m filename\u001b[38;5;241m.\u001b[39mendswith(\u001b[38;5;124m\"\u001b[39m\u001b[38;5;124m.json\u001b[39m\u001b[38;5;124m\"\u001b[39m) \u001b[38;5;129;01mand\u001b[39;00m filename\u001b[38;5;241m.\u001b[39mcount(\u001b[38;5;124m\"\u001b[39m\u001b[38;5;124m_\u001b[39m\u001b[38;5;124m\"\u001b[39m) \u001b[38;5;241m==\u001b[39m \u001b[38;5;241m1\u001b[39m:\n\u001b[1;32m      7\u001b[0m         file_path \u001b[38;5;241m=\u001b[39m os\u001b[38;5;241m.\u001b[39mpath\u001b[38;5;241m.\u001b[39mjoin(original_annotations, filename)\n",
      "\u001b[0;31mNameError\u001b[0m: name 'os' is not defined"
     ]
    }
   ],
   "source": [
    "original_annotations = \"../annotations/\"\n",
    "converted_annotations = \"../testing/annotations_attempts/\"\n",
    "\n",
    "for filename in os.listdir(original_annotations):\n",
    "    # if ends with json and contains one '_' only\n",
    "    if filename.endswith(\".json\") and filename.count(\"_\") == 1:\n",
    "        file_path = os.path.join(original_annotations, filename)\n",
    "        converted_file_path = os.path.join(converted_annotations, filename)\n",
    "\n",
    "        converted_original_data = convert_annotations(file_path)\n",
    "        converted_label_data = convert_label_annotations(converted_file_path)\n",
    "\n",
    "        colors = {\n",
    "            \"TEMP\": \"#ffb3ba\",\n",
    "            \"SOFTNAME\": \"#ffffba\",\n",
    "            \"SOFTVERS\": \"#orange\",\n",
    "            \"STIME\": \"#baffc9\",\n",
    "            \"MOL\": \"#bae1ff\",\n",
    "            \"FFM\": \"#cdb4db\",\n",
    "        }\n",
    "        options = {\"colors\": colors}\n",
    "        displacy.render(\n",
    "            converted_original_data, style=\"ent\", manual=True, options=options\n",
    "        )\n",
    "        print(\"----\")\n",
    "        displacy.render(converted_label_data, style=\"ent\", manual=True, options=options)\n",
    "        print(\"\\n=================================================================\\n\")"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "mdner-llm",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.11.11"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
